## To cite ChIP-Atlas in your publication:

- Paper
  - Zou, Z., Ohta, T., Miura, F., Oki, S. ChIP-Atlas 2021 update: a data-mining suite for exploring epigenomic landscapes by fully integrating ChIP-seq, ATAC-seq and Bisulfite-seq data. __Nucleic Acids Res__. 50(W1), W175-W182, 2022. http://dx.doi.org/10.1093/nar/gkac199
  - Oki, S., Ohta, T., Shioi, G., Hatanaka, H., Ogasawara, O., Okuda, Y., Kawaji, H., Nakaki, R., Sese, J., Meno, C. ChIP-Atlas: a data-mining suite powered by full integration of public ChIP-seq data. __EMBO Rep__. 19(12), e46255, 2018. http://dx.doi.org/10.15252/embr.201846255
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. https://chip-atlas.org

## Publications citing ChIP-Atlas:

1. Hino, Shinjiro et al., Chromatin Immunoprecipitation Sequencing (ChIP-seq) for Detecting Histone Modifications and Modifiers. __Methods Mol Biol__ (2023). [Link](https://doi.org/doi:10.1007/978-1-0716-2724-2_4).
2. Acevedo, Adriana Franco et al., BCL6 regulates the endothelial pro-immunogenic phenotype relevant to organ transplant rejection. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514941).
3. Acevedo, Adriana Franco et al., The DNA binding protein BCL6 regulates NFκB-controlled endothelial inflammatory gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514942).
4. Ambrosini, Giovanna et al., The Breast Cancer Epigenomics Track Hub. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.01.490187).
5. Anderson, Craig J. et al., Strand-resolved mutagenicity of DNA damage and repair. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.10.495644).
6. Anerillas, Carlos et al., A BDNF-TrkB autocrine loop enhances senescent cell viability. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-33709-8).
7. Atindaana, Edmond et al., Bimodal Expression Patterns, and Not Viral Burst Sizes, Predict the Effects of Vpr on HIV-1 Proviral Populations in Jurkat Cells. __mBio__ (2022). [Link](https://doi.org/doi:10.1128/MBIO.03748-21).
8. Awamleh, Zain et al., DNA methylation signature associated with Bohring-Opitz syndrome: a new tool for functional classification of variants in ASXL genes. __Eur J Hum Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41431-022-01083-0).
9. Baessler, Andrew et al., Tet2 coordinates with Foxo1 and Runx1 to balance T follicular helper cell and T helper 1 cell differentiation. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABM4982).
10. Barlier, Céline et al., RNetDys: identification of disease-related impaired regulatory interactions due to SNPs. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.08.511312).
11. Bideyan, Lara et al., Integrative analysis reveals multiple modes of LXR transcriptional regulation in liver. __Proc Natl Acad Sci U S A__ (2022). [Link](https://doi.org/doi:10.1073/PNAS.2122683119).
12. Booth, Lauren N. et al., Males induce premature demise of the opposite sex by multifaceted strategies. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00276-y).
13. Bujnicki, Janusz et al., Editorial: the 20th annual Nucleic Acids Research Web Server Issue 2022. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC525).
14. Butera, Alessio et al., p53-driven lipidome influences non-cell-autonomous lysophospholipids in pancreatic cancer. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00319-9).
15. Cao, Meng et al., The synergistic interaction landscape of chromatin regulators reveals their epigenetic regulation mechanisms across five cancer cell lines. __Comput Struct Biotechnol J__ (2022). [Link](https://doi.org/doi:10.1016/J.CSBJ.2022.09.008).
16. Chai, Gaoshang et al., Sevoflurane inhibits histone acetylation and contributes to cognitive dysfunction by enhancing the expression of ANP32A in aging mice. __Behav Brain Res__ (2022). [Link](https://doi.org/doi:10.1016/J.BBR.2022.113949).
17. Chandra, Omkar et al., Inferring functions of coding and non-coding genes using epigenomic patterns and deciphering the effect of combinatorics of transcription factors binding at promoters. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.17.488570).
18. Chauvistré, Heike et al., Persister state-directed transitioning and vulnerability in melanoma. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30641-9).
19. Chen, Fujia et al., Placental DNA methylation changes in gestational diabetes mellitus. __Epigenetics__ (2022). [Link](https://doi.org/doi:10.1080/15592294.2022.2110193).
20. Chen, Guangyi et al., Graph attention network for link prediction of gene regulations from single cell RNA-sequencing data. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC559).
21. Chen, Haobin et al., BET Inhibitors Target the SCLC-N subtype of Small Cell Lung Cancer by Blocking NEUROD1 Transactivation. __Mol Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-22-0594).
22. Chen, Jiayang et al., MYT1L is required for suppressing earlier neuronal development programs in the adult mouse brain. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.17.512591).
23. Chen, Ke Yun et al., Adipocyte-Specific Ablation of PU.1 Promotes Energy Expenditure and Ameliorates Metabolic Syndrome in Aging Mice. __Front Aging__ (2022). [Link](https://doi.org/doi:10.3389/FRAGI.2021.803482).
24. Chen, Ming et al., Integrative analyses for the identification of idiopathic pulmonary fibrosis-associated genes and shared loci with other diseases. __Thorax__ (2022). [Link](https://doi.org/doi:10.1136/THORAX-2021-217703).
25. Chen, Tao et al., GSTM3 deficiency impedes DNA mismatch repair to promote gastric tumorigenesis via CAND1/NRF2-KEAP1 signaling. __Cancer Lett__ (2022). [Link](https://doi.org/doi:10.1016/J.CANLET.2022.215692).
26. Chirichella, Michele et al., RFX transcription factors control a miR-150/PDAP1 axis that restrains the proliferation of human T cells. __PLoS Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PBIO.3001538).
27. Corleone, G et al., Enhancer plasticity sustains oncogenic transformation and progression of B-Cell Acute Lymphoblastic leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.18.492497).
28. Del Giudice, Marco et al., FOXA1 regulates alternative splicing in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111404).
29. Dong, Ce et al., Curcumin Attenuates Hydrocephalus via Activation of E2F Transcription Factor 4. __Cerebrovasc Dis__ (2022). [Link](https://doi.org/doi:10.1159/000523994).
30. Dufour, Catherine R. et al., The mTOR chromatin-bound interactome in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110534).
31. Eguchi, Ryohei et al., TRANSDIRE: data-driven direct reprogramming by a pioneer factor-guided trans-omics approach. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC209).
32. Escalante-Covarrubias, Q et al., Time-of-day defines the efficacy of NAD+ to treat diet-induced metabolic disease by adjusting oscillations of the hepatic circadian clock. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503453).
33. Eustice, Moriah et al., Nutrient sensing pathways regulating adult reproductive diapause in C. elegans. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0274076).
34. Fang, Xue-Quan et al., PGC1α Cooperates with FOXA1 to Regulate Epithelial Mesenchymal Transition through the TCF4-TWIST1. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS23158247).
35. Feng, Chenchen et al., Arborinine from Glycosmis parva leaf extract inhibits clear-cell renal cell carcinoma by inhibiting KDM1A/UBE2O signaling. __Food Nutr Res__ (2022). [Link](https://doi.org/doi:10.29219/FNR.V66.8714).
36. Filipovic, David et al., Predictive Models of Genome-wide Aryl Hydrocarbon Receptor DNA Binding Reveal Tissue Specific Binding Determinants. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.13.491754).
37. Freimer, Jacob W. et al., Systematic discovery and perturbation of regulatory genes in human T cells reveals the architecture of immune networks. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01106-Y).
38. Fu, Liang-Yu et al., ChIP-Hub provides an integrative platform for exploring plant regulome. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30770-1).
39. Gao, Yu et al., TRmir: A Comprehensive Resource for Human Transcriptional Regulatory Information of MiRNAs. __Front Genet__ (2022). [Link](https://doi.org/doi:10.3389/FGENE.2022.808950).
40. Gourisankar, Sai et al., Rewiring Cancer Drivers to Activate Apoptosis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.04.517548).
41. Guan, Jinting et al., SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference. __BMC Genomics__ (2022). [Link](https://doi.org/doi:10.1186/S12864-022-09020-7).
42. Gupta, Anuj et al., Dynamic Control Balancing Cell Proliferation and Inflammation is Crucial for an Effective Immune Response to Malaria. __Front Mol Biosci__ (2022). [Link](https://doi.org/doi:10.3389/FMOLB.2021.800721).
43. Habano, Wataru et al., Aryl hydrocarbon receptor as a DNA methylation reader in the stress response pathway. __CRC Crit Rev Toxicol__ (2022). [Link](https://doi.org/doi:10.1016/J.TOX.2022.153154).
44. Harada, Takeshi et al., Novel antimyeloma therapeutic option with inhibition of the HDAC1-IRF4 axis and PIM kinase. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022007155).
45. Hasegawa, Atsushi et al., Heterozygous variants in GATA2 contribute to DCML deficiency in mice by disrupting tandem protein binding. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-03316-W).
46. He, Shasha et al., The Role of the SOX9/lncRNA ANXA2P2/miR-361-3p/SOX9 Regulatory Loop in Cervical Cancer Cell Growth and Resistance to Cisplatin. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.784525).
47. He, Xujun et al., PBRM1 presents a potential prognostic marker and therapeutic target in duodenal papillary carcinoma. __Clin Transl Med__ (2022). [Link](https://doi.org/doi:10.1002/CTM2.1062).
48. Hersbach, Bob A et al., Probing cell identity hierarchies by fate titration and collision during direct reprogramming. __Mol Syst Biol__ (2022). [Link](https://doi.org/doi:10.15252/MSB.202211129).
49. Hinton, Antentor et al., OPA1 Downregulation in Skeletal Muscle Induces MERC formation in an ATF4-Dependent Manner. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.12.507669).
50. Hirata, Hirohito et al., A novel role of helix-loop-helix transcriptional factor Bhlhe40 in osteoclast activation. __J Cell Physiol__ (2022). [Link](https://doi.org/doi:10.1002/JCP.30844).
51. Hoffmann, Markus et al., TF-Prioritizer: a java pipeline to prioritize condition-specific transcription factors. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.19.512881).
52. Hsieh, Yu-Ting et al., Equilibrative Nucleoside Transporter 3 is an IFN-stimulated Gene that Facilitates Viral Genome Release. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.27.489746).
53. Hu, Xiuchuan et al., Aryl hydrocarbon receptor utilises cellular zinc signals to maintain the gut epithelial barrier. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.515052).
54. Hu, Yixiang et al., RUNX1 inhibits the antiviral immune response against influenza A virus through attenuating type I interferon signaling. __Virol J__ (2022). [Link](https://doi.org/doi:10.1186/S12985-022-01764-8).
55. Huang, Tinghua et al., Identification of upstream transcription factor binding sites in orthologous genes using mixed Student's t-test statistics. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009773).
56. Ideno, Hirosato et al., Human PSCs determine the competency of cerebral organoid differentiation via FGF signaling and epigenetic mechanisms. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105140).
57. Ito, Naoto et al., The Ccl17 gene encoding TARC is synergistically transactivated by PU.1 and IRF4 driven by the mammalian common promoter in dendritic cells. __Allergy__ (2022). [Link](https://doi.org/doi:10.1111/ALL.15184).
58. Itokazu, Maki et al., Adipose-derived exosomes block muscular stem cell proliferation in aged mouse by delivering miRNA Let-7d-3p that targets transcription factor HMGA2. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102098).
59. Iwata, Michio et al., Regulome-based characterization of drug activity across the human diseasome. __NPJ Syst Biol Appl__ (2022). [Link](https://doi.org/doi:10.1038/S41540-022-00255-4).
60. J, Huang et al., ChIPBase v3.0: the encyclopedia of transcriptional regulations of non-coding RNAs and protein-coding genes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC1067).
61. J, Wang et al., CohesinDB: a comprehensive database for decoding cohesin-related epigenomes, 3D genomes and transcriptomes in human cells. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC795).
62. Jiang, Junyao et al., IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105359).
63. Jing, Fang et al., Prediction of the transcription factor binding sites with meta-learning. __Methods__ (2022). [Link](https://doi.org/doi:10.1016/J.YMETH.2022.04.010).
64. Kaizuka, Takeshi et al., Developmental dynamics of the postsynaptic proteome to understand synaptic maturation and dysmaturation. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.05.490828).
65. Kanemoto, Yoshiaki et al., A long non-coding RNA as a direct vitamin D target transcribed from the anti-sense strand of the human HSD17B2 locus. __Biosci Rep__ (2022). [Link](https://doi.org/doi:10.1042/BSR20220321).
66. Kim, Dong Kyu et al., Aβ-induced mitochondrial dysfunction in neural progenitors controls KDM5A to influence neuronal differentiation. __Exp Mol Med__ (2022). [Link](https://doi.org/doi:10.1038/S12276-022-00841-W).
67. Kim, Iljin et al., AURKB, in concert with REST, acts as an oxygen-sensitive epigenetic regulator of the hypoxic induction of MDM2. __BMB Rep__ (2022). [Link](https://doi.org/doi:10.5483/BMBREP.2022.55.6.017).
68. Kitami, Kazuhisa et al., Peritoneal restoration by repurposing vitamin D inhibits ovarian cancer dissemination via blockade of the TGF-β1/thrombospondin-1 axis. __Matrix Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.MATBIO.2022.03.003).
69. Kobelyatskaya, Anastasiya A. et al., ALDH3A2, ODF2, QSOX2, and MicroRNA-503-5p Expression to Forecast Recurrence in TMPRSS2-ERG-Positive Prostate Cancer. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911695).
70. Kojima, Shin et al., Epigenome editing reveals core DNA methylation for imprinting control in the Dlk1-Dio3 imprinted domain. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC344).
71. Kokaji, Toshiya et al., Construction of an inter-organ transomic network for whole-body metabolism. __Curr Opin Endocr Metab Res__ (2022). [Link](https://doi.org/doi:10.1016/J.COEMR.2022.100361).
72. Kokaji, Toshiya et al., In vivo transomic analyses of glucose-responsive metabolism in skeletal muscle reveal core differences between the healthy and obese states. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.03.27.486003).
73. Kondo, Jumpei et al., Transcription factor SP1 regulates haptoglobin fucosylation via induction of GDP-fucose transporter 1 in the hepatoma cell line HepG2. __Biochem Biophys Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.BBREP.2022.101372).
74. Kong, Siyuan et al., Nucleosome-Omics: A Perspective on the Epigenetic Code and 3D Genome Landscape. __Genes__ (2022). [Link](https://doi.org/doi:10.3390/GENES13071114).
75. Kozlova, Alena A. et al., Assessment of DDAH1 and DDAH2 Contributions to Psychiatric Disorders via In Silico Methods. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911902).
76. Kreiß, Marius et al., Human 5-lipoxygenase regulates transcription by association to euchromatin. __Biochem Pharmacol__ (2022). [Link](https://doi.org/doi:10.1016/J.BCP.2022.115187).
77. Larsen, Erik G. et al., Transmembrane protein TMEM184B is necessary for interleukin-31-induced itch. __Pain__ (2022). [Link](https://doi.org/doi:10.1097/J.PAIN.0000000000002452).
78. Lee, Shi Hao et al., Activation function 1 of progesterone receptor is required for progesterone antagonism of oestrogen action in the uterus. __BMC Biol__ (2022). [Link](https://doi.org/doi:10.1186/S12915-022-01410-3).
79. Letson, Christopher T. et al., Targeting BET Proteins downregulates miR-33a to promote synergy with PIM inhibitors in CMML. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.01.514753).
80. Li, Ruizhi et al., Elucidation of an mTORC2-PKC-NRF2 pathway that sustains the ATF4 stress response and identification of Sirt5 as a key ATF4 effector. __Cell Death Discov__ (2022). [Link](https://doi.org/doi:10.1038/S41420-022-01156-5).
81. Li, Wei et al., YTHDC1 is downregulated by the YY1/HDAC2 complex and controls the sensitivity of ccRCC to sunitinib by targeting the ANXA1-MAPK pathway. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02460-9).
82. Li, Yafei et al., ARID3A promotes the chemosensitivity of colon cancer by inhibiting AKR1C3. __Cell Biol Int__ (2022). [Link](https://doi.org/doi:10.1002/CBIN.11789).
83. Li, Yaohua et al., Immunoglobulin superfamily 9 (IGSF9) is trans-activated by p53, inhibits breast cancer metastasis via FAK. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02459-8).
84. Liao, Yuandong et al., Downregulation of LNMAS orchestrates partial EMT and immune escape from macrophage phagocytosis to promote lymph node metastasis of cervical cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02202-3).
85. Lisacek, Frederique et al., Worldwide Glycoscience Informatics Infrastructure: The GlySpace Alliance. __JACS Au__ (2022). [Link](https://doi.org/doi:10.1021/JACSAU.2C00477).
86. Liu, Tianyuan et al., Computational identification of preneoplastic cells displaying high stemness and risk of cancer progression. __Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-22-0668).
87. Liu, Wentao et al., A novel FBW7/NFAT1 axis regulates cancer immunity in sunitinib-resistant renal cancer by inducing PD-L1 expression. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02253-0).
88. Liu, Zhen et al., JAK2/STAT3 inhibition attenuates intestinal ischemia-reperfusion injury via promoting autophagy: in vitro and in vivo study. __Mol Biol Rep__ (2022). [Link](https://doi.org/doi:10.1007/S11033-021-07099-X).
89. Lu, Yuanjun et al., Thioredoxin-interacting protein-activated intracellular potassium deprivation mediates the anti-tumour effect of a novel histone acetylation inhibitor HL23, a fangchinoline derivative, in human hepatocellular carcinoma. __J Adv Res__ (2022). [Link](https://doi.org/doi:10.1016/J.JARE.2022.10.017).
90. Lyu, Qingji et al., A small proportion of X-linked genes contribute to X chromosome upregulation in early embryos via BRD4-mediated transcriptional activation. __Curr Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.CUB.2022.08.059).
91. M, Watzky et al., Hexokinase 2 is a transcriptional target and a positive modulator of AHR signalling. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC360).
92. M, Zhang et al., MethBank 4.0: an updated database of DNA methylation across a variety of species. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC969).
93. Ma, Xiaolong et al., Single-cell RNA sequencing reveals B cell-T cell interactions in vascular adventitia of hyperhomocysteinemia-accelerated atherosclerosis. __Protein Cell__ (2022). [Link](https://doi.org/doi:10.1007/S13238-021-00904-0).
94. MacKay, Harry et al., Sex-specific epigenetic development in the mouse hypothalamic arcuate nucleus pinpoints human genomic regions associated with body mass index. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABO3991).
95. Maity, Alok K. et al., Inference of age-associated transcription factor regulatory activity changes in single cells. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00233-9).
96. Maixner, Nitzan et al., Visceral Adipose Tissue E2F1-miRNA206/210 Pathway Associates with Type 2 Diabetes in Humans with Extreme Obesity. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11193046).
97. Mamivand, Ali et al., Data mining of bulk and single-cell RNA sequencing introduces OBI1-AS1 as an astrocyte marker with possible role in glioma recurrence and progression. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01260-4).
98. Marques, Ines J. et al., Wt1 transcription factor impairs cardiomyocyte specification and drives a phenotypic switch from myocardium to epicardium. __Development__ (2022). [Link](https://doi.org/doi:10.1242/DEV.200375).
99. Martínez Corrales, Guillermo et al., Transcriptional memory of dFOXO activation in youth curtails later-life mortality through chromatin remodeling and Xbp1. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00312-x).
100. Masamoto, Yosuke et al., EVI1 exerts distinct roles in AML via ERG and cyclin D1 promoting a chemoresistance and immune-suppressive environment. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022008018).
101. Matsumoto, Akifumi et al., C11orf21, a novel RUNX1 target gene, is down-regulated by RUNX1-ETO. __BBA Adv__ (2022). [Link](https://doi.org/doi:10.1016/J.BBADVA.2022.100047).
102. McKowen, J Keller et al., The Drosophila BEAF Insulator Protein Interacts with the Polybromo Subunit of the PBAP Chromatin Remodeling Complex. __Genes Genomes Genet__ (2022). [Link](https://doi.org/doi:10.1093/G3JOURNAL/JKAC223).
103. Mei, Heng et al., Autophagy Facilitates Cancer-Intrinsic Evasion of CAR T Cell-Mediated Cytotoxicity in B-Cell Malignancies. __Blood__ (2022). [Link](https://doi.org/doi:10.1182/BLOOD-2022-168590).
104. Meller, Anna et al., Mining Proteomics Datasets to Uncover Functional Pseudogenes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2124-0_16).
105. Miao, Jiamin et al., Sp1 Inhibits PGC-1α via HDAC2-Catalyzed Histone Deacetylation in Chronic Constriction Injury-Induced Neuropathic Pain. __ACS Chem Neurosci__ (2022). [Link](https://doi.org/doi:10.1021/ACSCHEMNEURO.2C00440).
106. Migdał, Maciej et al., xcore: an R package for inference of gene expression regulators. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.23.481130).
107. Mori, Jinichi et al., Identification of cell cycle-associated and -unassociated regulators for expression of a hepatocellular carcinoma oncogene cyclin-dependent kinase inhibitor 3. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.07.088).
108. Mori, Tatsufumi et al., Depletion of NIMA-related kinase Nek2 induces aberrant self-renewal and apoptosis in stem/progenitor cells of aged muscular tissues. __Mech Ageing Dev__ (2022). [Link](https://doi.org/doi:10.1016/J.MAD.2022.111619).
109. Morin, Alexander et al., Characterizing the targets of transcription regulators by aggregating ChIP-seq and perturbation expression data sets. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.30.505909).
110. Morita, Tsuyoshi et al., Actin-related protein 5 functions as a novel modulator of MyoD and MyoG in skeletal muscle and in rhabdomyosarcoma. __Elife__ (2022). [Link](https://doi.org/doi:10.7554/ELIFE.77746).
111. Mouri, Kousuke et al., Prioritization of autoimmune disease-associated genetic variants that perturb regulatory element activity in T cells. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01056-5).
112. Muromoto, Ryuta et al., Regulation of NFKBIZ gene promoter activity by STAT3, C/EBPβ, and STAT1. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.140).
113. Nair, Praful R. et al., MLL1 regulates cytokine-driven cell migration and metastasis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.18.512715).
114. Nakamura, Hideaki et al., Hypoxia-inducible factor-1α and poly [ADP ribose] polymerase 1 cooperatively regulate Notch3 expression under hypoxia via a noncanonical mechanism. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102137).
115. Neal, Alice et al., Finding and Verifying Enhancers for Endothelial-Expressed Genes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2059-5_28).
116. Ng, Krystle Joy et al., Sox2 in the dermal papilla regulates hair follicle pigmentation. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111100).
117. Nie, Meng et al., Targeting acetylcholine signaling modulates persistent drug tolerance in EGFR-mutant lung cancer and impedes tumor relapse. __J Clin Invest__ (2022). [Link](https://doi.org/doi:10.1172/JCI160152).
118. Nishi, Kentaro et al., Novel estrogen-responsive genes (ERGs) for the evaluation of estrogenic activity. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0273164).
119. Ochsner, Scott A. et al., Transcriptional regulatory networks of circulating immune cells in type 1 diabetes: A community knowledgebase. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104581).
120. Ohnishi, Tomokazu et al., Periodontitis promotes the expression of gingival transmembrane serine protease 2 (TMPRSS2), a priming protease for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). __J Oral Biosci__ (2022). [Link](https://doi.org/doi:10.1016/J.JOB.2022.04.004).
121. Ono, Koya et al., Congenital sideroblastic anemia model due to ALAS2 mutation is susceptible to ferroptosis. __Sci Rep__ (2022). [Link](https://doi.org/doi:10.1038/S41598-022-12940-9).
122. Panatta, Emanuele et al., Metabolic regulation by p53 prevents R-loop-associated genomic instability. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111568).
123. Panatta, Emanuele et al., p53 regulates expression of nuclear envelope components in cancer cells. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00349-3).
124. Peng, Liang et al., Bombesin receptor-activated protein exacerbates cisplatin-induced AKI by regulating the degradation of SIRT2. __Nephrol Dial Transplant__ (2022). [Link](https://doi.org/doi:10.1093/NDT/GFAC164).
125. Pontis, Julien et al., Primate-specific transposable elements shape transcriptional networks during human development. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34800-W).
126. Q, Zhou et al., ChromLoops: a comprehensive database for specific protein-mediated chromatin loops in diverse organisms. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC893).
127. Reagor, Caleb C. et al., Depicting pseudotime-lagged causality across single-cell trajectories for accurate gene-regulatory inference. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.25.489377).
128. Ren, Jiannan et al., Downregulation of CBX7 induced by EZH2 upregulates FGFR3 expression to reduce sensitivity to cisplatin in bladder cancer. __Br J Cancer__ (2022). [Link](https://doi.org/doi:10.1038/S41416-022-02058-0).
129. Rhizobium, Growth-promoting Endophyte, TFSyntax: a database of transcription factors binding syntax in mammalian genomes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC849).
130. Richard, Daniel et al., Intronic regulation of SARS-CoV-2 receptor (ACE2) expression mediated by immune signaling and oxidative stress pathways. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104614).
131. Richard, Daniel et al., Lineage-Specific Differences and Regulatory Networks Governing Human Chondrocyte Development. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.25.493388).
132. Rykaczewska, Urszula et al., Plaque Evaluation by Ultrasound and Transcriptomics Reveals BCLAF1 as a Regulator of Smooth Muscle Cell Lipid Transdifferentiation in Atherosclerosis. __Arterioscler Thromb Vasc Biol__ (2022). [Link](https://doi.org/doi:10.1161/ATVBAHA.121.317018).
133. Sakai, Hiroshi et al., Uhrf1 governs the proliferation and differentiation of muscle satellite cells. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.103928).
134. Samelson, Avi J. et al., BRD2 inhibition blocks SARS-CoV-2 infection by reducing transcription of the host cell receptor ACE2. __Nat Cell Biol__ (2022). [Link](https://doi.org/doi:10.1038/S41556-021-00821-8).
135. Sasaki, Norihiko et al., Gp130-Mediated STAT3 Activation Contributes to the Aggressiveness of Pancreatic Cancer through H19 Long Non-Coding RNA Expression. __Cancers__ (2022). [Link](https://doi.org/doi:10.3390/CANCERS14092055).
136. Sasaki, Norihiko et al., PRC2-dependent regulation of ganglioside expression during dedifferentiation contributes to the proliferation and migration of vascular smooth muscle cells. __Front Cell Dev Biol__ (2022). [Link](https://doi.org/doi:10.3389/FCELL.2022.1003349).
137. Sato, Fumiaki et al., COL8A1 facilitates the growth of triple-negative breast cancer via FAK/Src activation. __Breast Cancer Res Treat__ (2022). [Link](https://doi.org/doi:10.1007/S10549-022-06635-Y).
138. Sawada, Takahiro et al., Antagonistic action of a synthetic androgen ligand mediated by chromatin remodeling in a human prostate cancer cell line. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.109).
139. Schihada, Hannes et al., Isoforms of GPR35 have distinct extracellular N-termini that allosterically modify receptor-transducer coupling and mediate intracellular pathway bias. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102328).
140. Serrano-Zayas, Candilianne et al., Regulation of cellular LDL uptake by PROX1 and CHD7. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.20.507601).
141. Shen, Shang-Hang et al., Bromodomain-containing protein 4 activates cardiotrophin-like cytokine factor 1, an unfavorable prognostic biomarker, and promotes glioblastoma in vitro. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-1164).
142. Shin, Jihoon et al., SARS-CoV-2 infection impairs the insulin/IGF signaling pathway in the lung, liver, adipose tissue, and pancreatic cells via IRF1. __Metabolism__ (2022). [Link](https://doi.org/doi:10.1016/J.METABOL.2022.155236).
143. Silvennoinen, Katri et al., SCN1A overexpression, associated with a genomic region marked by a risk variant for a common epilepsy, raises seizure susceptibility. __Acta Neuropathol__ (2022). [Link](https://doi.org/doi:10.1007/S00401-022-02429-0).
144. Singh, Anupama et al., Conserved neuropeptidergic regulation of intestinal integrity in invertebrate models of aging. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.24.481867).
145. Siswanto, Ferbian Milas et al., Chlorogenic acid activates Nrf2/SKN-1 and prolongs the lifespan of Caenorhabditis elegans via the Akt-FOXO3/DAF16a-DDB1 pathway and activation of DAF16f. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.1093/GERONA/GLAC062).
146. Sokolowska, Katarzyna Ewa et al., Identified in blood diet-related methylation changes stratify liver biopsies of NAFLD patients according to fibrosis grade. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01377-6).
147. Sokolowski, Dustin J. et al., Differential Expression Enrichment Tool (DEET): An interactive atlas of human differential gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.29.505468).
148. Sun, Jichun et al., The lncRNA CASC2 Modulates Hepatocellular Carcinoma Cell Sensitivity and Resistance to TRAIL Through Apoptotic and Non-Apoptotic Signaling. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.726622).
149. Tahara, Saeko et al., MOCCS profile analysis clarifies the cell type dependency of transcription factor-binding sequences and cis-regulatory SNPs in humans. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.08.487641).
150. Takayama, Shun et al., GATA6 regulates expression of annexin A10 (ANXA10) associated with epithelial-mesenchymal transition of oral squamous cell carcinoma. __Arch Oral Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.ARCHORALBIO.2022.105569).
151. Takeuchi, Chikara et al., Mod(mdg4) variants repress telomeric retrotransposon HeT-A by blocking subtelomeric enhancers. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.16.476534).
152. Talamanca, Lorenzo et al., Sex-dimorphic and age-dependent organization of 24 hour gene expression rhythms in human. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.13.494714).
153. Tanigawa, Yosuke et al., WhichTF is functionally important in your open chromatin data?. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1010378).
154. Terakawa, Akira et al., Trans-omics analysis of insulin action reveals a cell growth subnetwork which co-regulates anabolic processes. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104231).
155. Toyoda, Kosuke et al., Reprogramming of lactate metabolism is linked to the oncogenesis of the virus-induced leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503711).
156. Tsujino, Takuya et al., CRISPR screens reveal genetic determinants of PARP inhibitor sensitivity and resistance in prostate cancer. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.13.488115).
157. Vorontsova, Julia E. et al., A new insight into the aryl hydrocarbon receptor/cytochrome 450 signaling pathway in MG63, HOS, SAOS2, and U2OS cell lines. __Biochimie__ (2022). [Link](https://doi.org/doi:10.1016/J.BIOCHI.2022.10.018).
158. Wang, Yijie et al., NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-04226-7).
159. Wang, Zixuan et al., Towards a better understanding of TF-DNA binding prediction from genomic features. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105993).
160. Woogeng, Ivo Ngundu et al., Inducing human retinal pigment epithelium-like cells from somatic tissue. __Stem Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.12.008).
161. Wu, Beibei et al., ORAI1 Limits SARS-CoV-2 Infection by Regulating Tonic Type I IFN Signaling. __J Immunol__ (2022). [Link](https://doi.org/doi:10.4049/JIMMUNOL.2100742).
162. Wu, Shizhong et al., Bromodomain containing 4 transcriptionally activated Deltex E3 ubiquitin ligase 2 contributes to glioma progression and predicts an unfavorable prognosis. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-555).
163. Y, Hino et al., Mitochondrial stress induces AREG expression and epigenomic remodeling through c-JUN and YAP-mediated enhancer activation. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC735).
164. Yamada, Minami et al., KLF5-regulated extracellular matrix remodeling secures biliary epithelial tissue integrity against cholestatic liver injury. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.25.477619).
165. Yamamoto-Imoto, Hitomi et al., Age-associated decline of MondoA drives cellular senescence through impaired autophagy and mitochondrial homeostasis. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110444).
166. Yamashita, Yu et al., Osteoblast lineage cells-derived Sema3A regulates bone homeostasis independently of androgens. __Endocrinology__ (2022). [Link](https://doi.org/doi:10.1210/ENDOCR/BQAC126).
167. Yamazaki, Masaya et al., ROR1 plays a critical role in pancreatic tumor-initiating cells with a partial EMT signature. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.07.13.499868).
168. Yan, Bingyu et al., A comprehensive single cell data analysis of in lymphoblastoid cells reveals the role of Super-enhancers in maintaining EBV latency. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.10.503552).
169. Yang, Kaixin et al., Prognostic and Immunotherapeutic Roles of KRAS in Pan-Cancer. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11091427).
170. Yang, Lihua et al., Sevoflurane Offers Neuroprotection in a Cerebral Ischemia/Reperfusion Injury Rat Model Through the E2F1/EZH2/TIMP2 Regulatory Axis. __Mol Neurobiol__ (2022). [Link](https://doi.org/doi:10.1007/S12035-021-02602-8).
171. Yashar, William M. et al., GoPeaks: histone modification peak calling for CUT&Tag. __Genome Biol__ (2022). [Link](https://doi.org/doi:10.1186/S13059-022-02707-W).
172. Yheskel, Matanel et al., Proximity labeling reveals a new in vivo network of interactors for the histone demethylase KDM5. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.20.517232).
173. Yonezawa, Yoshiro et al., Identification of a Functional Susceptibility Variant for Adolescent Idiopathic Scoliosis that Upregulates Early Growth Response 1 (EGR1)-Mediated UNCX Expression. __J Bone Miner Res__ (2022). [Link](https://doi.org/doi:10.1002/JBMR.4738).
174. Zelenka, Tomas et al., The 3D enhancer network of the developing T cell genome is shaped by SATB1. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34345-Y).
175. Zhang, Huan et al., CBX3 Regulated By YBX1 Promotes Smoking-induced Pancreatic Cancer Progression via Inhibiting SMURF2 Expression. __Int J Biol Sci__ (2022). [Link](https://doi.org/doi:10.7150/IJBS.68995).
176. Zhang, Liang et al., STAT3 is a biologically relevant therapeutic target in H3K27M-mutant diffuse midline glioma. __Neuro Oncol__ (2022). [Link](https://doi.org/doi:10.1093/NEUONC/NOAC093).
177. Zhang, Ting et al., ADAR1 masks the cancer immunotherapeutic promise of ZBP1-driven necroptosis. __Nature__ (2022). [Link](https://doi.org/doi:10.1038/S41586-022-04753-7).
178. Zhang, Yongqing et al., Fusion prior gene network for high reliable single-cell gene regulatory network inference. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105279).
179. Zhao, Jianmei et al., CanMethdb: a database for genome-wide DNA methylation annotation in cancers. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC783).
180. Zhao, Mengyuan et al., A hybrid deep learning framework for gene regulatory network inference from single-cell transcriptomic data. __Brief Bioinform__ (2022). [Link](https://doi.org/doi:10.1093/BIB/BBAB568).
181. Zheng, Xianchong et al., Downregulation of HINFP induces senescence-associated secretory phenotype to promote metastasis in a non-cell-autonomous manner in bladder cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02371-1).
182. Zhong, Xiaoling et al., CBX3 is associated with metastasis and glutathione/ glycosphingolipid metabolism in colon adenocarcinoma. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.21037/JGO-22-97/COIF)).
183. Zhu, Tianyu et al., A pan-tissue DNA methylation atlas enables in silico decomposition of human tissue methylomes at cell-type resolution. __Nat Methods__ (2022). [Link](https://doi.org/doi:10.1038/S41592-022-01412-7).
184. Zou, Zhaonan et al., Epigenetic landscape of drug responses revealed through large-scale ChIP-seq data analyses. __BMC Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1186/S12859-022-04571-8).
185. A, Mamivand et al., Integrative Bioinformatics and Experimental Investigation Introduces OBI1-AS1 as a Key LncRNA in the Progression of Low-grade Glioma to Glioblastoma. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-215387/V1).
186. A, Meiler et al., AnnoMiner is a new web-tool to integrate epigenetics, transcription factor occupancy and transcriptomics data to predict transcriptional regulators. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-94805-1).
187. A, Nagalingam et al., Hyperleptinemia in obese state renders luminal breast cancers refractory to tamoxifen by coordinating a crosstalk between Med1, miR205 and ErbB. __NPJ Breast Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41523-021-00314-9).
188. A, Wanet et al., E-cadherin is regulated by GATA-2 and marks the early commitment of mouse hematopoietic progenitors to the basophil and mast cell fates. __Sci Immunol__ (2021). [Link](https://doi.org/doi:10.1126/SCIIMMUNOL.ABA0178).
189. AP, Wu et al., Bayesian information sharing enhances detection of regulatory associations in rare cell types. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB269).
190. Agudelo, Leandro Z. et al., Metabolic resilience is encoded in genome plasticity. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.25.449953).
191. Aizawa, Shiho et al., Early reactivation of clustered genes on the inactive X chromosome during somatic cell reprogramming. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.11.008).
192. Alfeghaly, Charbel et al.,  Implication of repeat insertion domains in the trans -activity of the long non-coding RNA ANRIL . __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/nar/gkab245).
193. Astudillo, Pablo, Analysis in silico of the functional interaction between WNT5A and YAP/TEAD signaling in cancer. __PeerJ__ (2021). [Link](https://doi.org/doi:10.7717/peerj.10869).
194. B. Tóth, Beáta et al., Regulatory modules of human thermogenic adipocytes: functional genomics of large cohort and Meta-analysis derived marker-genes. __BMC Genomics__ (2021). [Link](https://doi.org/doi:10.1186/S12864-021-08126-8).
195. Bang, Jeyoung et al., Identification of Signaling Pathways for Early Embryonic Lethality and Developmental Retardation in Sephs1 -/- Mice. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222111647).
196. Biani, M.C. et al., Small extracellular vesicles from iPSC-MSC lose their regenerative potential upon UV-C irradiation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.27.461979).
197. Bono, Hidemasa, Meta-analysis of oxidative transcriptomes in insects. __Antioxidants__ (2021). [Link](https://doi.org/doi:10.3390/antiox10030345).
198. C, Feng et al., PFKFB4 is overexpressed in clear-cell renal cell carcinoma promoting pentose phosphate pathway that mediates Sunitinib resistance. __J Exp Clin Cancer Res__ (2021). [Link](https://doi.org/doi:10.1186/S13046-021-02103-5).
199. CA, Anene et al., ACSNI: An unsupervised machine-learning tool for prediction of tissue-specific pathway components using gene expression profiles. __Patterns__ (2021). [Link](https://doi.org/doi:10.1016/J.PATTER.2021.100270).
200. Centenera, Margaret M. et al., ELOVL5 is a critical and targetable fatty acid elongase in prostate cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-2511).
201. Cerase, Andrea et al., Chd8 regulates X chromosome inactivation in mouse through fine-tuning control of Xist expression. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-01945-1).
202. Chen, Haobin et al., BET Bromodomain Inhibitors Target the NEUROD1-subtype SCLC by Blocking NEUROD1 Transactivation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.25.465771).
203. Chen, Ming et al., Integrative Analyses Reveal Novel Disease-associated Loci and Genes for Idiopathic Pulmonary Fibrosis. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.11.21257064).
204. Choi, Sangyong et al., Identification of a Putative Enhancer RNA for EGFR in Hyper-Accessible Regions in Esophageal Squamous Cell Carcinoma Cells by Analysis of Chromatin Accessibility Landscapes. __Front Oncol__ (2021). [Link](https://doi.org/doi:10.3389/FONC.2021.724687).
205. Cofer, Evan M. et al., Modeling transcriptional regulation of model species with deep learning. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/gr.266171.120).
206. De Smedt, Jonathan et al., PU.1 drives specification of pluripotent stem cell-derived endothelial cells to LSEC-like cells. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-020-03356-2).
207. Dhaka, Bhavya et al., Differential chromatin accessibility landscape of gain-of-function mutant p53 tumours. __BMC Cancer__ (2021). [Link](https://doi.org/doi:10.1186/s12885-021-08362-x).
208. Droin, Colas et al., Space-time logic of liver gene expression at sub-lobular scale. __Nat Metab__ (2021). [Link](https://doi.org/doi:10.1038/s42255-020-00323-1).
209. Durand, Sébastien et al., RSL24D1 sustains steady-state ribosome biogenesis and pluripotency translational programs in embryonic stem cells. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.27.443845).
210. E, Mammarella et al., NUAK2 and RCan2 participate in the p53 mutant pro-tumorigenic network. __Biol Direct__ (2021). [Link](https://doi.org/doi:10.1186/S13062-021-00296-5).
211. Feng, Haizhou et al., Identification of core miRNAs and regulatory pathways in breast cancer by integrated bioinformatics analysis. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/d0mo00171f).
212. Feng, Lei et al., Comprehensive Analysis of E3 Ubiquitin Ligases Reveals Ring Finger Protein 223 as a Novel Oncogene Activated by KLF4 in Pancreatic Cancer. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.738709).
213. Feng, Peiyuan et al., A machine learning-based framework for modeling transcription elongation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/pnas.2007450118).
214. Fernandez Lahore, Gonzalo et al., Polymorphic estrogen receptor binding site causes Cd2-dependent sex bias in the susceptibility to autoimmune diseases. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25828-5).
215. Franco, Elisa De et al., Primate-specific ZNF808 is essential for pancreatic development in humans. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.23.21262262).
216. Fujino, Kan et al., A human endogenous bornavirus-like nucleoprotein encodes a mitochondrial protein associated with cell viability. __J Virol__ (2021). [Link](https://doi.org/doi:10.1128/JVI.02030-20).
217. Fujino, Shiki et al., HNF1A regulates colorectal cancer progression and drug resistance as a downstream of POU5F1. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89126-2).
218. G, Devailly et al., Comprehensive analysis of epigenetic signatures of human transcription control. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/D0MO00130A).
219. Gallo, Alberto et al., NF-Y subunits overexpression in gastric adenocarcinomas (STAD). __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-03027-Y).
220. Gharavi, Erfaneh et al., Embeddings of genomic region sets capture rich biological associations in lower dimensions. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB439).
221. H, Doi et al., Early-life midazolam exposure persistently changes chromatin accessibility to impair adult hippocampal neurogenesis and cognition. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2107596118).
222. H, Mizuno et al., Evi1 upregulates Fbp1 and supports progression of acute myeloid leukemia through pentose phosphate pathway activation. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/CAS.15098).
223. H, Ohguchi et al., Lysine Demethylase 5A is Required for MYC Driven Transcription in Multiple Myeloma. __Blood Cancer Discov__ (2021). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0108).
224. Hamano, Momoko et al., Prediction of single-cell mechanisms for disease progression in hypertrophic remodelling by a trans-omics approach. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86821-y).
225. Hammal, Fayrouz et al., ReMap 2022: a database of Human, Mouse, Drosophila and Arabidopsis regulatory regions from an integrative analysis of DNA-binding sequencing experiments. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB996).
226. Hanaki, Shunsuke et al., PP1 regulatory subunit NIPP1 regulates transcription of E2F1 target genes following DNA damage. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/cas.14924).
227. Harada, Yukihiro et al., ETS‐dependent enhancers for endothelial‐specific expression of serum/glucocorticoid‐regulated kinase 1 during mouse embryo development. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/gtc.12874).
228. He, Lingli et al., An optimized two-step chromatin immunoprecipitation protocol to quantify the associations of two separate proteins and their common target DNA. __STAR Protoc__ (2021). [Link](https://doi.org/doi:10.1016/j.xpro.2021.100504).
229. Hernández-Nava, Elizabeth et al., Transcriptional and Epigenetic Bioinformatic Analysis of Claudin-9 Regulation in Gastric Cancer. __J Oncol__ (2021). [Link](https://doi.org/doi:10.1155/2021/5936905).
230. Hiebert, Paul, The Nrf2 transcription factor: A multifaceted regulator of the extracellular matrix. __Matrix Biol Plus__ (2021). [Link](https://doi.org/doi:10.1016/j.mbplus.2021.100057).
231. Hisamatsu, Yoji et al., Matrix Metalloproteinases in Human Decidualized Endometrial Stromal Cells. __Curr Issues Mol Biol__ (2021). [Link](https://doi.org/doi:10.3390/CIMB43030146).
232. Hisano, Osamu et al., Short single-stranded DNA with putative non-canonical structures comprises a novel class of plasma cell-free DNA. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.25.445702).
233. Hisano, Osamu et al., Short single-stranded DNAs with putative non-canonical structures comprise a new class of plasma cell-free DNA. __BMC Biol__ (2021). [Link](https://doi.org/doi:10.1186/S12915-021-01160-8).
234. Hosoi, Hiroki et al., Super-enhancers for RUNX3 are required for cell proliferation in EBV-infected B cell lines. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145421).
235. Hosoya, Tadashi et al., Chondroprotective effects of CDK4/6 inhibition via enhanced ubiquitin-dependent degradation of JUN in synovial fibroblasts. __Baillieres Clin Rheumatol__ (2021). [Link](https://doi.org/doi:10.1093/RHEUMATOLOGY/KEAB874).
236. Hossain, Sk Md Mosaddek et al., Discovering key transcriptomic regulators in pancreatic ductal adenocarcinoma using Dirichlet process Gaussian mixture model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87234-7).
237. Huang, Chia Chi Flora et al., Functional mapping of androgen receptor enhancer activity. __Genome Biol__ (2021). [Link](https://doi.org/doi:10.1186/s13059-021-02339-6).
238. Hudecova, Irena et al., Characteristics, origin, and potential for cancer diagnostics of ultrashort plasma cell-free DNA. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/GR.275691.121).
239. Inazumi, Hideaki et al., NRSF-GNAO1 Pathway Contributes to the Regulation of Cardiac Ca 2+ Homeostasis. __Circ Res__ (2021). [Link](https://doi.org/doi:10.1161/CIRCRESAHA.121.318898).
240. J, Dahlqvist et al., Systematic identification of genomic elements that regulate FCGR2A expression and harbor variants linked with autoimmune disease. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB372).
241. J, Studd et al., Cancer drivers and clonal dynamics in acute lymphoblastic leukaemia subtypes. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-366981/V1).
242. J, Wutschka et al., JUNB suppresses distant metastasis by influencing the initial metastatic stage. __Clin Exp Metastasis__ (2021). [Link](https://doi.org/doi:10.1007/S10585-021-10108-9).
243. Jang, Minjeong et al., Matrix stiffness epigenetically regulates the oncogenic activation of the Yes-associated protein in gastric cancer. __Nat Biomed Eng__ (2021). [Link](https://doi.org/doi:10.1038/s41551-020-00657-x).
244. Jarvis, Lorna B. et al., Therapeutically expanded human regulatory T-cells are super-suppressive due to HIF1A induced expression of CD73. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02721-X).
245. Jia, Yunlu et al., Super enhancer-mediated upregulation of HJURP promotes growth and survival of t(4;14)-positive multiple myeloma. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-0921).
246. Jiang, Jiayue-Clara et al., Integrated transcription factor profiling with transcriptome analysis identifies L1PA2 transposons as global regulatory modulators in a breast cancer model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86395-9).
247. Jiang, Jiayue-Clara et al., Widespread Exaptation of L1 Transposons for Transcription Factor Binding in Breast Cancer. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115625).
248. Jiang, Xinliang et al., Heat shock induces the nuclear accumulation of YAP1 via SRC. __Exp Cell Res__ (2021). [Link](https://doi.org/doi:10.1016/j.yexcr.2020.112439).
249. Jin, Xin et al., Smoking-associated upregulation of CBX3 suppresses ARHGAP24 expression to activate Rac1 signaling and promote tumor progression in lung adenocarcinoma. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/S41388-021-02114-8).
250. Jung, Sascha et al., A computer-guided design tool to increase the efficiency of cellular conversions. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/s41467-021-21801-4).
251. Jungwirth, Emilian et al., Meta-analysis and Consolidation of Farnesoid X Receptor Chromatin Immunoprecipitation Sequencing Data Across Different Species and Conditions. __Hepatol Commun__ (2021). [Link](https://doi.org/doi:10.1002/HEP4.1749).
252. K, Kitakaze et al., ATF4-mediated transcriptional regulation protects against β-cell loss during endoplasmic reticulum stress in a mouse model. __Mol Metab__ (2021). [Link](https://doi.org/doi:10.1016/J.MOLMET.2021.101338).
253. K, Nishimura et al., Profiling of Androgen-Dependent Enhancer RNAs Expression in Human Prostate Tumors: Search for Malignancy Transition Markers. __Res Rep Urol__ (2021). [Link](https://doi.org/doi:10.2147/RRU.S328661).
254. K, Sofiadis et al., HMGB1 coordinates SASP-related chromatin folding and RNA homeostasis on the path to senescence. __Mol Syst Biol__ (2021). [Link](https://doi.org/doi:10.15252/MSB.20209760).
255. Kaiser, Marina et al., Regulation of otocyst patterning by <i>Tbx2</i> and <i>Tbx3</i> is required for inner ear morphogenesis in the mouse. __Development__ (2021). [Link](https://doi.org/doi:10.1242/dev.195651).
256. Kalman, Zsofia E et al., A preliminary study on the cistrome of human postsynaptic density from an evolutionary and network-based perspective. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.01.25.428072).
257. Kanai, Masahiro et al., Insights from complex trait fine-mapping across diverse populations. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.03.21262975).
258. Kanamori, Yohei et al., Iron-rich Kupffer cells exhibit phenotypic changes during the development of liver fibrosis in NASH. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/j.isci.2020.102032).
259. Katada, Sayako et al., Neural stem/precursor cells dynamically change their epigenetic landscape to differentially respond to BMP signaling for fate switching during brain development. __Genes Dev__ (2021). [Link](https://doi.org/doi:10.1101/GAD.348797.121).
260. Kidwell, Ashley et al., Translation rescue by targeting Ppp1r15a upstream open reading frame in vivo. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.12.11.472232).
261. Kikutake, Chie et al., Pan-cancer analysis of non-coding recurrent mutations and their possible involvement in cancer pathogenesis. __NAR Cancer__ (2021). [Link](https://doi.org/doi:10.1093/narcan/zcab008).
262. Kim, Jessica et al., The RNA helicase DDX6 controls early mouse embryogenesis by repressing aberrant inhibition of BMP signaling through miRNA-mediated gene silencing. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.11.29.470397).
263. Kohrogi, Kensaku et al., LSD1 de fi nes erythroleukemia metabolism by controlling the lineage-speci fi c transcription factors GATA1 and C / EBP a. __Blood Adv__ (2021). [Link](https://doi.org/doi:10.1182/bloodadvances.2020003521).
264. Koike, Yuka et al., Age-related demethylation of the TDP-43 autoregulatory region in the human motor cortex. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02621-0).
265. Kosinsky, Robyn Laura et al., RNF20 and RNF40 regulate vitamin D receptor-dependent signaling in inflammatory bowel disease. __Cell Death Differ__ (2021). [Link](https://doi.org/doi:10.1038/s41418-021-00808-w).
266. Kwon, Andrew Tae-Jun et al., Development of p53 Knockout U87MG Cell Line for Unbiased Drug Delivery Testing System Using CRISPR-Cas9 and Transcriptomic Analysis. __J Biotechnol__ (2021). [Link](https://doi.org/doi:10.1016/j.jbiotec.2021.03.017).
267. L, Wang et al., The Ubiquitin Sensor and Adaptor Protein p62 Mediates Signal Transduction of a Viral Oncogenic Pathway. __mBio__ (2021). [Link](https://doi.org/doi:10.1128/MBIO.01097-21).
268. Labade, Ajay S. et al., Nup93 and CTCF modulate spatiotemporal dynamics and function of the HOXA gene locus during differentiation. __J Cell Sci__ (2021). [Link](https://doi.org/doi:10.1242/JCS.259307).
269. Lee, Hsiu-Chi et al., YAP1 overexpression contributes to the development of enzalutamide resistance by induction of cancer stemness and lipid metabolism in prostate cancer. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/s41388-021-01718-4).
270. Lee, Shin-Ae et al., METTL8 mRNA Methyltransferase Enhances Cancer Cell Migration via Direct Binding to ARID1A. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115432).
271. Li, Jie-Ning et al., Expression of SnoRNA U50A Is Associated with Better Prognosis and Prolonged Mitosis in Breast Cancer. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13246304).
272. Liu, Yi et al., Tumors exploit FTO-mediated regulation of glycolytic metabolism to evade immune surveillance. __Cell Metab__ (2021). [Link](https://doi.org/doi:10.1016/j.cmet.2021.04.001).
273. Luginbühl, Joachim et al., Decoding Neuronal Diversification by Multiplexed Single-cell RNA-Seq. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/j.stemcr.2021.02.006).
274. M, Francia et al., SUMO conjugation susceptibility of Akt/protein kinase B affects the expression of the pluripotency transcription factor Nanog in embryonic stem cells. __PLoS One__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0254447).
275. M, Saito et al., PHLDA3 is an important downstream mediator of p53 in squamous cell carcinogenesis. __J Invest Dermatol__ (2021). [Link](https://doi.org/doi:10.1016/J.JID.2021.09.007).
276. M, Yamagishi et al., Chronological genome and single-cell transcriptome integration characterizes the evolutionary process of adult T cell leukemia-lymphoma. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25101-9).
277. Ma, Shiyang et al., Powerful gene-based testing by integrating long-range chromatin interactions and knockoff genotypes. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.14.21260405).
278. Ma, Wenjing et al., BARTweb: a web server for transcriptional regulator association analysis. __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqab022).
279. Markouli, C. et al., Sustained intrinsic WNT and BMP4 activation impairs hESC differentiation to definitive endoderm and drives the cells towards extra-embryonic mesoderm. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87547-7).
280. Marques, Ines J. et al., Downregulation of WT1 transcription factor gene expression is required to promote myocardial fate. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.06.451274).
281. Matsuura, Norihiro et al., NOTCH3 limits the epithelial–mesenchymal transition and predicts a favorable clinical outcome in esophageal cancer. __Cancer Med__ (2021). [Link](https://doi.org/doi:10.1002/cam4.3933).
282. Milanesi, Elena et al., Whole Blood Expression Pattern of Inflammation and Redox Genes in Mild Alzheimer's Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S334337).
283. Minnoye, Liesbeth et al., Chromatin accessibility profiling methods. __Nat Rev Methods Primers__ (2021). [Link](https://doi.org/doi:10.1038/s43586-020-00008-9).
284. Misawa, Aya et al., Long Noncoding RNA HOXA11-AS and Transcription Factor HOXB13 Modulate the Expression of Bone Metastasis-Related Genes in Prostate Cancer. __Genes__ (2021). [Link](https://doi.org/doi:10.3390/genes12020182).
285. Miyake, Masato et al., Integrated stress response regulates GDF15 secretion from adipocytes, preferentially suppresses appetite for a high-fat diet and improves obesity. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103448).
286. Miyazawa, Kazuo et al., Trans-ancestry genome-wide analysis of atrial fibrillation provides new insights into disease biology and enables polygenic prediction of cardioembolic risk. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.06.21263189).
287. Morishita, Mayu et al., Characterization of mouse embryonic fibroblasts derived from Rassf6 knockout mice shows the implication of Rassf6 in the regulation of NF-κB signaling. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/GTC.12901).
288. Morrow, Alyssa Kramer et al., Epitome: predicting epigenetic events in novel cell types with multi-cell deep ensemble learning. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB676).
289. Mourao, Larissa et al., Hyperactive WNT/CTNNB1 signaling induces a competing cell proliferation and epidermal differentiation response in the mouse mammary epithelium. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.22.449461).
290. N, Osada et al., mTOR inhibitors sensitize multiple myeloma cells to venetoclax via IKZF3-and Blimp-1-mediated BCL-2 up-regulation. __Haematologica__ (2021). [Link](https://doi.org/doi:10.3324/HAEMATOL.2021.278506).
291. Nagai, Yasuki et al., Glucotoxicity-induced suppression of Cox6a2 expression provokes β-cell dysfunction via augmented ROS production. __Biochem Biophys Res Commun__ (2021). [Link](https://doi.org/doi:10.1016/j.bbrc.2021.03.148).
292. Namba, Shinichi et al., Transcript-targeted analysis reveals isoform alterations and double-hop fusions in breast cancer. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02833-4).
293. Nurtdinov, Ramil et al., Alu-mediated weak CEBPA binding and slow B cell transdifferentiation in human. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.28.466072).
294. Ogino, Takashi et al., Post-transcriptional repression of circadian component CLOCK regulates cancer-stemness in murine breast cancer cells. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/eLife.66155).
295. Oh, Taek In et al., PGC1α Loss Promotes Lung Cancer Metastasis through Epithelial-Mesenchymal Transition. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13081772).
296. Omiya, Hanae et al., BMP signaling suppresses Gemc1 expression and ependymal differentiation of mouse telencephalic progenitors. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79610-6).
297. P, McErlean et al., DNA Methylome Alterations are Associated with Airway Macrophage Differentiation and Phenotype During Lung Fibrosis. __Am J Respir Crit Care Med__ (2021). [Link](https://doi.org/doi:10.1164/RCCM.202101-0004OC).
298. P, Weidemüller et al., Transcription factors: Bridge between cell signaling and gene regulation. __Proteomics__ (2021). [Link](https://doi.org/doi:10.1002/PMIC.202000034).
299. PM, Tyler et al., Human autoinflammatory disease reveals ELF4 as a transcriptional regulator of inflammation. __Nat Immunol__ (2021). [Link](https://doi.org/doi:10.1038/S41590-021-00984-4).
300. Papagiannopoulos, Christos I et al., The histone methyltransferase inhibitor A-366 enhances hemoglobin expression in erythroleukemia cells upon co-exposure with chemical inducers in culture. __J Biol Res__ (2021). [Link](https://doi.org/doi:10.1186/s40709-020-00132-3).
301. Pecori, Federico et al., A defined glycosylation regulatory network modulates total glycome dynamics during pluripotency state transition. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79666-4).
302. Petrone, María Victoria et al., The pluripotency transcription factor OCT4 represses heme oxygenase-1 gene expression. __FEBS Lett__ (2021). [Link](https://doi.org/doi:10.1002/1873-3468.14138).
303. Piette, Benjamin L. et al., Comprehensive interactome profiling of the human Hsp70 network highlights functional differentiation of J domains. __Mol Cell__ (2021). [Link](https://doi.org/doi:10.1016/j.molcel.2021.04.012).
304. Q, Luo et al., SIGNET: single-cell RNA-seq-based gene regulatory network prediction using multiple-layer perceptron bagging. __Brief Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/BIB/BBAB547).
305. Quinn, Hazel M et al., YAP and β-catenin cooperate to drive oncogenesis in basal breast cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.can-20-2801).
306. RC, Zapata et al., Conserved immunomodulatory transcriptional networks underlie antipsychotic-induced weight gain. __Transl Psychiatry__ (2021). [Link](https://doi.org/doi:10.1038/S41398-021-01528-Y).
307. Reimegård, Johan et al., A combined approach for single-cell mRNA and intracellular protein expression analysis. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-02142-w).
308. Ruiz, José L et al.,  The regulatory genome of the malaria vector Anopheles gambiae : integrating chromatin accessibility and gene expression . __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqaa113).
309. S, Han et al., Targeting ATF4-dependent pro-survival autophagy to synergize glutaminolysis inhibition. __Theranostics__ (2021). [Link](https://doi.org/doi:10.7150/THNO.60028).
310. S, Li et al., Long noncoding RNA HOTAIR interacts with Y-Box Protein-1 (YBX1) to regulate cell proliferation. __Life Sci Alliance__ (2021). [Link](https://doi.org/doi:10.26508/LSA.202101139).
311. S, Mohr et al., The alternative serotonin transporter promoter P2 impacts gene function in females with irritable bowel syndrome. __J Cell Mol Med__ (2021). [Link](https://doi.org/doi:10.1111/JCMM.16736).
312. SY, Wu et al., MYC suppresses STING-dependent innate immunity by transcriptionally upregulating DNMT1 in triple-negative breast cancer. __J Immunother Cancer__ (2021). [Link](https://doi.org/doi:10.1136/JITC-2021-002528).
313. Sarkar, Arijita et al., Genome-wide DNA methylation and multi-omics study of human chondrocyte ontogeny and an epigenetic clock analysis of adult chondrocytes. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.02.454544).
314. Sekita, Yoichi et al., AKT signaling is associated with epigenetic reprogramming via the upregulation of TET and its cofactor, alpha-ketoglutarate during iPSC generation. __Stem Cell Res Ther__ (2021). [Link](https://doi.org/doi:10.1186/S13287-021-02578-1).
315. Sharma, Abhay, Inferring molecular mechanisms of dexamethasone therapy in severe COVID-19 from existing transcriptomic data. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145665).
316. Shimizu, Tatsuya et al., Osteocytes as main responders to low-intensity pulsed ultrasound treatment during fracture healing. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89672-9).
317. Shin, Jihoon et al., Possible Involvement of Adipose Tissue in Patients With Older Age, Obesity, and Diabetes With Coronavirus SARS-CoV-2 Infection (COVID-19) via GRP78 (BIP/HSPA5): Significance of Hyperinsulinemia Management in COVID-19. __Diabetes__ (2021). [Link](https://doi.org/doi:10.2337/DB20-1094).
318. Shinohara, Mitsuru et al., Upregulated expression of a subset of genes in <i>APP</i> ; <i>ob</i> / <i>ob</i> mice: Evidence of an interaction between diabetes‐linked obesity and Alzheimer's disease. __FASEB Bioadv__ (2021). [Link](https://doi.org/doi:10.1096/fba.2020-00151).
319. Shirai, Yuya et al., Elucidation of disease etiology by trans-layer omics analysis. __Inflamm Regen__ (2021). [Link](https://doi.org/doi:10.1186/s41232-021-00155-w).
320. Singh, Isha et al., Heterogeneity in the Epigenetic Landscape of Murine Testis-Specific Histone Variants TH2A and TH2B Sharing the Same Bi-Directional Promoter. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.755751).
321. Stirparo, Giuliano G. et al., OCT4 induces embryonic pluripotency via STAT3 signaling and metabolic mechanisms. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2008890118).
322. Stuparević, Igor et al., Regulation of the conserved 3′-5′ exoribonuclease EXOSC10/Rrp6 during cell division, development and cancer. __Biol Rev__ (2021). [Link](https://doi.org/doi:10.1111/brv.12693).
323. Su, Emily Y et al., Reconstruction of dynamic regulatory networks reveals signaling-induced topology 3 changes associated with germ layer specification 4 5 6. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.06.443021).
324. Sun, Yaping et al., Oleic Acid and Eicosapentaenoic Acid Reverse Palmitic Acid-induced Insulin Resistance in Human HepG2 Cells via the Reactive Oxygen Species / JUN Pathway. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/j.gpb.2019.06.005).
325. T, Sawada et al., Androgen-dependent and DNA binding-independent association of androgen receptor with chromatic regions coding androgen-induced non-coding RNAs. __Biosci Biotechnol Biochem__ (2021). [Link](https://doi.org/doi:10.1093/BBB/ZBAB135).
326. T, Yao et al., Saikosaponin-D Alleviates Renal Inflammation and Cell Apoptosis in a Mouse Model of Sepsis via TCF7/FOSL1/MMP9 Inhibition. __Mol Cell Biol__ (2021). [Link](https://doi.org/doi:10.1128/MCB.00332-21).
327. Takai, Jun et al., Gata2 heterozygous mutant mice exhibit reduced inflammatory responses and impaired bacterial clearance. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.102836).
328. Takeda, Takashi et al., A stem cell marker KLF5 regulates CCAT1 via three-dimensional genome structure in colorectal cancer cells. __Br J Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41416-021-01579-4).
329. Takeuchi, Yasuto et al., The membrane-linked adaptor FRS2β fashions a cytokine-rich inflammatory microenvironment that promotes breast cancer carcinogenesis. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2103658118).
330. Tan, Deng et al., Quantitative control of noise in mammalian gene expression by dynamic histone regulation. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/ELIFE.65654).
331. Teschendorff, Andrew E et al., Statistical mechanics meets single-cell biology. __Nat Rev Genet__ (2021). [Link](https://doi.org/doi:10.1038/s41576-021-00341-z).
332. Tian, Yijun et al., Novel role of prostate cancer risk variant rs7247241 on PPP1R14A isoform transition through allelic TF binding and CpG methylation. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB347).
333. Umarov, Ramzan et al., DeepCellState: An autoencoder-based framework for predicting cell type specific transcriptional states induced by drug treatment. __PLoS Comput Biol__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009465).
334. Unno, Kenji et al., Activated ALK Cooperates with N-Myc via Wnt/β-catenin Signaling to Induce Neuroendocrine Prostate Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-3351).
335. Wan, Ruosi et al., Direct Reconstruction of Gene Regulatory Networks underlying Cellular state Transitions without Pseudo-time Inference. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.12.443928).
336. Wang, Jun et al., HPV E7 affects the function of cervical cancer cells via the TAL1/lnc‑EBIC/KLHDC7B axis. __Oncol Rep__ (2021). [Link](https://doi.org/doi:10.3892/or.2021.8002).
337. Wang, Liang et al., ASXL1 promotes adrenocortical carcinoma and is associated with chemoresistance to EDP regimen. __Aging__ (2021). [Link](https://doi.org/doi:10.18632/AGING.203534).
338. Wei, Kai-Che et al., An innovative targeted therapy for fluoroscopy-induced chronic radiation dermatitis. __J Mol Med__ (2021). [Link](https://doi.org/doi:10.1007/S00109-021-02146-3).
339. Wen, Li et al., Selective EZH2 inhibitor zld1039 alleviates inflammation in cisplatin-induced acute kidney injury partially by enhancing RKIP and suppressing NF-κB p65 pathway. __Zhongguo Yao Li Xue Bao__ (2021). [Link](https://doi.org/doi:10.1038/S41401-021-00837-8).
340. Wierzbicka, Justyna M. et al., The Effects of Vitamin D on the Expression of IL-33 and Its Receptor ST2 in Skin Cells; Potential Implication for Psoriasis. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222312907).
341. Wu, Beibei et al., ORAI1 establishes resistance to SARS-CoV-2 infection by regulating tonic type I interferon signaling. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.04.442548).
342. X, Chen et al., Poly-L-arginine promotes asthma angiogenesis through induction of FGFBP1 in airway epithelial cells via activation of the mTORC1-STAT3 pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04055-2).
343. X, Li et al., Defining Proximity Proteomics of Histone Modifications by Antibody-mediated Protein A-APEX2 Labeling. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/J.GPB.2021.09.003).
344. X, Zhao et al., Combinatorial CRISPR/Cas9 Screening Reveals Epistatic Networks of Interacting Tumor Suppressor Genes and Therapeutic Targets in Human Breast Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-2555).
345. Xiang, Tian et al., The novel ZEB1-upregulated protein PRTG induced by Helicobacter pylori infection promotes gastric carcinogenesis through the cGMP/PKG signaling pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-021-03440-1).
346. Y, Hayashi et al., Norrie disease protein is essential for cochlear hair cell maturation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2106369118).
347. Y, Ono et al., Multi-Omic Meta-Analysis of Transcriptomes and the Bibliome Uncovers Novel Hypoxia-Inducible Genes. __Biomedicines__ (2021). [Link](https://doi.org/doi:10.3390/BIOMEDICINES9050582).
348. Y, Zhang et al., TcoFBase: a comprehensive database for decoding the regulatory transcription co-factors in human and mouse. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB950).
349. YH, Chen et al., The Induction of Alpha-1 Antitrypsin by Vitamin D in Human T Cells Is TGF-β Dependent: A Proposed Anti-inflammatory Role in Airway Disease. __Front Nutr__ (2021). [Link](https://doi.org/doi:10.3389/FNUT.2021.667203).
350. Yagi, Mikako et al., Mitochondrial translation deficiency impairs NAD <sup>+</sup> ‐mediated lysosomal acidification. __EMBO J__ (2021). [Link](https://doi.org/doi:10.15252/embj.2020105268).
351. Yao, Xiaolei et al., TP53/miR-129/MDM2/4/TP53 feedback loop modulates cell proliferation and apoptosis in retinoblastoma. __Cell Cycle__ (2021). [Link](https://doi.org/doi:10.1080/15384101.2021.1892321).
352. Yazar, Volkan et al., Integrative genome-wide analysis of dopaminergic neuron-specific PARIS expression in Drosophila dissects recognition of multiple PPAR-γ associated gene regulation. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-00858-7).
353. Yi, Lu et al., The FOXM1/RNF26/p57 axis regulates the cell cycle to promote the aggressiveness of bladder cancer. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04260-Z).
354. Yi, Xianfu et al., Interrogating cell type-specific cooperation of transcriptional regulators in 3D chromatin. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103468).
355. Yoneda, Mitsuhiro et al., Nucleosome assembly protein 1 (NAP-1) is a regulator of histone H1 acetylation. __J Biochem__ (2021). [Link](https://doi.org/doi:10.1093/JB/MVAB098).
356. Yoshizaki, Kaichi et al., Paternal age affects offspring via an epigenetic mechanism involving REST/NRSF. __EMBO Rep__ (2021). [Link](https://doi.org/doi:10.15252/embr.202051524).
357. Zampieri, Carlotta et al., p53 mutations define the chromatin landscape to confer drug tolerance in pancreatic cancer. __Mol Oncol__ (2021). [Link](https://doi.org/doi:10.1002/1878-0261.13161).
358. Zelenka, Tomas et al., The 3D enhancer network of the developing T cell genome is controlled by SATB1. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.09.451769).
359. Zhang, Shu et al., RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. __Sci Adv__ (2021). [Link](https://doi.org/doi:10.1126/SCIADV.ABG8205).
360. Zhao, Xiaolu et al., CircleBase: an integrated resource and analysis platform for human eccDNAs. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB1104).
361. Zhao, Ye et al., Identification of Unique Transcriptomic Signatures and Hub Genes Through RNA Sequencing and Integrated WGCNA and PPI Network Analysis in Nonerosive Reflux Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S340452).
362. de Roover, Astrid et al., Hypoxia induces DOT1L in articular cartilage to protect against osteoarthritis. __JCI Insight__ (2021). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.150451).
363. Łasut-Szyszka, Barbara et al., Transcriptome Analysis of Cells Exposed to Actinomycin D and Nutlin-3a Reveals New Candidate p53-Target Genes and Indicates That CHIR-98014 Is an Important Inhibitor of p53 Activity. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222011072).
364. AS, Knaupp et al., TINC- A Method to Dissect Regulatory Complexes at Single-Locus Resolution- Reveals an Extensive Protein Complex at the Nanog Promoter. __Stem Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.STEMCR.2020.11.005).
365. Ahmed, Mahmoud et al., Integrating binding and expression data to predict transcription factors combined function. __BMC Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12864-020-06977-1).
366. Azim, Kasum et al., Resolving the transcriptional transitions associated with oligodendrocyte generation from adult neural stem cells by 2 single cell sequencing Running Title: Oligodendroglial transcriptional networks 4 5 6. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.12.18.423285).
367. B, Zhu et al., Mi-2β-targeted inhibition induces immunotherapy response in melanoma. __Research Square__ (2020). [Link](https://doi.org/doi:10.21203/RS.3.RS-118153/V1).
368. Beknazarov, Nazar et al., Deep learning approach for predicting functional Z-DNA regions using omics data. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-76203-1).
369. Bono, Hidemasa et al., Meta-analysis of hypoxic transcriptomes from public databases. __Biomedicines__ (2020). [Link](https://doi.org/doi:10.3390/biomedicines8010010).
370. Booth, Lauren N et al., Males deploy multifaceted strategies and hijack longevity pathways to induce premature demise of the opposite sex. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.30.181008).
371. Bunina, Daria et al., Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons. __Cell Syst__ (2020). [Link](https://doi.org/doi:10.1016/j.cels.2020.05.003).
372. Caruana, Beth T et al., The NF-κB signalling pathway regulates GLUT6 expression in endometrial cancer. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109688).
373. Cataldo, Alessandra et al., MiR-302b as a Combinatorial Therapeutic Approach to Improve Cisplatin Chemotherapy Efficacy in Human Triple-Negative Breast Cancer. __Cancers__ (2020). [Link](https://doi.org/doi:10.3390/cancers12082261).
374. Cavalli, Marco et al., A Multi-Omics Approach to Liver Diseases: Integration of Single Nuclei Transcriptomics with Proteomics and HiCap Bulk Data in Human Liver. __OMICS__ (2020). [Link](https://doi.org/doi:10.1089/omi.2019.0215).
375. Charmpi, Konstantina et al., Convergent network effects along the axis of gene expression during prostate cancer progression. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02188-9).
376. Chauvistré, Heike et al., The Janus-faced role of KDM5B heterogeneity in melanoma: differentiation as a situational driver of both growth arrest and drug-resistance. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.01.999847).
377. Chen, Carol C L et al., Histone H3.3G34-Mutant Interneuron Progenitors Co-opt PDGFRA for Gliomagenesis. __Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.cell.2020.11.012).
378. Chen, Hengxing et al., PARK2 promotes mitochondrial pathway of apoptosis and antimicrotubule drugs chemosensitivity via degradation of phospho-BCL-2. __Theranostics__ (2020). [Link](https://doi.org/doi:10.7150/thno.47044).
379. Chen, Jiaxin et al., LncSEA: a platform for long non-coding RNA related sets and enrichment analysis. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa806).
380. Choi, Jang Hyun et al., Histone demethylase KDM4D cooperates with NFIB and MLL1 complex to regulate adipogenic differentiation of C3H10T1/2 mesenchymal stem cells. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-60049-8).
381. Choudhury, Abrar et al., Meningioma epigenetic grouping reveals biologic drivers and therapeutic vulnerabilities. __medRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.11.23.20237495).
382. Czipa, Erik et al., ChIPSummitDB: a ChIP-seq-based database of human transcription factor binding sites and the topological arrangements of the proteins bound to them. __Database__ (2020). [Link](https://doi.org/doi:10.1093/database/baz141).
383. Devailly, Guillaume et al., Comprehensive analysis of epigenetic signatures of human transcrip-tion control †. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.23.309625).
384. Dorotea, Debra et al., Recent Insights Into SREBP as a Direct Mediator of Kidney Fibrosis via Lipid-Independent Pathways. __Front Pharmacol__ (2020). [Link](https://doi.org/doi:10.3389/fphar.2020.00265).
385. Farahmand, Saman et al., ModEx: A text mining system for extracting mode of regulation of transcription factor-gene regulatory interaction. __J Biomed Inform__ (2020). [Link](https://doi.org/doi:10.1016/j.jbi.2019.103353).
386. Feng, Chenchen et al., KnockTF: A comprehensive human gene expression profile database with knockdown/knockout of transcription factors. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz881).
387. Fornes, Oriol et al., JASPAR 2020: Update of the open-Access database of transcription factor binding profiles. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1001).
388. Gao, Lihua et al., Ozone therapy promotes the differentiation of basal keratinocytes via increasing Tp63-mediated transcription of KRT10 to improve psoriasis. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15160).
389. Gerisch, Birgit et al., HLH-30/TFEB Is a Master Regulator of Reproductive Quiescence. __Dev Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.devcel.2020.03.014).
390. Grujic, Olivera et al., Identification and characterization of constrained non-exonic bases lacking predictive epigenomic and transcription factor binding annotations. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-19962-9).
391. Guan, Shuhong et al., TGF-β1 induces CREB1-mediated miR-1290 upregulation to antagonize lung fibrosis via Napsin A. __Int J Mol Med__ (2020). [Link](https://doi.org/doi:10.3892/ijmm.2020.4565).
392. Hayashi, Akimasa et al., A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma. __Nat Cancer__ (2020). [Link](https://doi.org/doi:10.1038/s43018-019-0010-1).
393. Hayashi, Akimasa et al., Evolutionary Dynamics of Non-Coding Regions in Pancreatic Ductal Adenocarcinoma. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.11.294389).
394. Hirano, Kazumi et al., Indonesian ginger (Bangle) extract promotes neurogenesis of human neural stem cells through WNT pathway activation. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134772).
395. Horn, Moritz et al., Hexosamine Pathway Activation Improves Protein Homeostasis through the Integrated Stress Response. __iScience__ (2020). [Link](https://doi.org/doi:10.1016/j.isci.2020.100887).
396. Hsiao, Wen-Yu et al., The Lipid Handling Capacity of Subcutaneous Fat Is Programmed by mTORC2 during Development. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108223).
397. Hänsel-Hertsch, Robert et al., Landscape of G-quadruplex DNA structural regions in breast cancer. __Nat Genet__ (2020). [Link](https://doi.org/doi:10.1038/s41588-020-0672-8).
398. Kaminuma, Eli et al., DDBJ data analysis challenge: A machine learning competition to predict arabidopsis chromatin feature annotations from DNA sequences. __Genes Genet Syst__ (2020). [Link](https://doi.org/doi:10.1266/ggs.19-00034).
399. Katahira, Jun et al., Human THO maintains the stability of repetitive DNA. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12760).
400. Kikuyama, Takahiro et al., 25(OH)D3 stimulates the expression of vitamin D target genes in renal tubular cells when Cyp27b1 is abrogated. __J Steroid Biochem Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jsbmb.2020.105593).
401. Kim, Yu Jin et al., UBE2C Overexpression Aggravates Patient Outcome by Promoting Estrogen-Dependent/Independent Cell Proliferation in Early Hormone Receptor-Positive and HER2-Negative Breast Cancer. __Front Oncol__ (2020). [Link](https://doi.org/doi:10.3389/fonc.2019.01574).
402. Koga, Tomoaki et al., Expression of leukotriene B4 receptor 1 defines functionally distinct DCs that control allergic skin inflammation. __Cell Mol Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41423-020-00559-7).
403. Kohmoto, Tomohiro et al., Claudin-6 is a single prognostic marker and functions as a tumor-promoting gene in a subgroup of intestinal type gastric cancer. __Gastric Cancer__ (2020). [Link](https://doi.org/doi:10.1007/s10120-019-01014-x).
404. Kokaji, Toshiya et al., Transomics analysis reveals allosteric and gene regulation axes for altered hepatic glucose-responsive metabolism in obesity. __Sci Signal__ (2020). [Link](https://doi.org/doi:10.1126/SCISIGNAL.AAZ1236).
405. Kolmykov, Semyon et al., GTRD: an integrated view of transcription regulation. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa1057).
406. Kremsky, Isaac et al., Protection from DNA re-methylation by transcription factors in primordial germ cells and pre-implantation embryos can explain trans-generational epigenetic inheritance. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02036-w).
407. Krześniak, Małgorzata et al., Synergistic activation of p53 by actinomycin D and nutlin-3a is associated with the upregulation of crucial regulators and effectors of innate immunity. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109552).
408. Kubota, Naoto et al., An integrated analysis of public genomic data unveils a possible functional mechanism of psoriasis risk via a long-range ERRFI1 enhancer. __BMC Med Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12920-020-0662-9).
409. Kutscher, Lena et al., Functional loss of a non-canonical BCOR-PRC1.1 complex accelerates SHH-driven medulloblastoma formation. __Genes Dev__ (2020). [Link](https://doi.org/doi:10.1101/2020.02.06.938035).
410. Kwiecien, Kamila et al., The methylation status of the chemerin promoter region located from − 252 to + 258 bp regulates constitutive but not acute-phase cytokine-inducible chemerin expression levels. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-70625-7).
411. Kwon, Andrew Tae-Jun et al., Efficient Development of Platform Cell Lines Using CRISPR-Cas9 and Transcriptomics Analysis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.16.299248).
412. L, He et al., A Regulation Loop between YAP and NR4A1 Balances Cell Proliferation and Apoptosis. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.CELREP.2020.108284).
413. L, Jarvis et al., Therapeutically expanded human regulatory T-cells are super- suppressive due to HIF1A induced expression of CD73. __Research Square__ (2020). [Link](https://doi.org/doi:10.21203/RS.3.RS-92789/V1).
414. LM, Koch et al., Cytosolic pH regulates proliferation and tumour growth by promoting expression of cyclin D1. __Nat Metab__ (2020). [Link](https://doi.org/doi:10.1038/S42255-020-00297-0).
415. Larsen, Erik G. et al., TMEM184B controls pruriceptor specification and function. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.01.25.919902).
416. Lei, Xiao et al., Crystal Structures of Ternary Complexes of MEF2 and NKX2–5 Bound to DNA Reveal a Disease Related Protein–Protein Interaction Interface. __J Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jmb.2020.07.004).
417. Leporcq, Clémentine et al., TFmotifView: a webserver for the visualization of transcription factor motifs in genomic regions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa252).
418. Li, Xin et al., ERRα activates SHMT2 transcription to enhance the resistance of breast cancer to lapatinib via modulating the mitochondrial metabolic adaption. __Biosci Rep__ (2020). [Link](https://doi.org/doi:10.1042/BSR20192465).
419. Li, Yanyu et al., TRlnc: a comprehensive database for human transcriptional regulatory information of lncRNAs. __Brief Bioinform__ (2020). [Link](https://doi.org/doi:10.1093/bib/bbaa011).
420. Li, Yi-Han et al., FGF9 is a downstream target of SRY and sufficient to determine male sex fate in ex vivo XX gonad culture. __Biol Reprod__ (2020). [Link](https://doi.org/doi:10.1093/biolre/ioaa154).
421. Li, Zhiqing et al., Runx2 (Runt-Related Transcription Factor 2)-Mediated Microcalcification Is a Novel Pathological Characteristic and Potential Mediator of Abdominal Aortic Aneurysm. __Arterioscler Thromb Vasc Biol__ (2020). [Link](https://doi.org/doi:10.1161/ATVBAHA.119.314113).
422. Lin, Quy Xiao Xuan et al., TFregulomeR reveals transcription factors' context-specific features and functions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1088).
423. Lorenzo, Jose Paulo et al., APOBEC2 binds Chromatin and Represses Transcription during Myoblast Differentiation. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.07.29.223594).
424. Lou, Shaoke et al., TopicNet: a framework for measuring transcriptional regulatory network change. __Bioinformatics__ (2020). [Link](https://doi.org/doi:10.1093/bioinformatics/btaa403).
425. Lyu, Junfang et al., Synthetic lethality of RB1 and aurora A is driven by stathmin-mediated disruption of microtubule dynamics. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18872-0).
426. Maharjan, Mukesh et al., Overlapping but Distinct Sequences Play Roles in the Insulator and Promoter Activities of the Drosophila BEAF-Dependent scs' Insulator. __Genetics__ (2020). [Link](https://doi.org/doi:10.1534/genetics.120.303344).
427. Mao, Ninghui et al., Oncogenic ERG represses PI3K signaling through downregulation of IRS2. __Cancer Res__ (2020). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-19-1394).
428. McErlean, Peter et al., Profiling of H3K27Ac Reveals the Influence of Asthma on the Epigenome of the Airway Epithelium. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.585746).
429. Meng, Yuhong et al., Transcriptomic landscape profiling of metformin-treated healthy mice: Implication for potential hypertension risk when prophylactically used. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15472).
430. Mercatelli, Daniele et al., Gene regulatory network inference resources: A practical overview. __Biochim Biophys Acta Gene Regul Mech__ (2020). [Link](https://doi.org/doi:10.1016/j.bbagrm.2019.194430).
431. Merrett, James E et al., Identification of DNA response elements regulating expression of CCAAT/enhancer-binding protein (C/EBP) β and δ and MAP kinase-interacting kinases during early adipogenesis. __Adipocyte__ (2020). [Link](https://doi.org/doi:10.1080/21623945.2020.1796361).
432. Miyamoto, Yoichi et al., Genetic loss of importin α4 causes abnormal sperm morphology and impacts on male fertility in mouse. __FASEB J__ (2020). [Link](https://doi.org/doi:10.1096/fj.202000768RR).
433. Nam, A. Reum et al., Alternative methylation of intron motifs is associated with cancer-related gene expression in both canine mammary tumor and human breast cancer. __Clin Epigenetics__ (2020). [Link](https://doi.org/doi:10.1186/s13148-020-00888-4).
434. Namekawa, Takeshi et al., ALDH1A1 in patient-derived bladder cancer spheroids activates retinoic acid signaling leading to TUBB3 overexpression and tumor progression. __Int J Cancer__ (2020). [Link](https://doi.org/doi:10.1002/ijc.32505).
435. Namekawa, Takeshi et al., HIF1α inhibitor 2-methoxyestradiol decreases NRN1 expression and represses in vivo and in vitro growth of patient-derived testicular germ cell tumor spheroids. __Cancer Lett__ (2020). [Link](https://doi.org/doi:10.1016/j.canlet.2020.05.040).
436. Ochiai, Kyoko et al., Chromatin Protein PC4 Orchestrates B Cell Differentiation by Collaborating with IKAROS and IRF4. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108517).
437. Ochsner, Scott A et al., A transcriptional regulatory atlas of coronavirus infection of human cells. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.24.059527).
438. Ochsner, Scott A. et al., Consensus transcriptional regulatory networks of coronavirus-infected human cells. __Sci Data__ (2020). [Link](https://doi.org/doi:10.1038/s41597-020-00628-6).
439. Ohishi, Hiroaki et al., Characterization of genetic-origin-dependent monoallelic expression in mouse embryonic stem cells. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12736).
440. Okada, Ryo et al., RNaseH2A Downregulation Drives Chromosomal DNA Fragmentation and Accumulation of RNA-DNA Hybrids in Senescent Cells. __SSRN__ (2020). [Link](https://doi.org/doi:10.2139/ssrn.3588831).
441. Okazaki, Yuri et al., Increased glucose metabolism in Arid5b-/- skeletal muscle is associated with the down-regulation of TBC1 domain family member 1 (TBC1D1). __Biol Res__ (2020). [Link](https://doi.org/doi:10.1186/s40659-020-00313-3).
442. P, Sahlén et al., Chromatin interactions in differentiating keratinocytes reveal novel atopic dermatitis- and psoriasis-associated genes. __J Allergy Clin Immunol__ (2020). [Link](https://doi.org/doi:10.1016/J.JACI.2020.09.035).
443. Pecori, Federico et al., Mucin-type O-glycosylation controls pluripotency in mouse embryonic stem cells via Wnt receptor endocytosis. __J Cell Sci__ (2020). [Link](https://doi.org/doi:10.1242/jcs.245845).
444. Pradhananga, Sailendra et al., Promoter anchored interaction landscape of THP-1 macrophages captures early immune response processes. __Cell Immunol__ (2020). [Link](https://doi.org/doi:10.1016/j.cellimm.2020.104148).
445. Pratapa, Aditya et al., Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data. __Nat Methods__ (2020). [Link](https://doi.org/doi:10.1038/s41592-019-0690-6).
446. Q, Pan et al., VARAdb: a comprehensive variation annotation database for human. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/NAR/GKAA922).
447. Ray, John P et al., Prioritizing disease and trait causal variants at the TNFAIP3 locus using functional and genomic features. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-15022-4).
448. Rikitake, Marika et al., Analysis of GPI-anchored proteins involved in germline stem cell proliferation in the Caenorhabditis elegans germline stem cell niche. __J Biochem__ (2020). [Link](https://doi.org/doi:10.1093/jb/mvaa075).
449. Roels, Juliette et al., Aging of preleukemic thymocytes drives CpG island hypermethylation in T-cell acute lymphoblastic leukemia. __Blood Cancer Discov__ (2020). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0059).
450. Roels, Juliette et al., Distinct and temporary-restricted epigenetic mechanisms regulate human αβ and γδ T cell development. __Nat Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41590-020-0747-9).
451. Ronzio, Mirko et al., Integrating Peak Colocalization and Motif Enrichment Analysis for the Discovery of Genome-Wide Regulatory Modules and Transcription Factor Recruitment Rules. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00072).
452. SN, Reisinger et al., STAT3 in the dorsal raphe gates behavioural reactivity and regulates gene networks associated with psychopathology. __Mol Psychiatry__ (2020). [Link](https://doi.org/doi:10.1038/S41380-020-00904-2).
453. Sakuratani, Takuji et al., Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2020). [Link](https://doi.org/doi:10.1007/s00428-020-02899-1).
454. Sharipov, Ruslan N et al., Assessment of transcriptional importance of cell line-specific features based on GTRD and FANTOM5 data. __PLoS One__ (2020). [Link](https://doi.org/doi:10.1371/journal.pone.0243332).
455. Shikov, Anton E et al., Phenome-wide functional dissection of pleiotropic effects highlights key molecular pathways for human complex traits. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-58040-4).
456. Sugasawa, Takehito et al., Influence of intermittent cold stimulations on CREB and its targeting genes in muscle: Investigations into molecular mechanisms of local cryotherapy. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134588).
457. Sukumaran, Abitha et al., Insight on Transcriptional Regulation of the Energy Sensing AMPK and Biosynthetic mTOR Pathway Genes. __Front Cell Dev Biol__ (2020). [Link](https://doi.org/doi:10.3389/fcell.2020.00671).
458. Swoboda, Alexander et al., STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pathway. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01584-6).
459. Tamura, Isao et al., Transcription factor C/EBPβ induces genome-wide H3K27ac and upregulates gene expression during decidualization of human endometrial stromal cells. __Molec Cell Endocr__ (2020). [Link](https://doi.org/doi:10.1016/j.mce.2020.111085).
460. Tanaka, Hiroshi et al., The NSD2/WHSC1/MMSET methyltransferase prevents cellular senescence-associated epigenomic remodeling. __Aging Cell__ (2020). [Link](https://doi.org/doi:10.1111/acel.13173).
461. Teschendorff, Andrew E et al., Improved detection of tumor suppressor events in single-cell RNA-Seq data. __NPJ Genom Med__ (2020). [Link](https://doi.org/doi:10.1038/s41525-020-00151-y).
462. Teschendorff, Andrew E. et al., EPISCORE: Cell type deconvolution of bulk tissue DNA methylomes from single-cell RNA-Seq data. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02126-9).
463. Thonsri, Unchalee et al., High glucose-ROS conditions enhance the progression in cholangiocarcinoma via upregulation of MAN2A2 and CHD8. __Cancer Sci__ (2020). [Link](https://doi.org/doi:10.1111/cas.14719).
464. Toro, Ayelén et al., Novel Interplay between p53 and HO-1 in Embryonic Stem Cells. __Cells__ (2020). [Link](https://doi.org/doi:10.3390/cells10010035).
465. Totani, Haruhito et al., Autocrine HGF/c-Met signaling pathway confers aggressiveness in lymph node adult T-cell leukemia/lymphoma. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01393-x).
466. Ueda, Mahoko Takahashi et al., Comprehensive genomic analysis reveals dynamic evolution of endogenous retroviruses that code for retroviral-like protein domains. __Mob DNA__ (2020). [Link](https://doi.org/doi:10.1186/s13100-020-00224-w).
467. Wang, Allen et al., Single-cell multiomic profiling of human lungs reveals cell-type-specific and age-dynamic control of SARS-CoV2 host genes. __Elife__ (2020). [Link](https://doi.org/doi:10.7554/ELIFE.62522).
468. Wang, Rong et al., Transient IGF-1R inhibition combined with osimertinib eradicates AXL-low expressing EGFR mutated lung cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18442-4).
469. Wu, Xue et al., Regulation of cellular sterol homeostasis by the oxygen responsive noncoding RNA lincNORS. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18411-x).
470. X, Li et al., The role of MYB proto-oncogene like 2 in tamoxifen resistance in breast cancer. __J Mol Histol__ (2020). [Link](https://doi.org/doi:10.1007/S10735-020-09920-6).
471. Y, Li et al., Single-Cell Transcriptome Analysis Reveals Dynamic Cell Populations and Differential Gene Expression Patterns in Control and Aneurysmal Human Aortic Tissue. __Circulation__ (2020). [Link](https://doi.org/doi:10.1161/CIRCULATIONAHA.120.046528).
472. Y, Sera et al., UTX maintains functional integrity of murine hematopoietic system by globally regulating aging-associated genes. __Blood__ (2020). [Link](https://doi.org/doi:10.1182/BLOOD.2019001044).
473. Y, Yasukawa et al., SAA1 is upregulated in gastric cancer-associated fibroblasts possibly by its enhancer activation. __Carcinogenesis__ (2020). [Link](https://doi.org/doi:10.1093/CARCIN/BGAA131).
474. Yumimoto, Kanae et al., Recent insight into the role of FBXW7 as a tumor suppressor. __Semin Cancer Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.semcancer.2020.02.017).
475. Zhao, Wei et al., Epigenetic Regulation of m6A Modifications in Human Cancer. __Mol Ther Nucleic Acids__ (2020). [Link](https://doi.org/doi:10.1016/j.omtn.2019.11.022).
476. Zhou, Stanley et al., Noncoding mutations target cis-regulatory elements of the FOXA1 plexus in prostate cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-14318-9).
477. Zhou, Yao et al., epiCOLOC: Integrating Large-Scale and Context-Dependent Epigenomics Features for Comprehensive Colocalization Analysis. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00053).
478. Abugessaisa, Imad et al., refTSS: A Reference Data Set for Human and Mouse Transcription Start Sites. __J Mol Biol__ (2019). [Link](https://doi.org/doi:10.1016/j.jmb.2019.04.045).
479. Ahn, Hongryul et al., Propanet: Time-varying condition-specific transcriptional network construction by network propagation. __Front Plant Sci__ (2019). [Link](https://doi.org/doi:10.3389/fpls.2019.00698).
480. Ando, Mizuo et al., Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-09937-w).
481. Aono, Shiori et al., β-catenin/TCF4 complex-mediated induction of the NRF3 (NFE2L3) gene in cancer cells. __Int J Mol Sci__ (2019). [Link](https://doi.org/doi:10.3390/ijms20133344).
482. Arata, Yoshiyuki et al., Defective induction of the proteasome associated with T-cell receptor signaling underlies T-cell senescence. __Genes Cells__ (2019). [Link](https://doi.org/doi:10.1111/gtc.12728).
483. Bakhmet, Evgeny I et al., hnRNP-K Targets Open Chromatin in Mouse Embryonic Stem Cells in Concert with Multiple Regulators. __Stem Cells__ (2019). [Link](https://doi.org/doi:10.1002/stem.3025).
484. Banerjee, Sharmi et al., Identifying transcriptional regulatory modules among different chromatin states in mouse neural stem cells. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
485. Bocci, Matteo et al., Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__ (2019). [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
486. Bravo González-Blas, Carmen et al., cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. __Nat Methods__ (2019). [Link](https://doi.org/doi:10.1038/s41592-019-0367-1).
487. Chang, Hao Chun et al., Investigating the role of super-enhancer RNAs underlying embryonic stem cell differentiation. __BMC Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12864-019-6293-x).
488. Chereji, Răzvan V et al., Quantitative MNase-seq accurately maps nucleosome occupancy levels. __Genome Biol__ (2019). [Link](https://doi.org/doi:10.1186/s13059-019-1815-z).
489. Chow, Kwan T. et al., IRF5 regulates unique subset of genes in dendritic cells during West Nile virus infection. __J Leukoc Biol__ (2019). [Link](https://doi.org/doi:10.1002/JLB.MA0318-136RRR).
490. Delicado, Diana et al., Multilocus phylogeny, species delimitation and biogeography of Iberian valvatiform springsnails (Caenogastropoda: Hydrobiidae), with the description of a new genus. __Zool J Linn Soc__ (2019). [Link](https://doi.org/doi:10.1093/zoolinnean/zly093).
491. Duforestel, Manon et al., Glyphosate primes mammary cells for tumorigenesis by reprogramming the epigenome in a TET3-dependent manner. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2019.00885).
492. Farahmand, Saman et al., Causal Inference Engine: A platform for directional gene set enrichment analysis and inference of active transcriptional regulators. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz1046).
493. Ferris, Elliott et al., Parallel Accelerated Evolution in Distant Hibernators Reveals Candidate Cis Elements and Genetic Circuits Regulating Mammalian Obesity. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.10.102).
494. Fleming, Aaron M. et al., Location dependence of the transcriptional response of a potential G-quadruplex in gene promoters under oxidative stress. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz207).
495. Fontela, Miguel G et al., The conserved non-coding sequence 2 (CNS2) enhances CD69 transcription through cooperation between the transcription factors oct1 and RUNX1. __Genes__ (2019). [Link](https://doi.org/doi:10.3390/genes10090651).
496. Horie, Kenta et al., Down-regulation of GATA1-dependent erythrocyte-related genes in the spleens of mice exposed to a space travel. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-44067-9).
497. Hotta, Masahiro et al., Farnesoid X receptor induces cell death and sensitizes to TRAIL-induced inhibition of growth in colorectal cancer cells through the up-regulation of death receptor 5. __Biochem Biophys Res Commun__ (2019). [Link](https://doi.org/doi:10.1016/j.bbrc.2019.09.033).
498. Hurtz, Christian et al., Rationale for targeting BCL6 in MLL-rearranged acute lymphoblastic leukemia. __Genes Dev__ (2019). [Link](https://doi.org/doi:10.1101/gad.327593.119).
499. Ishijima, Yasushi et al., The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Molec Cell Endocr__ (2019). [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
500. Kang, Hyeji et al., HSF1 regulates mevalonate and cholesterol biosynthesis pathways. __Cancers__ (2019). [Link](https://doi.org/doi:10.3390/cancers11091363).
501. Kehl, Tim et al., The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int J Cancer__ (2019). [Link](https://doi.org/doi:10.1002/ijc.31834).
502. Kikuchi, Masataka et al., Enhancer variants associated with Alzheimer's disease affect gene expression via chromatin looping. __BMC Med Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12920-019-0574-8).
503. Klein, Jason C et al., Functional testing of thousands of osteoarthritis-associated variants for regulatory activity. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-10439-y).
504. Kojima, Machiko et al., The histone methyltransferase WHSC1 is regulated by EZH2 and is important for ovarian clear cell carcinoma cell proliferation. __BMC Cancer__ (2019). [Link](https://doi.org/doi:10.1186/s12885-019-5638-9).
505. Kolmykov, Semyon K et al., Population size estimation for quality control of ChIP-Seq datasets. __PLoS One__ (2019). [Link](https://doi.org/doi:10.1371/journal.pone.0221760).
506. Kondo, Takefumi et al., Two-step regulation of trachealess ensures tight coupling of cell fate with morphogenesis in the drosophila trachea. __Elife__ (2019). [Link](https://doi.org/doi:10.7554/eLife.45145).
507. Krug, Brian et al., Pervasive H3K27 Acetylation Leads to ERV Expression and a Therapeutic Vulnerability in H3K27M Gliomas. __Cancer Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.ccell.2019.04.004).
508. Kulyté, Agné et al., MicroRNA-27a/b-3p and PPARG regulate SCAMP3 through a feed-forward loop during adipogenesis. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-50210-3).
509. Kunii, Yasuto et al., Differential protein expression of DARPP-32 versus Calcineurin in the prefrontal cortex and nucleus accumbens in schizophrenia and bipolar disorder. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-51456-7).
510. Kuno, Akihiro et al., Conserved DNA-binding motif loci reflect functional transcription factor binding sites. __Research Square__ (2019). [Link](https://doi.org/doi:10.21203/rs.2.15866/v1).
511. Lin, Siyuan et al., Transcriptome and DNA methylome signatures associated with retinal Müller glia development, injury response, and aging. __Invest Ophthalmol Vis Sci__ (2019). [Link](https://doi.org/doi:10.1167/iovs.19-27361).
512. Lizio, Marina et al., Update of the FANTOM web resource: Expansion to provide additional transcriptome atlases. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1099).
513. Maedera, Shotaro et al., GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13298).
514. Matsuda, Taito et al., Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__ (2019). [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
515. Menzel, Michael et al., Enhort: A platform for deep analysis of genomic positions. __PeerJ Comput Sci__ (2019). [Link](https://doi.org/doi:10.7717/peerj-cs.198).
516. Murayama, Yuki et al., Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13328).
517. Neiman, Gabriel et al., Integrin alpha-5 subunit is critical for the early stages of human pluripotent stem cell cardiac differentiation. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-54352-2).
518. Ochsner, Scott A et al., The Signaling Pathways Project, an integrated ‘omics knowledgebase for mammalian cellular signaling pathways. __Sci Data__ (2019). [Link](https://doi.org/doi:10.1038/s41597-019-0193-4).
519. Oulès, Bénédicte et al., Mutant Lef1 controls Gata6 in sebaceous gland development and cancer. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018100526).
520. Peggion, Caterina et al., The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol Neurobiol__ (2019). [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
521. Qian, Feng Cui et al., SEanalysis: A web tool for super-enhancer associated regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz302).
522. Ranjit, Melissa et al., Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
523. Rauschmeier, René et al., Bhlhe40 and Bhlhe41 transcription factors regulate alveolar macrophage self‐renewal and identity. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018101233).
524. Robinson, Claire M et al., AHypoxia-inducible HIF1-GAL3ST1-sulfatide axis enhances ccRCC immune evasion via increased tumor cell-platelet binding. __Mol Cancer Res__ (2019). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-19-0461).
525. Saito, Megumi et al., A Polymorphic Variant in p19Arf Confers Resistance to Chemically Induced Skin Tumors by Activating the p53 Pathway. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
526. Salameti, Vasiliki et al., NOTCH1 signaling in oral squamous cell carcinoma via a TEL2/ SERPINE1 axis. __Oncotarget__ (2019). [Link](https://doi.org/doi:10.18632/oncotarget.27306).
527. Simak, Maria et al., Boolean function network analysis of time course liver transcriptome data to reveal novel circadian transcriptional regulators in mammals. __J Chin Med Assoc__ (2019). [Link](https://doi.org/doi:10.1097/JCMA.0000000000000180).
528. Singh, Rajbir et al., Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
529. Solari, Claudia et al., The pluripotency transcription factor Nanog represses glutathione reductase gene expression in mouse embryonic stem cells. __BMC Res Notes__ (2019). [Link](https://doi.org/doi:10.1186/s13104-019-4411-0).
530. Studd, James B et al., Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__ (2019). [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
531. Suda, Kojiro et al., Reduction of Rpd3 suppresses defects in locomotive ability and neuronal morphology induced by the knockdown of Drosophila SLC25A46 via an epigenetic pathway. __Exp Cell Res__ (2019). [Link](https://doi.org/doi:10.1016/j.yexcr.2019.111673).
532. Tano, Ayami et al., The juvenility-associated long noncoding RNA Gm14230 maintains cellular juvenescence. __J Cell Sci__ (2019). [Link](https://doi.org/doi:10.1242/jcs.227801).
533. Tong, Zhan et al., TransmiR v2.0: An updated transcription factor-microRNA regulation database. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1023).
534. Ulirsch, Jacob C et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __Nat Genet__ (2019). [Link](https://doi.org/doi:10.1038/s41588-019-0362-6).
535. Velychko, Sergiy et al., Excluding Oct4 from Yamanaka Cocktail Unleashes the Developmental Potential of iPSCs. __Cell Stem Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.stem.2019.10.002).
536. Vijayakrishnan, Jayaram et al., Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-13069-6).
537. Vogel, Felix C E et al., Targeting the H3K4 Demethylase KDM5B Reprograms the Metabolome and Phenotype of Melanoma Cells. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2019.06.124).
538. Wu, Cheng Jang et al., MiR-23~27~24–mediated control of humoral immunity reveals a TOX-driven regulatory circuit in follicular helper T cell differentiation. __Sci Adv__ (2019). [Link](https://doi.org/doi:10.1126/sciadv.aaw1715).
539. Wu, Zhaoming et al., Cis-control of Six1 expression in neural crest cells during craniofacial development. __Dev Dyn__ (2019). [Link](https://doi.org/doi:10.1002/dvdy.109).
540. Yamada, Toshimichi et al., Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __WIREs RNA__ (2019). [Link](https://doi.org/doi:10.1002/wrna.1508).
541. Yashiro, Takuya et al., PU.1 plays a pivotal role in dendritic cell migration from the periphery to secondary lymphoid organs via regulating CCR7 expression. __FASEB J__ (2019). [Link](https://doi.org/doi:10.1096/fj.201900379RR).
542. Yevshin, Ivan et al., GTRD: A database on gene transcription regulation - 2019 update. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1128).
543. Yoshizaki, Kaichi et al., Paternal Age Affects Offspring'S Behavior Possibly Via Epigenetic Mechanism Recruiting a Transcriptional Repressor Rest. __Eur Neuropsychopharmacol__ (2019). [Link](https://doi.org/doi:10.1016/j.euroneuro.2017.08.089).
544. Zhang, Lei et al., DeepHINT: Understanding HIV-1 integration via deep learning with attention. __Bioinformatics__ (2019). [Link](https://doi.org/doi:10.1093/bioinformatics/bty842).
545. Zheng, Rongbin et al., Cistrome Data Browser: Expanded datasets and new tools for gene regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1094).
546. Åkerborg, Örjan et al., High-Resolution Regulatory Maps Connect Vascular Risk Variants to Disease-Related Pathways. __Circulation__ (2019). [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
547. Anan, Kotaro et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gky234).
548. Baldi, Sandro et al., Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol Cell__ (2018). [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
549. Chatterjee, Shatakshee et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res Treat__ (2018). [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
550. Chen, Yu et al., PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci__ (2018). [Link](https://doi.org/doi:10.1111/cas.13796).
551. Chishima, Takafumi et al., Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes__ (2018). [Link](https://doi.org/doi:10.3390/genes9010023).
552. Chèneby, Jeanne et al., ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx1092).
553. Cui, Chunmei et al., Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics Proteomics Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
554. Dréos, René et al., MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx995).
555. Egashira, Yoshihiro et al., Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
556. Ferris, Elliott et al., Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
557. Fiziev, Petko et al., ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
558. Groff, Abigail F. et al., Enhancers in the Peril lincRNA locus regulate distant but not local genes. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1589-8).
559. Imrichova, Hana et al., ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/388934).
560. Kawaji, Hideya, dirHub: a trackHub configurator with directory structure projection. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/314807).
561. Kawata, Kentaro et al., Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__ (2018). [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
562. Kehl, Tim et al., REGGAE: A novel approach for the identification of key transcriptional regulators. __Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
563. Lasar, David et al., Peroxisome Proliferator Activated Receptor Gamma Controls Mature Brown Adipocyte Inducibility through Glycerol Kinase. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2017.12.067).
564. Mao, Fengbiao et al., EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx918).
565. Misawa, Aya et al., lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif Tissue Int__ (2018). [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
566. Miura, Itaru et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J Psychiatr Res__ (2018). [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
567. Mochizuki, Kentaro et al., SETDB1 is essential for mouse primordial germ cell fate determination by ensuring bmp signaling. __Development__ (2018). [Link](https://doi.org/doi:10.1242/dev.164160).
568. Mourad, Raphaël et al., Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
569. Nishizawa, Yujiro et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__ (2018). [Link](https://doi.org/doi:10.18632/oncotarget.23554).
570. Onodera, Yuta et al., Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__ (2018). [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
571. Polouliakh, Natalia et al., Sequence homology in eukaryotes (SHOE): Interactive visual tool for promoter analysis 06 Biological Sciences 0604 Genetics. __BMC Genomics__ (2018). [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
572. Raffield, Laura M et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet__ (2018). [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
573. Romero-Cordoba, Sandra L et al., Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
574. Wang, Huabo et al., Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J Biol Chem__ (2018). [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
575. Chen, Yuting et al., Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol__ (2017). [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
576. Govaere, O et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__ (2017). [Link](https://doi.org/doi:10.1038/onc.2017.260).
577. Guoa, Michael H et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc Natl Acad Sci U S A__ (2017). [Link](https://doi.org/doi:10.1073/pnas.1619052114).
578. Ishigaki, Kazuyoshi et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat Genet__ (2017). [Link](https://doi.org/doi:10.1038/ng.3885).
579. Kalender Atak, Zeynep et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med__ (2017). [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
580. Kehl, Tim et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkx350).
581. Matsuda, Kazunari et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__ (2017). [Link](https://doi.org/doi:10.1242/dev.143479).
582. Naderi, Ali, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__ (2017). [Link](https://doi.org/doi:10.18632/oncotarget.17826).
583. Onodera, Yuta et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__ (2017). [Link](https://doi.org/doi:10.1111/acel.12680).
584. Sanosaka, Tsukasa et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
585. Tanegashima, Kosuke et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes Cells__ (2017). [Link](https://doi.org/doi:10.1111/gtc.12456).
586. Turrell, Frances K et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev__ (2017). [Link](https://doi.org/doi:10.1101/gad.298463.117).
587. Umeyama, Taichi et al., DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
588. Yevshin, Ivan et al., GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkw951).
589. Yoshihara, Masahito et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
590. Sugeno, Naoto et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci Rep__ (2016). [Link](https://doi.org/doi:10.1038/srep36328).
