## To cite ChIP-Atlas in your publication:

- Paper
  - Zou, Z., Ohta, T., Miura, F., Oki, S. ChIP-Atlas 2021 update: a data-mining suite for exploring epigenomic landscapes by fully integrating ChIP-seq, ATAC-seq and Bisulfite-seq data. __Nucleic Acids Res__. 50(W1), W175-W182, 2022. http://dx.doi.org/10.1093/nar/gkac199
  - Oki, S., Ohta, T., Shioi, G., Hatanaka, H., Ogasawara, O., Okuda, Y., Kawaji, H., Nakaki, R., Sese, J., Meno, C. ChIP-Atlas: a data-mining suite powered by full integration of public ChIP-seq data. __EMBO Rep__. 19(12), e46255, 2018. http://dx.doi.org/10.15252/embr.201846255
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. https://chip-atlas.org

## Publications citing ChIP-Atlas:

1. Anzawa, H., et al. C4S DB: Comprehensive Collection and Comparison for ChIP-Seq Database. __J Mol Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.JMB.2023.168157).
2. Arimoto, K., et al. Expansion of interferon inducible gene pool via USP18 inhibition promotes cancer cell pyroptosis. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-022-35348-5).
3. Asavasupreechar, T., et al. PRB inhibited cell proliferation through let-7b-E2F1 in breast cancer. __Endocr Relat Cancer__ (2023). [Link](https://doi.org/doi:10.1530/ERC-22-0204).
4. Azzi, A., et al. The circadian clock protein Cryptochrome 1 is a direct target and feedback regulator of the Hippo pathway. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.107449).
5. BP, P., et al. Pcgf1 gene disruption reveals primary involvement of epigenetic mechanism in neuronal subtype specification in the enteric nervous system. __Dev Growth Differ__ (2023). [Link](https://doi.org/doi:10.1111/DGD.12880).
6. Badia-i-Mompel, P., et al. Gene regulatory network inference in the era of single-cell multi-omics. __Nat Rev Genet__ (2023). [Link](https://doi.org/doi:10.1038/S41576-023-00618-5).
7. Bhardwaj, V., et al. HIV-1 Vpr induces ciTRAN to prevent transcriptional repression of the provirus. __Sci Adv__ (2023). [Link](https://doi.org/doi:10.1126/SCIADV.ADH9170).
8. Bhin, J., et al. MYC is a clinically significant driver of mTOR inhibitor resistance in breast cancer. __J Exp Med__ (2023). [Link](https://doi.org/doi:10.1084/JEM.20211743).
9. Bi, X., et al. Pan-cancer analyses reveal multi-omic signatures and clinical implementations of the forkhead-box gene family. __Cancer Med__ (2023). [Link](https://doi.org/doi:10.1002/CAM4.6312).
10. Bossardi Ramos, R., et al. Shock drives a STAT3 and JunB-mediated coordinated transcriptional and DNA methylation response in the endothelium. __J Cell Sci__ (2023). [Link](https://doi.org/doi:10.1242/JCS.261323).
11. Bu, Q., et al. Stanniocalcin 2 (STC2) is a potent biomarker of hepatocellular carcinoma with its expression being augmented in Nrf1α-deficient cells, but diminished in Nrf2-deficient cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.15.540796).
12. Bustin, KA.., et al. Phenelzine-based probes reveal Secernin-3 is involved in thermal nociception. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.02.526866).
13. C, B., et al. Transmicron: accurate prediction of insertion probabilities improves detection of cancer driver genes from transposon mutagenesis screens. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAC1215).
14. Castillo, Hé., et al. Architecture and evolutionary conservation of Xenopus tropicalis osteoblast-specific regulatory regions shed light on bone diseases and early skeletal evolution. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.30.555543).
15. Chandra, A., et al. Quantitative control of Ets1 dosage by a multi-enhancer hub promotes Th1 cell differentiation and protects from allergic inflammation. __Immunity__ (2023). [Link](https://doi.org/doi:10.1016/J.IMMUNI.2023.05.004).
16. Chandra, O., et al. Patterns of transcription factor binding and epigenome at promoters allow interpretable predictability of multiple functions of non-coding and coding genes. __Comput Struct Biotechnol J__ (2023). [Link](https://doi.org/doi:10.1016/J.CSBJ.2023.07.014).
17. Chen, L., et al. NPM3 as an unfavorable prognostic biomarker involved in oncogenic pathways of lung adenocarcinoma via MYC translational activation. __Comb Chem High Throughput Screen__ (2023). [Link](https://doi.org/doi:10.2174/1386207326666230419080531).
18. Chen, W., et al. Mechanic Insight into the Distinct and Common Roles of Ovariectomy Versus Adrenalectomy on Adipose Tissue Remodeling in Female Mice. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24032308).
19. Chen, WC., et al. Genome-wide association study of esophageal squamous cell cancer identifies shared and distinct risk variants in African and Chinese populations. __Am J Hum Genet__ (2023). [Link](https://doi.org/doi:10.1016/J.AJHG.2023.08.007).
20. Chow, CC.T.., et al. ZBTB2 is Recruited to a Specific Subset of HIF-1 Target Loci to Facilitate Full Gene Expression Under Hypoxia. __J Mol Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.JMB.2023.168162).
21. Chow, CN., et al. Databases and prospects of dynamic gene regulation in eukaryotes: A mini review. __Comput Struct Biotechnol J__ (2023). [Link](https://doi.org/doi:10.1016/J.CSBJ.2023.03.032).
22. Dai, C., et al. Comprehensive analysis of the diagnostic and therapeutic value of the hypoxia-related gene PLAUR in the progression of atherosclerosis. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-35548-Z).
23. Dong, L., et al. A functional crosstalk between the H3K9 methylation writers and their reader HP1 in safeguarding embryonic stem cell identity. __Stem Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.STEMCR.2023.08.004).
24. Escalante-Covarrubias, Q., et al. Time-of-day defines NAD+ efficacy to treat diet-induced metabolic disease by synchronizing the hepatic clock in mice. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-37286-2).
25. Etoh, K., et al. A web-based integrative transcriptome analysis, RNAseqChef, uncovers cell/tissue type-dependent action of sulforaphane. __J Biol Chem__ (2023). [Link](https://doi.org/doi:10.1016/J.JBC.2023.104810).
26. FC, Q., et al. SEanalysis 2.0: a comprehensive super-enhancer regulatory network analysis tool for human and mouse. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD408).
27. Feng, G., et al. CellPolaris: Decoding Cell Fate through Generalization Transfer Learning of Gene Regulatory Networks. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.25.559244).
28. Filipovic, D., et al. Interpretable predictive models of genome-wide aryl hydrocarbon receptor-DNA binding reveal tissue specific binding determinants. __Toxicol Sci__ (2023). [Link](https://doi.org/doi:10.1093/TOXSCI/KFAD094).
29. Freeman, TF.., et al. Ribosome biogenesis disruption mediated chromatin structure changes revealed by SRAtac, a customizable end to end analysis pipeline for ATAC-seq. __BMC Genomics__ (2023). [Link](https://doi.org/doi:10.1186/S12864-023-09576-Y).
30. Frenkel, M., et al. Discovering chromatin dysregulation induced by protein-coding perturbations at scale. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.20.555752).
31. Fu, Y., et al. Fam72a functions as a cell-cycle-controlled gene during proliferation and antagonizes apoptosis through reprogramming PP2A substrates. __Dev Cell__ (2023). [Link](https://doi.org/doi:10.1016/J.DEVCEL.2023.02.006).
32. Gao, Z., et al. Phase Separation Modulates the Formation and Stabilities of DNA Guanine Quadruplex. __JACS Au__ (2023). [Link](https://doi.org/doi:10.1021/JACSAU.3C00106).
33. García‐Hernández, V., et al. β-Catenin activity induces an RNA biosynthesis program promoting therapy resistance in T-cell acute lymphoblastic leukemia. __EMBO Mol Med__ (2023). [Link](https://doi.org/doi:10.15252/EMMM.202216554).
34. Ghaibour, K., et al. An Efficient Protocol for CUT&RUN Analysis of FACS-Isolated Mouse Satellite Cells. __J Vis Exp__ (2023). [Link](https://doi.org/doi:10.3791/65215).
35. Gong, B., et al. Epigenetic and transcriptional activation of the secretory kinase FAM20C as an oncogene in glioma. __J Genet Genom__ (2023). [Link](https://doi.org/doi:10.1016/J.JGG.2023.01.008).
36. Grey, W., et al. The CKS1/CKS2 Proteostasis Axis Is Crucial to Maintain Hematopoietic Stem Cell Function. __Hemasphere__ (2023). [Link](https://doi.org/doi:10.1097/HS9.0000000000000853).
37. Hagemann, S., et al. IGF2BP1 induces high-risk neuroblastoma and forms a druggable feedforward loop with MYCN promoting 17q oncogene expression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.18.533148).
38. Hagemann, S., et al. IGF2BP1 induces neuroblastoma via a druggable feedforward loop with MYCN promoting 17q oncogene expression. __Mol Cancer__ (2023). [Link](https://doi.org/doi:10.1186/S12943-023-01792-0).
39. Hamba, Y., et al. Topologically associating domain underlies tissue specific expression of long intergenic non-coding RNAs. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.106640).
40. Hanaki, S., et al. Impact of FKBP52 on cell proliferation and hormone-dependent cancers. __Cancer Sci__ (2023). [Link](https://doi.org/doi:10.1111/CAS.15811).
41. Harada, S., et al. The Lysophospholipase PNPLA7 Controls Hepatic Choline and Methionine Metabolism. __Biomolecules__ (2023). [Link](https://doi.org/doi:10.3390/BIOM13030471).
42. Herbert, A. Flipons and small RNAs accentuate the asymmetries of pervasive transcription by the reset and sequence-specific microcoding of promoter conformation. __J Biol Chem__ (2023). [Link](https://doi.org/doi:10.1016/J.JBC.2023.105140).
43. Hino, S., et al. Chromatin Immunoprecipitation Sequencing (ChIP-seq) for Detecting Histone Modifications and Modifiers. __Methods Mol Biol__ (2023). [Link](https://doi.org/doi:10.1007/978-1-0716-2724-2_4).
44. Hou, Y., et al. Transcriptome sequencing analysis reveals a potential role of lncRNA NONMMUT058932.2 and NONMMUT029203.2 in abnormal myelin development of male offspring following prenatal PM2.5 exposure. __Sci Total Environ__ (2023). [Link](https://doi.org/doi:10.1016/J.SCITOTENV.2023.165004).
45. Hsieh, Y‐T., et al. IFN-stimulated metabolite transporter ENT3 facilitates viral genome release. __EMBO Rep__ (2023). [Link](https://doi.org/doi:10.15252/EMBR.202255286).
46. Hu, Q., et al. GPX3 expression was down-regulated but positively correlated with poor outcome in human cancers. __Front Oncol__ (2023). [Link](https://doi.org/doi:10.3389/FONC.2023.990551).
47. Huang, D., et al. MAX transcriptionally enhances PD-L1 to inhibit CD8+ T cell-mediated killing of lung adenocarcinoma cells. __Cell Immunol__ (2023). [Link](https://doi.org/doi:10.1016/J.CELLIMM.2023.104706).
48. Huang, W., et al. NFAT and NF-κB dynamically co-regulate TCR and CAR signaling responses in human T cells. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112663).
49. Hughes, LA.., et al. Copy number variation in tRNA isodecoder genes impairs mammalian development and balanced translation. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-37843-9).
50. Huo, S., et al. ATF3/SPI1/SLC31A1 Signaling Promotes Cuproptosis Induced by Advanced Glycosylation End Products in Diabetic Myocardial Injury. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24021667).
51. Inagaki, T., et al. Prolonged Kaposi’s Sarcoma-associated Herpesvirus vIL-6 Exposure Enhances Inflammatory Responses by Epigenetic Reprogramming. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.06.25.546454).
52. Ishikawa, M., et al. Inference of gene regulatory networks using time-series single-cell RNA-seq data with CRISPR perturbations. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.23.554254).
53. Itakura, H., et al. Tumor-suppressive role of the musculoaponeurotic fibrosarcoma gene in colorectal cancer. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/j.isci.2023.106478).
54. Ito, S., et al. Androgen receptor binding sites enabling genetic prediction of mortality due to prostate cancer in cancer-free subjects. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-39858-8).
55. Iyer, DR.., et al. Cyclic-AMP response element binding protein (CREB) and microRNA miR-29b regulate renalase gene expression under catecholamine excess conditions. __Life Sci__ (2023). [Link](https://doi.org/doi:10.1016/J.LFS.2023.121859).
56. Iyer, P., et al. MGA deletion leads to Richter’s transformation via modulation of mitochondrial OXPHOS. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.07.527502).
57. Jain, L., et al. The growth hormone receptor interacts with transcriptional regulator HMGN1 upon GH-induced nuclear translocation. __J Cell Commun Signal__ (2023). [Link](https://doi.org/doi:10.1007/S12079-023-00741-2).
58. Janacova, L., et al. Catechol-O-methyl transferase suppresses cell invasion and interplays with MET signaling in estrogen dependent breast cancer. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-28078-1).
59. Kaller, M., et al. c-MYC-Induced AP4 Attenuates DREAM-Mediated Repression by p53. __Cancers__ (2023). [Link](https://doi.org/doi:10.3390/CANCERS15041162).
60. Kamimoto, K., et al. Dissecting cell identity via network inference and in silico gene perturbation. __Nature__ (2023). [Link](https://doi.org/doi:10.1038/S41586-022-05688-9).
61. Kawamura, G., et al. Optogenetic decoding of Akt2-regulated metabolic signaling pathways in skeletal muscle cells using transomics analysis. __Sci Signal__ (2023). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ABN0782).
62. Kernfeld, E., et al. Model-X knockoffs reveal data-dependent limits on regulatory network identification. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.23.541948).
63. Kirtana, R.., et al. KDM5A noncanonically binds antagonists MLL1/2 to mediate gene regulation and promotes epithelial to mesenchymal transition. __Biochim Biophys Acta__ (2023). [Link](https://doi.org/doi:10.1016/J.BBAGRM.2023.194986).
64. Kojima, H., et al. MEF2C/p300-mediated epigenetic remodeling promotes the maturation of induced cardiomyocytes. __Stem Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.STEMCR.2023.05.001).
65. Koyama, Y., et al. Increased RUNX3 expression mediates tumor-promoting ability of human breast cancer-associated fibroblasts. __Cancer Med__ (2023). [Link](https://doi.org/doi:10.1002/CAM4.6421).
66. Kubo, A., et al. Protective effects of Mkl1/2 against lipodystrophy and muscle atrophy via PI3K/AKT-independent FoxO repression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.09.552644).
67. Kumegawa, K., et al. Chromatin profile-based identification of a novel ER-positive breast cancer subgroup with reduced ER-responsive element accessibility. __Br J Cancer__ (2023). [Link](https://doi.org/doi:10.1038/S41416-023-02178-1).
68. Kurosawa, K., et al. ncBAF enhances PXR-mediated transcriptional activation in the human and mouse liver. __Biochem Pharmacol__ (2023). [Link](https://doi.org/doi:10.1016/J.BCP.2023.115733).
69. Kurosawa, K., et al. ncBAF, a chromatin remodeler, enhances PXR-mediated transcriptional activation in the human and mouse liver. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.03.527063).
70. Lacher, SE.., et al. An antioxidant response element regulates the HIF1α axis in breast cancer cells. __Free Radic Biol Med__ (2023). [Link](https://doi.org/doi:10.1016/J.FREERADBIOMED.2023.05.003).
71. Latyshev, P., et al. Unsupervised domain adaptation methods for cross-species transfer of regulatory code signals. __Front Big Data__ (2023). [Link](https://doi.org/doi:10.3389/FDATA.2023.1140663).
72. Le Ciclé, C., et al. The Neurod1/4-Ntrk3-Src pathway regulates gonadotrope cell adhesion and motility. __Cell Death Discov__ (2023). [Link](https://doi.org/doi:10.1038/S41420-023-01615-7).
73. Lee, J., et al. Dnmt1/Tet2-mediated changes in Cmip methylation regulate the development of nonalcoholic fatty liver disease by controlling the Gbp2-Pparγ-CD36 axis. __Exp Mol Med__ (2023). [Link](https://doi.org/doi:10.1038/S12276-022-00919-5).
74. Lee, SA., et al. FUS Facilitates Gene Body Export from Nuclear Condensates. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.25.542265).
75. Li, F., et al. VGLL4 and MENIN function as TEAD1 corepressors to block pancreatic β cell proliferation. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111904).
76. Li, J., et al. Overexpression of Ultrabithorax Changes the Development of Silk Gland and the Expression of Fibroin Genes in Bombyx mori. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24076670).
77. Li, J., et al. The Recurrent-Specific Regulation Network of Prognostic Stemness-Related Signatures in Low-Grade Glioma. __Dis Markers__ (2023). [Link](https://doi.org/doi:10.1155/2023/2243928).
78. Lim, JM., et al. CREB-Regulated Transcriptional Coactivator 2 Proteome Landscape is Modulated by SREBF1. __Mol Cell Proteom__ (2023). [Link](https://doi.org/doi:10.1016/J.MCPRO.2023.100637).
79. Lin, SC., et al. Targeting YAP1 ameliorates progesterone resistance in endometriosis. __Hum Reprod__ (2023). [Link](https://doi.org/doi:10.1093/HUMREP/DEAD071).
80. Liu, Z., et al. NFYC-37 promotes tumor growth by activating the mevalonate pathway in bladder cancer. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112963).
81. Lu, Y., et al. ER stress mediates Angiotensin II-augmented innate immunity memory and facilitates distinct susceptibilities of thoracic from abdominal aorta to aneurysm development. __Front Immunol__ (2023). [Link](https://doi.org/doi:10.3389/FIMMU.2023.1268916).
82. Lu-Culligan, WJ.., et al. Acetyl-methyllysine marks chromatin at active transcription start sites. __Nature__ (2023). [Link](https://doi.org/doi:10.1038/S41586-023-06565-9).
83. Ma, C., et al. Super Enhancer-Associated Circular RNA-CircKrt4 Regulates Hypoxic Pulmonary Artery Endothelial Cell Dysfunction in Mice. __Arterioscler Thromb Vasc Biol__ (2023). [Link](https://doi.org/doi:10.1161/ATVBAHA.122.318842).
84. Mangoni, D., et al. LINE-1 regulates cortical development by acting as long non-coding RNAs. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-40743-7).
85. Mannion, AJ.., et al. The junctional mechanosensor AmotL2 regulates YAP promotor accessibility. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.01.13.523596).
86. Mao, G., et al. Gene Regulatory Network Inference Using Convolutional Neural Networks from scRNA-seq Data. __J Comput Biol__ (2023). [Link](https://doi.org/doi:10.1089/CMB.2022.0355).
87. Marku, M., et al. From time-series transcriptomics to gene regulatory networks: A review on inference methods. __PLoS Comput Biol__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1011254).
88. Mbebi, AJ.., et al. Gene regulatory network inference using mixed-norms regularized multivariate model with covariance selection. __PLoS Comput Biol__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1010832).
89. Meng, Q., et al. Consensus peaks of chromatin accessibility in the human genome. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.30.542889).
90. Moreira, P., et al. Nuclear Factor-Y is a Pervasive Regulator of Neuronal Gene Expression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.14.528575).
91. Morita, T., et al. Actin-related protein 5 suppresses the cooperative activation of cardiac gene transcription by myocardin and MEF2. __FEBS Open Bio__ (2023). [Link](https://doi.org/doi:10.1002/2211-5463.13549).
92. Mu, Q., et al. Identifying predictors of glioma evolution from longitudinal sequencing. __Sci Transl Med__ (2023). [Link](https://doi.org/doi:10.1126/SCITRANSLMED.ADH4181).
93. Murao, N., et al. Homeostatic Regulation of Seizure Susceptibility and Cognitive Function by ER Quality Controller Derlin-1 through Maintenance of Adult Neurogenesis. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.27.538634).
94. Nagao, S., et al. Reduced DEFA5 Expression and STAT3 Activation Underlie the Submucosal Invasion of Early Gastric Cancers. __Digestion__ (2023). [Link](https://doi.org/doi:10.1159/000531790).
95. Nakamura, J., et al. TFEB-mediated lysosomal exocytosis alleviates high fat diet-induced lipotoxicity in the kidney. __JCI Insight__ (2023). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.162498).
96. Navarrete-López, P., et al. Loss of the importin Kpna2 causes infertility in male mice by disrupting the translocation of testis-specific transcription factors. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.107134).
97. Neu, D., et al. Oncogenic reactivation of young L1s is a hallmark of colon cancer. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.17.541189).
98. Odongoo, R., et al. Overlapping role of c-Jun N-terminal kinase (JNK) 1 and 2 in imidazole ketone erastin-induced ferroptosis. __Gene Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.GENREP.2023.101813).
99. Oey, NE.., et al. A Proteome-Wide Effect of PHF8 Knockdown on Cortical Neurons Shows Downregulation of Parkinson&rsquo;s Disease-Associated Protein Alpha-Synuclein and Its Interactors. __Biomedicines__ (2023). [Link](https://doi.org/doi:10.3390/BIOMEDICINES11020486).
100. Ohata, H., et al. PROX1 induction by autolysosomal activity stabilizes persister-like state of colon cancer via feedback repression of the NOX1-mTORC1 pathway. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112519).
101. Ojewunmi, OO.., et al. The genetic dissection of fetal haemoglobin persistence in sickle cell disease in Nigeria. __medRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.16.23289851).
102. Oliveros, W., et al. Systematic characterization of regulatory variants of blood pressure genes ll Systematic characterization of regulatory variants of blood pressure genes. __Cell Genom__ (2023). [Link](https://doi.org/doi:10.1016/j.xgen.2023.100330).
103. Osada, N., et al. c-FOS is an integral component of the IKZF1 transactivator complex and mediates lenalidomide resistance in multiple myeloma. __Clin Transl Med__ (2023). [Link](https://doi.org/doi:10.1002/CTM2.1364).
104. Ota, H., et al. Switching mechanism from AR to EGFR signaling via 3-O-sulfated heparan sulfate in castration-resistant prostate cancer. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-38746-X).
105. Peña-Martínez, EG.., et al. Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach. __medRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.01.23294951).
106. Peñuelas‐Haro, I., et al. A novel empeROR of pancreatic malignancy. __EMBO J__ (2023). [Link](https://doi.org/doi:10.15252/EMBJ.2023114282).
107. Po, A., et al. An econometric lens resolves cell-state parallax. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.02.530529).
108. Raicu, AM., et al. Retinoblastoma protein activity revealed by CRISPRi study of divergent Rbf1 and Rbf2 paralogs. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.19.541454).
109. Rakotopare, J., et al. A systematic approach identifies p53-DREAM target genes associated with blood or brain abnormalities. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.06.21.545932).
110. Rommelfanger, MK.., et al. Gene regulatory network inference with popInfer reveals dynamic regulation of hematopoietic stem cell quiescence upon diet restriction and aging. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.18.537360).
111. Ross, MO., et al. PTPN2 copper-sensing rapidly relays copper level fluctuations into EGFR/CREB activation and associated CTR1 transcriptional repression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.29.555401).
112. Ryan, TA.J.., et al. Dimethyl fumarate and 4-octyl itaconate are anticoagulants that suppress Tissue Factor in macrophages via inhibition of Type I Interferon. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-39174-1).
113. Rymuza, J., et al. Methods for constructing and evaluating consensus genomic interval sets. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.03.551899).
114. Sakai, K., et al. Testicular histone hyperacetylation in mice by valproic acid administration affects the next generation by changes in sperm DNA methylation. __PLoS One__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0282898).
115. Sanchez-Hernandez, ES;., et al. Glucocorticoid Receptor Regulates and Interacts with LEDGF/p75 to Promote Docetaxel Resistance in Prostate Cancer Cells. __Cells__ (2023). [Link](https://doi.org/doi:10.3390/CELLS12162046).
116. Sekine, K., et al. Transposons contribute to the acquisition of cell type-specific cis-elements in the brain. __Commun Biol__ (2023). [Link](https://doi.org/doi:10.1038/S42003-023-04989-7).
117. Shan, W., et al. Identification of canonical pyroptosis-related genes, associated regulation axis, and related traditional Chinese medicine in spinal cord injury. __Front Aging Neurosci__ (2023). [Link](https://doi.org/doi:10.3389/FNAGI.2023.1152297).
118. Shen, J., et al. Current insights into the oncogenic roles of lncRNA LINC00355. __Cancer Innov__ (2023). [Link](https://doi.org/doi:10.1002/CAI2.91).
119. Shimoda, M., et al. KSHV uses viral IL6 to expand infected immunosuppressive macrophages. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.05.531224).
120. Shinzawa, K., et al. GREB1 isoform 4 is specifically transcribed by MITF and required for melanoma proliferation. __Oncogene__ (2023). [Link](https://doi.org/doi:10.1038/S41388-023-02803-6).
121. Silva-Llanes, I., et al. The Transcription Factor NRF2 Has Epigenetic Regulatory Functions Modulating HDACs, DNMTs, and miRNA Biogenesis. __Antioxidants__ (2023). [Link](https://doi.org/doi:10.3390/ANTIOX12030641).
122. Sokolov, DK.., et al. Dexamethasone Inhibits Heparan Sulfate Biosynthetic System and Decreases Heparan Sulfate Content in Orthotopic Glioblastoma Tumors in Mice. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS241210243).
123. Sun, W., et al. Catalytic domain-dependent and -independent transcriptional activities of the tumour suppressor histone H3K27 demethylase UTX/KDM6A in specific cancer types. __Epigenetics__ (2023). [Link](https://doi.org/doi:10.1080/15592294.2023.2222245).
124. Suzuki, A., et al. Ablation of Max expression induces meiotic onset in sexually undifferentiated germ cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.06.10.544477).
125. Tabata, S., et al. NFκB nuclear dynamics orchestrate inflammatory aging. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.18.536673).
126. Tabe-Bordbar, S., et al. Integrative modeling of lncRNA-chromatin interaction maps reveals diverse mechanisms of nuclear retention. __BMC Genomics__ (2023). [Link](https://doi.org/doi:10.1186/S12864-023-09498-9).
127. Tagore, M., et al. GABA regulates electrical activity and tumor initiation in melanoma. __Cancer Discov__ (2023). [Link](https://doi.org/doi:10.1158/2159-8290.CD-23-0389).
128. Tamura, I., et al. Genome-wide analysis of histone modifications that underlie the dynamic changes in gene expression during decidualization in human endometrial stromal cells. __Mol Hum Reprod__ (2023). [Link](https://doi.org/doi:10.1093/MOLEHR/GAAD019).
129. Tanaka, Y., et al. Ror1 promotes PPARα-mediated fatty acid metabolism in astrocytes. __Genes Cells__ (2023). [Link](https://doi.org/doi:10.1111/GTC.13013).
130. Toyoda, K., et al. HTLV-1 bZIP factor-induced reprogramming of lactate metabolism and epigenetic status promote leukemic cell expansion. __Blood Cancer Discov__ (2023). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-22-0139).
131. Trovato, M., et al. Histone H3.3 lysine 9 and 27 control repressive chromatin states at cryptic cis-regulatory elements and bivalent promoters in mouse embryonic stem cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.08.539859).
132. Vahab, N., et al. Uncovering Co-regulatory Modules and Gene Regulatory Networks in the Heart through Machine Learning-based Analysis of Large-scale Epigenomic Data. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.28.538783).
133. Waisman, A., et al. The transcription factor OCT6 promotes the dissolution of the naïve pluripotent state by repressing Nanog and activating a formative state gene regulatory network. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.07.13.548918).
134. Wang, J., et al. Inferring gene regulatory network from single-cell transcriptomes with graph autoencoder model. __PLoS Genet__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PGEN.1010942).
135. Wang, X., et al. Decoding key cell sub-populations and molecular alterations in glioblastoma at recurrence by single-cell analysis. __Acta Neuropathol Commun__ (2023). [Link](https://doi.org/doi:10.1186/S40478-023-01613-X).
136. Watabe, S., et al. Regulation of 4-HNE via SMARCA4 Is Associated with Worse Clinical Outcomes in Hepatocellular Carcinoma. __Biomedicines__ (2023). [Link](https://doi.org/doi:10.3390/BIOMEDICINES11082278).
137. Watanabe, M., et al. The kinetics of signaling through the common FcRγ chain determine cytokine profiles in dendritic cells. __Sci Signal__ (2023). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ABN9909).
138. Weinstock, JS.., et al. Gene regulatory network inference from CRISPR perturbations in primary CD4+ T cells elucidates the genomic basis of immune disease. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.17.557749).
139. Wen, L., et al. Tubular aryl hydratocarbon receptor upregulates EZH2 to promote cellular senescence in cisplatin-induced acute kidney injury. __Cell Death Dis__ (2023). [Link](https://doi.org/doi:10.1038/S41419-022-05492-3).
140. Wernig-Zorc, S., et al. The long non-coding RNA MALAT1 modulates NR4A1 expression through a downstream regulatory element in specific cancer-cell-types. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.09.531856).
141. Xu, H., et al. Inferring CTCF binding patterns and anchored loops across human tissues and cell types. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2022.06.15.496356).
142. Xu, J., et al. STGRNS: an interpretable transformer-based method for inferring gene regulatory networks from single-cell transcriptomic data. __Bioinformatics__ (2023). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAD165).
143. Xue, B., et al. Opportunities and challenges in sharing and reusing genomic interval data. __Front Genet__ (2023). [Link](https://doi.org/doi:10.3389/FGENE.2023.1155809).
144. Yadav, P., et al. Hypoxia-induced loss of SRSF2-dependent DNA methylation promotes CTCF-mediated alternative splicing of VEGFA in breast cancer. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.106804).
145. Yamakado, N., et al. Chemical inhibition of LSD1 leads to epithelial to mesenchymal transition in vitro of an oral squamous cell carcinoma OM-1 cell line via release from LSD1-dependent suppression of ZEB1. __Biochem Biophys Res Commun__ (2023). [Link](https://doi.org/doi:10.1016/J.BBRC.2023.01.062).
146. Yamamoto, R., et al. Computer model of IL-6 dependent rheumatoid arthritis in F759 mice. __Int Immunol__ (2023). [Link](https://doi.org/doi:10.1093/INTIMM/DXAD016).
147. Yamazaki, M., et al. YAP/BRD4-controlled ROR1 promotes tumor-initiating cells and hyperproliferation in pancreatic cancer. __EMBO J__ (2023). [Link](https://doi.org/doi:10.15252/EMBJ.2022112614).
148. Yang, C., et al. TRIM15 forms a regulatory loop with the AKT/FOXO1 axis and LASP1 to modulate the sensitivity of HCC cells to TKIs. __Cell Death Dis__ (2023). [Link](https://doi.org/doi:10.1038/S41419-023-05577-7).
149. Yari, S., et al. JAK inhibition ameliorates bone destruction by simultaneously targeting mature osteoclasts and their precursors. __Inflamm Regen__ (2023). [Link](https://doi.org/doi:10.1186/S41232-023-00268-4).
150. Yonezawa, S., et al. Meta-analysis of heat-stressed transcriptomes using the public gene expression database from human and mouse samples. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.30.537112).
151. Yumimoto, K., et al. Molecular evolution of Keap1 was essential for adaptation of vertebrates to terrestrial life. __Sci Adv__ (2023). [Link](https://doi.org/doi:10.1126/SCIADV.ADG2379).
152. Zhang, N., et al. CRISPR screening reveals gleason score and castration resistance related oncodriver ring finger protein 19 A (RNF19A) in prostate cancer. __Drug Resist Update__ (2023). [Link](https://doi.org/doi:10.1016/J.DRUP.2022.100912).
153. Zhang, R., et al. G-Quadruplex Structures Are Key Modulators of Somatic Structural Variants in Cancers. __Cancer Res__ (2023). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-22-3089).
154. Zhang, Z., et al. Therapeutic potency of compound RMY-205 for pulmonary fibrosis induced by SARS-CoV-2 nucleocapsid protein. __Cell Chem Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.CHEMBIOL.2023.02.004).
155. Zou, Z., et al. Elucidating disease-associated mechanisms triggered by pollutants via the epigenetic landscape using large-scale ChIP-Seq data. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.18.541391).
156. Abdelfattah, N., et al. Sex-biased Yap1 oncogene function. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.21.521494).
157. Acevedo, AF., et al. BCL6 regulates the endothelial pro-immunogenic phenotype relevant to organ transplant rejection. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514941).
158. Acevedo, AF., et al. The DNA binding protein BCL6 regulates NFκB-controlled endothelial inflammatory gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514942).
159. Ambrosini, G., et al. The Breast Cancer Epigenomics Track Hub. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.01.490187).
160. Anderson, CJ.., et al. Strand-resolved mutagenicity of DNA damage and repair. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.10.495644).
161. Anerillas, C., et al. A BDNF-TrkB autocrine loop enhances senescent cell viability. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-33709-8).
162. Atindaana, E., et al. Bimodal Expression Patterns, and Not Viral Burst Sizes, Predict the Effects of Vpr on HIV-1 Proviral Populations in Jurkat Cells. __mBio__ (2022). [Link](https://doi.org/doi:10.1128/MBIO.03748-21).
163. Awamleh, Z., et al. DNA methylation signature associated with Bohring-Opitz syndrome: a new tool for functional classification of variants in ASXL genes. __Eur J Hum Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41431-022-01083-0).
164. Baessler, A., et al. Tet2 coordinates with Foxo1 and Runx1 to balance T follicular helper cell and T helper 1 cell differentiation. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABM4982).
165. Barlier, Cé., et al. RNetDys: identification of disease-related impaired regulatory interactions due to SNPs. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.08.511312).
166. Bideyan, L., et al. Hepatic GATA4 regulates cholesterol and triglyceride homeostasis in collaboration with LXRs. __Genes Dev__ (2022). [Link](https://doi.org/doi:10.1101/GAD.350145.122).
167. Bideyan, L., et al. Integrative analysis reveals multiple modes of LXR transcriptional regulation in liver. __Proc Natl Acad Sci U S A__ (2022). [Link](https://doi.org/doi:10.1073/PNAS.2122683119).
168. Booth, LN.., et al. Males induce premature demise of the opposite sex by multifaceted strategies. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00276-y).
169. Bujnicki, J., et al. Editorial: the 20th annual Nucleic Acids Research Web Server Issue 2022. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC525).
170. Butera, A., et al. p53-driven lipidome influences non-cell-autonomous lysophospholipids in pancreatic cancer. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00319-9).
171. Cao, M., et al. The synergistic interaction landscape of chromatin regulators reveals their epigenetic regulation mechanisms across five cancer cell lines. __Comput Struct Biotechnol J__ (2022). [Link](https://doi.org/doi:10.1016/J.CSBJ.2022.09.008).
172. Chai, G., et al. Sevoflurane inhibits histone acetylation and contributes to cognitive dysfunction by enhancing the expression of ANP32A in aging mice. __Behav Brain Res__ (2022). [Link](https://doi.org/doi:10.1016/J.BBR.2022.113949).
173. Chandra, O., et al. Inferring functions of coding and non-coding genes using epigenomic patterns and deciphering the effect of combinatorics of transcription factors binding at promoters. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.17.488570).
174. Chauvistré, H., et al. Persister state-directed transitioning and vulnerability in melanoma. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30641-9).
175. Chen, F., et al. Placental DNA methylation changes in gestational diabetes mellitus. __Epigenetics__ (2022). [Link](https://doi.org/doi:10.1080/15592294.2022.2110193).
176. Chen, G., et al. Graph attention network for link prediction of gene regulations from single cell RNA-sequencing data. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC559).
177. Chen, H., et al. BET Inhibitors Target the SCLC-N subtype of Small Cell Lung Cancer by Blocking NEUROD1 Transactivation. __Mol Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-22-0594).
178. Chen, J., et al. MYT1L is required for suppressing earlier neuronal development programs in the adult mouse brain. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.17.512591).
179. Chen, KY., et al. Adipocyte-Specific Ablation of PU.1 Promotes Energy Expenditure and Ameliorates Metabolic Syndrome in Aging Mice. __Front Aging__ (2022). [Link](https://doi.org/doi:10.3389/FRAGI.2021.803482).
180. Chen, M., et al. Integrative analyses for the identification of idiopathic pulmonary fibrosis-associated genes and shared loci with other diseases. __Thorax__ (2022). [Link](https://doi.org/doi:10.1136/THORAX-2021-217703).
181. Chen, T., et al. GSTM3 deficiency impedes DNA mismatch repair to promote gastric tumorigenesis via CAND1/NRF2-KEAP1 signaling. __Cancer Lett__ (2022). [Link](https://doi.org/doi:10.1016/J.CANLET.2022.215692).
182. Chirichella, M., et al. RFX transcription factors control a miR-150/PDAP1 axis that restrains the proliferation of human T cells. __PLoS Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PBIO.3001538).
183. Corleone, G., et al. Enhancer plasticity sustains oncogenic transformation and progression of B-Cell Acute Lymphoblastic leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.18.492497).
184. Del Giudice, M., et al. FOXA1 regulates alternative splicing in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111404).
185. Dong, C., et al. Curcumin Attenuates Hydrocephalus via Activation of E2F Transcription Factor 4. __Cerebrovasc Dis__ (2022). [Link](https://doi.org/doi:10.1159/000523994).
186. Dufour, CR.., et al. The mTOR chromatin-bound interactome in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110534).
187. Eguchi, R., et al. TRANSDIRE: data-driven direct reprogramming by a pioneer factor-guided trans-omics approach. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC209).
188. Escalante-Covarrubias, Q., et al. Time-of-day defines the efficacy of NAD+ to treat diet-induced metabolic disease by adjusting oscillations of the hepatic circadian clock. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503453).
189. Eustice, M., et al. Nutrient sensing pathways regulating adult reproductive diapause in C. elegans. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0274076).
190. Fang, XQ., et al. PGC1α Cooperates with FOXA1 to Regulate Epithelial Mesenchymal Transition through the TCF4-TWIST1. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS23158247).
191. Feng, C., et al. Arborinine from Glycosmis parva leaf extract inhibits clear-cell renal cell carcinoma by inhibiting KDM1A/UBE2O signaling. __Food Nutr Res__ (2022). [Link](https://doi.org/doi:10.29219/FNR.V66.8714).
192. Filipovic, D., et al. Predictive Models of Genome-wide Aryl Hydrocarbon Receptor DNA Binding Reveal Tissue Specific Binding Determinants. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.13.491754).
193. Freimer, JW.., et al. Systematic discovery and perturbation of regulatory genes in human T cells reveals the architecture of immune networks. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01106-Y).
194. Fu, LY., et al. ChIP-Hub provides an integrative platform for exploring plant regulome. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30770-1).
195. Gao, Y., et al. TRmir: A Comprehensive Resource for Human Transcriptional Regulatory Information of MiRNAs. __Front Genet__ (2022). [Link](https://doi.org/doi:10.3389/FGENE.2022.808950).
196. Gourisankar, S., et al. Rewiring Cancer Drivers to Activate Apoptosis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.04.517548).
197. Guan, J., et al. SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference. __BMC Genomics__ (2022). [Link](https://doi.org/doi:10.1186/S12864-022-09020-7).
198. Gupta, A., et al. Dynamic Control Balancing Cell Proliferation and Inflammation is Crucial for an Effective Immune Response to Malaria. __Front Mol Biosci__ (2022). [Link](https://doi.org/doi:10.3389/FMOLB.2021.800721).
199. Habano, W., et al. Aryl hydrocarbon receptor as a DNA methylation reader in the stress response pathway. __CRC Crit Rev Toxicol__ (2022). [Link](https://doi.org/doi:10.1016/J.TOX.2022.153154).
200. Harada, T., et al. Novel antimyeloma therapeutic option with inhibition of the HDAC1-IRF4 axis and PIM kinase. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022007155).
201. Hasegawa, A., et al. Heterozygous variants in GATA2 contribute to DCML deficiency in mice by disrupting tandem protein binding. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-03316-W).
202. Hassan, HM.., et al. Regulation of Chromatin Accessibility by the Farnesoid X Receptor Is Essential for Circadian and Bile Acid Homeostasis In Vivo. __Cancers__ (2022). [Link](https://doi.org/doi:10.3390/CANCERS14246191).
203. He, S., et al. The Role of the SOX9/lncRNA ANXA2P2/miR-361-3p/SOX9 Regulatory Loop in Cervical Cancer Cell Growth and Resistance to Cisplatin. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.784525).
204. He, X., et al. PBRM1 presents a potential prognostic marker and therapeutic target in duodenal papillary carcinoma. __Clin Transl Med__ (2022). [Link](https://doi.org/doi:10.1002/CTM2.1062).
205. Hersbach, BA., et al. Probing cell identity hierarchies by fate titration and collision during direct reprogramming. __Mol Syst Biol__ (2022). [Link](https://doi.org/doi:10.15252/MSB.202211129).
206. Hinton, A., et al. OPA1 Downregulation in Skeletal Muscle Induces MERC formation in an ATF4-Dependent Manner. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.12.507669).
207. Hirata, H., et al. A novel role of helix-loop-helix transcriptional factor Bhlhe40 in osteoclast activation. __J Cell Physiol__ (2022). [Link](https://doi.org/doi:10.1002/JCP.30844).
208. Hoffmann, M., et al. TF-Prioritizer: a java pipeline to prioritize condition-specific transcription factors. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.19.512881).
209. Hsieh, YT., et al. Equilibrative Nucleoside Transporter 3 is an IFN-stimulated Gene that Facilitates Viral Genome Release. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.27.489746).
210. Hu, X., et al. Aryl hydrocarbon receptor utilises cellular zinc signals to maintain the gut epithelial barrier. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.515052).
211. Hu, Y., et al. RUNX1 inhibits the antiviral immune response against influenza A virus through attenuating type I interferon signaling. __Virol J__ (2022). [Link](https://doi.org/doi:10.1186/S12985-022-01764-8).
212. Huang, T., et al. Identification of upstream transcription factor binding sites in orthologous genes using mixed Student's t-test statistics. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009773).
213. Ideno, H., et al. Human PSCs determine the competency of cerebral organoid differentiation via FGF signaling and epigenetic mechanisms. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105140).
214. Ito, N., et al. The Ccl17 gene encoding TARC is synergistically transactivated by PU.1 and IRF4 driven by the mammalian common promoter in dendritic cells. __Allergy__ (2022). [Link](https://doi.org/doi:10.1111/ALL.15184).
215. Itokazu, M., et al. Adipose-derived exosomes block muscular stem cell proliferation in aged mouse by delivering miRNA Let-7d-3p that targets transcription factor HMGA2. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102098).
216. Iwata, M., et al. Regulome-based characterization of drug activity across the human diseasome. __NPJ Syst Biol Appl__ (2022). [Link](https://doi.org/doi:10.1038/S41540-022-00255-4).
217. J, H., et al. ChIPBase v3.0: the encyclopedia of transcriptional regulations of non-coding RNAs and protein-coding genes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC1067).
218. J, W., et al. CohesinDB: a comprehensive database for decoding cohesin-related epigenomes, 3D genomes and transcriptomes in human cells. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC795).
219. Jiang, J., et al. IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105359).
220. Jing, F., et al. Prediction of the transcription factor binding sites with meta-learning. __Methods__ (2022). [Link](https://doi.org/doi:10.1016/J.YMETH.2022.04.010).
221. Kaizuka, T., et al. Developmental dynamics of the postsynaptic proteome to understand synaptic maturation and dysmaturation. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.05.490828).
222. Kanemoto, Y., et al. A long non-coding RNA as a direct vitamin D target transcribed from the anti-sense strand of the human HSD17B2 locus. __Biosci Rep__ (2022). [Link](https://doi.org/doi:10.1042/BSR20220321).
223. Kim, DK., et al. Aβ-induced mitochondrial dysfunction in neural progenitors controls KDM5A to influence neuronal differentiation. __Exp Mol Med__ (2022). [Link](https://doi.org/doi:10.1038/S12276-022-00841-W).
224. Kim, I., et al. AURKB, in concert with REST, acts as an oxygen-sensitive epigenetic regulator of the hypoxic induction of MDM2. __BMB Rep__ (2022). [Link](https://doi.org/doi:10.5483/BMBREP.2022.55.6.017).
225. Kitami, K., et al. Peritoneal restoration by repurposing vitamin D inhibits ovarian cancer dissemination via blockade of the TGF-β1/thrombospondin-1 axis. __Matrix Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.MATBIO.2022.03.003).
226. Kobelyatskaya, AA.., et al. ALDH3A2, ODF2, QSOX2, and MicroRNA-503-5p Expression to Forecast Recurrence in TMPRSS2-ERG-Positive Prostate Cancer. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911695).
227. Kojima, S., et al. Epigenome editing reveals core DNA methylation for imprinting control in the Dlk1-Dio3 imprinted domain. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC344).
228. Kokaji, T., et al. Construction of an inter-organ transomic network for whole-body metabolism. __Curr Opin Endocr Metab Res__ (2022). [Link](https://doi.org/doi:10.1016/J.COEMR.2022.100361).
229. Kokaji, T., et al. In vivo transomic analyses of glucose-responsive metabolism in skeletal muscle reveal core differences between the healthy and obese states. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.03.27.486003).
230. Kondo, J., et al. Transcription factor SP1 regulates haptoglobin fucosylation via induction of GDP-fucose transporter 1 in the hepatoma cell line HepG2. __Biochem Biophys Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.BBREP.2022.101372).
231. Kong, S., et al. Nucleosome-Omics: A Perspective on the Epigenetic Code and 3D Genome Landscape. __Genes__ (2022). [Link](https://doi.org/doi:10.3390/GENES13071114).
232. Kozlova, AA.., et al. Assessment of DDAH1 and DDAH2 Contributions to Psychiatric Disorders via In Silico Methods. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911902).
233. Kreiß, M., et al. Human 5-lipoxygenase regulates transcription by association to euchromatin. __Biochem Pharmacol__ (2022). [Link](https://doi.org/doi:10.1016/J.BCP.2022.115187).
234. Labaf, M., et al. Increased AR expression in castration-resistant prostate cancer rapidly induces AR signaling reprogramming with the collaboration of EZH2. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2022.1021845).
235. Larsen, EG.., et al. Transmembrane protein TMEM184B is necessary for interleukin-31-induced itch. __Pain__ (2022). [Link](https://doi.org/doi:10.1097/J.PAIN.0000000000002452).
236. Lavrekha, VV.., et al. CisCross: A gene list enrichment analysis to predict upstream regulators in Arabidopsis thaliana. __Front Plant Sci__ (2022). [Link](https://doi.org/doi:10.3389/FPLS.2022.942710).
237. Lee, SH., et al. Activation function 1 of progesterone receptor is required for progesterone antagonism of oestrogen action in the uterus. __BMC Biol__ (2022). [Link](https://doi.org/doi:10.1186/S12915-022-01410-3).
238. Letson, CT.., et al. Targeting BET Proteins downregulates miR-33a to promote synergy with PIM inhibitors in CMML. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.01.514753).
239. Li, F., et al. TEAD1 regulates cell proliferation through a pocket-independent transcription repression mechanism. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC1063).
240. Li, R., et al. Elucidation of an mTORC2-PKC-NRF2 pathway that sustains the ATF4 stress response and identification of Sirt5 as a key ATF4 effector. __Cell Death Discov__ (2022). [Link](https://doi.org/doi:10.1038/S41420-022-01156-5).
241. Li, W., et al. YTHDC1 is downregulated by the YY1/HDAC2 complex and controls the sensitivity of ccRCC to sunitinib by targeting the ANXA1-MAPK pathway. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02460-9).
242. Li, Y., et al. ARID3A promotes the chemosensitivity of colon cancer by inhibiting AKR1C3. __Cell Biol Int__ (2022). [Link](https://doi.org/doi:10.1002/CBIN.11789).
243. Li, Y., et al. Immunoglobulin superfamily 9 (IGSF9) is trans-activated by p53, inhibits breast cancer metastasis via FAK. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02459-8).
244. Liao, Y., et al. Downregulation of LNMAS orchestrates partial EMT and immune escape from macrophage phagocytosis to promote lymph node metastasis of cervical cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02202-3).
245. Lisacek, F., et al. Worldwide Glycoscience Informatics Infrastructure: The GlySpace Alliance. __JACS Au__ (2022). [Link](https://doi.org/doi:10.1021/JACSAU.2C00477).
246. Liu, T., et al. Computational identification of preneoplastic cells displaying high stemness and risk of cancer progression. __Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-22-0668).
247. Liu, W., et al. A novel FBW7/NFAT1 axis regulates cancer immunity in sunitinib-resistant renal cancer by inducing PD-L1 expression. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02253-0).
248. Liu, Z., et al. JAK2/STAT3 inhibition attenuates intestinal ischemia-reperfusion injury via promoting autophagy: in vitro and in vivo study. __Mol Biol Rep__ (2022). [Link](https://doi.org/doi:10.1007/S11033-021-07099-X).
249. Lu, Y., et al. Thioredoxin-interacting protein-activated intracellular potassium deprivation mediates the anti-tumour effect of a novel histone acetylation inhibitor HL23, a fangchinoline derivative, in human hepatocellular carcinoma. __J Adv Res__ (2022). [Link](https://doi.org/doi:10.1016/J.JARE.2022.10.017).
250. Lyu, Q., et al. A small proportion of X-linked genes contribute to X chromosome upregulation in early embryos via BRD4-mediated transcriptional activation. __Curr Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.CUB.2022.08.059).
251. M, W., et al. Hexokinase 2 is a transcriptional target and a positive modulator of AHR signalling. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC360).
252. M, Z., et al. MethBank 4.0: an updated database of DNA methylation across a variety of species. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC969).
253. Ma, X., et al. Single-cell RNA sequencing reveals B cell-T cell interactions in vascular adventitia of hyperhomocysteinemia-accelerated atherosclerosis. __Protein Cell__ (2022). [Link](https://doi.org/doi:10.1007/S13238-021-00904-0).
254. MacKay, H., et al. Sex-specific epigenetic development in the mouse hypothalamic arcuate nucleus pinpoints human genomic regions associated with body mass index. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABO3991).
255. Maity, AK.., et al. Inference of age-associated transcription factor regulatory activity changes in single cells. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00233-9).
256. Maixner, N., et al. Visceral Adipose Tissue E2F1-miRNA206/210 Pathway Associates with Type 2 Diabetes in Humans with Extreme Obesity. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11193046).
257. Mamivand, A., et al. Data mining of bulk and single-cell RNA sequencing introduces OBI1-AS1 as an astrocyte marker with possible role in glioma recurrence and progression. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01260-4).
258. Marques, IJ.., et al. Wt1 transcription factor impairs cardiomyocyte specification and drives a phenotypic switch from myocardium to epicardium. __Development__ (2022). [Link](https://doi.org/doi:10.1242/DEV.200375).
259. Martínez Corrales, G., et al. Transcriptional memory of dFOXO activation in youth curtails later-life mortality through chromatin remodeling and Xbp1. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00312-x).
260. Masamoto, Y., et al. EVI1 exerts distinct roles in AML via ERG and cyclin D1 promoting a chemoresistance and immune-suppressive environment. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022008018).
261. Matsumoto, A., et al. C11orf21, a novel RUNX1 target gene, is down-regulated by RUNX1-ETO. __BBA Adv__ (2022). [Link](https://doi.org/doi:10.1016/J.BBADVA.2022.100047).
262. Mbebi, AJ.., et al. Gene regulatory network inference using mixed-norms regularized multivariate model with covariance selection. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.21.521538).
263. McKowen, JK., et al. The Drosophila BEAF Insulator Protein Interacts with the Polybromo Subunit of the PBAP Chromatin Remodeling Complex. __Genes Genomes Genet__ (2022). [Link](https://doi.org/doi:10.1093/G3JOURNAL/JKAC223).
264. Mei, H., et al. Autophagy Facilitates Cancer-Intrinsic Evasion of CAR T Cell-Mediated Cytotoxicity in B-Cell Malignancies. __Blood__ (2022). [Link](https://doi.org/doi:10.1182/BLOOD-2022-168590).
265. Meller, A., et al. Mining Proteomics Datasets to Uncover Functional Pseudogenes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2124-0_16).
266. Metcalf, MG.., et al. Cell non-autonomous control of autophagy and metabolism by glial cells. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.15.520639).
267. Miao, J., et al. Sp1 Inhibits PGC-1α via HDAC2-Catalyzed Histone Deacetylation in Chronic Constriction Injury-Induced Neuropathic Pain. __ACS Chem Neurosci__ (2022). [Link](https://doi.org/doi:10.1021/ACSCHEMNEURO.2C00440).
268. Migdał, M., et al. xcore: an R package for inference of gene expression regulators. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.23.481130).
269. Mori, J., et al. Identification of cell cycle-associated and -unassociated regulators for expression of a hepatocellular carcinoma oncogene cyclin-dependent kinase inhibitor 3. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.07.088).
270. Mori, T., et al. Depletion of NIMA-related kinase Nek2 induces aberrant self-renewal and apoptosis in stem/progenitor cells of aged muscular tissues. __Mech Ageing Dev__ (2022). [Link](https://doi.org/doi:10.1016/J.MAD.2022.111619).
271. Morin, A., et al. Characterizing the targets of transcription regulators by aggregating ChIP-seq and perturbation expression data sets. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.30.505909).
272. Morita, T., et al. Actin-related protein 5 functions as a novel modulator of MyoD and MyoG in skeletal muscle and in rhabdomyosarcoma. __Elife__ (2022). [Link](https://doi.org/doi:10.7554/ELIFE.77746).
273. Mouri, K., et al. Prioritization of autoimmune disease-associated genetic variants that perturb regulatory element activity in T cells. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01056-5).
274. Muromoto, R., et al. Regulation of NFKBIZ gene promoter activity by STAT3, C/EBPβ, and STAT1. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.140).
275. Nair, PR.., et al. MLL1 regulates cytokine-driven cell migration and metastasis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.18.512715).
276. Nakamura, H., et al. Hypoxia-inducible factor-1α and poly [ADP ribose] polymerase 1 cooperatively regulate Notch3 expression under hypoxia via a noncanonical mechanism. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102137).
277. Neal, A., et al. Finding and Verifying Enhancers for Endothelial-Expressed Genes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2059-5_28).
278. Ng, KJ., et al. Sox2 in the dermal papilla regulates hair follicle pigmentation. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111100).
279. Nie, M., et al. Targeting acetylcholine signaling modulates persistent drug tolerance in EGFR-mutant lung cancer and impedes tumor relapse. __J Clin Invest__ (2022). [Link](https://doi.org/doi:10.1172/JCI160152).
280. Nishi, K., et al. Novel estrogen-responsive genes (ERGs) for the evaluation of estrogenic activity. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0273164).
281. Ochsner, SA.., et al. Transcriptional regulatory networks of circulating immune cells in type 1 diabetes: A community knowledgebase. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104581).
282. Ohnishi, T., et al. Periodontitis promotes the expression of gingival transmembrane serine protease 2 (TMPRSS2), a priming protease for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). __J Oral Biosci__ (2022). [Link](https://doi.org/doi:10.1016/J.JOB.2022.04.004).
283. Ono, K., et al. Congenital sideroblastic anemia model due to ALAS2 mutation is susceptible to ferroptosis. __Sci Rep__ (2022). [Link](https://doi.org/doi:10.1038/S41598-022-12940-9).
284. Panatta, E., et al. Metabolic regulation by p53 prevents R-loop-associated genomic instability. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111568).
285. Panatta, E., et al. p53 regulates expression of nuclear envelope components in cancer cells. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00349-3).
286. Peng, L., et al. Bombesin receptor-activated protein exacerbates cisplatin-induced AKI by regulating the degradation of SIRT2. __Nephrol Dial Transplant__ (2022). [Link](https://doi.org/doi:10.1093/NDT/GFAC164).
287. Pontis, J., et al. Primate-specific transposable elements shape transcriptional networks during human development. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34800-W).
288. Q, Z., et al. ChromLoops: a comprehensive database for specific protein-mediated chromatin loops in diverse organisms. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC893).
289. Reagor, CC.., et al. Depicting pseudotime-lagged causality across single-cell trajectories for accurate gene-regulatory inference. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.25.489377).
290. Ren, J., et al. Downregulation of CBX7 induced by EZH2 upregulates FGFR3 expression to reduce sensitivity to cisplatin in bladder cancer. __Br J Cancer__ (2022). [Link](https://doi.org/doi:10.1038/S41416-022-02058-0).
291. Rhizobium, GE. TFSyntax: a database of transcription factors binding syntax in mammalian genomes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC849).
292. Richard, D., et al. Intronic regulation of SARS-CoV-2 receptor (ACE2) expression mediated by immune signaling and oxidative stress pathways. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104614).
293. Richard, D., et al. Lineage-Specific Differences and Regulatory Networks Governing Human Chondrocyte Development. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.25.493388).
294. Rykaczewska, U., et al. Plaque Evaluation by Ultrasound and Transcriptomics Reveals BCLAF1 as a Regulator of Smooth Muscle Cell Lipid Transdifferentiation in Atherosclerosis. __Arterioscler Thromb Vasc Biol__ (2022). [Link](https://doi.org/doi:10.1161/ATVBAHA.121.317018).
295. Saadey, AA.., et al. Rebalancing TGFβ1/BMP signals in exhausted T cells unlocks responsiveness to immune checkpoint blockade therapy. __Nat Immunol__ (2022). [Link](https://doi.org/doi:10.1038/S41590-022-01384-Y).
296. Sakai, H., et al. Uhrf1 governs the proliferation and differentiation of muscle satellite cells. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.103928).
297. Samelson, AJ.., et al. BRD2 inhibition blocks SARS-CoV-2 infection by reducing transcription of the host cell receptor ACE2. __Nat Cell Biol__ (2022). [Link](https://doi.org/doi:10.1038/S41556-021-00821-8).
298. Sasaki, N., et al. Gp130-Mediated STAT3 Activation Contributes to the Aggressiveness of Pancreatic Cancer through H19 Long Non-Coding RNA Expression. __Cancers__ (2022). [Link](https://doi.org/doi:10.3390/CANCERS14092055).
299. Sasaki, N., et al. PRC2-dependent regulation of ganglioside expression during dedifferentiation contributes to the proliferation and migration of vascular smooth muscle cells. __Front Cell Dev Biol__ (2022). [Link](https://doi.org/doi:10.3389/FCELL.2022.1003349).
300. Sato, F., et al. COL8A1 facilitates the growth of triple-negative breast cancer via FAK/Src activation. __Breast Cancer Res Treat__ (2022). [Link](https://doi.org/doi:10.1007/S10549-022-06635-Y).
301. Sawada, T., et al. Antagonistic action of a synthetic androgen ligand mediated by chromatin remodeling in a human prostate cancer cell line. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.109).
302. Schihada, H., et al. Isoforms of GPR35 have distinct extracellular N-termini that allosterically modify receptor-transducer coupling and mediate intracellular pathway bias. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102328).
303. Schuhwerk, H., et al. The EMT transcription factor ZEB1 governs a fitness-promoting but vulnerable DNA replication stress response. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111819).
304. Serrano-Zayas, C., et al. Regulation of cellular LDL uptake by PROX1 and CHD7. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.20.507601).
305. Shen, SH., et al. Bromodomain-containing protein 4 activates cardiotrophin-like cytokine factor 1, an unfavorable prognostic biomarker, and promotes glioblastoma in vitro. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-1164).
306. Shin, J., et al. SARS-CoV-2 infection impairs the insulin/IGF signaling pathway in the lung, liver, adipose tissue, and pancreatic cells via IRF1. __Metabolism__ (2022). [Link](https://doi.org/doi:10.1016/J.METABOL.2022.155236).
307. Silvennoinen, K., et al. SCN1A overexpression, associated with a genomic region marked by a risk variant for a common epilepsy, raises seizure susceptibility. __Acta Neuropathol__ (2022). [Link](https://doi.org/doi:10.1007/S00401-022-02429-0).
308. Singh, A., et al. Conserved neuropeptidergic regulation of intestinal integrity in invertebrate models of aging. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.24.481867).
309. Siswanto, FM., et al. Chlorogenic acid activates Nrf2/SKN-1 and prolongs the lifespan of Caenorhabditis elegans via the Akt-FOXO3/DAF16a-DDB1 pathway and activation of DAF16f. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.1093/GERONA/GLAC062).
310. Smirnov, D., et al. SIRT6 is a key regulator of mitochondrial function in the brain. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.08.515572).
311. Sokolowska, KE., et al. Identified in blood diet-related methylation changes stratify liver biopsies of NAFLD patients according to fibrosis grade. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01377-6).
312. Sokolowski, DJ.., et al. Differential Expression Enrichment Tool (DEET): An interactive atlas of human differential gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.29.505468).
313. Su, Y., et al. Epigenetic histone acetylation modulating prenatal Poly I:C induced neuroinflammation in the prefrontal cortex of rats: a study in a maternal immune activation model. __Front Cell Neurosci__ (2022). [Link](https://doi.org/doi:10.3389/FNCEL.2022.1037105).
314. Sugawara, S., et al. RNaseH2A downregulation drives inflammatory gene expression via genomic DNA fragmentation in senescent and cancer cells. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-04369-7).
315. Sun, J., et al. The lncRNA CASC2 Modulates Hepatocellular Carcinoma Cell Sensitivity and Resistance to TRAIL Through Apoptotic and Non-Apoptotic Signaling. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.726622).
316. Tahara, S., et al. MOCCS profile analysis clarifies the cell type dependency of transcription factor-binding sequences and cis-regulatory SNPs in humans. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.08.487641).
317. Takayama, S., et al. GATA6 regulates expression of annexin A10 (ANXA10) associated with epithelial-mesenchymal transition of oral squamous cell carcinoma. __Arch Oral Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.ARCHORALBIO.2022.105569).
318. Takeuchi, C., et al. Mod(mdg4) variants repress telomeric retrotransposon HeT-A by blocking subtelomeric enhancers. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.16.476534).
319. Talamanca, L., et al. Sex-dimorphic and age-dependent organization of 24 hour gene expression rhythms in human. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.13.494714).
320. Tanigawa, Y., et al. WhichTF is functionally important in your open chromatin data?. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1010378).
321. Terakawa, A., et al. Trans-omics analysis of insulin action reveals a cell growth subnetwork which co-regulates anabolic processes. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104231).
322. Toyoda, K., et al. Reprogramming of lactate metabolism is linked to the oncogenesis of the virus-induced leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503711).
323. Tsujino, T., et al. CRISPR screens reveal genetic determinants of PARP inhibitor sensitivity and resistance in prostate cancer. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.13.488115).
324. Vorontsova, JE.., et al. A new insight into the aryl hydrocarbon receptor/cytochrome 450 signaling pathway in MG63, HOS, SAOS2, and U2OS cell lines. __Biochimie__ (2022). [Link](https://doi.org/doi:10.1016/J.BIOCHI.2022.10.018).
325. Wang, Y., et al. NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-04226-7).
326. Wang, Z., et al. Towards a better understanding of TF-DNA binding prediction from genomic features. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105993).
327. Woogeng, IN., et al. Inducing human retinal pigment epithelium-like cells from somatic tissue. __Stem Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.12.008).
328. Wu, B., et al. ORAI1 Limits SARS-CoV-2 Infection by Regulating Tonic Type I IFN Signaling. __J Immunol__ (2022). [Link](https://doi.org/doi:10.4049/JIMMUNOL.2100742).
329. Wu, L., et al. Natural Coevolution of Tumor and Immunoenvironment in Glioblastoma. __Cancer Discov__ (2022). [Link](https://doi.org/doi:10.1158/2159-8290.CD-22-0196).
330. Wu, S., et al. Bromodomain containing 4 transcriptionally activated Deltex E3 ubiquitin ligase 2 contributes to glioma progression and predicts an unfavorable prognosis. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-555).
331. Y, H., et al. Mitochondrial stress induces AREG expression and epigenomic remodeling through c-JUN and YAP-mediated enhancer activation. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC735).
332. Yamada, M., et al. KLF5-regulated extracellular matrix remodeling secures biliary epithelial tissue integrity against cholestatic liver injury. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.25.477619).
333. Yamamoto-Imoto, H., et al. Age-associated decline of MondoA drives cellular senescence through impaired autophagy and mitochondrial homeostasis. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110444).
334. Yamashita, Y., et al. Osteoblast lineage cells-derived Sema3A regulates bone homeostasis independently of androgens. __Endocrinology__ (2022). [Link](https://doi.org/doi:10.1210/ENDOCR/BQAC126).
335. Yamazaki, M., et al. ROR1 plays a critical role in pancreatic tumor-initiating cells with a partial EMT signature. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.07.13.499868).
336. Yan, B., et al. A comprehensive single cell data analysis of in lymphoblastoid cells reveals the role of Super-enhancers in maintaining EBV latency. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.10.503552).
337. Yang, K., et al. Prognostic and Immunotherapeutic Roles of KRAS in Pan-Cancer. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11091427).
338. Yang, L., et al. Sevoflurane Offers Neuroprotection in a Cerebral Ischemia/Reperfusion Injury Rat Model Through the E2F1/EZH2/TIMP2 Regulatory Axis. __Mol Neurobiol__ (2022). [Link](https://doi.org/doi:10.1007/S12035-021-02602-8).
339. Yashar, WM.., et al. GoPeaks: histone modification peak calling for CUT&Tag. __Genome Biol__ (2022). [Link](https://doi.org/doi:10.1186/S13059-022-02707-W).
340. Yheskel, M., et al. Proximity labeling reveals a new in vivo network of interactors for the histone demethylase KDM5. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.20.517232).
341. Yonezawa, Y., et al. Identification of a Functional Susceptibility Variant for Adolescent Idiopathic Scoliosis that Upregulates Early Growth Response 1 (EGR1)-Mediated UNCX Expression. __J Bone Miner Res__ (2022). [Link](https://doi.org/doi:10.1002/JBMR.4738).
342. Zelenka, T., et al. The 3D enhancer network of the developing T cell genome is shaped by SATB1. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34345-Y).
343. Zhang, H., et al. CBX3 Regulated By YBX1 Promotes Smoking-induced Pancreatic Cancer Progression via Inhibiting SMURF2 Expression. __Int J Biol Sci__ (2022). [Link](https://doi.org/doi:10.7150/IJBS.68995).
344. Zhang, L., et al. STAT3 is a biologically relevant therapeutic target in H3K27M-mutant diffuse midline glioma. __Neuro Oncol__ (2022). [Link](https://doi.org/doi:10.1093/NEUONC/NOAC093).
345. Zhang, T., et al. ADAR1 masks the cancer immunotherapeutic promise of ZBP1-driven necroptosis. __Nature__ (2022). [Link](https://doi.org/doi:10.1038/S41586-022-04753-7).
346. Zhang, Y., et al. Fusion prior gene network for high reliable single-cell gene regulatory network inference. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105279).
347. Zhao, J., et al. CanMethdb: a database for genome-wide DNA methylation annotation in cancers. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC783).
348. Zhao, M., et al. A hybrid deep learning framework for gene regulatory network inference from single-cell transcriptomic data. __Brief Bioinform__ (2022). [Link](https://doi.org/doi:10.1093/BIB/BBAB568).
349. Zheng, X., et al. Downregulation of HINFP induces senescence-associated secretory phenotype to promote metastasis in a non-cell-autonomous manner in bladder cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02371-1).
350. Zhong, X., et al. CBX3 is associated with metastasis and glutathione/ glycosphingolipid metabolism in colon adenocarcinoma. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.21037/JGO-22-97/COIF)).
351. Zhu, T., et al. A pan-tissue DNA methylation atlas enables in silico decomposition of human tissue methylomes at cell-type resolution. __Nat Methods__ (2022). [Link](https://doi.org/doi:10.1038/S41592-022-01412-7).
352. Zou, Z., et al. Epigenetic landscape of drug responses revealed through large-scale ChIP-seq data analyses. __BMC Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1186/S12859-022-04571-8).
353. A, M., et al. AnnoMiner is a new web-tool to integrate epigenetics, transcription factor occupancy and transcriptomics data to predict transcriptional regulators. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-94805-1).
354. A, M., et al. Integrative Bioinformatics and Experimental Investigation Introduces OBI1-AS1 as a Key LncRNA in the Progression of Low-grade Glioma to Glioblastoma. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-215387/V1).
355. A, N., et al. Hyperleptinemia in obese state renders luminal breast cancers refractory to tamoxifen by coordinating a crosstalk between Med1, miR205 and ErbB. __NPJ Breast Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41523-021-00314-9).
356. A, W., et al. E-cadherin is regulated by GATA-2 and marks the early commitment of mouse hematopoietic progenitors to the basophil and mast cell fates. __Sci Immunol__ (2021). [Link](https://doi.org/doi:10.1126/SCIIMMUNOL.ABA0178).
357. AP, W., et al. Bayesian information sharing enhances detection of regulatory associations in rare cell types. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB269).
358. Agudelo, LZ.., et al. Metabolic resilience is encoded in genome plasticity. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.25.449953).
359. Aizawa, S., et al. Early reactivation of clustered genes on the inactive X chromosome during somatic cell reprogramming. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.11.008).
360. Alfeghaly, C., et al.  Implication of repeat insertion domains in the trans -activity of the long non-coding RNA ANRIL . __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/nar/gkab245).
361. Astudillo, P. Analysis in silico of the functional interaction between WNT5A and YAP/TEAD signaling in cancer. __PeerJ__ (2021). [Link](https://doi.org/doi:10.7717/peerj.10869).
362. B. Tóth, Bá., et al. Regulatory modules of human thermogenic adipocytes: functional genomics of large cohort and Meta-analysis derived marker-genes. __BMC Genomics__ (2021). [Link](https://doi.org/doi:10.1186/S12864-021-08126-8).
363. Bang, J., et al. Identification of Signaling Pathways for Early Embryonic Lethality and Developmental Retardation in Sephs1 -/- Mice. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222111647).
364. Biani, M.C.., et al. Small extracellular vesicles from iPSC-MSC lose their regenerative potential upon UV-C irradiation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.27.461979).
365. Bono, H. Meta-analysis of oxidative transcriptomes in insects. __Antioxidants__ (2021). [Link](https://doi.org/doi:10.3390/antiox10030345).
366. C, F., et al. PFKFB4 is overexpressed in clear-cell renal cell carcinoma promoting pentose phosphate pathway that mediates Sunitinib resistance. __J Exp Clin Cancer Res__ (2021). [Link](https://doi.org/doi:10.1186/S13046-021-02103-5).
367. CA, A., et al. ACSNI: An unsupervised machine-learning tool for prediction of tissue-specific pathway components using gene expression profiles. __Patterns__ (2021). [Link](https://doi.org/doi:10.1016/J.PATTER.2021.100270).
368. Centenera, MM.., et al. ELOVL5 is a critical and targetable fatty acid elongase in prostate cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-2511).
369. Cerase, A., et al. Chd8 regulates X chromosome inactivation in mouse through fine-tuning control of Xist expression. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-01945-1).
370. Chen, H., et al. BET Bromodomain Inhibitors Target the NEUROD1-subtype SCLC by Blocking NEUROD1 Transactivation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.25.465771).
371. Chen, M., et al. Integrative Analyses Reveal Novel Disease-associated Loci and Genes for Idiopathic Pulmonary Fibrosis. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.11.21257064).
372. Choi, S., et al. Identification of a Putative Enhancer RNA for EGFR in Hyper-Accessible Regions in Esophageal Squamous Cell Carcinoma Cells by Analysis of Chromatin Accessibility Landscapes. __Front Oncol__ (2021). [Link](https://doi.org/doi:10.3389/FONC.2021.724687).
373. Cofer, EM.., et al. Modeling transcriptional regulation of model species with deep learning. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/gr.266171.120).
374. De Smedt, J., et al. PU.1 drives specification of pluripotent stem cell-derived endothelial cells to LSEC-like cells. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-020-03356-2).
375. Dhaka, B., et al. Differential chromatin accessibility landscape of gain-of-function mutant p53 tumours. __BMC Cancer__ (2021). [Link](https://doi.org/doi:10.1186/s12885-021-08362-x).
376. Doi, H., et al. Early-life midazolam exposure persistently changes chromatin accessibility to impair adult hippocampal neurogenesis and cognition. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2107596118/-/DCSUPPLEMENTAL).
377. Droin, C., et al. Space-time logic of liver gene expression at sub-lobular scale. __Nat Metab__ (2021). [Link](https://doi.org/doi:10.1038/s42255-020-00323-1).
378. Durand, Sé., et al. RSL24D1 sustains steady-state ribosome biogenesis and pluripotency translational programs in embryonic stem cells. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.27.443845).
379. E, M., et al. NUAK2 and RCan2 participate in the p53 mutant pro-tumorigenic network. __Biol Direct__ (2021). [Link](https://doi.org/doi:10.1186/S13062-021-00296-5).
380. Feng, H., et al. Identification of core miRNAs and regulatory pathways in breast cancer by integrated bioinformatics analysis. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/d0mo00171f).
381. Feng, L., et al. Comprehensive Analysis of E3 Ubiquitin Ligases Reveals Ring Finger Protein 223 as a Novel Oncogene Activated by KLF4 in Pancreatic Cancer. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.738709).
382. Feng, P., et al. A machine learning-based framework for modeling transcription elongation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/pnas.2007450118).
383. Fernandez Lahore, G., et al. Polymorphic estrogen receptor binding site causes Cd2-dependent sex bias in the susceptibility to autoimmune diseases. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25828-5).
384. Franco, ED., et al. Primate-specific ZNF808 is essential for pancreatic development in humans. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.23.21262262).
385. Fujino, K., et al. A human endogenous bornavirus-like nucleoprotein encodes a mitochondrial protein associated with cell viability. __J Virol__ (2021). [Link](https://doi.org/doi:10.1128/JVI.02030-20).
386. Fujino, S., et al. HNF1A regulates colorectal cancer progression and drug resistance as a downstream of POU5F1. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89126-2).
387. G, D., et al. Comprehensive analysis of epigenetic signatures of human transcription control. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/D0MO00130A).
388. Gallo, A., et al. NF-Y subunits overexpression in gastric adenocarcinomas (STAD). __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-03027-Y).
389. Gharavi, E., et al. Embeddings of genomic region sets capture rich biological associations in lower dimensions. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB439).
390. H, D., et al. Early-life midazolam exposure persistently changes chromatin accessibility to impair adult hippocampal neurogenesis and cognition. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2107596118).
391. H, M., et al. Evi1 upregulates Fbp1 and supports progression of acute myeloid leukemia through pentose phosphate pathway activation. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/CAS.15098).
392. H, O., et al. Lysine Demethylase 5A is Required for MYC Driven Transcription in Multiple Myeloma. __Blood Cancer Discov__ (2021). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0108).
393. Hamano, M., et al. Prediction of single-cell mechanisms for disease progression in hypertrophic remodelling by a trans-omics approach. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86821-y).
394. Hammal, F., et al. ReMap 2022: a database of Human, Mouse, Drosophila and Arabidopsis regulatory regions from an integrative analysis of DNA-binding sequencing experiments. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB996).
395. Hanaki, S., et al. PP1 regulatory subunit NIPP1 regulates transcription of E2F1 target genes following DNA damage. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/cas.14924).
396. Harada, Y., et al. ETS‐dependent enhancers for endothelial‐specific expression of serum/glucocorticoid‐regulated kinase 1 during mouse embryo development. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/gtc.12874).
397. He, L., et al. An optimized two-step chromatin immunoprecipitation protocol to quantify the associations of two separate proteins and their common target DNA. __STAR Protoc__ (2021). [Link](https://doi.org/doi:10.1016/j.xpro.2021.100504).
398. Hernández-Nava, E., et al. Transcriptional and Epigenetic Bioinformatic Analysis of Claudin-9 Regulation in Gastric Cancer. __J Oncol__ (2021). [Link](https://doi.org/doi:10.1155/2021/5936905).
399. Hiebert, P. The Nrf2 transcription factor: A multifaceted regulator of the extracellular matrix. __Matrix Biol Plus__ (2021). [Link](https://doi.org/doi:10.1016/j.mbplus.2021.100057).
400. Hisamatsu, Y., et al. Matrix Metalloproteinases in Human Decidualized Endometrial Stromal Cells. __Curr Issues Mol Biol__ (2021). [Link](https://doi.org/doi:10.3390/CIMB43030146).
401. Hisano, O., et al. Short single-stranded DNAs with putative non-canonical structures comprise a new class of plasma cell-free DNA. __BMC Biol__ (2021). [Link](https://doi.org/doi:10.1186/S12915-021-01160-8).
402. Hosoi, H., et al. Super-enhancers for RUNX3 are required for cell proliferation in EBV-infected B cell lines. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145421).
403. Hosoya, T., et al. Chondroprotective effects of CDK4/6 inhibition via enhanced ubiquitin-dependent degradation of JUN in synovial fibroblasts. __Baillieres Clin Rheumatol__ (2021). [Link](https://doi.org/doi:10.1093/RHEUMATOLOGY/KEAB874).
404. Hossain, SMM., et al. Discovering key transcriptomic regulators in pancreatic ductal adenocarcinoma using Dirichlet process Gaussian mixture model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87234-7).
405. Huang, CCF., et al. Functional mapping of androgen receptor enhancer activity. __Genome Biol__ (2021). [Link](https://doi.org/doi:10.1186/s13059-021-02339-6).
406. Hudecova, I., et al. Characteristics, origin, and potential for cancer diagnostics of ultrashort plasma cell-free DNA. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/GR.275691.121).
407. Inazumi, H., et al. NRSF-GNAO1 Pathway Contributes to the Regulation of Cardiac Ca 2+ Homeostasis. __Circ Res__ (2021). [Link](https://doi.org/doi:10.1161/CIRCRESAHA.121.318898).
408. J, D., et al. Systematic identification of genomic elements that regulate FCGR2A expression and harbor variants linked with autoimmune disease. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB372).
409. J, S., et al. Cancer drivers and clonal dynamics in acute lymphoblastic leukaemia subtypes. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-366981/V1).
410. J, W., et al. JUNB suppresses distant metastasis by influencing the initial metastatic stage. __Clin Exp Metastasis__ (2021). [Link](https://doi.org/doi:10.1007/S10585-021-10108-9).
411. Jang, M., et al. Matrix stiffness epigenetically regulates the oncogenic activation of the Yes-associated protein in gastric cancer. __Nat Biomed Eng__ (2021). [Link](https://doi.org/doi:10.1038/s41551-020-00657-x).
412. Jarvis, LB.., et al. Therapeutically expanded human regulatory T-cells are super-suppressive due to HIF1A induced expression of CD73. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02721-X).
413. Jia, Y., et al. Super enhancer-mediated upregulation of HJURP promotes growth and survival of t(4;14)-positive multiple myeloma. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-0921).
414. Jiang, JC., et al. Integrated transcription factor profiling with transcriptome analysis identifies L1PA2 transposons as global regulatory modulators in a breast cancer model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86395-9).
415. Jiang, JC., et al. Widespread Exaptation of L1 Transposons for Transcription Factor Binding in Breast Cancer. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115625).
416. Jiang, X., et al. Heat shock induces the nuclear accumulation of YAP1 via SRC. __Exp Cell Res__ (2021). [Link](https://doi.org/doi:10.1016/j.yexcr.2020.112439).
417. Jin, X., et al. Smoking-associated upregulation of CBX3 suppresses ARHGAP24 expression to activate Rac1 signaling and promote tumor progression in lung adenocarcinoma. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/S41388-021-02114-8).
418. Jung, S., et al. A computer-guided design tool to increase the efficiency of cellular conversions. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/s41467-021-21801-4).
419. Jungwirth, E., et al. Meta-analysis and Consolidation of Farnesoid X Receptor Chromatin Immunoprecipitation Sequencing Data Across Different Species and Conditions. __Hepatol Commun__ (2021). [Link](https://doi.org/doi:10.1002/HEP4.1749).
420. K, K., et al. ATF4-mediated transcriptional regulation protects against β-cell loss during endoplasmic reticulum stress in a mouse model. __Mol Metab__ (2021). [Link](https://doi.org/doi:10.1016/J.MOLMET.2021.101338).
421. K, N., et al. Profiling of Androgen-Dependent Enhancer RNAs Expression in Human Prostate Tumors: Search for Malignancy Transition Markers. __Res Rep Urol__ (2021). [Link](https://doi.org/doi:10.2147/RRU.S328661).
422. K, S., et al. HMGB1 coordinates SASP-related chromatin folding and RNA homeostasis on the path to senescence. __Mol Syst Biol__ (2021). [Link](https://doi.org/doi:10.15252/MSB.20209760).
423. Kaiser, M., et al. Regulation of otocyst patterning by <i>Tbx2</i> and <i>Tbx3</i> is required for inner ear morphogenesis in the mouse. __Development__ (2021). [Link](https://doi.org/doi:10.1242/dev.195651).
424. Kalman, ZE., et al. A preliminary study on the cistrome of human postsynaptic density from an evolutionary and network-based perspective. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.01.25.428072).
425. Kanai, M., et al. Insights from complex trait fine-mapping across diverse populations. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.03.21262975).
426. Kanamori, Y., et al. Iron-rich Kupffer cells exhibit phenotypic changes during the development of liver fibrosis in NASH. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/j.isci.2020.102032).
427. Katada, S., et al. Neural stem/precursor cells dynamically change their epigenetic landscape to differentially respond to BMP signaling for fate switching during brain development. __Genes Dev__ (2021). [Link](https://doi.org/doi:10.1101/GAD.348797.121).
428. Kidwell, A., et al. Translation rescue by targeting Ppp1r15a upstream open reading frame in vivo. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.12.11.472232).
429. Kikutake, C., et al. Pan-cancer analysis of non-coding recurrent mutations and their possible involvement in cancer pathogenesis. __NAR Cancer__ (2021). [Link](https://doi.org/doi:10.1093/narcan/zcab008).
430. Kim, J., et al. The RNA helicase DDX6 controls early mouse embryogenesis by repressing aberrant inhibition of BMP signaling through miRNA-mediated gene silencing. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.11.29.470397).
431. Kohrogi, K., et al. LSD1 de fi nes erythroleukemia metabolism by controlling the lineage-speci fi c transcription factors GATA1 and C / EBP a. __Blood Adv__ (2021). [Link](https://doi.org/doi:10.1182/bloodadvances.2020003521).
432. Koike, Y., et al. Age-related demethylation of the TDP-43 autoregulatory region in the human motor cortex. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02621-0).
433. Kosinsky, RL., et al. RNF20 and RNF40 regulate vitamin D receptor-dependent signaling in inflammatory bowel disease. __Cell Death Differ__ (2021). [Link](https://doi.org/doi:10.1038/s41418-021-00808-w).
434. Kwon, ATJ., et al. Development of p53 Knockout U87MG Cell Line for Unbiased Drug Delivery Testing System Using CRISPR-Cas9 and Transcriptomic Analysis. __J Biotechnol__ (2021). [Link](https://doi.org/doi:10.1016/j.jbiotec.2021.03.017).
435. L, W., et al. The Ubiquitin Sensor and Adaptor Protein p62 Mediates Signal Transduction of a Viral Oncogenic Pathway. __mBio__ (2021). [Link](https://doi.org/doi:10.1128/MBIO.01097-21).
436. Labade, AS.., et al. Nup93 and CTCF modulate spatiotemporal dynamics and function of the HOXA gene locus during differentiation. __J Cell Sci__ (2021). [Link](https://doi.org/doi:10.1242/JCS.259307).
437. Lee, HC., et al. YAP1 overexpression contributes to the development of enzalutamide resistance by induction of cancer stemness and lipid metabolism in prostate cancer. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/s41388-021-01718-4).
438. Lee, SA., et al. METTL8 mRNA Methyltransferase Enhances Cancer Cell Migration via Direct Binding to ARID1A. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115432).
439. Li, JN., et al. Expression of SnoRNA U50A Is Associated with Better Prognosis and Prolonged Mitosis in Breast Cancer. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13246304).
440. Liu, Y., et al. Tumors exploit FTO-mediated regulation of glycolytic metabolism to evade immune surveillance. __Cell Metab__ (2021). [Link](https://doi.org/doi:10.1016/j.cmet.2021.04.001).
441. Luginbühl, J., et al. Decoding Neuronal Diversification by Multiplexed Single-cell RNA-Seq. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/j.stemcr.2021.02.006).
442. M, F., et al. SUMO conjugation susceptibility of Akt/protein kinase B affects the expression of the pluripotency transcription factor Nanog in embryonic stem cells. __PLoS One__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0254447).
443. M, S., et al. PHLDA3 is an important downstream mediator of p53 in squamous cell carcinogenesis. __J Invest Dermatol__ (2021). [Link](https://doi.org/doi:10.1016/J.JID.2021.09.007).
444. M, Y., et al. Chronological genome and single-cell transcriptome integration characterizes the evolutionary process of adult T cell leukemia-lymphoma. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25101-9).
445. Ma, S., et al. Powerful gene-based testing by integrating long-range chromatin interactions and knockoff genotypes. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.14.21260405).
446. Ma, W., et al. BARTweb: a web server for transcriptional regulator association analysis. __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqab022).
447. Markouli, C.., et al. Sustained intrinsic WNT and BMP4 activation impairs hESC differentiation to definitive endoderm and drives the cells towards extra-embryonic mesoderm. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87547-7).
448. Marques, IJ.., et al. Downregulation of WT1 transcription factor gene expression is required to promote myocardial fate. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.06.451274).
449. Matsuura, N., et al. NOTCH3 limits the epithelial–mesenchymal transition and predicts a favorable clinical outcome in esophageal cancer. __Cancer Med__ (2021). [Link](https://doi.org/doi:10.1002/cam4.3933).
450. Milanesi, E., et al. Whole Blood Expression Pattern of Inflammation and Redox Genes in Mild Alzheimer's Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S334337).
451. Minnoye, L., et al. Chromatin accessibility profiling methods. __Nat Rev Methods Primers__ (2021). [Link](https://doi.org/doi:10.1038/s43586-020-00008-9).
452. Misawa, A., et al. Long Noncoding RNA HOXA11-AS and Transcription Factor HOXB13 Modulate the Expression of Bone Metastasis-Related Genes in Prostate Cancer. __Genes__ (2021). [Link](https://doi.org/doi:10.3390/genes12020182).
453. Miyake, M., et al. Integrated stress response regulates GDF15 secretion from adipocytes, preferentially suppresses appetite for a high-fat diet and improves obesity. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103448).
454. Miyazawa, K., et al. Trans-ancestry genome-wide analysis of atrial fibrillation provides new insights into disease biology and enables polygenic prediction of cardioembolic risk. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.06.21263189).
455. Morishita, M., et al. Characterization of mouse embryonic fibroblasts derived from Rassf6 knockout mice shows the implication of Rassf6 in the regulation of NF-κB signaling. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/GTC.12901).
456. Morrow, AK., et al. Epitome: predicting epigenetic events in novel cell types with multi-cell deep ensemble learning. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB676).
457. Mourao, L., et al. Hyperactive WNT/CTNNB1 signaling induces a competing cell proliferation and epidermal differentiation response in the mouse mammary epithelium. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.22.449461).
458. N, O., et al. mTOR inhibitors sensitize multiple myeloma cells to venetoclax via IKZF3-and Blimp-1-mediated BCL-2 up-regulation. __Haematologica__ (2021). [Link](https://doi.org/doi:10.3324/HAEMATOL.2021.278506).
459. Nagai, Y., et al. Glucotoxicity-induced suppression of Cox6a2 expression provokes β-cell dysfunction via augmented ROS production. __Biochem Biophys Res Commun__ (2021). [Link](https://doi.org/doi:10.1016/j.bbrc.2021.03.148).
460. Namba, S., et al. Transcript-targeted analysis reveals isoform alterations and double-hop fusions in breast cancer. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02833-4).
461. Nurtdinov, R., et al. Alu-mediated weak CEBPA binding and slow B cell transdifferentiation in human. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.28.466072).
462. Ogino, T., et al. Post-transcriptional repression of circadian component CLOCK regulates cancer-stemness in murine breast cancer cells. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/eLife.66155).
463. Oh, TI., et al. PGC1α Loss Promotes Lung Cancer Metastasis through Epithelial-Mesenchymal Transition. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13081772).
464. Omiya, H., et al. BMP signaling suppresses Gemc1 expression and ependymal differentiation of mouse telencephalic progenitors. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79610-6).
465. P, ME., et al. DNA Methylome Alterations are Associated with Airway Macrophage Differentiation and Phenotype During Lung Fibrosis. __Am J Respir Crit Care Med__ (2021). [Link](https://doi.org/doi:10.1164/RCCM.202101-0004OC).
466. P, Wü., et al. Transcription factors: Bridge between cell signaling and gene regulation. __Proteomics__ (2021). [Link](https://doi.org/doi:10.1002/PMIC.202000034).
467. PM, T., et al. Human autoinflammatory disease reveals ELF4 as a transcriptional regulator of inflammation. __Nat Immunol__ (2021). [Link](https://doi.org/doi:10.1038/S41590-021-00984-4).
468. Papagiannopoulos, CI., et al. The histone methyltransferase inhibitor A-366 enhances hemoglobin expression in erythroleukemia cells upon co-exposure with chemical inducers in culture. __J Biol Res__ (2021). [Link](https://doi.org/doi:10.1186/s40709-020-00132-3).
469. Pecori, F., et al. A defined glycosylation regulatory network modulates total glycome dynamics during pluripotency state transition. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79666-4).
470. Petrone, MíV., et al. The pluripotency transcription factor OCT4 represses heme oxygenase-1 gene expression. __FEBS Lett__ (2021). [Link](https://doi.org/doi:10.1002/1873-3468.14138).
471. Piette, BL.., et al. Comprehensive interactome profiling of the human Hsp70 network highlights functional differentiation of J domains. __Mol Cell__ (2021). [Link](https://doi.org/doi:10.1016/j.molcel.2021.04.012).
472. Q, L., et al. SIGNET: single-cell RNA-seq-based gene regulatory network prediction using multiple-layer perceptron bagging. __Brief Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/BIB/BBAB547).
473. Quinn, HM., et al. YAP and β-catenin cooperate to drive oncogenesis in basal breast cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.can-20-2801).
474. RC, Z., et al. Conserved immunomodulatory transcriptional networks underlie antipsychotic-induced weight gain. __Transl Psychiatry__ (2021). [Link](https://doi.org/doi:10.1038/S41398-021-01528-Y).
475. Reimegård, J., et al. A combined approach for single-cell mRNA and intracellular protein expression analysis. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-02142-w).
476. Ruiz, JéL., et al.  The regulatory genome of the malaria vector Anopheles gambiae : integrating chromatin accessibility and gene expression . __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqaa113).
477. S, H., et al. Targeting ATF4-dependent pro-survival autophagy to synergize glutaminolysis inhibition. __Theranostics__ (2021). [Link](https://doi.org/doi:10.7150/THNO.60028).
478. S, L., et al. Long noncoding RNA HOTAIR interacts with Y-Box Protein-1 (YBX1) to regulate cell proliferation. __Life Sci Alliance__ (2021). [Link](https://doi.org/doi:10.26508/LSA.202101139).
479. S, M., et al. The alternative serotonin transporter promoter P2 impacts gene function in females with irritable bowel syndrome. __J Cell Mol Med__ (2021). [Link](https://doi.org/doi:10.1111/JCMM.16736).
480. SY, W., et al. MYC suppresses STING-dependent innate immunity by transcriptionally upregulating DNMT1 in triple-negative breast cancer. __J Immunother Cancer__ (2021). [Link](https://doi.org/doi:10.1136/JITC-2021-002528).
481. Sakuratani, T., et al. Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2021). [Link](https://doi.org/doi:10.1007/S00428-020-02899-1).
482. Sarkar, A., et al. Genome-wide DNA methylation and multi-omics study of human chondrocyte ontogeny and an epigenetic clock analysis of adult chondrocytes. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.02.454544).
483. Sekita, Y., et al. AKT signaling is associated with epigenetic reprogramming via the upregulation of TET and its cofactor, alpha-ketoglutarate during iPSC generation. __Stem Cell Res Ther__ (2021). [Link](https://doi.org/doi:10.1186/S13287-021-02578-1).
484. Sharma, A. Inferring molecular mechanisms of dexamethasone therapy in severe COVID-19 from existing transcriptomic data. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145665).
485. Shimizu, T., et al. Osteocytes as main responders to low-intensity pulsed ultrasound treatment during fracture healing. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89672-9).
486. Shin, J., et al. Possible Involvement of Adipose Tissue in Patients With Older Age, Obesity, and Diabetes With Coronavirus SARS-CoV-2 Infection (COVID-19) via GRP78 (BIP/HSPA5): Significance of Hyperinsulinemia Management in COVID-19. __Diabetes__ (2021). [Link](https://doi.org/doi:10.2337/DB20-1094).
487. Shinohara, M., et al. Upregulated expression of a subset of genes in <i>APP</i> ; <i>ob</i> / <i>ob</i> mice: Evidence of an interaction between diabetes‐linked obesity and Alzheimer's disease. __FASEB Bioadv__ (2021). [Link](https://doi.org/doi:10.1096/fba.2020-00151).
488. Shirai, Y., et al. Elucidation of disease etiology by trans-layer omics analysis. __Inflamm Regen__ (2021). [Link](https://doi.org/doi:10.1186/s41232-021-00155-w).
489. Singh, I., et al. Heterogeneity in the Epigenetic Landscape of Murine Testis-Specific Histone Variants TH2A and TH2B Sharing the Same Bi-Directional Promoter. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.755751).
490. Stirparo, GG.., et al. OCT4 induces embryonic pluripotency via STAT3 signaling and metabolic mechanisms. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2008890118).
491. Stuparević, I., et al. Regulation of the conserved 3′-5′ exoribonuclease EXOSC10/Rrp6 during cell division, development and cancer. __Biol Rev__ (2021). [Link](https://doi.org/doi:10.1111/brv.12693).
492. Su, EY., et al. Reconstruction of dynamic regulatory networks reveals signaling-induced topology 3 changes associated with germ layer specification 4 5 6. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.06.443021).
493. Sun, Y., et al. Oleic Acid and Eicosapentaenoic Acid Reverse Palmitic Acid-induced Insulin Resistance in Human HepG2 Cells via the Reactive Oxygen Species / JUN Pathway. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/j.gpb.2019.06.005).
494. T, S., et al. Androgen-dependent and DNA binding-independent association of androgen receptor with chromatic regions coding androgen-induced non-coding RNAs. __Biosci Biotechnol Biochem__ (2021). [Link](https://doi.org/doi:10.1093/BBB/ZBAB135).
495. T, Y., et al. Saikosaponin-D Alleviates Renal Inflammation and Cell Apoptosis in a Mouse Model of Sepsis via TCF7/FOSL1/MMP9 Inhibition. __Mol Cell Biol__ (2021). [Link](https://doi.org/doi:10.1128/MCB.00332-21).
496. Takai, J., et al. Gata2 heterozygous mutant mice exhibit reduced inflammatory responses and impaired bacterial clearance. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.102836).
497. Takeda, T., et al. A stem cell marker KLF5 regulates CCAT1 via three-dimensional genome structure in colorectal cancer cells. __Br J Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41416-021-01579-4).
498. Takeuchi, Y., et al. The membrane-linked adaptor FRS2β fashions a cytokine-rich inflammatory microenvironment that promotes breast cancer carcinogenesis. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2103658118).
499. Tan, D., et al. Quantitative control of noise in mammalian gene expression by dynamic histone regulation. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/ELIFE.65654).
500. Teschendorff, AE., et al. Statistical mechanics meets single-cell biology. __Nat Rev Genet__ (2021). [Link](https://doi.org/doi:10.1038/s41576-021-00341-z).
501. Tian, Y., et al. Novel role of prostate cancer risk variant rs7247241 on PPP1R14A isoform transition through allelic TF binding and CpG methylation. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB347).
502. Umarov, R., et al. DeepCellState: An autoencoder-based framework for predicting cell type specific transcriptional states induced by drug treatment. __PLoS Comput Biol__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009465).
503. Unno, K., et al. Activated ALK Cooperates with N-Myc via Wnt/β-catenin Signaling to Induce Neuroendocrine Prostate Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-3351).
504. Wan, R., et al. Direct Reconstruction of Gene Regulatory Networks underlying Cellular state Transitions without Pseudo-time Inference. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.12.443928).
505. Wang, J., et al. HPV E7 affects the function of cervical cancer cells via the TAL1/lnc‑EBIC/KLHDC7B axis. __Oncol Rep__ (2021). [Link](https://doi.org/doi:10.3892/or.2021.8002).
506. Wang, L., et al. ASXL1 promotes adrenocortical carcinoma and is associated with chemoresistance to EDP regimen. __Aging__ (2021). [Link](https://doi.org/doi:10.18632/AGING.203534).
507. Wei, KC., et al. An innovative targeted therapy for fluoroscopy-induced chronic radiation dermatitis. __J Mol Med__ (2021). [Link](https://doi.org/doi:10.1007/S00109-021-02146-3).
508. Wen, L., et al. Selective EZH2 inhibitor zld1039 alleviates inflammation in cisplatin-induced acute kidney injury partially by enhancing RKIP and suppressing NF-κB p65 pathway. __Zhongguo Yao Li Xue Bao__ (2021). [Link](https://doi.org/doi:10.1038/S41401-021-00837-8).
509. Wierzbicka, JM.., et al. The Effects of Vitamin D on the Expression of IL-33 and Its Receptor ST2 in Skin Cells; Potential Implication for Psoriasis. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222312907).
510. Wu, B., et al. ORAI1 establishes resistance to SARS-CoV-2 infection by regulating tonic type I interferon signaling. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.04.442548).
511. X, C., et al. Poly-L-arginine promotes asthma angiogenesis through induction of FGFBP1 in airway epithelial cells via activation of the mTORC1-STAT3 pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04055-2).
512. X, L., et al. Defining Proximity Proteomics of Histone Modifications by Antibody-mediated Protein A-APEX2 Labeling. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/J.GPB.2021.09.003).
513. X, Z., et al. Combinatorial CRISPR/Cas9 Screening Reveals Epistatic Networks of Interacting Tumor Suppressor Genes and Therapeutic Targets in Human Breast Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-2555).
514. Xiang, T., et al. The novel ZEB1-upregulated protein PRTG induced by Helicobacter pylori infection promotes gastric carcinogenesis through the cGMP/PKG signaling pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-021-03440-1).
515. Y, H., et al. Norrie disease protein is essential for cochlear hair cell maturation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2106369118).
516. Y, O., et al. Multi-Omic Meta-Analysis of Transcriptomes and the Bibliome Uncovers Novel Hypoxia-Inducible Genes. __Biomedicines__ (2021). [Link](https://doi.org/doi:10.3390/BIOMEDICINES9050582).
517. Y, Z., et al. TcoFBase: a comprehensive database for decoding the regulatory transcription co-factors in human and mouse. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB950).
518. YH, C., et al. The Induction of Alpha-1 Antitrypsin by Vitamin D in Human T Cells Is TGF-β Dependent: A Proposed Anti-inflammatory Role in Airway Disease. __Front Nutr__ (2021). [Link](https://doi.org/doi:10.3389/FNUT.2021.667203).
519. Yagi, M., et al. Mitochondrial translation deficiency impairs NAD <sup>+</sup> ‐mediated lysosomal acidification. __EMBO J__ (2021). [Link](https://doi.org/doi:10.15252/embj.2020105268).
520. Yao, X., et al. TP53/miR-129/MDM2/4/TP53 feedback loop modulates cell proliferation and apoptosis in retinoblastoma. __Cell Cycle__ (2021). [Link](https://doi.org/doi:10.1080/15384101.2021.1892321).
521. Yazar, V., et al. Integrative genome-wide analysis of dopaminergic neuron-specific PARIS expression in Drosophila dissects recognition of multiple PPAR-γ associated gene regulation. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-00858-7).
522. Yi, L., et al. The FOXM1/RNF26/p57 axis regulates the cell cycle to promote the aggressiveness of bladder cancer. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04260-Z).
523. Yi, X., et al. Interrogating cell type-specific cooperation of transcriptional regulators in 3D chromatin. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103468).
524. Yoneda, M., et al. Nucleosome assembly protein 1 (NAP-1) is a regulator of histone H1 acetylation. __J Biochem__ (2021). [Link](https://doi.org/doi:10.1093/JB/MVAB098).
525. Yoshizaki, K., et al. Paternal age affects offspring via an epigenetic mechanism involving REST/NRSF. __EMBO Rep__ (2021). [Link](https://doi.org/doi:10.15252/embr.202051524).
526. Zampieri, C., et al. p53 mutations define the chromatin landscape to confer drug tolerance in pancreatic cancer. __Mol Oncol__ (2021). [Link](https://doi.org/doi:10.1002/1878-0261.13161).
527. Zhang, S., et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. __Sci Adv__ (2021). [Link](https://doi.org/doi:10.1126/SCIADV.ABG8205).
528. Zhao, X., et al. CircleBase: an integrated resource and analysis platform for human eccDNAs. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB1104).
529. Zhao, Y., et al. Identification of Unique Transcriptomic Signatures and Hub Genes Through RNA Sequencing and Integrated WGCNA and PPI Network Analysis in Nonerosive Reflux Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S340452).
530. de Roover, A., et al. Hypoxia induces DOT1L in articular cartilage to protect against osteoarthritis. __JCI Insight__ (2021). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.150451).
531. Łasut-Szyszka, B., et al. Transcriptome Analysis of Cells Exposed to Actinomycin D and Nutlin-3a Reveals New Candidate p53-Target Genes and Indicates That CHIR-98014 Is an Important Inhibitor of p53 Activity. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222011072).
532. AS, K., et al. TINC- A Method to Dissect Regulatory Complexes at Single-Locus Resolution- Reveals an Extensive Protein Complex at the Nanog Promoter. __Stem Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.STEMCR.2020.11.005).
533. Ahmed, M., et al. Integrating binding and expression data to predict transcription factors combined function. __BMC Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12864-020-06977-1).
534. Azim, K., et al. Resolving the transcriptional transitions associated with oligodendrocyte generation from adult neural stem cells by 2 single cell sequencing Running Title: Oligodendroglial transcriptional networks 4 5 6. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.12.18.423285).
535. B, Z., et al. Mi-2β-targeted inhibition induces immunotherapy response in melanoma. __Research Square__ (2020). [Link](https://doi.org/doi:10.21203/RS.3.RS-118153/V1).
536. Beknazarov, N., et al. Deep learning approach for predicting functional Z-DNA regions using omics data. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-76203-1).
537. Bono, H., et al. Meta-analysis of hypoxic transcriptomes from public databases. __Biomedicines__ (2020). [Link](https://doi.org/doi:10.3390/biomedicines8010010).
538. Booth, LN., et al. Males deploy multifaceted strategies and hijack longevity pathways to induce premature demise of the opposite sex. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.30.181008).
539. Bunina, D., et al. Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons. __Cell Syst__ (2020). [Link](https://doi.org/doi:10.1016/j.cels.2020.05.003).
540. Caruana, BT., et al. The NF-κB signalling pathway regulates GLUT6 expression in endometrial cancer. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109688).
541. Cataldo, A., et al. MiR-302b as a Combinatorial Therapeutic Approach to Improve Cisplatin Chemotherapy Efficacy in Human Triple-Negative Breast Cancer. __Cancers__ (2020). [Link](https://doi.org/doi:10.3390/cancers12082261).
542. Cavalli, M., et al. A Multi-Omics Approach to Liver Diseases: Integration of Single Nuclei Transcriptomics with Proteomics and HiCap Bulk Data in Human Liver. __OMICS__ (2020). [Link](https://doi.org/doi:10.1089/omi.2019.0215).
543. Charmpi, K., et al. Convergent network effects along the axis of gene expression during prostate cancer progression. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02188-9).
544. Chauvistré, H., et al. The Janus-faced role of KDM5B heterogeneity in melanoma: differentiation as a situational driver of both growth arrest and drug-resistance. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.01.999847).
545. Chen, CCL., et al. Histone H3.3G34-Mutant Interneuron Progenitors Co-opt PDGFRA for Gliomagenesis. __Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.cell.2020.11.012).
546. Chen, H., et al. PARK2 promotes mitochondrial pathway of apoptosis and antimicrotubule drugs chemosensitivity via degradation of phospho-BCL-2. __Theranostics__ (2020). [Link](https://doi.org/doi:10.7150/thno.47044).
547. Chen, J., et al. LncSEA: a platform for long non-coding RNA related sets and enrichment analysis. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa806).
548. Choi, JH., et al. Histone demethylase KDM4D cooperates with NFIB and MLL1 complex to regulate adipogenic differentiation of C3H10T1/2 mesenchymal stem cells. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-60049-8).
549. Choudhury, A., et al. Meningioma epigenetic grouping reveals biologic drivers and therapeutic vulnerabilities. __medRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.11.23.20237495).
550. Czipa, E., et al. ChIPSummitDB: a ChIP-seq-based database of human transcription factor binding sites and the topological arrangements of the proteins bound to them. __Database__ (2020). [Link](https://doi.org/doi:10.1093/database/baz141).
551. Devailly, G., et al. Comprehensive analysis of epigenetic signatures of human transcrip-tion control †. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.23.309625).
552. Dorotea, D., et al. Recent Insights Into SREBP as a Direct Mediator of Kidney Fibrosis via Lipid-Independent Pathways. __Front Pharmacol__ (2020). [Link](https://doi.org/doi:10.3389/fphar.2020.00265).
553. Farahmand, S., et al. ModEx: A text mining system for extracting mode of regulation of transcription factor-gene regulatory interaction. __J Biomed Inform__ (2020). [Link](https://doi.org/doi:10.1016/j.jbi.2019.103353).
554. Feng, C., et al. KnockTF: A comprehensive human gene expression profile database with knockdown/knockout of transcription factors. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz881).
555. Fornes, O., et al. JASPAR 2020: Update of the open-Access database of transcription factor binding profiles. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1001).
556. Gao, L., et al. Ozone therapy promotes the differentiation of basal keratinocytes via increasing Tp63-mediated transcription of KRT10 to improve psoriasis. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15160).
557. Gerisch, B., et al. HLH-30/TFEB Is a Master Regulator of Reproductive Quiescence. __Dev Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.devcel.2020.03.014).
558. Grujic, O., et al. Identification and characterization of constrained non-exonic bases lacking predictive epigenomic and transcription factor binding annotations. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-19962-9).
559. Guan, S., et al. TGF-β1 induces CREB1-mediated miR-1290 upregulation to antagonize lung fibrosis via Napsin A. __Int J Mol Med__ (2020). [Link](https://doi.org/doi:10.3892/ijmm.2020.4565).
560. Hayashi, A., et al. A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma. __Nat Cancer__ (2020). [Link](https://doi.org/doi:10.1038/s43018-019-0010-1).
561. Hayashi, A., et al. Evolutionary Dynamics of Non-Coding Regions in Pancreatic Ductal Adenocarcinoma. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.11.294389).
562. Hirano, K., et al. Indonesian ginger (Bangle) extract promotes neurogenesis of human neural stem cells through WNT pathway activation. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134772).
563. Horn, M., et al. Hexosamine Pathway Activation Improves Protein Homeostasis through the Integrated Stress Response. __iScience__ (2020). [Link](https://doi.org/doi:10.1016/j.isci.2020.100887).
564. Hsiao, WY., et al. The Lipid Handling Capacity of Subcutaneous Fat Is Programmed by mTORC2 during Development. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108223).
565. Hänsel-Hertsch, R., et al. Landscape of G-quadruplex DNA structural regions in breast cancer. __Nat Genet__ (2020). [Link](https://doi.org/doi:10.1038/s41588-020-0672-8).
566. Kaminuma, E., et al. DDBJ data analysis challenge: A machine learning competition to predict arabidopsis chromatin feature annotations from DNA sequences. __Genes Genet Syst__ (2020). [Link](https://doi.org/doi:10.1266/ggs.19-00034).
567. Katahira, J., et al. Human THO maintains the stability of repetitive DNA. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12760).
568. Kikuyama, T., et al. 25(OH)D3 stimulates the expression of vitamin D target genes in renal tubular cells when Cyp27b1 is abrogated. __J Steroid Biochem Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jsbmb.2020.105593).
569. Kim, YJ., et al. UBE2C Overexpression Aggravates Patient Outcome by Promoting Estrogen-Dependent/Independent Cell Proliferation in Early Hormone Receptor-Positive and HER2-Negative Breast Cancer. __Front Oncol__ (2020). [Link](https://doi.org/doi:10.3389/fonc.2019.01574).
570. Koga, T., et al. Expression of leukotriene B4 receptor 1 defines functionally distinct DCs that control allergic skin inflammation. __Cell Mol Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41423-020-00559-7).
571. Kohmoto, T., et al. Claudin-6 is a single prognostic marker and functions as a tumor-promoting gene in a subgroup of intestinal type gastric cancer. __Gastric Cancer__ (2020). [Link](https://doi.org/doi:10.1007/s10120-019-01014-x).
572. Kokaji, T., et al. Transomics analysis reveals allosteric and gene regulation axes for altered hepatic glucose-responsive metabolism in obesity. __Sci Signal__ (2020). [Link](https://doi.org/doi:10.1126/SCISIGNAL.AAZ1236).
573. Kolmykov, S., et al. GTRD: an integrated view of transcription regulation. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa1057).
574. Kremsky, I., et al. Protection from DNA re-methylation by transcription factors in primordial germ cells and pre-implantation embryos can explain trans-generational epigenetic inheritance. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02036-w).
575. Krześniak, Mł., et al. Synergistic activation of p53 by actinomycin D and nutlin-3a is associated with the upregulation of crucial regulators and effectors of innate immunity. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109552).
576. Kubota, N., et al. An integrated analysis of public genomic data unveils a possible functional mechanism of psoriasis risk via a long-range ERRFI1 enhancer. __BMC Med Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12920-020-0662-9).
577. Kutscher, L., et al. Functional loss of a non-canonical BCOR-PRC1.1 complex accelerates SHH-driven medulloblastoma formation. __Genes Dev__ (2020). [Link](https://doi.org/doi:10.1101/2020.02.06.938035).
578. Kwiecien, K., et al. The methylation status of the chemerin promoter region located from − 252 to + 258 bp regulates constitutive but not acute-phase cytokine-inducible chemerin expression levels. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-70625-7).
579. Kwon, ATJ., et al. Efficient Development of Platform Cell Lines Using CRISPR-Cas9 and Transcriptomics Analysis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.16.299248).
580. L, H., et al. A Regulation Loop between YAP and NR4A1 Balances Cell Proliferation and Apoptosis. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.CELREP.2020.108284).
581. LM, K., et al. Cytosolic pH regulates proliferation and tumour growth by promoting expression of cyclin D1. __Nat Metab__ (2020). [Link](https://doi.org/doi:10.1038/S42255-020-00297-0).
582. Larsen, EG.., et al. TMEM184B controls pruriceptor specification and function. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.01.25.919902).
583. Lei, X., et al. Crystal Structures of Ternary Complexes of MEF2 and NKX2–5 Bound to DNA Reveal a Disease Related Protein–Protein Interaction Interface. __J Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jmb.2020.07.004).
584. Leporcq, Cé., et al. TFmotifView: a webserver for the visualization of transcription factor motifs in genomic regions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa252).
585. Li, X., et al. ERRα activates SHMT2 transcription to enhance the resistance of breast cancer to lapatinib via modulating the mitochondrial metabolic adaption. __Biosci Rep__ (2020). [Link](https://doi.org/doi:10.1042/BSR20192465).
586. Li, Y., et al. TRlnc: a comprehensive database for human transcriptional regulatory information of lncRNAs. __Brief Bioinform__ (2020). [Link](https://doi.org/doi:10.1093/bib/bbaa011).
587. Li, YH., et al. FGF9 is a downstream target of SRY and sufficient to determine male sex fate in ex vivo XX gonad culture. __Biol Reprod__ (2020). [Link](https://doi.org/doi:10.1093/biolre/ioaa154).
588. Li, Z., et al. Runx2 (Runt-Related Transcription Factor 2)-Mediated Microcalcification Is a Novel Pathological Characteristic and Potential Mediator of Abdominal Aortic Aneurysm. __Arterioscler Thromb Vasc Biol__ (2020). [Link](https://doi.org/doi:10.1161/ATVBAHA.119.314113).
589. Lin, QXX., et al. TFregulomeR reveals transcription factors' context-specific features and functions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1088).
590. Lorenzo, JP., et al. APOBEC2 binds Chromatin and Represses Transcription during Myoblast Differentiation. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.07.29.223594).
591. Lou, S., et al. TopicNet: a framework for measuring transcriptional regulatory network change. __Bioinformatics__ (2020). [Link](https://doi.org/doi:10.1093/bioinformatics/btaa403).
592. Lyu, J., et al. Synthetic lethality of RB1 and aurora A is driven by stathmin-mediated disruption of microtubule dynamics. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18872-0).
593. Maharjan, M., et al. Overlapping but Distinct Sequences Play Roles in the Insulator and Promoter Activities of the Drosophila BEAF-Dependent scs' Insulator. __Genetics__ (2020). [Link](https://doi.org/doi:10.1534/genetics.120.303344).
594. Mao, N., et al. Oncogenic ERG represses PI3K signaling through downregulation of IRS2. __Cancer Res__ (2020). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-19-1394).
595. McErlean, P., et al. Profiling of H3K27Ac Reveals the Influence of Asthma on the Epigenome of the Airway Epithelium. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.585746).
596. Meng, Y., et al. Transcriptomic landscape profiling of metformin-treated healthy mice: Implication for potential hypertension risk when prophylactically used. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15472).
597. Mercatelli, D., et al. Gene regulatory network inference resources: A practical overview. __Biochim Biophys Acta Gene Regul Mech__ (2020). [Link](https://doi.org/doi:10.1016/j.bbagrm.2019.194430).
598. Merrett, JE., et al. Identification of DNA response elements regulating expression of CCAAT/enhancer-binding protein (C/EBP) β and δ and MAP kinase-interacting kinases during early adipogenesis. __Adipocyte__ (2020). [Link](https://doi.org/doi:10.1080/21623945.2020.1796361).
599. Miyamoto, Y., et al. Genetic loss of importin α4 causes abnormal sperm morphology and impacts on male fertility in mouse. __FASEB J__ (2020). [Link](https://doi.org/doi:10.1096/fj.202000768RR).
600. Nam, A.R., et al. Alternative methylation of intron motifs is associated with cancer-related gene expression in both canine mammary tumor and human breast cancer. __Clin Epigenetics__ (2020). [Link](https://doi.org/doi:10.1186/s13148-020-00888-4).
601. Namekawa, T., et al. ALDH1A1 in patient-derived bladder cancer spheroids activates retinoic acid signaling leading to TUBB3 overexpression and tumor progression. __Int J Cancer__ (2020). [Link](https://doi.org/doi:10.1002/ijc.32505).
602. Namekawa, T., et al. HIF1α inhibitor 2-methoxyestradiol decreases NRN1 expression and represses in vivo and in vitro growth of patient-derived testicular germ cell tumor spheroids. __Cancer Lett__ (2020). [Link](https://doi.org/doi:10.1016/j.canlet.2020.05.040).
603. Ochiai, K., et al. Chromatin Protein PC4 Orchestrates B Cell Differentiation by Collaborating with IKAROS and IRF4. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108517).
604. Ochsner, SA., et al. A transcriptional regulatory atlas of coronavirus infection of human cells. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.24.059527).
605. Ochsner, SA.., et al. Consensus transcriptional regulatory networks of coronavirus-infected human cells. __Sci Data__ (2020). [Link](https://doi.org/doi:10.1038/s41597-020-00628-6).
606. Ohishi, H., et al. Characterization of genetic-origin-dependent monoallelic expression in mouse embryonic stem cells. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12736).
607. Okada, R., et al. RNaseH2A Downregulation Drives Chromosomal DNA Fragmentation and Accumulation of RNA-DNA Hybrids in Senescent Cells. __SSRN__ (2020). [Link](https://doi.org/doi:10.2139/ssrn.3588831).
608. Okazaki, Y., et al. Increased glucose metabolism in Arid5b-/- skeletal muscle is associated with the down-regulation of TBC1 domain family member 1 (TBC1D1). __Biol Res__ (2020). [Link](https://doi.org/doi:10.1186/s40659-020-00313-3).
609. P, Sé., et al. Chromatin interactions in differentiating keratinocytes reveal novel atopic dermatitis- and psoriasis-associated genes. __J Allergy Clin Immunol__ (2020). [Link](https://doi.org/doi:10.1016/J.JACI.2020.09.035).
610. Pecori, F., et al. Mucin-type O-glycosylation controls pluripotency in mouse embryonic stem cells via Wnt receptor endocytosis. __J Cell Sci__ (2020). [Link](https://doi.org/doi:10.1242/jcs.245845).
611. Pradhananga, S., et al. Promoter anchored interaction landscape of THP-1 macrophages captures early immune response processes. __Cell Immunol__ (2020). [Link](https://doi.org/doi:10.1016/j.cellimm.2020.104148).
612. Pratapa, A., et al. Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data. __Nat Methods__ (2020). [Link](https://doi.org/doi:10.1038/s41592-019-0690-6).
613. Q, P., et al. VARAdb: a comprehensive variation annotation database for human. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/NAR/GKAA922).
614. Ray, JP., et al. Prioritizing disease and trait causal variants at the TNFAIP3 locus using functional and genomic features. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-15022-4).
615. Rikitake, M., et al. Analysis of GPI-anchored proteins involved in germline stem cell proliferation in the Caenorhabditis elegans germline stem cell niche. __J Biochem__ (2020). [Link](https://doi.org/doi:10.1093/jb/mvaa075).
616. Roels, J., et al. Aging of preleukemic thymocytes drives CpG island hypermethylation in T-cell acute lymphoblastic leukemia. __Blood Cancer Discov__ (2020). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0059).
617. Roels, J., et al. Distinct and temporary-restricted epigenetic mechanisms regulate human αβ and γδ T cell development. __Nat Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41590-020-0747-9).
618. Ronzio, M., et al. Integrating Peak Colocalization and Motif Enrichment Analysis for the Discovery of Genome-Wide Regulatory Modules and Transcription Factor Recruitment Rules. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00072).
619. SN, R., et al. STAT3 in the dorsal raphe gates behavioural reactivity and regulates gene networks associated with psychopathology. __Mol Psychiatry__ (2020). [Link](https://doi.org/doi:10.1038/S41380-020-00904-2).
620. Sakuratani, T., et al. Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2020). [Link](https://doi.org/doi:10.1007/s00428-020-02899-1).
621. Sharipov, RN., et al. Assessment of transcriptional importance of cell line-specific features based on GTRD and FANTOM5 data. __PLoS One__ (2020). [Link](https://doi.org/doi:10.1371/journal.pone.0243332).
622. Shikov, AE., et al. Phenome-wide functional dissection of pleiotropic effects highlights key molecular pathways for human complex traits. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-58040-4).
623. Sugasawa, T., et al. Influence of intermittent cold stimulations on CREB and its targeting genes in muscle: Investigations into molecular mechanisms of local cryotherapy. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134588).
624. Sukumaran, A., et al. Insight on Transcriptional Regulation of the Energy Sensing AMPK and Biosynthetic mTOR Pathway Genes. __Front Cell Dev Biol__ (2020). [Link](https://doi.org/doi:10.3389/fcell.2020.00671).
625. Swoboda, A., et al. STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pathway. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01584-6).
626. Tamura, I., et al. Transcription factor C/EBPβ induces genome-wide H3K27ac and upregulates gene expression during decidualization of human endometrial stromal cells. __Molec Cell Endocr__ (2020). [Link](https://doi.org/doi:10.1016/j.mce.2020.111085).
627. Tanaka, H., et al. The NSD2/WHSC1/MMSET methyltransferase prevents cellular senescence-associated epigenomic remodeling. __Aging Cell__ (2020). [Link](https://doi.org/doi:10.1111/acel.13173).
628. Teschendorff, AE., et al. Improved detection of tumor suppressor events in single-cell RNA-Seq data. __NPJ Genom Med__ (2020). [Link](https://doi.org/doi:10.1038/s41525-020-00151-y).
629. Teschendorff, AE.., et al. EPISCORE: Cell type deconvolution of bulk tissue DNA methylomes from single-cell RNA-Seq data. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02126-9).
630. Thonsri, U., et al. High glucose-ROS conditions enhance the progression in cholangiocarcinoma via upregulation of MAN2A2 and CHD8. __Cancer Sci__ (2020). [Link](https://doi.org/doi:10.1111/cas.14719).
631. Toro, Aé., et al. Novel Interplay between p53 and HO-1 in Embryonic Stem Cells. __Cells__ (2020). [Link](https://doi.org/doi:10.3390/cells10010035).
632. Totani, H., et al. Autocrine HGF/c-Met signaling pathway confers aggressiveness in lymph node adult T-cell leukemia/lymphoma. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01393-x).
633. Ueda, MT., et al. Comprehensive genomic analysis reveals dynamic evolution of endogenous retroviruses that code for retroviral-like protein domains. __Mob DNA__ (2020). [Link](https://doi.org/doi:10.1186/s13100-020-00224-w).
634. Wang, A., et al. Single-cell multiomic profiling of human lungs reveals cell-type-specific and age-dynamic control of SARS-CoV2 host genes. __Elife__ (2020). [Link](https://doi.org/doi:10.7554/ELIFE.62522).
635. Wang, R., et al. Transient IGF-1R inhibition combined with osimertinib eradicates AXL-low expressing EGFR mutated lung cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18442-4).
636. Wu, X., et al. Regulation of cellular sterol homeostasis by the oxygen responsive noncoding RNA lincNORS. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18411-x).
637. X, L., et al. The role of MYB proto-oncogene like 2 in tamoxifen resistance in breast cancer. __J Mol Histol__ (2020). [Link](https://doi.org/doi:10.1007/S10735-020-09920-6).
638. Y, L., et al. Single-Cell Transcriptome Analysis Reveals Dynamic Cell Populations and Differential Gene Expression Patterns in Control and Aneurysmal Human Aortic Tissue. __Circulation__ (2020). [Link](https://doi.org/doi:10.1161/CIRCULATIONAHA.120.046528).
639. Y, S., et al. UTX maintains functional integrity of murine hematopoietic system by globally regulating aging-associated genes. __Blood__ (2020). [Link](https://doi.org/doi:10.1182/BLOOD.2019001044).
640. Y, Y., et al. SAA1 is upregulated in gastric cancer-associated fibroblasts possibly by its enhancer activation. __Carcinogenesis__ (2020). [Link](https://doi.org/doi:10.1093/CARCIN/BGAA131).
641. Yumimoto, K., et al. Recent insight into the role of FBXW7 as a tumor suppressor. __Semin Cancer Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.semcancer.2020.02.017).
642. Zhao, W., et al. Epigenetic Regulation of m6A Modifications in Human Cancer. __Mol Ther Nucleic Acids__ (2020). [Link](https://doi.org/doi:10.1016/j.omtn.2019.11.022).
643. Zhou, S., et al. Noncoding mutations target cis-regulatory elements of the FOXA1 plexus in prostate cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-14318-9).
644. Zhou, Y., et al. epiCOLOC: Integrating Large-Scale and Context-Dependent Epigenomics Features for Comprehensive Colocalization Analysis. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00053).
645. Abugessaisa, I., et al. refTSS: A Reference Data Set for Human and Mouse Transcription Start Sites. __J Mol Biol__ (2019). [Link](https://doi.org/doi:10.1016/j.jmb.2019.04.045).
646. Ahn, H., et al. Propanet: Time-varying condition-specific transcriptional network construction by network propagation. __Front Plant Sci__ (2019). [Link](https://doi.org/doi:10.3389/fpls.2019.00698).
647. Ando, M., et al. Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-09937-w).
648. Aono, S., et al. β-catenin/TCF4 complex-mediated induction of the NRF3 (NFE2L3) gene in cancer cells. __Int J Mol Sci__ (2019). [Link](https://doi.org/doi:10.3390/ijms20133344).
649. Arata, Y., et al. Defective induction of the proteasome associated with T-cell receptor signaling underlies T-cell senescence. __Genes Cells__ (2019). [Link](https://doi.org/doi:10.1111/gtc.12728).
650. Bakhmet, EI., et al. hnRNP-K Targets Open Chromatin in Mouse Embryonic Stem Cells in Concert with Multiple Regulators. __Stem Cells__ (2019). [Link](https://doi.org/doi:10.1002/stem.3025).
651. Banerjee, S., et al. Identifying transcriptional regulatory modules among different chromatin states in mouse neural stem cells. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
652. Bocci, M., et al. Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__ (2019). [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
653. Bravo González-Blas, C., et al. cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. __Nat Methods__ (2019). [Link](https://doi.org/doi:10.1038/s41592-019-0367-1).
654. Chang, HC., et al. Investigating the role of super-enhancer RNAs underlying embryonic stem cell differentiation. __BMC Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12864-019-6293-x).
655. Chereji, RăV., et al. Quantitative MNase-seq accurately maps nucleosome occupancy levels. __Genome Biol__ (2019). [Link](https://doi.org/doi:10.1186/s13059-019-1815-z).
656. Chow, KT.., et al. IRF5 regulates unique subset of genes in dendritic cells during West Nile virus infection. __J Leukoc Biol__ (2019). [Link](https://doi.org/doi:10.1002/JLB.MA0318-136RRR).
657. Delicado, D., et al. Multilocus phylogeny, species delimitation and biogeography of Iberian valvatiform springsnails (Caenogastropoda: Hydrobiidae), with the description of a new genus. __Zool J Linn Soc__ (2019). [Link](https://doi.org/doi:10.1093/zoolinnean/zly093).
658. Duforestel, M., et al. Glyphosate primes mammary cells for tumorigenesis by reprogramming the epigenome in a TET3-dependent manner. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2019.00885).
659. Farahmand, S., et al. Causal Inference Engine: A platform for directional gene set enrichment analysis and inference of active transcriptional regulators. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz1046).
660. Ferris, E., et al. Parallel Accelerated Evolution in Distant Hibernators Reveals Candidate Cis Elements and Genetic Circuits Regulating Mammalian Obesity. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.10.102).
661. Fleming, AM.., et al. Location dependence of the transcriptional response of a potential G-quadruplex in gene promoters under oxidative stress. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz207).
662. Fontela, MG., et al. The conserved non-coding sequence 2 (CNS2) enhances CD69 transcription through cooperation between the transcription factors oct1 and RUNX1. __Genes__ (2019). [Link](https://doi.org/doi:10.3390/genes10090651).
663. Horie, K., et al. Down-regulation of GATA1-dependent erythrocyte-related genes in the spleens of mice exposed to a space travel. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-44067-9).
664. Hotta, M., et al. Farnesoid X receptor induces cell death and sensitizes to TRAIL-induced inhibition of growth in colorectal cancer cells through the up-regulation of death receptor 5. __Biochem Biophys Res Commun__ (2019). [Link](https://doi.org/doi:10.1016/j.bbrc.2019.09.033).
665. Hurtz, C., et al. Rationale for targeting BCL6 in MLL-rearranged acute lymphoblastic leukemia. __Genes Dev__ (2019). [Link](https://doi.org/doi:10.1101/gad.327593.119).
666. Ishijima, Y., et al. The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Molec Cell Endocr__ (2019). [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
667. Kang, H., et al. HSF1 regulates mevalonate and cholesterol biosynthesis pathways. __Cancers__ (2019). [Link](https://doi.org/doi:10.3390/cancers11091363).
668. Kehl, T., et al. The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int J Cancer__ (2019). [Link](https://doi.org/doi:10.1002/ijc.31834).
669. Kikuchi, M., et al. Enhancer variants associated with Alzheimer's disease affect gene expression via chromatin looping. __BMC Med Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12920-019-0574-8).
670. Klein, JC., et al. Functional testing of thousands of osteoarthritis-associated variants for regulatory activity. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-10439-y).
671. Kojima, M., et al. The histone methyltransferase WHSC1 is regulated by EZH2 and is important for ovarian clear cell carcinoma cell proliferation. __BMC Cancer__ (2019). [Link](https://doi.org/doi:10.1186/s12885-019-5638-9).
672. Kolmykov, SK., et al. Population size estimation for quality control of ChIP-Seq datasets. __PLoS One__ (2019). [Link](https://doi.org/doi:10.1371/journal.pone.0221760).
673. Kondo, T., et al. Two-step regulation of trachealess ensures tight coupling of cell fate with morphogenesis in the drosophila trachea. __Elife__ (2019). [Link](https://doi.org/doi:10.7554/eLife.45145).
674. Krug, B., et al. Pervasive H3K27 Acetylation Leads to ERV Expression and a Therapeutic Vulnerability in H3K27M Gliomas. __Cancer Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.ccell.2019.04.004).
675. Kulyté, Aé., et al. MicroRNA-27a/b-3p and PPARG regulate SCAMP3 through a feed-forward loop during adipogenesis. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-50210-3).
676. Kunii, Y., et al. Differential protein expression of DARPP-32 versus Calcineurin in the prefrontal cortex and nucleus accumbens in schizophrenia and bipolar disorder. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-51456-7).
677. Kuno, A., et al. Conserved DNA-binding motif loci reflect functional transcription factor binding sites. __Research Square__ (2019). [Link](https://doi.org/doi:10.21203/rs.2.15866/v1).
678. Lin, S., et al. Transcriptome and DNA methylome signatures associated with retinal Müller glia development, injury response, and aging. __Invest Ophthalmol Vis Sci__ (2019). [Link](https://doi.org/doi:10.1167/iovs.19-27361).
679. Lizio, M., et al. Update of the FANTOM web resource: Expansion to provide additional transcriptome atlases. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1099).
680. Maedera, S., et al. GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13298).
681. Matsuda, T., et al. Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__ (2019). [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
682. Menzel, M., et al. Enhort: A platform for deep analysis of genomic positions. __PeerJ Comput Sci__ (2019). [Link](https://doi.org/doi:10.7717/peerj-cs.198).
683. Murayama, Y., et al. Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13328).
684. Neiman, G., et al. Integrin alpha-5 subunit is critical for the early stages of human pluripotent stem cell cardiac differentiation. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-54352-2).
685. Ochsner, SA., et al. The Signaling Pathways Project, an integrated ‘omics knowledgebase for mammalian cellular signaling pathways. __Sci Data__ (2019). [Link](https://doi.org/doi:10.1038/s41597-019-0193-4).
686. Oulès, Béé., et al. Mutant Lef1 controls Gata6 in sebaceous gland development and cancer. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018100526).
687. Peggion, C., et al. The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol Neurobiol__ (2019). [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
688. Qian, FC., et al. SEanalysis: A web tool for super-enhancer associated regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz302).
689. Ranjit, M., et al. Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
690. Rauschmeier, Ré., et al. Bhlhe40 and Bhlhe41 transcription factors regulate alveolar macrophage self‐renewal and identity. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018101233).
691. Robinson, CM., et al. AHypoxia-inducible HIF1-GAL3ST1-sulfatide axis enhances ccRCC immune evasion via increased tumor cell-platelet binding. __Mol Cancer Res__ (2019). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-19-0461).
692. Saito, M., et al. A Polymorphic Variant in p19Arf Confers Resistance to Chemically Induced Skin Tumors by Activating the p53 Pathway. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
693. Salameti, V., et al. NOTCH1 signaling in oral squamous cell carcinoma via a TEL2/ SERPINE1 axis. __Oncotarget__ (2019). [Link](https://doi.org/doi:10.18632/oncotarget.27306).
694. Simak, M., et al. Boolean function network analysis of time course liver transcriptome data to reveal novel circadian transcriptional regulators in mammals. __J Chin Med Assoc__ (2019). [Link](https://doi.org/doi:10.1097/JCMA.0000000000000180).
695. Singh, R., et al. Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
696. Solari, C., et al. The pluripotency transcription factor Nanog represses glutathione reductase gene expression in mouse embryonic stem cells. __BMC Res Notes__ (2019). [Link](https://doi.org/doi:10.1186/s13104-019-4411-0).
697. Studd, JB., et al. Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__ (2019). [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
698. Suda, K., et al. Reduction of Rpd3 suppresses defects in locomotive ability and neuronal morphology induced by the knockdown of Drosophila SLC25A46 via an epigenetic pathway. __Exp Cell Res__ (2019). [Link](https://doi.org/doi:10.1016/j.yexcr.2019.111673).
699. Tano, A., et al. The juvenility-associated long noncoding RNA Gm14230 maintains cellular juvenescence. __J Cell Sci__ (2019). [Link](https://doi.org/doi:10.1242/jcs.227801).
700. Tong, Z., et al. TransmiR v2.0: An updated transcription factor-microRNA regulation database. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1023).
701. Ulirsch, JC., et al. Interrogation of human hematopoiesis at single-cell and single-variant resolution. __Nat Genet__ (2019). [Link](https://doi.org/doi:10.1038/s41588-019-0362-6).
702. Velychko, S., et al. Excluding Oct4 from Yamanaka Cocktail Unleashes the Developmental Potential of iPSCs. __Cell Stem Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.stem.2019.10.002).
703. Vijayakrishnan, J., et al. Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-13069-6).
704. Vogel, FCE., et al. Targeting the H3K4 Demethylase KDM5B Reprograms the Metabolome and Phenotype of Melanoma Cells. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2019.06.124).
705. Wu, CJ., et al. MiR-23~27~24–mediated control of humoral immunity reveals a TOX-driven regulatory circuit in follicular helper T cell differentiation. __Sci Adv__ (2019). [Link](https://doi.org/doi:10.1126/sciadv.aaw1715).
706. Wu, Z., et al. Cis-control of Six1 expression in neural crest cells during craniofacial development. __Dev Dyn__ (2019). [Link](https://doi.org/doi:10.1002/dvdy.109).
707. Yamada, T., et al. Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __WIREs RNA__ (2019). [Link](https://doi.org/doi:10.1002/wrna.1508).
708. Yashiro, T., et al. PU.1 plays a pivotal role in dendritic cell migration from the periphery to secondary lymphoid organs via regulating CCR7 expression. __FASEB J__ (2019). [Link](https://doi.org/doi:10.1096/fj.201900379RR).
709. Yevshin, I., et al. GTRD: A database on gene transcription regulation - 2019 update. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1128).
710. Yoshizaki, K., et al. Paternal Age Affects Offspring'S Behavior Possibly Via Epigenetic Mechanism Recruiting a Transcriptional Repressor Rest. __Eur Neuropsychopharmacol__ (2019). [Link](https://doi.org/doi:10.1016/j.euroneuro.2017.08.089).
711. Zhang, L., et al. DeepHINT: Understanding HIV-1 integration via deep learning with attention. __Bioinformatics__ (2019). [Link](https://doi.org/doi:10.1093/bioinformatics/bty842).
712. Zheng, R., et al. Cistrome Data Browser: Expanded datasets and new tools for gene regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1094).
713. Åkerborg, Ö., et al. High-Resolution Regulatory Maps Connect Vascular Risk Variants to Disease-Related Pathways. __Circulation__ (2019). [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
714. Anan, K., et al. LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gky234).
715. Baldi, S., et al. Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol Cell__ (2018). [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
716. Chatterjee, S., et al. Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res Treat__ (2018). [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
717. Chen, Y., et al. PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci__ (2018). [Link](https://doi.org/doi:10.1111/cas.13796).
718. Chishima, T., et al. Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes__ (2018). [Link](https://doi.org/doi:10.3390/genes9010023).
719. Chèneby, J., et al. ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx1092).
720. Cui, C., et al. Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics Proteomics Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
721. Dréos, Ré., et al. MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx995).
722. Egashira, Y., et al. Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
723. Ferris, E., et al. Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
724. Fiziev, P., et al. ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
725. Groff, AF.., et al. Enhancers in the Peril lincRNA locus regulate distant but not local genes. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1589-8).
726. Imrichova, H., et al. ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/388934).
727. Kawaji, H. dirHub: a trackHub configurator with directory structure projection. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/314807).
728. Kawata, K., et al. Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__ (2018). [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
729. Kehl, T., et al. REGGAE: A novel approach for the identification of key transcriptional regulators. __Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
730. Lasar, D., et al. Peroxisome Proliferator Activated Receptor Gamma Controls Mature Brown Adipocyte Inducibility through Glycerol Kinase. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2017.12.067).
731. Mao, F., et al. EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx918).
732. Misawa, A., et al. lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif Tissue Int__ (2018). [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
733. Miura, I., et al. DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J Psychiatr Res__ (2018). [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
734. Mochizuki, K., et al. SETDB1 is essential for mouse primordial germ cell fate determination by ensuring bmp signaling. __Development__ (2018). [Link](https://doi.org/doi:10.1242/dev.164160).
735. Mourad, Rë., et al. Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
736. Nishizawa, Y., et al. Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__ (2018). [Link](https://doi.org/doi:10.18632/oncotarget.23554).
737. Onodera, Y., et al. Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__ (2018). [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
738. Polouliakh, N., et al. Sequence homology in eukaryotes (SHOE): Interactive visual tool for promoter analysis 06 Biological Sciences 0604 Genetics. __BMC Genomics__ (2018). [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
739. Raffield, LM., et al. Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet__ (2018). [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
740. Romero-Cordoba, SL., et al. Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
741. Wang, H., et al. Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J Biol Chem__ (2018). [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
742. Chen, Y., et al. Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol__ (2017). [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
743. Govaere, O., et al. The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__ (2017). [Link](https://doi.org/doi:10.1038/onc.2017.260).
744. Guoa, MH., et al. Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc Natl Acad Sci U S A__ (2017). [Link](https://doi.org/doi:10.1073/pnas.1619052114).
745. Ishigaki, K., et al. Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat Genet__ (2017). [Link](https://doi.org/doi:10.1038/ng.3885).
746. Kalender Atak, Z., et al. Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med__ (2017). [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
747. Kehl, T., et al. RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkx350).
748. Matsuda, K., et al. ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__ (2017). [Link](https://doi.org/doi:10.1242/dev.143479).
749. Naderi, A. C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__ (2017). [Link](https://doi.org/doi:10.18632/oncotarget.17826).
750. Onodera, Y., et al. miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__ (2017). [Link](https://doi.org/doi:10.1111/acel.12680).
751. Sanosaka, T., et al. DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
752. Tanegashima, K., et al. Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes Cells__ (2017). [Link](https://doi.org/doi:10.1111/gtc.12456).
753. Turrell, FK., et al. Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev__ (2017). [Link](https://doi.org/doi:10.1101/gad.298463.117).
754. Umeyama, T., et al. DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
755. Yevshin, I., et al. GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkw951).
756. Yoshihara, M., et al. Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
757. Sugeno, N., et al. α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci Rep__ (2016). [Link](https://doi.org/doi:10.1038/srep36328).
