## To cite ChIP-Atlas in your publication:

- Paper
  - Zou, Z., Ohta, T., Oki, S. ChIP-Atlas 3.0: a data-mining suite to explore chromosome architecture together with large-scale regulome data. __Nucleic Acids Res__. 52(W1), W45-W53, 2024. http://dx.doi.org/10.1093/nar/gkae358
  - Zou, Z., Ohta, T., Miura, F., Oki, S. ChIP-Atlas 2021 update: a data-mining suite for exploring epigenomic landscapes by fully integrating ChIP-seq, ATAC-seq and Bisulfite-seq data. __Nucleic Acids Res__. 50(W1), W175-W182, 2022. http://dx.doi.org/10.1093/nar/gkac199
  - Oki, S., Ohta, T., Shioi, G., Hatanaka, H., Ogasawara, O., Okuda, Y., Kawaji, H., Nakaki, R., Sese, J., Meno, C. ChIP-Atlas: a data-mining suite powered by full integration of public ChIP-seq data. __EMBO Rep__. 19(12), e46255, 2018. http://dx.doi.org/10.15252/embr.201846255
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. https://chip-atlas.org

## Publications citing ChIP-Atlas:

1. A, G., et al. SUMO operates from a unique long tandem repeat to keep innate immunity in check. __Nucleic Acids Res__ (2025). [Link](https://doi.org/doi:10.1093/NAR/GKAF750).
2. Abentung, A., et al. DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition. __Cell Mol Life Sci__ (2025). [Link](https://doi.org/doi:10.1007/S00018-025-05730-9).
3. Agarwal, G., et al. Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.24.645017).
4. Aggrey-Fynn, JE.., et al. Therapeutic targeting of FOSL1 and RELA-dependent transcriptional mechanisms to suppress pancreatic cancer metastasis. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07810-X).
5. Agrawal, R., et al. p53 regulates DREAM complex-mediated repression in a p21-independent manner. __EMBO J__ (2025). [Link](https://doi.org/doi:10.1038/S44318-025-00402-7).
6. Alaeva, A., et al. OmiXAI: An Ensemble XAI Pipeline for Interpretable Deep Learning in Omics Data. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.28.651097).
7. Anderson, SM., et al. High-throughput discovery of transmembrane helix dimers from human single-pass membrane proteins with TOXGREEN sort-seq. __PNAS Nexus__ (2025). [Link](https://doi.org/doi:10.1093/PNASNEXUS/PGAF305).
8. Arase, M., et al. Multi-omics uncovers transcriptional programs of gut-resident memory CD4+ T cells in Crohn's disease. __J Exp Med__ (2025). [Link](https://doi.org/doi:10.1084/JEM.20242106).
9. Arya, SK., et al. N-alpha-acetyltransferase 40 modulates ecdysteroid action through chromatin accessibility changes near the promoters of 20-hydroxyecdysone response genes in Tribolium Castaneum TcA cells. __Insect Biochem Mol Biol__ (2025). [Link](https://doi.org/doi:10.1016/J.IBMB.2025.104285).
10. Baba, Y., et al. Tcf21 modulates fibroblast activation and promotes cardiac fibrosis after injury via Pdgfrb signaling. __Sci Rep__ (2025). [Link](https://doi.org/doi:10.1038/S41598-025-13102-3).
11. Barker, HR.., et al. TFBSFootprinter: a multiomics tool for prediction of transcription factor binding sites in vertebrate species. __Transcription__ (2025). [Link](https://doi.org/doi:10.1080/21541264.2025.2521764).
12. Bashkatov, A., et al. Deep learning deciphers the related role of master regulators and G-quadruplexes in tissue specification. __Sci Rep__ (2025). [Link](https://doi.org/doi:10.1038/S41598-025-07579-1).
13. Beknazarov, N., et al. OmicsFUSION: A Pretrained Hyena-Based Framework for Encoding, Reconstruction, and Representation of the Omics and DNA Data. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.09.05.674329).
14. Bergmann, PE.., et al. Sex-Specific Differential DNA Methylation in Mild Cognitive Impairment and Alzheimer’s Disease. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.05.15.654279).
15. Brown, AC.., et al. The cAMP responsive element modulator (CREM) transcription factor influences susceptibility to undernutrition and infection. __mBio__ (2025). [Link](https://doi.org/doi:10.1128/MBIO.01390-25).
16. Bryan, AF.., et al. Cohesin stabilization at promoters and enhancers by common transcription factors and chromatin regulators. __Epigenetics Chromatin__ (2025). [Link](https://doi.org/doi:10.1186/S13072-025-00598-2/FIGURES/6).
17. Butts, JC., et al. Identifying non-coding variant effects at scale via machine learning models of cis-regulatory reporter assays. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.16.648420).
18. Cai, J., et al. Liberation of galactose from lactose by gut microbial β-galactosidase prevents uterine bacterial infection. __ISME J__ (2025). [Link](https://doi.org/doi:10.1093/ISMEJO/WRAF185).
19. Calle, X., et al. 17-beta estradiol prevents cardiac myocyte hypertrophy by regulating mitochondrial E3 ubiquitin ligase 1. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07389-3).
20. Che, M., et al. Radiation-induced upregulation of itaconate in macrophages promotes the radioresistance of non-small cell lung cancer by stabilizing NRF2 protein and suppressing immune response. __Redox Biol__ (2025). [Link](https://doi.org/doi:10.1016/J.REDOX.2025.103711).
21. Chen, G., et al. Predicting the molecular mechanism of ginger targeting PRMT1/BTG2 axis to inhibit gastric cancer based on WGCNA and machine algorithms. __Phytomedicine__ (2025). [Link](https://doi.org/doi:10.1016/J.PHYMED.2025.156892).
22. Chen, H., et al. Maternal behavior promotes resilience to adolescent stress in mice through a microglia-neuron axis. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-57810-W).
23. Chen, N., et al. Prenatal BPB/BPAF Exposure Induces Depression-like Behavior in Male Mice Offspring via Compound-Specific Transcriptional Dysregulation and Neurodevelopmental Pathway Alterations. __Neurotoxicology__ (2025). [Link](https://doi.org/doi:10.1016/J.NEURO.2025.103322).
24. Chen, Z., et al. EBBP-Mediated Integrated Stress Response Attenuates Anthracycline-Induced Cardiotoxicity by Inhibiting the Ferroptosis of Cardiomyocytes. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202502726).
25. Chen, Z., et al. YTHDF2 promotes ATP synthesis and immune evasion in B cell malignancies. __Cell__ (2025). [Link](https://doi.org/doi:10.1016/J.CELL.2024.11.007).
26. Chevalley, M., et al. A large-scale benchmark for network inference from single-cell perturbation data. __Commun Biol__ (2025). [Link](https://doi.org/doi:10.1038/S42003-025-07764-Y).
27. Chiaro, PD., et al. A framework to mine laser microdissection-based omics data and uncover regulators of pancreatic cancer heterogeneity. __Gigascience__ (2025). [Link](https://doi.org/doi:10.1093/GIGASCIENCE/GIAF101).
28. Chuang, TD., et al. Comparative Analysis of Differentially Expressed Long Non-Coding RNA in Pre- and Postmenopausal Fibroids. __Int J Mol Sci__ (2025). [Link](https://doi.org/doi:10.3390/IJMS26146798).
29. Corton, J.C., et al. Characterization of a gene expression biomarker predictive of hypoxia-inducible factor-1 modulation. __Chem Biol Interact__ (2025). [Link](https://doi.org/doi:10.1016/J.CBI.2025.111595).
30. Cui, T., et al. Integrated bioinformatics and experimental validation identify ATF3 as a key gene in secondary brain damage after intracerebral hemorrhage. __PLoS One__ (2025). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0328530).
31. Dang, T., et al. ICAT drives lactylation of tumor-associated macrophages via the c-Myc-ENO1 axis to promote cervical cancer progression. __Free Radic Biol Med__ (2025). [Link](https://doi.org/doi:10.1016/J.FREERADBIOMED.2025.09.031).
32. del Burgo, FGé., et al. The chromatin reader Dido3 is a regulator of the gene network that controls B cell differentiation. __Cell Biosci__ (2025). [Link](https://doi.org/doi:10.1186/S13578-025-01394-X,).
33. Deng, Y., et al. Targeting the HLA-E–NKG2A axis in combination with MS-275 enhances NK cell-based immunotherapy against DMG. __J Exp Clin Cancer Res__ (2025). [Link](https://doi.org/doi:10.1186/S13046-025-03390-Y,).
34. Dilliott, AA.., et al. The Neurodegenerative Disease Knowledge Portal: Propelling Discovery Through the Sharing of Neurodegenerative Disease Genomic Resources. __Neurol Genet__ (2025). [Link](https://doi.org/doi:10.1212/NXG.0000000000200246).
35. Dong, W., et al. scGraphETM: Graph-Based Deep Learning Approach for Unraveling Cell Type-Specific Gene Regulatory Networks from Single-Cell Multi-Omics Data. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.01.24.634773).
36. Dong, Z., et al. Methyltransferase-like 3–catalysed N6-methyladenosine methylation facilitates the contribution of vascular smooth muscle cells to atherosclerosis. __Cardiovasc Res__ (2025). [Link](https://doi.org/doi:10.1093/CVR/CVAF029).
37. Du, C., et al. SUMO-Specific Peptidase 5 Promotes Oesophageal Squamous Cell Carcinoma Growth through the NF-κB- SLC1A3 Axis. __Front Biosci__ (2025). [Link](https://doi.org/doi:10.31083/FBL27047).
38. Dutta, S., et al. Tubulin hyperacetylation drives HMGB1 nuclear exit via the ROS-PARP1 axis leading to rotenone-induced G2/M Arrest. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.25.645152).
39. Espín, R., et al. Harnessing transcriptional regulation of alternative end-joining to predict cancer treatment. __NAR Cancer__ (2025). [Link](https://doi.org/doi:10.1093/NARCAN/ZCAF007).
40. Estrada, MD.., et al. Transcriptional regulation of Ligase IV by an intronic regulatory element directs thymocyte development. __Genes Immun__ (2025). [Link](https://doi.org/doi:10.1038/S41435-025-00353-3).
41. Feng, Y., et al. Overexpression of OTX2 in human neural cells links depression risk genes. __Transl Psychiatry__ (2025). [Link](https://doi.org/doi:10.1038/S41398-025-03320-8).
42. Fierro, C., et al. The WWP1-JARID1B axis sustains acute myeloid leukemia chemoresistance. __Proc Natl Acad Sci U S A__ (2025). [Link](https://doi.org/doi:10.1073/PNAS.2421159122).
43. Fu, X., et al. A foundation model of transcription across human cell types. __Nature__ (2025). [Link](https://doi.org/doi:10.1038/S41586-024-08391-Z).
44. Fujita, S., et al. Super-Enhancer-Mediated DLX5 Activation Defines Regulatory Mechanisms in Human Embryonic Stem Cell-Derived Osteoblasts. __Dev Growth Differ__ (2025). [Link](https://doi.org/doi:10.1111/DGD.70023).
45. Gim, DH., et al. Deciphering Cell-Type and Temporal-Specific Matrisome Expression Signatures in Human Cortical Development and Neurodevelopmental Disorders via scRNA-Seq Meta-Analysis. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.24.639826).
46. Ginting, RP., et al. Regulation of redox homeostasis by ATF4-MTHFD2 axis during white adipose tissue browning. __Redox Biol__ (2025). [Link](https://doi.org/doi:10.1016/J.REDOX.2025.103715).
47. Gothwal, SK., et al. DNA replication stress-induced transcriptome of Human Burkitt’s lymphoma identifies MBD1 as a novel suppressor of BCL6 rearrangements in germinal center derived B-lymphomagenesis. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.13.643172).
48. Grift, YBC., et al. GRHL and PGR control WNT4 expression in the mammary gland via 3D looping of conserved and species-specific enhancers. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.11.648333).
49. Gu, Q., et al. Single-cell and spatial transcriptome analyses reveal MAZ(+) NPC-like clusters as key role contributing to glioma recurrence and drug resistance. __J Transl Med__ (2025). [Link](https://doi.org/doi:10.1186/S12967-025-06706-W).
50. Guo, Y., et al. Network analysis reveals potential mechanisms that determine the cellular identity of keratinocytes and corneal epithelial cells through the Hox/Gtl2-Dio3 miRNA axis. __Front Cell Dev Biol__ (2025). [Link](https://doi.org/doi:10.3389/FCELL.2025.1475334).
51. Hamanaka, K., et al. Genome sequencing provides high diagnostic yield and new etiological insights for intellectual disability and developmental delay. __NPJ Genom Med__ (2025). [Link](https://doi.org/doi:10.1038/S41525-025-00521-4).
52. Hamdan, FH.., et al. Intestinal Stem Cells from Patients with Inflammatory Bowel Disease Retain an Epigenetic Memory of Inflammation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.05.24.655923).
53. Han, CW., et al. RUNX2 is a cell-intrinsic brake on Th17 pathogenicity driven by neutrophil extracellular traps. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.26.661866).
54. Han, J., et al. Cardiomyocyte-derived USP13 protects hearts from hypertrophy via deubiquitinating and stabilizing STAT1 in male mice. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-61028-1).
55. Harada, J., et al. H3K27me3-mediated regulation of PD-L1 expression in triple-negative breast cancer (TNBC). __Pathol Res Pract__ (2025). [Link](https://doi.org/doi:10.1016/J.PRP.2025.155872).
56. Hashimoto, T., et al. The oncoprotein DEK controls growth-regulated gene expression by enhancing the DNA-binding activity of basic leucine zipper transcription factors. __FEBS J__ (2025). [Link](https://doi.org/doi:10.1111/FEBS.70124).
57. Hattori, N., et al. Epigenetic disruption of adipogenic gene enhancers in dedifferentiated liposarcomas and its therapeutic value. __Front Oncol__ (2025). [Link](https://doi.org/doi:10.3389/FONC.2025.1419877).
58. Hayakawa, K., et al. A Novel O-GlcNAcylation at Threonine 71 of Histone H4 Is a Component of Heterochromatin. __J Biochem__ (2025). [Link](https://doi.org/doi:10.1093/JB/MVAF044).
59. Hayashi, A., et al. The Non-Coding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions. __Cancer Res Commun__ (2025). [Link](https://doi.org/doi:10.1158/2767-9764.CRC-24-0167).
60. He, B., et al. Helicobacter pylori CagA elevates FTO to induce gastric cancer progression via a "hit-and-run" paradigm. __Cancer Commun__ (2025). [Link](https://doi.org/doi:10.1002/CAC2.70004).
61. Hikichi, H., et al. Mafb Deficiency in Myeloid Cells Increases Susceptibility to Mycobacterium tuberculosis Infection in Mice. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.07.06.663407).
62. Hirade, K., et al. Inhibiting KRAS with CD47 and immune checkpoint overcomes intrinsic resistance to combined KRAS and immune checkpoint inhibitor therapy. __Cell Rep Med__ (2025). [Link](https://doi.org/doi:10.1016/J.XCRM.2025.102317).
63. Honda, N., et al. Implications of Mutant SOD1 on RNA Processing and Interferon Responses in Amyotrophic Lateral Sclerosis: Omics Data Analysis. __Cureus__ (2025). [Link](https://doi.org/doi:10.7759/CUREUS.81045).
64. Honda, S., et al. Shortening and optimization of MEF2C and GATA4 promote cardiac reprogramming. __Biochem Biophys Res Commun__ (2025). [Link](https://doi.org/doi:10.1016/J.BBRC.2025.152511).
65. Hong, Y., et al. LRRK2 reduces the sensitivity to TKI and PD-1 blockade in ccRCC via activating LPCAT1. __Oncogene__ (2025). [Link](https://doi.org/doi:10.1038/S41388-025-03289-0).
66. Hu, Q., et al. GPX3 promotes cisplatin resistance in TNBC by manipulating ROS-TGFB1-ZEB2. __Cell Commun Signal__ (2025). [Link](https://doi.org/doi:10.1186/S12964-025-02356-Z).
67. Hu, W., et al. RUNX1 promotes denervation-induced muscle atrophy by activating the JUNB/NF-κB pathway and driving M1 macrophage polarization. __Open Life Sci__ (2025). [Link](https://doi.org/doi:10.1515/BIOL-2025-1157).
68. Huang, S., et al. PLAGL2-STAU1-NCOA4 axis enhances gastric cancer peritoneal metastasis by resisting ferroptosis via ferritinophagy. __Apoptosis__ (2025). [Link](https://doi.org/doi:10.1007/S10495-025-02083-3).
69. Huang, S., et al. Prediction of NR3C1 as a methylation marker for the prevention and treatment of gastric cancer in Rhizoma Atractylodis Macrocephalae based on machine learning algorithm and bioinformatics analysis. __Front Genet__ (2025). [Link](https://doi.org/doi:10.3389/FGENE.2025.1584986).
70. Huang, SJ., et al. KAT6A Promotes Macrophage Inflammation and Periodontitis. __J Clin Periodontol__ (2025). [Link](https://doi.org/doi:10.1111/JCPE.14203).
71. Iannucci, A., et al. Loss of Adenosine Deaminase Acting on RNA 1 Induces Panoptosis and Immune Response in Ulcerative Colitis Gut Mucosa. __MedComm (2020)__ (2025). [Link](https://doi.org/doi:10.1002/MCO2.70212).
72. Ikeda, S., et al. Extraction of biological terms using large language models enhances the usability of metadata in the BioSample database. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.17.638570).
73. Ishiguro, K., et al. DOT1L inhibition reprograms innate immunity to potentiate immunomodulatory drug responses in multiple myeloma. __Cancer Lett__ (2025). [Link](https://doi.org/doi:10.1016/J.CANLET.2025.217941).
74. Ishihara, T., et al. Transcriptomic and lipidomic analysis of aging-associated inflammatory signature in mouse liver. __Inflamm Regen__ (2025). [Link](https://doi.org/doi:10.1186/S41232-025-00377-2,).
75. Ishii, K., et al. PU.1 and TGF-β signaling activate the cell-surface expression of CD103 in mast cells and dendritic cells: opposite roles of GATA2 in the expression of mucosal mast cell genes. __FEBS J__ (2025). [Link](https://doi.org/doi:10.1111/FEBS.70252).
76. Ishii, K., et al. PU.1 and TGF-β signaling transactivate CD103 expression in mast cells and dendritic cells: Opposite roles of GATA2 in the expression of mucosal mast cell-specific genes. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.10.637371).
77. J, K., et al. Single-nucleotide-resolution genomic maps of O6-methylguanine from the glioblastoma drug temozolomide. __Nucleic Acids Res__ (2025). [Link](https://doi.org/doi:10.1093/NAR/GKAE1320).
78. Jang, M., et al. Extracellular matrix glycation epigenetically regulates brain aging and neurodegeneration in the in vitro aged neurovascular model. __Biomaterials__ (2025). [Link](https://doi.org/doi:10.1016/J.BIOMATERIALS.2025.123504).
79. Jatzlau, J., et al. Rare but specific: 5-bp composite motifs define SMAD binding in BMP signaling. __BMC Biol__ (2025). [Link](https://doi.org/doi:10.1186/S12915-025-02183-1).
80. Ji, X., et al. Novel Evolutionarily Conserved Oncogene COA4 is Driven by KRAS Mutant and Promotes Cancer Metastasis Through Dual Mitochondrial Metabolism-Dependent and -Independent Mechanisms. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202507533).
81. Jiang, D., et al. A feedback loop between Paxillin and Yorkie sustains Drosophila intestinal homeostasis and regeneration. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-024-55255-1).
82. Jin, X., et al. ZDHHC12 Palmitoylates HDAC8 to Promote the Progression of Hepatocellular Carcinoma Associated with a Diet High in Saturated Fatty Acids. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202505702).
83. Jit, S., et al. Alternative Promoters Drive Transcriptomic Reprogramming and Prognostic Stratification in TNBC. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.19.643579).
84. Johansen, VBI., et al. Regulation of LEAP2 by insulin and glucagon in mice and humans. __Cell Rep Med__ (2025). [Link](https://doi.org/doi:10.1016/J.XCRM.2025.101996).
85. Kaminaka, R., et al. SOX10 Regulates Melanoma Metastasis Through the IRF1-ITGA3/EphA2-FAK Pathway. __Cancer Sci__ (2025). [Link](https://doi.org/doi:10.1111/CAS.70173).
86. Kayama, K., et al. High GRWD1 expression may predict clinically aggressive lower grade glioma, skin cutaneous melanoma, and kidney renal clear cell carcinoma carrying wild-type p53: a systematic study based on TCGA data analysis. __J Biochem__ (2025). [Link](https://doi.org/doi:10.1093/JB/MVAF004).
87. Kempynck, N., et al. CREsted: modeling genomic and synthetic cell type-specific enhancers across tissues and species. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.02.646812).
88. Kitahama, K., et al. Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma. __Sci Rep__ (2025). [Link](https://doi.org/doi:10.1038/S41598-025-13919-Y).
89. Kobayashi, Y., et al. Distinct dermal fibroblasts direct mechano-chemical signaling to the epidermis during pregnancy. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.09.11.675503).
90. Kommu, S., et al. Prediction of gene regulatory connections with joint single-cell foundation models and graph-based learning. __Bioinformatics__ (2025). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAF217).
91. Koslowski, S., et al. LIPTER, a cardiomyocyte-enriched long noncoding RNA, controls cardiac cytoskeletal maturation and is regulated by a cardiomyocyte-specific enhancer. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.08.10.669544).
92. Kratochvilová, Lš., et al. Chromatin Immunoprecipitation Reveals p53 Binding to G-Quadruplex DNA Sequences in Myeloid Leukemia Cell Lines. __ACS Bio Med Chem Au__ (2025). [Link](https://doi.org/doi:10.1021/ACSBIOMEDCHEMAU.4C00124/ASSET/IMAGES/LARGE/BG4C00124_0008.JPEG).
93. Lambourne, L., et al. Widespread variation in molecular interactions and regulatory properties among transcription factor isoforms. __Mol Cell__ (2025). [Link](https://doi.org/doi:10.1016/J.MOLCEL.2025.03.004).
94. Lapohos, O., et al. AmalgaMo: flexible DNA motif merging. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.07.29.667561).
95. Lax, C., et al. Genome-wide exploration of the transcriptional regulatory landscape in the early-diverging fungus R. microsporus reveals pervasive DNA methyl adenine regulatory network. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.01.24.634581).
96. Lecluze, E., et al. Ubiquitous dysregulation of the Wnt pathway in immune thrombocytopenia genetic susceptibility. __Blood Vessel Thromb Hemost__ (2025). [Link](https://doi.org/doi:10.1016/J.BVTH.2025.100107).
97. Lee, JH., et al. CREB Regulates Cisplatin Resistance by Targeting TNKS and KDM6A in NSCLC cell-Derived Tumor Spheroid. __Int J Biol Sci__ (2025). [Link](https://doi.org/doi:10.7150/IJBS.109419).
98. Lemée, MV., et al. Disrupted transcriptional networks regulated by CHD1L during neurodevelopment underlie the mirrored neuroanatomical and growth phenotypes of the 1q21.1 copy number variant. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.18.638841).
99. Li, D., et al. Global loss of responsiveness in key regulator metabolites and elevated enzyme proteins as metabolic dysregulation in skeletal muscle and liver of obese mice during starvation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.01.22.634184).
100. Li, J., et al. Estrogen receptor β alleviates colitis in intestinal epithelial cells and activates HIF-1a and ATG-9a-mediated autophagy. __Exp Cell Res__ (2025). [Link](https://doi.org/doi:10.1016/J.YEXCR.2025.114520).
101. Li, JP., et al. Comprehensive analytical approach identifies a subtype of CTCF+ tumor-associated neutrophils associated with CRC development and as a biomarker for immunotherapy. __Int J Biol Sci__ (2025). [Link](https://doi.org/doi:10.7150/IJBS.111529).
102. Li, Q., et al. Molecular Subtypes and Biomarkers of Ulcerative Colitis Revealed by Sphingolipid Metabolism-Related Genes: Insights from Machine Learning and Molecular Dynamics. __Curr Issues Mol Biol__ (2025). [Link](https://doi.org/doi:10.3390/CIMB47080616).
103. Li, Z., et al. Runx2-NLRP3 Axis Orchestrates Matrix Stiffness-evoked Vascular Smooth Muscle Cell Inflammation. __Am J Physiol__ (2025). [Link](https://doi.org/doi:10.1152/AJPCELL.00448.2024).
104. Liao, L., et al. Targeting the ceramidase ACER3 attenuates cholestasis in mice by mitigating bile acid overload via unsaturated ceramide-mediated LXRβ signaling transduction. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-57330-7).
105. Lin, SC., et al. The long noncoding RNA lncZBTB10 facilitates AR function via S-palmitoylation to promote prostate cancer progression and abiraterone resistance. __Br J Cancer__ (2025). [Link](https://doi.org/doi:10.1038/S41416-025-02938-1).
106. Liu, F., et al. BRD4 Signaling Maintains the Differentiated State of β Cells. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202505659).
107. Liu, J., et al. Identification of biomarkers associated with programmed cell death in liver ischemia-reperfusion injury: insights from machine learning frameworks and molecular docking in multiple cohorts. __Kusuri No Chishiki__ (2025). [Link](https://doi.org/doi:10.3389/FMED.2025.1501467).
108. Liu, J., et al. MED10 as a Novel Oncogenic Driver in HCC: Promoting Cell Cycle Progression and Proliferation Through RAF1 Activation. __Front Biosci__ (2025). [Link](https://doi.org/doi:10.31083/FBL39944).
109. Liu, PC., et al. Suppression of NF-κB and downstream XBP1 by DcR3 contributes to a decrease in antibody secretion. __J Immunol__ (2025). [Link](https://doi.org/doi:10.1093/JIMMUN/VKAE005).
110. Liu, Y., et al. LARP4-mediated hypertranslation drives T cell dysfunction in tumors. __Nat Immunol__ (2025). [Link](https://doi.org/doi:10.1038/S41590-025-02232-5).
111. Liu, Y., et al. TAF15 in tumor-associated macrophages enhances pro-tumorigenic polarization and promotes cholangiocarcinoma progression. __JHEP Rep__ (2025). [Link](https://doi.org/doi:10.1016/J.JHEPR.2025.101545).
112. Lizarazo, S., et al. An RNA polymerase III tissue and tumor atlas uncovers context-specific activities linked to 3D epigenome regulatory mechanisms. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.08.28.672650).
113. Lorenzo-Martín, L.F., et al. Ribosomal protein deficiencies linked to Diamond-Blackfan anemia induce distinctive alterations of ATF4 expression. __iScience__ (2025). [Link](https://doi.org/doi:10.1016/J.ISCI.2025.112138/ASSET/F18D3154-D322-4CE4-A8AB-C2F73C03EC2F/MAIN.ASSETS/GR6_LRG.JPG).
114. Lorenzo, D., et al. Loss of HNF1b drives pancreatic Intraductal Papillary Mucinous Neoplasms (IPMN) initiation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.12.659269).
115. Loureiro, JR.., et al. The Insertion of an ATTTC Repeat in an Alu Element Hyperactivates a Primate-Specific Neurodevelopmental Enhancer in Spinocerebellar Ataxia Type 37. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.03.657633).
116. Luo, X., et al. Outer membrane vesicles of Porphyromonas gingivalis impede bone regeneration by inducing ferroptosis via the Hippo-YAP signaling pathway. __J Nanobiotechnology__ (2025). [Link](https://doi.org/doi:10.1186/S12951-025-03457-0).
117. Lázaro-Carot, L., et al. TET3 protects the Dlk1-Dio3 Imprinted Locus from DNA hypomethylation during adult NSC Reprogramming. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.13.638077).
118. Ma, L., et al. Prenatal PM2.5 exposure affects embryonic hematopoietic development through SOX2-regulated gene expression. __J Hazard Mater__ (2025). [Link](https://doi.org/doi:10.1016/J.JHAZMAT.2025.137193).
119. Martin, F., et al. Structure of the nucleosome-bound human BCL7A. __Nucleic Acids Res__ (2025). [Link](https://doi.org/doi:10.1093/NAR/GKAF273).
120. Martino, N., et al. Endothelial STING and STAT1 mediate interferon-independent effects of IL-6 in an endotoxemia-induced model of shock. __J Clin Invest__ (2025). [Link](https://doi.org/doi:10.1172/JCI189570).
121. McKenna, NJ.., et al. Single cell transcriptomics in a treatment-segregated cohort exposes a STAT3-regulated therapeutic gap in idiopathic pulmonary fibrosis. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.16.659944).
122. Meng, X., et al. FN1 Immunoregulation in Glioblastoma: Insights From Neutrophil-Centric Studies. __Cancer Sci__ (2025). [Link](https://doi.org/doi:10.1111/CAS.70041).
123. Mori, J., et al. Assessment of chromatin remodeling of acute myeloid leukemia cells treated with gilteritinib: a case report. __J Med Case Rep__ (2025). [Link](https://doi.org/doi:10.1186/S13256-025-05186-2).
124. Morita, K., et al. Structural robustness and temporal vulnerability of the starvation-responsive metabolic network in healthy and obese mouse liver. __Sci Signal__ (2025). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ADS2547).
125. Mouren, JC., et al. Exonic enhancers are a widespread class of dual-function regulatory elements. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.27.645641).
126. Nantakeeratipat, T., et al. Temporal Transcriptomic Analysis of Periodontal Disease Progression and Its Molecular Links to Systemic Diseases. __Int J Mol Sci__ (2025). [Link](https://doi.org/doi:10.3390/IJMS26051998).
127. Negedu, S., et al. Characterization of core promoter activation by the Drosophila insulator-binding protein BEAF. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.25.661594).
128. Nishida, Y., et al. Dual MYC and GSPT1 Protein Degrader for MYC-Driven Cancers. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.24.650490).
129. Nishimura, T., et al. Cholesterol restriction primes antiviral innate immunity via SREBP1-driven noncanonical type I IFNs. __EMBO Rep__ (2025). [Link](https://doi.org/doi:10.1038/S44319-024-00346-9).
130. Nishio, M., et al. MOB1 deletion in murine mature adipocytes ameliorates obesity and diabetes. __Proc Natl Acad Sci U S A__ (2025). [Link](https://doi.org/doi:10.1073/PNAS.2424741122).
131. Nix, MN.., et al. A Bivalent Molecular Glue Linking Lysine Acetyltransferases to Oncogene-induced Cell Death. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.14.643404).
132. Nomura, T., et al. GLI3 in Archaic Humans: Possible Contributions to Human Skeletal and Brain Evolution. __Dev Growth Differ__ (2025). [Link](https://doi.org/doi:10.1111/DGD.70015).
133. Nornes, S., et al. Evaluating the transcriptional regulators of arterial gene expression via a catalogue of characterized arterial enhancers. __Elife__ (2025). [Link](https://doi.org/doi:10.7554/ELIFE.102440).
134. Northoff, BH., et al. CircRNAs increase during vascular cell differentiation and are biomarkers for vascular disease. __Cardiovasc Res__ (2025). [Link](https://doi.org/doi:10.1093/CVR/CVAF013).
135. Nurtdinov, RN.., et al. EPIraction - an atlas of candidate enhancer-gene interactions in human tissues and cell lines. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.18.638885).
136. Okazaki, H., et al. Expression of Lymphoid Enhancer-Binding Factor 1 in Cancer-Associated Fibroblasts Mediates Tumor Growth and Transdifferentiation Toward Squamous Cell Carcinoma in Human Breast Cancer. __Cancer Med__ (2025). [Link](https://doi.org/doi:10.1002/CAM4.70627).
137. Oliveros, W., et al. The impact of sex, age, and genetic ancestry on DNA methylation across tissues. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.29.651179).
138. Oyabu, M., et al. Dnmt3a overexpression disrupts skeletal muscle homeostasis, promotes an aging-like phenotype, and reduces metabolic elasticity. __iScience__ (2025). [Link](https://doi.org/doi:10.1016/J.ISCI.2025.112144).
139. Paul, S., et al. Cooperation between the Hippo and MAPK pathway activation drives acquired resistance to TEAD inhibition. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-56634-Y).
140. Peng, B., et al. Saliva-derived extracellular vesicles: a promising therapeutic approach for salivary gland fibrosis. __J Transl Med__ (2025). [Link](https://doi.org/doi:10.1186/S12967-025-06620-1).
141. Peng, R., et al. Dissecting dynamic gene regulatory network using transformer-based temporal causality analysis. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.02.05.636766).
142. Philips, RL.., et al. An activating Stat1 mutant disrupts normal STAT4 innate lymphocyte programs during viral infection. __Sci Immunol__ (2025). [Link](https://doi.org/doi:10.1126/SCIIMMUNOL.ADO5986).
143. Poovitha, S., et al. Srebf and Runx3 regulate hepatic HMG-CoA reductase and intracellular cholesterol under hypoxia. __Cell Mol Life Sci__ (2025). [Link](https://doi.org/doi:10.1007/S00018-025-05736-3).
144. Raby, J., et al. Leveraging the largest harmonized epigenomic data collection for metadata prediction validated and augmented over 350,000 public epigenomic datasets. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.09.04.670545).
145. Ran, R., et al. A Ligand-Centered Framework for γδ T Cell Activation in Colorectal Cancer Revealed by Single-Cell and Transformer-Based Perturbation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.16.660020).
146. Ribeiro, JH., et al. A human-specific, concerted repression of microcephaly genes contributes to radiation-induced growth defects in cortical organoids. __iScience__ (2025). [Link](https://doi.org/doi:10.1016/J.ISCI.2025.111853).
147. Roskams-Hieter, B., et al. baal-nf identifies motif-disrupting variants that decrease transcription factor binding affinity. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.01.17.633399).
148. Rucli, S., et al. Functional genomic profiling of O-GlcNAc reveals its context-specific interplay with RNA polymerase II. __Genome Biol__ (2025). [Link](https://doi.org/doi:10.1186/S13059-025-03537-2).
149. Rusch, J., et al. Fat cadherin cleavage releases a transcriptionally active nuclear fragment to regulate target gene expression. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.05.08.652952).
150. Schwayer, C.., et al. Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.01.14.632683).
151. Seasock, MJ.., et al. Let-7 restrains an epigenetic circuit in AT2 cells to prevent fibrogenic intermediates in pulmonary fibrosis. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-59641-1).
152. Settas, N., et al. Increased METTL3 Expression and m6A Methylation in Myoblasts of Facioscapulohumeral Muscular Dystrophy. __Int J Mol Sci__ (2025). [Link](https://doi.org/doi:10.3390/IJMS26115170).
153. Shalaby, O., et al. Folliculin Deletion in the Mouse Kidney Results in Cystogenesis of the Loops of Henle via Aberrant TFEB Activation. __Am J Pathol__ (2025). [Link](https://doi.org/doi:10.1016/J.AJPATH.2025.05.010).
154. Shalev, D., et al. High-throughput transcriptome analysis reveals a developmental increase in Acvr1c which mediates epigenetic repression of the gene encoding the pubertal brake, Makorin ring finger protein 3. __NAR Mol Med__ (2025). [Link](https://doi.org/doi:10.1093/NARMME/UGAF008).
155. Shi, X., et al. Deubiquitinase MYSM1 drives myocardial ischemia/reperfusion injury by stabilizing STAT1 in cardiomyocytes. __Theranostics__ (2025). [Link](https://doi.org/doi:10.7150/THNO.100097).
156. Shi, X., et al. Guanine nucleotide biosynthesis blockade impairs MLL complex formation and sensitizes leukemias to menin inhibition. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-57544-9).
157. Shi, Y., et al. Myeloid deficiency of Z-DNA binding protein 1 restricts septic cardiomyopathy via promoting macrophage polarisation towards the M2-subtype. __Clin Transl Med__ (2025). [Link](https://doi.org/doi:10.1002/CTM2.70315).
158. Shimizu, T., et al. Upregulation of YY1/EZH2 and MLH1 as Therapeutic Targets for Adult T-Cell Leukemia/Lymphoma. __Cancer Sci__ (2025). [Link](https://doi.org/doi:10.1111/CAS.70095).
159. Shirai, M., et al. Cadmium promotes hyaluronan synthesis by inducing hyaluronan synthase 3 expression in cultured vascular endothelial cells via the c-Jun N-terminal kinase-c-Jun pathway. __CRC Crit Rev Toxicol__ (2025). [Link](https://doi.org/doi:10.1016/J.TOX.2025.154062).
160. Shoeb, MR.., et al. A stem cell differentiation model reveals two alternative fates in CBFA2T3::GLIS2-driven acute megakaryoblastic leukemia initiation. __Commun Biol__ (2025). [Link](https://doi.org/doi:10.1038/S42003-025-08730-4).
161. Shu, H., et al. The Relevance of G-Quadruplexes in Gene Promoters and the First Introns Associated with Transcriptional Regulation in Breast Cancer. __Int J Mol Sci__ (2025). [Link](https://doi.org/doi:10.3390/IJMS26146874).
162. Simonis, A., et al. Persistent epigenetic memory of SARS-CoV-2 mRNA vaccination in monocyte-derived macrophages. __Mol Syst Biol__ (2025). [Link](https://doi.org/doi:10.1038/S44320-025-00093-6).
163. Solleiro-Villavicencio, H., et al. Inflammation: a key mechanism connecting metabolic-associated steatotic liver disease and systemic arterial hypertension. __Front Immunol__ (2025). [Link](https://doi.org/doi:10.3389/FIMMU.2025.1620585).
164. Sompub, K., et al. NF-κB RelA regulates temporal oligodendrocyte differentiation in the postnatal brains. __Front Cell Neurosci__ (2025). [Link](https://doi.org/doi:10.3389/FNCEL.2025.1622874).
165. Stefanie Egetemaier, A., et al. Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.10.641595).
166. Su, G., et al. Inferring gene regulatory networks by hypergraph generative model. __Cell Rep Methods__ (2025). [Link](https://doi.org/doi:10.1016/J.CRMETH.2025.101026).
167. Sudo, G., et al. Concurrent hypermethylation of CpG islands and hypomethylation of CpG-poor regions are associated with gastric cancer risk after Helicobacter pylori eradication. __Gastric Cancer__ (2025). [Link](https://doi.org/doi:10.1007/S10120-025-01646-2).
168. Sullivan, KA.., et al. Omics Integration Uncovers Mechanisms Associated with HIV Viral Load and Potential Therapeutic Insights. __medRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.07.29.25332397).
169. Sun, B., et al. Actin Networking Collapse Under STAT5A Deficiency Drives Mitochondrial Dysfunction and Autocrine IFN-β Production. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.05.26.656095).
170. Sun, H., et al. DNA remnants in red blood cells enable early detection of cancer. __Cell Res__ (2025). [Link](https://doi.org/doi:10.1038/S41422-025-01122-7).
171. Supino, D., et al. Monocyte-macrophage membrane expression of IL-1R2 is a severity biomarker in sepsis. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07597-X).
172. Suwa, T., et al. UPR-induced intracellular C5aR1 promotes adaptation to the hypoxic tumour microenvironment. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07862-Z).
173. Tang, S., et al. Novel mutations of SRPX facilitate the stemness and malignant progression of glioma. __Br J Cancer__ (2025). [Link](https://doi.org/doi:10.1038/S41416-025-03091-5).
174. Teng, F., et al. Targeting RAS-mutant cancer cells using a synthetic RAS-activated cancer killing system. __Trends Biotechnol__ (2025). [Link](https://doi.org/doi:10.1016/J.TIBTECH.2025.07.031).
175. Tiedje, V., et al. Loss of tumor cell MHC Class II drives MAPK-inhibitor insensitivity of BRAF-mutant anaplastic thyroid cancers. __J Clin Invest__ (2025). [Link](https://doi.org/doi:10.1172/JCI191781).
176. Tóth, BáB.., et al. In silico analysis of UCP promoters predicts conserved and human specific regulators of adipocyte thermogenesis. __iScience__ (2025). [Link](https://doi.org/doi:10.1016/J.ISCI.2025.112969).
177. Uno, Y., et al. O-GlcNAcylation on serine 40 of histone H2A promotes proliferation and invasion in triple-negative breast cancer. __Sci Rep__ (2025). [Link](https://doi.org/doi:10.1038/S41598-025-95394-Z).
178. Walters, BW., et al. Profiling Histone Modifications in Differentiating Mouse Spermatogonia with CUT&Tag. __Methods Mol Biol__ (2025). [Link](https://doi.org/doi:10.1007/978-1-0716-4698-4_1).
179. Wang, A., et al. Endoplasmic reticulum stress-related CLIP4 plays a procarcinogenic role in hepatocellular carcinoma: an integrated analysis. __BMC Cancer__ (2025). [Link](https://doi.org/doi:10.1186/S12885-025-13537-X).
180. Wang, H., et al. MAGEA6 Engages a YY1-Dependent Transcription to Dictate Perineural Invasion in Colorectal Cancer. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202501119).
181. Wang, H., et al. Targeting EGFR-binding protein SLC7A11 enhancing antitumor immunity of T cells via inducing MHC-I antigen presentation in nasopharyngeal carcinoma. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-024-07327-9).
182. Wang, J., et al. Early-life inflammation increases aggressive behavior in adult male mice through an astrocyte-neuron signaling. __Mol Psychiatry__ (2025). [Link](https://doi.org/doi:10.1038/S41380-025-03260-1).
183. Wang, N., et al. Distinct mismatch-repair complex genes set neuronal CAG-repeat expansion rate to drive selective pathogenesis in HD mice. __Cell__ (2025). [Link](https://doi.org/doi:10.1016/J.CELL.2025.01.031).
184. Wang, W., et al. BET inhibitor in combination with BCG vaccine enhances antitumor efficacy and orchestrates T cell reprogramming for melanoma. __Cell Rep Med__ (2025). [Link](https://doi.org/doi:10.1016/J.XCRM.2025.101995).
185. Wang, W., et al. CircDIAPH1 Promotes Liver Metastasis and Development of Colorectal Cancer by Initiation of CEACAM6 Expression. __Mol Carcinog__ (2025). [Link](https://doi.org/doi:10.1002/MC.23896).
186. Wang, W., et al. Enzalutamide-Resistant STEAP4+ MyoCAF Secrete Phosphatidylcholine to Foster Progression by Activating Stemness in Hormone-Sensitive Prostate Cancer. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202510602).
187. Wang, Y., et al. Identification and validation of pyroptosis-related genes in Alzheimer’s disease based on multi-transcriptome and machine learning. __Front Aging Neurosci__ (2025). [Link](https://doi.org/doi:10.3389/FNAGI.2025.1568337).
188. Wang, Y., et al. The DYT6 dystonia causative protein THAP1 is responsible for proteasome activity via PSMB5 transcriptional regulation. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-56867-X).
189. Wang, Z., et al. Targeting MTPN sensitizes pancreatic cancer of wild-type BRCA1/2 to Cisplatin-based chemotherapy. __Cancer Gene Ther__ (2025). [Link](https://doi.org/doi:10.1038/S41417-025-00925-5).
190. Watanabe, Y., et al. ALS-associated RNA-binding proteins promote UNC13A transcription through REST downregulation. __EMBO J__ (2025). [Link](https://doi.org/doi:10.1038/S44318-025-00506-0).
191. Wei, PJ., et al. GAEDGRN: reconstruction of gene regulatory networks based on gravity-inspired graph autoencoders. __Brief Bioinform__ (2025). [Link](https://doi.org/doi:10.1093/BIB/BBAF232).
192. Wen, X., et al. VSMC-specific TRPC1 deletion attenuates angiotensin II-induced hypertension and cardiovascular remodeling. __J Mol Med__ (2025). [Link](https://doi.org/doi:10.1007/S00109-024-02509-6).
193. Wijesinghe, KM.., et al. Plasma membrane accessible cholesterol is regulated by ACC1 and lipid droplets. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.08.21.671640).
194. Włodarczyk, T., et al. Epiregulon: Single-cell transcription factor activity inference to predict drug response and drivers of cell states. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-62252-5).
195. Woo, SR., et al. Blocking YAP1-Liprin-β2 interaction impedes metastasis and promotes tumor suppression in head and neck squamous carcinoma. __Sci Rep__ (2025). [Link](https://doi.org/doi:10.1038/S41598-025-11652-0).
196. Worpenberg, L., et al. Codon-specific ribosome stalling reshapes translational dynamics during branched-chain amino acid starvation. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.03.27.645654).
197. Wu, A., et al. FDX1 facilitates elesclomol-induced cuproptosis and promotes glioblastoma development via transcription factor NFKB1. __Biochem Pharmacol__ (2025). [Link](https://doi.org/doi:10.1016/J.BCP.2025.117186).
198. Wu, MJ., et al. KDM4C works in concert with GATA1 to regulate heme metabolism in head and neck squamous cell carcinoma. __Cell Mol Life Sci__ (2025). [Link](https://doi.org/doi:10.1007/S00018-025-05693-X).
199. Wu, R., et al. Dual Inference Routes for Natural Language Hypothesis Generation in Multimodal Drug Discovery: An Early Experimentation Study. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.09.02.673688).
200. Wu, X., et al. AKR1B10 dictates c-Myc stability to suppress colorectal cancer metastasis via PP2A nitration. __Sci Adv__ (2025). [Link](https://doi.org/doi:10.1126/SCIADV.ADV6937).
201. Wu, Y., et al. 5-Aminolevulinic Acid Photodynamic Therapy Modulates Lipid Production by Akt/JunD-mediated NR4A1 Activation in the Treatment of Acne Vulgaris. __J Invest Dermatol__ (2025). [Link](https://doi.org/doi:10.1016/J.JID.2025.01.018).
202. Xing, Z., et al. High-glucose-associated YTHDC1 lactylation reduces the sensitivity of bladder cancer to enfortumab vedotin therapy. __Cell Rep__ (2025). [Link](https://doi.org/doi:10.1016/J.CELREP.2025.115545).
203. Xu, K., et al. Pancreatic Cancer-Derived Extracellular Vesicles Enriched with miR-223-5p Promote Skeletal Muscle Wasting Associated with Cachexia. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202504064).
204. Xu, N., et al. Genome-wide CRISPR screen identifies Menin and SUZ12 as regulators of human developmental timing. __Nat Cell Biol__ (2025). [Link](https://doi.org/doi:10.1038/S41556-025-01751-5).
205. Xu, Y., et al. Spi1 aggravates neuropathic pain by modulating Clec7a-mediated neuroinflammation and microglial phagocytosis. __J Headache Pain__ (2025). [Link](https://doi.org/doi:10.1186/S10194-025-02089-X).
206. Xuan, Y., et al. Hydroxychloroquine cures autoimmune myocarditis by inhibiting the innate immune system via the C-X-C motif chemokine ligand 16 and C-X-C motif receptor 6 axis between macrophages and T cells. __Br J Pharmacol__ (2025). [Link](https://doi.org/doi:10.1111/BPH.70028).
207. Xue, F., et al. An Integrated Database for Exploring Alternative Promoters in Animals. __Sci Data__ (2025). [Link](https://doi.org/doi:10.1038/S41597-025-04548-1).
208. Y, Y., et al. Targeting macrophage circadian rhythms with microcurrent stimulation to activate cancer immunity through phagocytic defense. __Theranostics__ (2025). [Link](https://doi.org/doi:10.7150/THNO.100748).
209. Yamashita, Y., et al. Fam102a translocates Runx2 and Rbpjl to facilitate Osterix expression and bone formation. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-024-55451-Z).
210. Yang, F., et al. GATA1-mediated Notch signaling augment antitumor immunity of CD11b+CD27- natural killer cells maturation via BCL9/β-catenin signal. __Cell Rep__ (2025). [Link](https://doi.org/doi:10.1016/J.CELREP.2025.115708).
211. Yang, G., et al. Histone demethylase PHF2 regulates inflammatory genes in Alzheimer's disease. __Mol Psychiatry__ (2025). [Link](https://doi.org/doi:10.1038/S41380-025-03181-Z).
212. Yang, L., et al. Transcription factor ATF3 aggravates kidney fibrosis by maintaining the state of histone H3 lysine 27 acetylation. __Zhonghua Yi Xue Za Zhi__ (2025). [Link](https://doi.org/doi:10.1097/CM9.0000000000003425).
213. Yang, M., et al. Dimer-Specific FokT-seq Reveals DNA-Binding Dimerization and Novel Genomic Targets of TDP-43. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202508902).
214. Yang, R., et al. Integrative transcriptomics analysis reveals convergent toxicological effects of perfluorooctanoic acid and perfluorooctane sulfonate on human liver: Evidence from multiple models. __J Hazard Mater__ (2025). [Link](https://doi.org/doi:10.1016/J.JHAZMAT.2025.138112,).
215. Yang, Y., et al. Identifying Lactylation-related biomarkers and therapeutic drugs in ulcerative colitis: insights from machine learning and molecular docking. __BMC Pharmacol Toxicol__ (2025). [Link](https://doi.org/doi:10.1186/S40360-025-00939-7).
216. Yang, Y., et al. Inhibition of UBE2C Promotes Parkin-Mediated K63-Linked Ubiquitination of TOP2A to Induce Senescence and Increase Sensitivity of Doxorubicin in Breast Cancer. __Adv Sci__ (2025). [Link](https://doi.org/doi:10.1002/ADVS.202417348).
217. Ye, D., et al. Neurogranin facilitates maintaining the immunosuppressive state of hepatocellular carcinoma by promoting TGF-β1 secretion. __Int J Biol Macromol__ (2025). [Link](https://doi.org/doi:10.1016/J.IJBIOMAC.2025.143716,).
218. Ye, Y., et al. CellLoop: Identifying single-cell 3D genome chromatin loops. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.08.647893).
219. Yildirim, Z., et al. Parp3 assists muscle function and skeletal muscle differentiation by selectively adjusting H3K27me3 enrichment. __iScience__ (2025). [Link](https://doi.org/doi:10.1016/j.isci.2025.112267).
220. Yildiz, U., et al. High-throughput single-cell CRISPRi screens stratify neurodevelopmental functions of schizophrenia-associated genes. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.13.659629).
221. Yin, B., et al. Genetic compensation response contributes to Pleurodeles waltl limb regeneration. __NPJ Regen Med__ (2025). [Link](https://doi.org/doi:10.1038/S41536-025-00417-Y).
222. Yin, B., et al. Investigating the Oncogenic and Immunological Implications of YTHDF1 in Ovarian Cancer. __Biologics__ (2025). [Link](https://doi.org/doi:10.2147/BTT.S542488).
223. Yu, J., et al. AZIN1-dependent polyamine synthesis accelerates tumor cell cycle progression and impairs effector T-cell function in osteosarcoma. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07640-X).
224. Zhang, G., et al. TRAPT: a multi-stage fused deep learning framework for predicting transcriptional regulators based on large-scale epigenomic data. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-58921-0).
225. Zhang, H., et al. N6-methyladenosine RNA modification regulates the transcription of SLC7A11 through KDM6B and GATA3 to modulate ferroptosis. __J Biomed Sci__ (2025). [Link](https://doi.org/doi:10.1186/S12929-024-01100-Y).
226. Zhang, H., et al. NCF2 drives tumor progression and metastasis through NR2F2/LATS2/YAP1 axis in esophageal squamous cell carcinoma. __Cancer Lett__ (2025). [Link](https://doi.org/doi:10.1016/J.CANLET.2025.217902).
227. Zhang, M., et al. BRD1 deficiency affects SREBF1-related lipid metabolism through regulating H3K9ac/H3K9me3 transition to inhibit HCC progression. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07404-7).
228. Zhang, QH., et al. FNDC5/Irisin exacerbates APAP-induced acute liver injury through activating JNK/NF-κB and inflammatory response. __Zhongguo Yao Li Xue Bao__ (2025). [Link](https://doi.org/doi:10.1038/S41401-025-01509-7).
229. Zhang, Z., et al. Simultaneous epigenomic profiling and regulatory activity measurement using e2MPRA. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.06.26.661849).
230. zhao, J., et al. CanASM: A comprehensive database for genome-wide allele-specific DNA methylation identification and annotation in cancer. __bioRxiv__ (2025). [Link](https://doi.org/doi:10.1101/2025.04.03.646980).
231. Zhao, T., et al. Decoding SFRP2 progenitors in sustaining tooth growth at single-cell resolution. __Stem Cell Res Ther__ (2025). [Link](https://doi.org/doi:10.1186/S13287-025-04190-Z).
232. Zhou, J., et al. SWI/SNF ATPase silenced HLF potentiates lung metastasis in solid cancers. __Nat Commun__ (2025). [Link](https://doi.org/doi:10.1038/S41467-025-60329-9).
233. Zhou, Y., et al. Cellular crosstalk mediated by Meteorin-like regulating hepatic stellate cell activation during hepatic fibrosis. __Cell Death Dis__ (2025). [Link](https://doi.org/doi:10.1038/S41419-025-07734-6).
234. Zhou, Y., et al. The miR-1305/KLF5 negative regulatory loop affects pancreatic cancer cell proliferation and apoptosis. __Hum Cell__ (2025). [Link](https://doi.org/doi:10.1007/S13577-025-01173-3).
235. Zhu, Z., et al. Conjugated Bile Acid-Driven CD14+CD16+ Monocyte Infiltration Promotes Cholestatic Liver Injury by Enhancing Hepatocyte Necroptosis. __JHEP Rep__ (2025). [Link](https://doi.org/doi:10.1016/J.JHEPR.2025.101517).
236. Zhuang, C., et al. Rbm39 ameliorates metabolic dysfunction-associated steatotic liver disease by regulating Apob and Fabp4. __Biochim Biophys Acta Mol Basis Dis__ (2025). [Link](https://doi.org/doi:10.1016/J.BBADIS.2025.167815).
237. Abdrabou, AM.., et al. Identification of VISTA regulators in macrophages mediating cancer cell survival. __Sci Adv__ (2024). [Link](https://doi.org/doi:10.1126/SCIADV.ADQ8122).
238. Abudahab, S., et al. Genome-wide analysis of hepatic DNA methylation reveals impact of epigenetic aging on xenobiotic metabolism and transport genes in an aged mouse model. __Geroscience__ (2024). [Link](https://doi.org/doi:10.1007/S11357-024-01137-9).
239. Adams, M., et al. Generation and analysis of a mouse multi-tissue genome annotation atlas. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.31.578267).
240. Agostino, SD., et al. The Strong Activation of p53 Tumor Suppressor Drives the Synthesis of the Enigmatic Isoform of DUSP13 Protein. __Biomedicines__ (2024). [Link](https://doi.org/doi:10.3390/BIOMEDICINES12071449).
241. Ahn, JY., et al. Dual function of PHF16 in reinstating homeostasis of murine intestinal epithelium after crypt regeneration. __Dev Cell__ (2024). [Link](https://doi.org/doi:10.1016/J.DEVCEL.2024.08.009).
242. Al-Hadeethi, T., et al. Association Between Periodontitis and COVID-19. __Curr Oral Health Rep__ (2024). [Link](https://doi.org/doi:10.1007/S40496-023-00361-0).
243. Ali, S., et al. BATF controls IFN I production via DC-SCRIPT in plasmacytoid dendritic cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.11.574638).
244. An, J., et al. Identification of chemical inhibitors targeting long noncoding RNA through gene signature-based high throughput screening. __Int J Biol Macromol__ (2024). [Link](https://doi.org/doi:10.1016/J.IJBIOMAC.2024.139119).
245. Arce, MM.., et al. Central control of dynamic gene circuits governs T cell rest and activation. __Nature__ (2024). [Link](https://doi.org/doi:10.1038/S41586-024-08314-Y).
246. Arimura, Y., et al. MagIC-Cryo-EM: Structural determination on magnetic beads for scarce macromolecules in heterogeneous samples. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.21.576499).
247. Łasut-Szyszka, B., et al. Transcriptomic and proteomic study of cancer cell lines exposed to actinomycin D and nutlin-3a reveals numerous, novel candidates for p53-regulated genes. __Chem Biol Interact__ (2024). [Link](https://doi.org/doi:10.1016/J.CBI.2024.110946).
248. Badia-i-Mompel, P., et al. Comparison and evaluation of methods to infer gene regulatory networks from multimodal single-cell data. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.20.629764).
249. Bai, D., et al. scLong: A Billion-Parameter Foundation Model for Capturing Long-Range Gene Context in Single-Cell Transcriptomics. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.11.09.622759).
250. Bai, X., et al. METTL14 suppresses the expression of YAP1 and the stemness of triple-negative breast cancer. __J Exp Clin Cancer Res__ (2024). [Link](https://doi.org/doi:10.1186/S13046-024-03225-2).
251. Bakambamba, K., et al. IRE1-dependent GOLIM4 expression controls protein secretion to modulate glioblastoma cell adhesion and migration. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.10.22.619629).
252. Balachandra, V., et al. DNAJC9 prevents CENP-A mislocalization and chromosomal instability by maintaining the fidelity of histone supply chains. __EMBO J__ (2024). [Link](https://doi.org/doi:10.1038/S44318-024-00093-6).
253. Barnes, PE., et al. Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.09.05.611436).
254. Beknazarov, N., et al. Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-68439-Y).
255. Bergin, CJ.., et al. The dopamine transporter antagonist vanoxerine inhibits G9a and suppresses cancer stem cell functions in colon tumors. __Nat Cancer__ (2024). [Link](https://doi.org/doi:10.1038/S43018-024-00727-Y).
256. Bhattacharyya, S., et al. Identifying genetic variants associated with chromatin looping and genome function. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-52296-4).
257. Brown, M., et al. Concordant B and T Cell Heterogeneity Inferred from the Multiomic Landscape of Peripheral Blood Mononuclear Cells in a Crohn's Disease Cohort. __J Crohns Colitis__ (2024). [Link](https://doi.org/doi:10.1093/ECCO-JCC/JJAE055).
258. Brunie, M., et al. The Activation of the CCND1 Promoter by AP-1 and SOX Transcription Factors in PC3 Prostate Cancer Cells Can Be Prevented by Anacardic Acid Analogs. __Cell Biochem Biophys__ (2024). [Link](https://doi.org/doi:10.1007/S12013-024-01646-6).
259. Bryce-Smith, S., et al. TDP-43 loss induces extensive cryptic polyadenylation in ALS/FTD. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.22.576625).
260. Caporali, S., et al. Selective metabolic regulations by p53 mutant variants in pancreatic cancer. __J Exp Clin Cancer Res__ (2024). [Link](https://doi.org/doi:10.1186/S13046-024-03232-3).
261. Chandra, O., et al. Explainable models using transcription factor binding and epigenome patterns at promoters reveal disease-associated genes and their regulators in the context of cell-types. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.05.06.592622).
262. Chang, YH., et al. SETDB1 suppresses NK cell-mediated immunosurveillance in acute myeloid leukemia with granulo-monocytic differentiation. __Cell Rep__ (2024). [Link](https://doi.org/doi:10.1016/J.CELREP.2024.114536).
263. Chen, B., et al. LINC02454-CCT complex interaction is essential for telomerase activity and cell proliferation in head and neck squamous cell carcinoma. __Cancer Lett__ (2024). [Link](https://doi.org/doi:10.1016/J.CANLET.2024.216734).
264. Chen, H., et al. MiR-375 impairs breast cancer cell stemness by targeting the KLF5/G6PD signaling axis. __Environ Toxicol__ (2024). [Link](https://doi.org/doi:10.1002/TOX.24204).
265. Chen, X., et al. Canonical androgen response element motifs are tumor suppressive regulatory elements in the prostate. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-53734-Z).
266. Chen, X., et al. Characterization of dipyridamole as a novel ferroptosis inhibitor and its therapeutic potential in acute respiratory distress syndrome management. __Theranostics__ (2024). [Link](https://doi.org/doi:10.7150/THNO.102318).
267. Chen, X., et al. Integrating Metabolic RNA Labeling-Based Time-Resolved Single-Cell RNA Sequencing with Spatial Transcriptomics for Spatiotemporal Transcriptomic Analysis. __Small Methods__ (2024). [Link](https://doi.org/doi:10.1002/SMTD.202401297).
268. Cheng, S., et al. Review and Evaluate the Bioinformatics Analysis Strategies of ATAC-seq and CUT&Tag Data. __Genomics Proteomics Bioinformatics__ (2024). [Link](https://doi.org/doi:10.1093/GPBJNL/QZAE054).
269. Christensen, LM.., et al. T-regulatory cells require Sin3a for stable expression of Foxp3. __Front Immunol__ (2024). [Link](https://doi.org/doi:10.3389/FIMMU.2024.1444937).
270. Cinquin, O. ChIP-GPT: a managed large language model for robust data extraction from biomedical database records. __Brief Bioinform__ (2024). [Link](https://doi.org/doi:10.1093/BIB/BBAD535).
271. Cisneros-Soberanis, F., et al. Near millimolar concentration of nucleosomes in mitotic chromosomes from late prometaphase into anaphase. __J Cell Biol__ (2024). [Link](https://doi.org/doi:10.1083/JCB.202403165).
272. Corton, J.C., et al. A 50-gene biomarker identifies estrogen receptor-modulating chemicals in a microarray compendium. __Chem Biol Interact__ (2024). [Link](https://doi.org/doi:10.1016/J.CBI.2024.110952).
273. Cui, M., et al. HKDC1, a target of TFEB, is essential to maintain both mitochondrial and lysosomal homeostasis, preventing cellular senescence. __Proc Natl Acad Sci U S A__ (2024). [Link](https://doi.org/doi:10.1073/PNAS.2306454120).
274. Cui, W., et al. Gut microbial metabolite facilitates colorectal cancer development via ferroptosis inhibition. __Nat Cell Biol__ (2024). [Link](https://doi.org/doi:10.1038/S41556-023-01314-6).
275. da Silva, JéEH.., et al. A Data-Distribution and Successive Spline Points based discretization approach for evolving gene regulatory networks from scRNA-Seq time-series data using Cartesian Genetic Programming. __Biosystems__ (2024). [Link](https://doi.org/doi:10.1016/J.BIOSYSTEMS.2024.105126).
276. De Paolis, V., et al. Unconventional p65/p52 NF-κB module regulates key tumor microenvironment-related genes in breast tumor-associated macrophages (TAMs). __C R Acad Sci III__ (2024). [Link](https://doi.org/doi:10.1016/J.LFS.2024.123059).
277. Deng, Q., et al. SMARCA4 is a haploinsufficient B cell lymphoma tumor suppressor that fine-tunes centrocyte cell fate decisions. __Cancer Cell__ (2024). [Link](https://doi.org/doi:10.1016/J.CCELL.2024.02.011).
278. Dennis, M., et al. Her2 amplification, Rel-A, and Bach1 can influence APOBEC3A expression in breast cancer cells. __PLoS Genet__ (2024). [Link](https://doi.org/doi:10.1371/JOURNAL.PGEN.1011293).
279. Diawara, M., et al. The transcription factors NR5A1 and JUNB cooperate to activate the Axl promoter in mouse Sertoli cell lines. __Mol Biol Rep__ (2024). [Link](https://doi.org/doi:10.1007/S11033-024-09934-3).
280. Ding, L., et al. Maternal high fat diet induces circadian clock-independent endocrine alterations impacting the metabolism of the offspring. __iScience__ (2024). [Link](https://doi.org/doi:10.1016/J.ISCI.2024.110343).
281. Dominguez-Alonso, S., et al. Deciphering the Non-Coding Genome in Autism Spectrum Disorders (ASD): A Study of De Novo and Rare Inherited Variants through Targeted Sequencing in Regulatory Regions. __medRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.10.14.24315434).
282. Du, Z., et al. RRM2 Promotes Liver Metastasis of Pancreatic Cancer by Stabilizing YBX1 and Activating the TGF-beta Pathway. __iScience__ (2024). [Link](https://doi.org/doi:10.1016/J.ISCI.2024.110864).
283. Ekstrom, TL.., et al. USP44 overexpression drives a MYC-like gene expression program in neuroblastoma through epigenetic reprogramming. __Mol Cancer Res__ (2024). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-23-0454).
284. Escribano-Núñez, A., et al. IGF1 drives Wnt-induced joint damage and is a potential therapeutic target for osteoarthritis. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-53604-8).
285. Fajac, A., et al. Mutant mice lacking alternatively spliced p53 isoforms unveil Ackr4 as a male-specific prognostic factor in Myc-driven B-cell lymphomas. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2023.10.13.562180).
286. Fernández-Ginés, R., et al. Specific targeting of the NRF2/β-TrCP axis promotes beneficial effects in NASH. __Redox Biol__ (2024). [Link](https://doi.org/doi:10.1016/J.REDOX.2024.103027).
287. Franco Acevedo, A., et al. The transcriptional repressor BCL6 regulates CXCR3 chemokine and HLA II expression in endothelial cells. __Am J Transplant__ (2024). [Link](https://doi.org/doi:10.1016/J.AJT.2024.07.026).
288. Frenkel, M., et al. Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants. __Nat Biotechnol__ (2024). [Link](https://doi.org/doi:10.1038/S41587-024-02347-4).
289. Fujiwara, K., et al. Spatial organizations of heterochromatin underpin nuclear structural integrity of ventricular cardiomyocytes against mechanical stress. __Cell Rep__ (2024). [Link](https://doi.org/doi:10.1016/J.CELREP.2024.115048).
290. Gan, Y., et al. Inferring gene regulatory networks from single-cell transcriptomics based on graph embedding. __Bioinformatics__ (2024). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAE291).
291. Gancedo, S.N.., et al. The xenobiotic transporter ABCC4/MRP4 promotes epithelial mesenchymal transition in pancreatic cancer. __Front Pharmacol__ (2024). [Link](https://doi.org/doi:10.3389/FPHAR.2024.1432851).
292. Gao, L., et al. Caffeine upregulates SIRT3 expression to ameliorate astrocytes-mediated HIV-1 Tat neurotoxicity via suppression of EGR1 signaling pathway. __J Neurovirol__ (2024). [Link](https://doi.org/doi:10.1007/S13365-024-01222-X).
293. Garg, S., et al. BRD9 regulates normal human hematopoietic stem cell function and lineage differentiation. __Cell Death Differ__ (2024). [Link](https://doi.org/doi:10.1038/S41418-024-01306-5).
294. Gothwal, SK., et al. Stress Induced Signaling Pathways in Burkitt’s Lymphoma Play Novel Mechanisms in Fate Determination and Pathogenesis of Germinal Center-Derived B-Lymphomas. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.19.628635).
295. Grześ, M., et al. A common druggable signature of oncogenic c-Myc, mutant KRAS and mutant p53 reveals functional redundancy and competition among oncogenes in cancer. __Cell Death Dis__ (2024). [Link](https://doi.org/doi:10.1038/S41419-024-06965-3).
296. Gu, J., et al. Noncanonical functions of microRNAs in the nucleus. __Sheng Wu Hua Xue Yu Sheng Wu Wu Li Xue Bao (Shanghai)__ (2024). [Link](https://doi.org/doi:10.3724/ABBS.2023268).
297. Gualdrini, F., et al. An integrative epigenome-based strategy for unbiased functional profiling of clinical kinase inhibitors. __Mol Syst Biol__ (2024). [Link](https://doi.org/doi:10.1038/S44320-024-00040-X).
298. Guo, H., et al. Sp1 Regulates the M1 Polarization of Microglia Through the HuR/NF-κB Axis after Spinal Cord Injury. __Int J Neurosci__ (2024). [Link](https://doi.org/doi:10.1016/J.NEUROSCIENCE.2024.02.014).
299. Han, F., et al. GLTSCR1 deficiency promotes colorectal cancer development through regulating non-homologous end joining. __Oncogene__ (2024). [Link](https://doi.org/doi:10.1038/S41388-024-03179-X).
300. Han, S., et al. E2F1 modulates RCCD1 expression to participate in the initiation and progression of EMT in colorectal cancer. __Pathol Res Pract__ (2024). [Link](https://doi.org/doi:10.1016/J.PRP.2024.155429).
301. Hara, S., et al. Identification of responsible sequences which mutations cause maternal H19-ICR hypermethylation with Beckwith-Wiedemann syndrome-like overgrowth. __Commun Biol__ (2024). [Link](https://doi.org/doi:10.1038/S42003-024-07323-X).
302. Hara, T., et al. High N6-methyladenosine-activated TCEAL8 mRNA is a novel pancreatic cancer marker. __Cancer Sci__ (2024). [Link](https://doi.org/doi:10.1111/CAS.16152).
303. He, H., et al. GINS2 regulates temozolomide chemosensitivity via the EGR1/ECT2 axis in gliomas. __Cell Death Dis__ (2024). [Link](https://doi.org/doi:10.1038/S41419-024-06586-W).
304. He, W., et al. Npy transcription is regulated by noncanonical STAT3 signaling in hypothalamic neurons: Implication with lipotoxicity and obesity. __Molec Cell Endocr__ (2024). [Link](https://doi.org/doi:10.1016/J.MCE.2024.112179).
305. Hosoi, H., et al. IGLL5 controlled by super-enhancer affects cell survival and MYC expression in mature B-cell lymphoma. __Leuk Res Rep__ (2024). [Link](https://doi.org/doi:10.1016/J.LRR.2024.100451).
306. Huang, T., et al. Correlating gene expression levels with transcription factor binding sites facilitates identification of key transcription factors from transcriptome data. __Front Genet__ (2024). [Link](https://doi.org/doi:10.3389/FGENE.2024.1511456).
307. Iida, M., et al. Dysregulated synaptic gene expression in oligodendrocytes of spinal and bulbar muscular atrophy. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.11.575248).
308. Inoue, T., et al. The c.1617del variant of TMEM260 is identified as the most frequent single gene determinant for Japanese patients with a specific type of congenital heart disease. __J Hum Genet__ (2024). [Link](https://doi.org/doi:10.1038/S10038-024-01225-W).
309. Ito, M., et al. Protein acylations induced by a ketogenic diet demonstrate diverse patterns depending on organs and differ between histones and global proteins. __Biochem Biophys Res Commun__ (2024). [Link](https://doi.org/doi:10.1016/J.BBRC.2024.149960).
310. Iyer, P., et al. MGA deletion leads to Richter's transformation by modulating mitochondrial OXPHOS. __Sci Transl Med__ (2024). [Link](https://doi.org/doi:10.1126/SCITRANSLMED.ADG7915).
311. J, X., et al. miRStart 2.0: enhancing miRNA regulatory insights through deep learning-based TSS identification. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE1086).
312. Jabeen, S., et al. A Comprehensive Evaluation of Self Attention for Detecting Feature Interactions. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.08.23.609428).
313. Jia, Y., et al. Oncogenic HJURP enhancer promotes the aggressive behavior of triple-negative breast cancer in association with p53/E2F1/FOXM1-axis. __Cancer Lett__ (2024). [Link](https://doi.org/doi:10.1016/J.CANLET.2024.217423).
314. Jones, AG.., et al. Biological sex affects functional variation across the human genome. __medRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.09.03.24313025).
315. Kaizuka, T., et al. Remodeling of the postsynaptic proteome in male mice and marmosets during synapse development. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/s41467-024-46529-9).
316. Kalyanaraman, H., et al. A plasma membrane-associated form of the androgen receptor enhances nuclear androgen signaling in osteoblasts and prostate cancer cells. __Sci Signal__ (2024). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ADI7861).
317. Kano, N., et al. Identification and characterization of putative enhancer regions that direct Il6 transcription in murine macrophages. __Int Immunol__ (2024). [Link](https://doi.org/doi:10.1093/INTIMM/DXAE024).
318. Karamveer, .. Approaches for Benchmarking Single-Cell Gene Regulatory Network Methods. __Bioinform Biol Insights__ (2024). [Link](https://doi.org/doi:10.1177/11779322241287120).
319. Karkoutly, S., et al. FoxO transcription factors regulate urea cycle through Ass1. __Biochem Biophys Res Commun__ (2024). [Link](https://doi.org/doi:10.1016/J.BBRC.2024.150594).
320. Katagiri, M., et al. NR4A3 deficiency ameliorates contact hypersensitivity and exacerbates psoriasis by regulating gene expression in dendritic cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.02.20.581116).
321. Kawakatsu, R., et al. Venetoclax efficacy on acute myeloid leukemia is enhanced by the combination with butyrate. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-55286-0).
322. Kawataki, S., et al. GADD45β-MTK1 signaling axis mediates oncogenic stress-induced activation of the p38 and JNK pathways. __Cancer Sci__ (2024). [Link](https://doi.org/doi:10.1111/CAS.16389).
323. Kim, GD., et al. Cell Type- and Age-Specific Expression of lncRNAs across Kidney Cell Types. __J Am Soc Nephrol__ (2024). [Link](https://doi.org/doi:10.1681/ASN.0000000000000354).
324. Kirtana, R., et al. Modulation of H3K4 trimethylation by KDM5A and MLLs impacts metabolic adaptability in prostate and cervical cancer cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.05.02.592178).
325. Kittke, V., et al. RLS-associated MEIS transcription factors control distinct processes in human neural stem cells. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-80266-9).
326. Koga, D., et al. Identification and characterization of an enhancer element regulating expression of Cdkn1c (p57 gene). __Genes Cells__ (2024). [Link](https://doi.org/doi:10.1111/GTC.13173).
327. Kohnke, T., et al. Human ASXL1-mutant hematopoiesis is driven by a truncated protein associated with aberrant de-ubiquitination of H2AK119. __Blood Cancer Discov__ (2024). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-23-0235).
328. Kommu, S., et al. Gene Regulatory Network Inference with Joint Representation from Graph Neural Network and Single-Cell Foundation Model. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.16.628715).
329. Kosuge, M., et al. Landscape of evolutionary arms races between transposable elements and KRAB-ZFP family. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-73752-7).
330. Kuga, T., et al. Enhanced GATA4 expression in senescent systemic lupus erythematosus monocytes promotes high levels of IFNα production. __Front Immunol__ (2024). [Link](https://doi.org/doi:10.3389/FIMMU.2024.1320444).
331. Lambourne, L., et al. Widespread variation in molecular interactions and regulatory properties among transcription factor isoforms. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.03.12.584681).
332. Lee, PW.T.., et al. Hypoxia- and Postirradiation reoxygenation-induced HMHA1/ARHGAP45 expression contributes to cancer cell invasion in a HIF-dependent manner. __Br J Cancer__ (2024). [Link](https://doi.org/doi:10.1038/S41416-024-02691-X).
333. Leonard, MR.., et al. Aiolos promotes CXCR3 expression on TH1 cells via positive regulation of IFNγ/STAT1 signaling. __JCI Insight__ (2024). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.180287).
334. Li-Bao, L., et al. Regulation of Myc transcription by an enhancer cluster dedicated to pluripotency and early embryonic expression. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-48258-5).
335. Li, C., et al. Mi-2β promotes immune evasion in melanoma by activating EZH2 methylation. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-46422-5).
336. Li, J., et al. Stabilization of KPNB1 by deubiquitinase USP7 promotes glioblastoma progression through the YBX1-NLGN3 axis. __J Exp Clin Cancer Res__ (2024). [Link](https://doi.org/doi:10.1186/S13046-024-02954-8).
337. Li, JN., et al. SnoRNA U50A mediates everolimus resistance in breast cancer through mTOR downregulation. __Transl Oncol__ (2024). [Link](https://doi.org/doi:10.1016/J.TRANON.2024.102062).
338. Li, R., et al. Mef2c Exacerbates Neuron Necroptosis via Modulating Alternative Splicing of Cflar in Ischemic Stroke With Hyperlipidemia. __CNS Neurosci Ther__ (2024). [Link](https://doi.org/doi:10.1111/CNS.70144).
339. Li, X., et al. LINE-1 transcription activates long-range gene expression. __Nat Genet__ (2024). [Link](https://doi.org/doi:10.1038/S41588-024-01789-5).
340. Li, XC., et al. TF-High-Evolutionary: in vivo mutagenesis of gene regulatory networks for the study of the genetics and evolution of the Drosophila regulatory genome. __Mol Biol Evol__ (2024). [Link](https://doi.org/doi:10.1093/MOLBEV/MSAE167).
341. Liao, TT., et al. Hypoxia-induced long non-coding RNA HIF1A‑AS2 regulates stability of MHC class I protein in head and neck cancer. __Cancer Immunol Res__ (2024). [Link](https://doi.org/doi:10.1158/2326-6066.CIR-23-0622).
342. Liguoro, D., et al. The MITF/mir-579-3p regulatory axis dictates BRAF-mutated melanoma cell fate in response to MAPK inhibitors. __Cell Death Dis__ (2024). [Link](https://doi.org/doi:10.1038/S41419-024-06580-2).
343. Lim, CH., et al. ER Stress-Activated HSF1 Governs Cancer Cell Resistance to USP7 Inhibitor-Based Chemotherapy through the PERK Pathway. __Int J Mol Sci__ (2024). [Link](https://doi.org/doi:10.3390/IJMS25052768).
344. Lin, HY., et al. The TLK-ASF1 histone chaperone pathway plays a critical role in IL-1b-mediated AML progression. __Blood__ (2024). [Link](https://doi.org/doi:10.1182/BLOOD.2023022079).
345. Lin, SC., et al. Un-methylation of NUDT21 represses docosahexaenoic acid biosynthesis contributing to enzalutamide resistance in prostate cancer. __Drug Resist Update__ (2024). [Link](https://doi.org/doi:10.1016/J.DRUP.2024.101144).
346. Lin, YS., et al. Overexpression of NUDT16L1 sustains proper function of mitochondria and leads to ferroptosis insensitivity in colorectal cancer. __Redox Biol__ (2024). [Link](https://doi.org/doi:10.1016/J.REDOX.2024.103358).
347. Liu, K., et al. HMGA2 overexpression activates IGF2BP2 to stabilize APLP2 via m6A modification and promote pancreatic cancer progression. __Heliyon__ (2024). [Link](https://doi.org/doi:10.1016/J.HELIYON.2024.E27268).
348. Liu, N., et al. RBM39 enhances cholangiocarcinoma growth through EZH2-mediated WNT7B/β-catenin pathway. __Cell Mol Gastroenterol Hepatol__ (2024). [Link](https://doi.org/doi:10.1016/J.JCMGH.2024.101404).
349. Liu, X., et al. Discovering common pathogenetic processes between tuberculosis and COVID-19 by bioinformatics and system biology approach. __Heliyon__ (2024). [Link](https://doi.org/doi:10.1016/J.HELIYON.2024.E28664).
350. Liu, Y., et al. ETS1-mediated Regulation of SOAT1 Enhances the Malignant Phenotype of Oral Squamous Cell Carcinoma and Induces Tumor-associated Macrophages M2-like Polarization. __Int J Biol Sci__ (2024). [Link](https://doi.org/doi:10.7150/IJBS.93815).
351. Lo Sardo, F., et al. The oncogenic axis YAP/MYC/EZH2 impairs PTEN tumor suppression activity enhancing lung tumorigenicity. __Cell Death Discov__ (2024). [Link](https://doi.org/doi:10.1038/S41420-024-02216-8).
352. Lodi, MK., et al. COFFEE: Consensus Single Cell-Type Specific Inference for Gene Regulatory Networks. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.05.574445).
353. Lorenzo, J.P., et al. APOBEC2 safeguards skeletal muscle cell fate through binding chromatin and regulating transcription of non-muscle genes during myoblast differentiation. __Proc Natl Acad Sci U S A__ (2024). [Link](https://doi.org/doi:10.1073/PNAS.2312330121).
354. Loupe, JM.., et al. Multiomic profiling of transcription factor binding and function in human brain. __Nat Neurosci__ (2024). [Link](https://doi.org/doi:10.1038/S41593-024-01658-8).
355. Lu, F., et al. Oncogenic Gene CNOT7 Promotes Progression and Induces Poor Prognosis of Glioma. __Mol Biotechnol__ (2024). [Link](https://doi.org/doi:10.1007/S12033-024-01223-5).
356. Lu, Z., et al. Assessing next-generation sequencing-based computational methods for predicting transcriptional regulators with query gene sets. __Brief Bioinform__ (2024). [Link](https://doi.org/doi:10.1093/BIB/BBAE366).
357. Lu, Z., et al. Assessing NGS-based computational methods for predicting transcriptional regulators with query gene sets. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.02.01.578316).
358. Lu, Z., et al. BIT: Bayesian Identification of Transcriptional Regulators. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.06.02.597061).
359. M, A., et al. Chem-CRISPR/dCas9FCPF: a platform for chemically induced epigenome editing. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE798).
360. M, L., et al. TransmiR v3.0: an updated transcription factor-microRNA regulation database. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE1081).
361. M, SP., et al. Genomic profiling of six human somatic histone H1 variants denotes that H1X accumulates at recently incorporated transposable elements. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE014).
362. Ma, W., et al. Orphan Nuclear Receptor NR4A3 Promotes Vascular Calcification via Histone Lactylation. __Circ Res__ (2024). [Link](https://doi.org/doi:10.1161/CIRCRESAHA.123.323699).
363. MacFawn, I., et al. Grainyhead-like-2, an epithelial master programmer, promotes interferon induction and suppresses breast cancer recurrence. __Mol Immunol__ (2024). [Link](https://doi.org/doi:10.1016/J.MOLIMM.2024.04.012).
364. Mahmood, N., et al. Methyl-CpG binding domain protein 2 (Mbd2) drives breast cancer progression through the modulation of epithelial-to-mesenchymal transition. __Exp Mol Med__ (2024). [Link](https://doi.org/doi:10.1038/S12276-024-01205-2).
365. Mahmood, N., et al. The ISR downstream target ATF4 represses long-term memory in a cell type-specific manner. __Proc Natl Acad Sci U S A__ (2024). [Link](https://doi.org/doi:10.1073/PNAS.2407472121).
366. Malekpour, SA., et al. Single-cell multi-omics analysis identifies context-specific gene regulatory gates and mechanisms. __Brief Bioinform__ (2024). [Link](https://doi.org/doi:10.1093/BIB/BBAE180).
367. Mannens, CC.A.., et al. Chromatin accessibility during human first-trimester neurodevelopment. __Nature__ (2024). [Link](https://doi.org/doi:10.1038/S41586-024-07234-1).
368. Mannion, AJ.., et al. Regulation of YAP Promotor Accessibility in Endothelial Mechanotransduction. __Arterioscler Thromb Vasc Biol__ (2024). [Link](https://doi.org/doi:10.1161/ATVBAHA.123.320300).
369. Martin, A., et al. IFN-γ licenses normal and pathogenic ALPK1/TIFA pathway in human monocytes. __iScience__ (2024). [Link](https://doi.org/doi:10.1016/J.ISCI.2024.111563).
370. Matsuoka, Y., et al. Expression Profiles of Brain-Derived Neurotrophic Factor Splice Variants in the Hippocampus of Alzheimer's Disease Model Mouse. __Biol Pharm Bull__ (2024). [Link](https://doi.org/doi:10.1248/BPB.B24-00446).
371. Miao, M., et al. IL-13 facilitates ferroptotic death in asthmatic epithelial cells via SOCS1-mediated ubiquitinated degradation of SLC7A11. __Redox Biol__ (2024). [Link](https://doi.org/doi:10.1016/J.REDOX.2024.103100).
372. Minaeva, M., et al. Specifying cellular context of transcription factor regulons for exploring context-specific gene regulation programs. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2023.12.31.573765).
373. Mokhtaridoost, M., et al. Inter-chromosomal contacts demarcate genome topology along a spatial gradient. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-53983-Y).
374. Montero-Hidalgo, AJ.., et al. SRSF6 modulates histone-chaperone HIRA splicing to orchestrate AR and E2F activity in prostate cancer. __Sci Adv__ (2024). [Link](https://doi.org/doi:10.1126/SCIADV.ADO8231).
375. Moreira, P., et al. Nuclear factor Y, a key player in neuronal gene regulation. __Sci Prog__ (2024). [Link](https://doi.org/doi:10.1177/00368504241264998).
376. Mukherjee, A., et al. DAB2IP Loss in Luminal A Breast Cancer Leads to NF-kB Associated Aggressive Oncogenic Phenotypes. __JCI Insight__ (2024). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.171705).
377. Murao, N., et al. The Derlin-1-Stat5b axis maintains homeostasis of adult hippocampal neurogenesis. __EMBO Rep__ (2024). [Link](https://doi.org/doi:10.1038/S44319-024-00205-7).
378. Mutnuru, SA., et al. PRMT5-mediated histone methylation regulates alternative splicing via MECP2-PTBP1 to promote EMT in breast cancer hypoxia. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.10.16.618678).
379. Nakamura, T., et al. Topologically associating domains define the impact of de novo promoter variants on autism spectrum disorder risk. __Cell Genom__ (2024). [Link](https://doi.org/doi:10.1016/J.XGEN.2024.100488).
380. Nakanishi, T., et al. Complex intrinsic abnormalities in osteoblast lineage cells of X-linked hypophosphatemia: Analysis of human iPS cell models generated by CRISPR/Cas9-mediated gene ablation. __Bone__ (2024). [Link](https://doi.org/doi:10.1016/J.BONE.2024.117044).
381. Nguyen, A., et al. Jumonji histone demethylases are therapeutic targets in small cell lung cancer. __Oncogene__ (2024). [Link](https://doi.org/doi:10.1038/S41388-024-03125-X).
382. Niimi, K., et al. Macrophages play a crucial role in vascular smooth muscle cell coverage. __Development__ (2024). [Link](https://doi.org/doi:10.1242/DEV.203080).
383. Nornes, S., et al. A catalogue of verified and characterized arterial enhancers for key arterial identity genes. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.04.30.591717).
384. Nose, K., et al. γ-Glutamylcyclotransferase is transcriptionally regulated by c-Jun and controls proliferation of glioblastoma stem cells through Notch1 levels. __Cancer Gene Ther__ (2024). [Link](https://doi.org/doi:10.1038/S41417-024-00835-Y).
385. Ochiai, N., et al. The pivotal role of the Hes1/Piezo1 pathway in the pathophysiology of glucocorticoid-induced osteoporosis. __JCI Insight__ (2024). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.179963).
386. Oguchi, A., et al. An atlas of transcribed enhancers across helper T cell diversity for decoding human diseases. __Science__ (2024). [Link](https://doi.org/doi:10.1126/SCIENCE.ADD8394).
387. Ohishi, H., et al. Transcription-coupled changes in genomic region proximities during transcriptional bursting. __Sci Adv__ (2024). [Link](https://doi.org/doi:10.1126/SCIADV.ADN0020).
388. Olszewska, A.M.., et al. Genomic and non-genomic action of vitamin D on ion channels - Targeting mitochondria. __Mitochondrion__ (2024). [Link](https://doi.org/doi:10.1016/J.MITO.2024.101891).
389. Pandey, A., et al. CTCF and BORIS-mediated Autophagy regulation via Alternative splicing of BNIP3L in Breast Cancer. __J Biol Chem__ (2024). [Link](https://doi.org/doi:10.1016/J.JBC.2024.107416).
390. Paolis, VD., et al. An antisense-long-noncoding-RNA modulates p75NTR expression levels during neuronal polarization. __iScience__ (2024). [Link](https://doi.org/doi:10.1016/J.ISCI.2024.111566).
391. Pathania, AS.., et al. SAP30, a novel autophagy regulatory gene in neuroblastoma. __Mol Ther Oncol__ (2024). [Link](https://doi.org/doi:10.1016/J.OMTON.2024.200916).
392. Peña-Martínez, EG.., et al. Cardiovascular Disease-Associated Non-Coding Variants Disrupt GATA4-DNA Binding and Regulatory Functions. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.09.19.613959).
393. Peng, G., et al. TSCRE: a comprehensive database for tumor-specific cis-regulatory elements. __NAR Cancer__ (2024). [Link](https://doi.org/doi:10.1093/NARCAN/ZCAD063).
394. Peng, Q., et al. Analysis of blood methylation quantitative trait loci in East Asians reveals ancestry-specific impacts on complex traits. __Nat Genet__ (2024). [Link](https://doi.org/doi:10.1038/S41588-023-01494-9).
395. Petersen, RM.., et al. Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.08.23.609412).
396. Pollin, G., et al. Ehmt2 inactivation in pancreatic epithelial cells shapes the transcriptional landscape and inflammation response of the whole pancreas. __Front Genet__ (2024). [Link](https://doi.org/doi:10.3389/FGENE.2024.1412767).
397. Qin, L., et al. Dissecting the functions and regulatory mechanisms of disulfidoptosis-related RPN1 in pan-cancer: modulation of immune microenvironment and cellular senescence. __Front Immunol__ (2024). [Link](https://doi.org/doi:10.3389/FIMMU.2024.1512445).
398. Qin, L., et al. Maternal NO2 exposure and fetal growth restriction: Hypoxia transmission and lncRNAs-proinflammation-mediated abnormal hematopoiesis. __Proc Natl Acad Sci U S A__ (2024). [Link](https://doi.org/doi:10.1073/PNAS.2409597121).
399. Rachid Zaim, S., et al. MOCHA's advanced statistical modeling of scATAC-seq data enables functional genomic inference in large human cohorts. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-50612-6).
400. Rajendra, KC., et al. Evidence of RNA polymerase III recruitment and transcription at protein-coding gene promoters. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.06.08.598009).
401. Ramirez, JM., et al. The molecular impact of cigarette smoking resembles aging across tissues. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.03.14.585016).
402. Refael, T., et al. An i-motif-regulated enhancer, eRNA and adjacent lncRNA affect Lhb expression through distinct mechanisms in a sex-specific context. __Cell Mol Life Sci__ (2024). [Link](https://doi.org/doi:10.1007/S00018-024-05398-7).
403. Ren, Y., et al. Hyper-methylation and DNMT3A mediated LTC4S downregulation promoted lung adenocarcinoma tumorigenesis via mTORC1 signaling pathway. __Heliyon__ (2024). [Link](https://doi.org/doi:10.1016/j.heliyon.2024.e33203).
404. Rhizobium, GE. The 22nd annual Nucleic Acids Research Web Server Issue 2024. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE492).
405. Ribeiro, JH., et al. A human-specific, concerted repression of microcephaly genes contributes to radiation-induced growth defects in forebrain organoids. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.06.27.600564).
406. Ruscitti, C., et al. Recruited atypical Ly6G+ macrophages license alveolar regeneration after lung injury. __Sci Immunol__ (2024). [Link](https://doi.org/doi:10.1126/SCIIMMUNOL.ADO1227).
407. Ryu, JE., et al. Circadian regulation of endoplasmic reticulum calcium response in mouse cultured astrocytes. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.02.02.578616).
408. Safaeesirat, A., et al. Inference of Transcriptional Regulation From STARR-seq Data. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.03.06.583826).
409. Sakai, S., et al. In vivo transition in chromatin accessibility during differentiation of deep-layer excitatory neurons in the neocortex. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.08.26.609258).
410. Sarkar, D., et al. PARylation of GCN5 by PARP1 mediates its recruitment to DSBs and facilitates both HR and NHEJ Repair. __Cell Mol Life Sci__ (2024). [Link](https://doi.org/doi:10.1007/S00018-024-05469-9).
411. Sarott, RC., et al. Relocalizing transcriptional kinases to activate apoptosis. __Science__ (2024). [Link](https://doi.org/doi:10.1126/SCIENCE.ADL5361).
412. Sato, Y., et al. Calcineurin/NFATc1 pathway represses cellular cytotoxicity by modulating histone H3 expression. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-65769-9).
413. Scarafía, MíA., et al. SHEN-LONG, a novel cardiac regulatory locus, regulates cardiac master transcription factor NKX2-5 in pluripotent stem cell derived cardiomyocytes. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.24.630259).
414. Sekiguchi, K., et al. Rho-associated, coiled-coil-containing protein kinase 2 regulates expression of mineralocorticoid receptor to mediate sodium reabsorption in mice. __Biochem Biophys Res Commun__ (2024). [Link](https://doi.org/doi:10.1016/J.BBRC.2024.150874).
415. Seo, SH., et al. C/EBPβ Regulates HIF-1α-Driven Invasion of Non-Small-Cell Lung Cancer Cells. __Biomolecules__ (2024). [Link](https://doi.org/doi:10.3390/BIOM15010036).
416. Sergi, B., et al. Epidrug Screening Identifies Type I PRMT Inhibitors as Modulators of Lysosomal Exocytosis and Drug Sensitivity in Cancers. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.08.26.609671).
417. Shen, X., et al. TRIM33 plays a critical role in regulating dendritic cell differentiation and homeostasis by modulating Irf8 and Bcl2l11 transcription. __Cell Mol Immunol__ (2024). [Link](https://doi.org/doi:10.1038/S41423-024-01179-1).
418. Shrestha, R., et al. An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression. __Cancer Res__ (2024). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-24-0890).
419. Singh, DK.., et al. MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF. __Mol Cell Biol__ (2024). [Link](https://doi.org/doi:10.1080/10985549.2024.2383773).
420. Siraj, L., et al. Functional dissection of complex and molecular trait variants at single nucleotide resolution. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.05.05.592437).
421. Sizer, RE., et al. Selective Occupation by E2F and RB of Loci Expressed by RNA Polymerase III. __Cancers__ (2024). [Link](https://doi.org/doi:10.3390/CANCERS16030481).
422. Sonnemann, HM.., et al. Placental co-transcriptional activator Vestigial-like 1 (VGLL1) drives tumorigenesis via increasing transcription of proliferation and invasion genes. __Front Oncol__ (2024). [Link](https://doi.org/doi:10.3389/FONC.2024.1403052).
423. Spence, JP.., et al. Specificity, length, and luck: How genes are prioritized by rare and common variant association studies. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.12.628073).
424. Stadler, M., et al. asteRIa enables robust interaction modeling between chromatin modifications and epigenetic readers. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.03.15.585146).
425. Statello, L., et al. The chromatin-associated lncREST ensures effective replication stress response by promoting the assembly of fork signaling factors. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-45183-5).
426. Su, G., et al. Inferring gene regulatory networks by hypergraph variational autoencoder. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.04.01.586509).
427. Suwa, T., et al. UPR-induced intracellular C5aR1 promotes adaptation to the hypoxic tumour microenvironment by regulating tumour cell fate. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.09.27.615431).
428. Suzuki, H., et al. Mutant α-synuclein causes death of human cortical neurons via ERK1/2 and JNK activation. __Mol Brain__ (2024). [Link](https://doi.org/doi:10.1186/S13041-024-01086-6).
429. Szántó, PéR.., et al. Thermosensory neurons control genetic inheritance through regulation of germline transposons. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.02.626368).
430. T, N., et al. Update of the FANTOM web resource: enhancement for studying noncoding genomes. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE1047).
431. Tabata, S., et al. NFκB dynamics-dependent epigenetic changes modulate inflammatory gene expression and induce cellular senescence. __FEBS J__ (2024). [Link](https://doi.org/doi:10.1111/FEBS.17227).
432. Takahashi, M., et al. Kaempferol Exerts Anti-Inflammatory Effects by Accelerating Treg Development via Aryl Hydrocarbon Receptor-Mediated and PU.1/IRF4-Dependent Transactivation of the Aldh1a2/Raldh2 Gene in Dendritic Cells. __Allergy__ (2024). [Link](https://doi.org/doi:10.1111/ALL.16410).
433. Takasugi, M., et al. An atlas of the aging mouse proteome reveals the features of age-related post-transcriptional dysregulation. __Nat Commun__ (2024). [Link](https://doi.org/doi:10.1038/S41467-024-52845-X).
434. Takhaveev, V., et al. Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.07.21.604463).
435. Tan, H., et al. TSPYL1 as a Critical Regulator of TGFβ Signaling through Repression of TGFBR1 and TSPYL2. __Adv Sci__ (2024). [Link](https://doi.org/doi:10.1002/ADVS.202306486).
436. Tang, L., et al. Targeting autophagy overcomes cancer-intrinsic resistance to CAR-T immunotherapy in B-cell malignancies. __Cancer Commun__ (2024). [Link](https://doi.org/doi:10.1002/CAC2.12525).
437. Thapa, R., et al. The Long Non-Coding RNA Gene AC027288.3 Plays a Role in Human Endometrial Stromal Fibroblast Decidualization. __Cells__ (2024). [Link](https://doi.org/doi:10.3390/CELLS13090778).
438. Tirlet, Y., et al. Generic and queryable data integration schema for transcriptomics and epigenomics studies. __Comput Struct Biotechnol J__ (2024). [Link](https://doi.org/doi:10.1016/j.csbj.2024.11.022).
439. Tomita-Naito, S., et al. Insidious chromatin change with a propensity to exhaust intestinal stem cells during aging. __iScience__ (2024). [Link](https://doi.org/doi:10.1016/J.ISCI.2024.110793).
440. Tomiyasu, H., et al. FOXO1 promotes cancer cell growth through MDM2-mediated p53 degradation. __J Biol Chem__ (2024). [Link](https://doi.org/doi:10.1016/J.JBC.2024.107209).
441. Ueda, H., et al. Glycolysis-mTORC1 crosstalk drives proliferation of patient-derived endometrial cancer spheroid cells with ALDH activity. __Cell Death Discov__ (2024). [Link](https://doi.org/doi:10.1038/S41420-024-02204-Y).
442. Ueno, T., et al. Myc upregulates Ggct, γ-glutamylcyclotransferase to promote development of p53-deficient osteosarcoma. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.03.18.585455).
443. Ulfig, A., et al. Redox heterogeneity in mouse embryonic stem cells individualizes cell fate decisions. __Dev Cell__ (2024). [Link](https://doi.org/doi:10.1016/J.DEVCEL.2024.07.008).
444. Umhoefer, JM.., et al. Deciphering regulation of FOXP3 expression in human conventional T cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.08.30.610436).
445. Uota, A., et al. ARMC5 selectively degrades SCAP-free SREBF1 and is essential for fatty acid desaturation in adipocytes. __J Biol Chem__ (2024). [Link](https://doi.org/doi:10.1016/J.JBC.2024.107953).
446. V, L., et al. Integrated multi-omics analysis of PBX1 in mouse adult neural stem- and progenitor cells identifies a transcriptional module that functionally links PBX1 to TCF3/4. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE864).
447. Vareli, A., et al. Systems biology-enabled targeting of NF-κB and BCL2 overcomes microenvironment-mediated BH3-mimetic resistance in DLBCL. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.11.30.626166).
448. Vishnevsky, OV.., et al. Peak Scores Significantly Depend on the Relationships between Contextual Signals in ChIP-Seq Peaks. __Int J Mol Sci__ (2024). [Link](https://doi.org/doi:10.3390/IJMS25021011).
449. Wan, L., et al. Identification of a molecular network regulated by multiple ASD high risk genes. __Hum Mol Genet__ (2024). [Link](https://doi.org/doi:10.1093/HMG/DDAE058).
450. Wan, R., et al. Unveiling gene regulatory networks during cellular state transitions without linkage across time points. __Sci Rep__ (2024). [Link](https://doi.org/doi:10.1038/S41598-024-62850-1).
451. Wan, Y., et al. Brusatol induces ferroptosis to inhibit hepatocellular carcinoma progression by targeting ATF3. __Chem Biol Drug Des__ (2024). [Link](https://doi.org/doi:10.1111/CBDD.14565).
452. Wang, A., et al. KLF13 promotes SLE pathogenesis by modifying chromatin accessibility of key proinflammatory cytokine genes. __Commun Biol__ (2024). [Link](https://doi.org/doi:10.1038/S42003-024-07099-0).
453. Wang, C., et al. The Oncogenic Role of TNFRSF12A in Colorectal Cancer and Pan-Cancer Bioinformatics Analysis. __Cancer Res Treat__ (2024). [Link](https://doi.org/doi:10.4143/CRT.2024.408).
454. Wang, J. TFTF: An R-Based Integrative Tool for Decoding Human Transcription Factor–Target Interactions. __Biomolecules__ (2024). [Link](https://doi.org/doi:10.3390/BIOM14070749).
455. Wang, JC., et al. GRACE: Unveiling Gene Regulatory Networks With Causal Mechanistic Graph Neural Networks in Single-Cell RNA-Sequencing Data. __IEEE Trans Neural Netw Learn Syst__ (2024). [Link](https://doi.org/doi:10.1109/TNNLS.2024.3412753).
456. Wang, W., et al. RegVelo: gene-regulatory-informed dynamics of single cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.11.627935).
457. Wang, W., et al. Scalable identification of lineage-specific gene regulatory networks from metacells with NetID. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.09.08.611796).
458. Watanabe, Y., et al. ALS-associated RNA binding proteins converge on UNC13A transcription through regulation of REST. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.10.22.619761).
459. Wei, T., et al. FOXM1/DEPDC1 feedback loop promotes hepatocarcinogenesis and represents promising targets for cancer therapy. __Cancer Sci__ (2024). [Link](https://doi.org/doi:10.1111/CAS.16273).
460. Wei, X., et al. Competitive-like binding between carbon black and CTNNB1 to ΔNp63 interpreting the abnormal respiratory epithelial repair after injury. __Sci Total Environ__ (2024). [Link](https://doi.org/doi:10.1016/J.SCITOTENV.2024.172652).
461. Wu, B., et al. Overloading And unpacKing (OAK) - droplet-based combinatorial indexing for ultra-high throughput single-cell multiomic profiling. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.01.23.576918).
462. Wu, F., et al. NUP43 promotes PD-L1/nPD-L1/PD-L1 feedback loop via TM4SF1/JAK/STAT3 pathway in colorectal cancer progression and metastatsis. __Cell Death Discov__ (2024). [Link](https://doi.org/doi:10.1038/S41420-024-02025-Z).
463. Xia, K., et al. SETDB1 targeting SESN2 regulates mitochondrial damage and oxidative stress in renal ischemia-reperfusion injury. __BMC Biol__ (2024). [Link](https://doi.org/doi:10.1186/S12915-024-02048-Z).
464. Xie, M., et al. CBX3 Downregulates HLTF to Activate PI3K/AKT Signaling Promoting Cholangiocarcinoma. __Adv Biol (Weinh)__ (2024). [Link](https://doi.org/doi:10.1002/ADBI.202400413).
465. Yang, W., et al. Revealing long-range heterogeneous organization of nucleoproteins with N6-methyladenine footprinting. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.12.05.627052).
466. Ye, P., et al. Genomics-FM: Universal Foundation Model for Versatile and Data-Efficient Functional Genomic Analysis. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.07.16.603653).
467. Yi, W., et al. Molecular Signatures of CB-6644 Inhibition of the RUVBL1/2 Complex in Multiple Myeloma. __Int J Mol Sci__ (2024). [Link](https://doi.org/doi:10.3390/IJMS25169022).
468. Yoshioka, H., et al. Alteration of Gene Expression in Pathological Keratinization of the Ocular Surface. __Invest Ophthalmol Vis Sci__ (2024). [Link](https://doi.org/doi:10.1167/IOVS.65.6.37).
469. YOSHIZAWA, MASATO., et al. Depletion of DNTTIP2 Induces Cell Cycle Arrest in Pancreatic Cancer Cells. __Cancer Genomics Proteomics__ (2024). [Link](https://doi.org/doi:10.21873/CGP.20426).
470. Yu, C., et al. PRDM1 promotes nucleus pulposus cell pyroptosis leading to intervertebral disc degeneration via activating CASP1 transcription. __Cell Biol Toxicol__ (2024). [Link](https://doi.org/doi:10.1007/S10565-024-09932-Y).
471. Yu, X., et al. Time-course swRNA-seq uncovers a hierarchical gene regulatory network in controlling the response-repair-remodeling after wounding. __Commun Biol__ (2024). [Link](https://doi.org/doi:10.1038/S42003-024-06352-W).
472. Z, Z., et al. ChIP-Atlas 3.0: a data-mining suite to explore chromosome architecture together with large-scale regulome data. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE358).
473. Zandigohar, M., et al. Transcription Factor Activity Regulating Macrophage Heterogeneity during Skin Wound Healing. __J Immunol__ (2024). [Link](https://doi.org/doi:10.4049/JIMMUNOL.2400172).
474. Zaretsky, A., et al. SIRT6-dependent functional switch via K494 modifications of RE-1 silencing transcription factor. __Cell Death Dis__ (2024). [Link](https://doi.org/doi:10.1038/S41419-024-07160-0).
475. Zee, A., et al. The Circadian Isoform Landscape of Mouse Livers. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.10.16.618716).
476. Zeng, Y., et al. CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.06.04.597369).
477. Zhang, A., et al. FADS1 inhibition protects retinal pigment epithelium cells from ferroptosis in age related macular degeneration. __Eur J Pharmacol__ (2024). [Link](https://doi.org/doi:10.1016/J.EJPHAR.2024.177227).
478. Zhang, D., et al. LogicGep: Boolean networks inference using symbolic regression from time-series transcriptomic profiling data. __Brief Bioinform__ (2024). [Link](https://doi.org/doi:10.1093/BIB/BBAE286).
479. Zhang, G., et al. TRAPT: A multi-stage fused deep learning framework for transcriptional regulators prediction via integrating large-scale epigenomic data. __bioRxiv__ (2024). [Link](https://doi.org/doi:10.1101/2024.05.17.594242).
480. Zhang, Q., et al. ZDHHC1 downregulates LIPG and inhibits colorectal cancer growth via IGF2BP1 Palmitoylation. __Cancer Gene Ther__ (2024). [Link](https://doi.org/doi:10.1038/S41417-024-00808-1).
481. Zhang, S., et al. Endogenous sulfur dioxide deficiency as a driver of cardiomyocyte senescence through abolishing sulphenylation of STAT3 at cysteine 259. __Redox Biol__ (2024). [Link](https://doi.org/doi:10.1016/J.REDOX.2024.103124).
482. Zhang, W., et al. HDAC7 is a potential therapeutic target in acute erythroid leukemia. __Leukemia__ (2024). [Link](https://doi.org/doi:10.1038/S41375-024-02394-5).
483. Zhang, Y., et al. Trichosanthin elicits antitumor activity via MICU3 mediated mitochondria calcium influx. __J Adv Res__ (2024). [Link](https://doi.org/doi:10.1016/J.JARE.2024.11.001).
484. Zhao, X., et al. Identification the immune related marker genes and transcription-factor network in ruptured cerebral aneurysms using bioinformatics analysis and machine-learning strategies. __Biomed Signal Process Control__ (2024). [Link](https://doi.org/doi:10.1016/J.BSPC.2023.105611).
485. Zhao, ZP., et al. Modulating Treg cell activity in prostate cancer via chitosan nanoparticles loaded with si-BATF/PRDM1. __Int Immunopharmacol__ (2024). [Link](https://doi.org/doi:10.1016/J.INTIMP.2024.113445).
486. Zhou, X., et al. LncPepAtlas: a comprehensive resource for exploring the translational landscape of long non-coding RNAs. __Nucleic Acids Res__ (2024). [Link](https://doi.org/doi:10.1093/NAR/GKAE905).
487. Zhou, X., et al. TFTG: A comprehensive database for human transcription factors and their targets. __Comput Struct Biotechnol J__ (2024). [Link](https://doi.org/doi:10.1016/J.CSBJ.2024.04.036).
488. Zhou, Z., et al. FBW7/GSK3β mediated degradation of IGF2BP2 inhibits IGF2BP2-SLC7A5 positive feedback loop and radioresistance in lung cancer. __J Exp Clin Cancer Res__ (2024). [Link](https://doi.org/doi:10.1186/S13046-024-02959-3).
489. Zhu, X., et al. Genomic and transcriptomic features of androgen receptor signaling inhibitor resistance in metastatic castration-resistant prostate cancer. __J Clin Invest__ (2024). [Link](https://doi.org/doi:10.1172/JCI178604).
490. Abashkin, DA.., et al. ASCL1 Is Involved in the Pathogenesis of Schizophrenia by Regulation of Genes Related to Cell Proliferation, Neuronal Signature Formation, and Neuroplasticity. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS242115746).
491. Anzawa, H., et al. C4S DB: Comprehensive Collection and Comparison for ChIP-Seq Database. __J Mol Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.JMB.2023.168157).
492. Arimoto, K., et al. Expansion of interferon inducible gene pool via USP18 inhibition promotes cancer cell pyroptosis. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-022-35348-5).
493. Armaos, A., et al. The PENGUIN approach to reconstruct protein interactions at enhancer-promoter regions and its application to prostate cancer. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-43767-1).
494. Arriojas, A., et al. A Bayesian Noisy Logic Model for Inference of Transcription Factor Activity from Single Cell and Bulk Transcriptomic Data. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.03.539308).
495. Asavasupreechar, T., et al. PRB inhibited cell proliferation through let-7b-E2F1 in breast cancer. __Endocr Relat Cancer__ (2023). [Link](https://doi.org/doi:10.1530/ERC-22-0204).
496. Azzi, A., et al. The circadian clock protein Cryptochrome 1 is a direct target and feedback regulator of the Hippo pathway. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.107449).
497. Badia-i-Mompel, P., et al. Gene regulatory network inference in the era of single-cell multi-omics. __Nat Rev Genet__ (2023). [Link](https://doi.org/doi:10.1038/S41576-023-00618-5).
498. Bai, Y., et al. Trans-omic analysis reveals opposite metabolic dysregulation between feeding and fasting in liver associated with obesity. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.11.15.567316).
499. Barvaux, S., et al. SinCMat: A single-cell-based method for predicting functional maturation transcription factors. __Stem Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.STEMCR.2023.12.006).
500. Büchel, J., et al. Single-nucleotide-resolution genomic maps of O6-methylguanine from the glioblastoma drug temozolomide. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.12.12.571283).
501. Beknazarov, N., et al. Z-Flipons conserved between human and mouse are associated with increased transcription initiation rates. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.10.31.564984).
502. Bertonnier-Brouty, L., et al. E2F transcription factors promote tumorigenicity in pancreatic ductal adenocarcinoma. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.11.10.566445).
503. Bhardwaj, V., et al. HIV-1 Vpr induces ciTRAN to prevent transcriptional repression of the provirus. __Sci Adv__ (2023). [Link](https://doi.org/doi:10.1126/SCIADV.ADH9170).
504. Bhin, J., et al. MYC is a clinically significant driver of mTOR inhibitor resistance in breast cancer. __J Exp Med__ (2023). [Link](https://doi.org/doi:10.1084/JEM.20211743).
505. Bi, X., et al. Pan-cancer analyses reveal multi-omic signatures and clinical implementations of the forkhead-box gene family. __Cancer Med__ (2023). [Link](https://doi.org/doi:10.1002/CAM4.6312).
506. Bossardi Ramos, R., et al. Shock drives a STAT3 and JunB-mediated coordinated transcriptional and DNA methylation response in the endothelium. __J Cell Sci__ (2023). [Link](https://doi.org/doi:10.1242/JCS.261323).
507. BP, P., et al. Pcgf1 gene disruption reveals primary involvement of epigenetic mechanism in neuronal subtype specification in the enteric nervous system. __Dev Growth Differ__ (2023). [Link](https://doi.org/doi:10.1111/DGD.12880).
508. Bu, Q., et al. Stanniocalcin 2 (STC2) is a potent biomarker of hepatocellular carcinoma with its expression being augmented in Nrf1α-deficient cells, but diminished in Nrf2-deficient cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.15.540796).
509. Bu, Q., et al. STC2 is a potential biomarker of hepatocellular carcinoma with its expression being upregulated in Nrf1α-deficient cells, but downregulated in Nrf2-deficient cells. __Int J Biol Macromol__ (2023). [Link](https://doi.org/doi:10.1016/J.IJBIOMAC.2023.127575).
510. Bustin, KA.., et al. Phenelzine-based probes reveal Secernin-3 is involved in thermal nociception. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.02.526866).
511. C, B., et al. Transmicron: accurate prediction of insertion probabilities improves detection of cancer driver genes from transposon mutagenesis screens. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAC1215).
512. Castillo, Hé., et al. Architecture and evolutionary conservation of Xenopus tropicalis osteoblast-specific regulatory regions shed light on bone diseases and early skeletal evolution. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.30.555543).
513. Chandra, A., et al. Quantitative control of Ets1 dosage by a multi-enhancer hub promotes Th1 cell differentiation and protects from allergic inflammation. __Immunity__ (2023). [Link](https://doi.org/doi:10.1016/J.IMMUNI.2023.05.004).
514. Chandra, O., et al. Patterns of transcription factor binding and epigenome at promoters allow interpretable predictability of multiple functions of non-coding and coding genes. __Comput Struct Biotechnol J__ (2023). [Link](https://doi.org/doi:10.1016/J.CSBJ.2023.07.014).
515. Chen, L., et al. NPM3 as an unfavorable prognostic biomarker involved in oncogenic pathways of lung adenocarcinoma via MYC translational activation. __Comb Chem High Throughput Screen__ (2023). [Link](https://doi.org/doi:10.2174/1386207326666230419080531).
516. Chen, W., et al. Mechanic Insight into the Distinct and Common Roles of Ovariectomy Versus Adrenalectomy on Adipose Tissue Remodeling in Female Mice. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24032308).
517. Chen, WC., et al. Genome-wide association study of esophageal squamous cell cancer identifies shared and distinct risk variants in African and Chinese populations. __Am J Hum Genet__ (2023). [Link](https://doi.org/doi:10.1016/J.AJHG.2023.08.007).
518. Chen, Y., et al. Canonical and nuclear mTOR specify distinct transcriptional programs in androgen-dependent prostate cancer cells. __Mol Cancer Res__ (2023). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-23-0087).
519. Cheng, R., et al. A Combinatorial Regulatory Platform Determines Expression of RNA Polymerase III Subunit RPC7α ( POLR3G) in Cancer. __Cancers__ (2023). [Link](https://doi.org/doi:10.3390/CANCERS15204995).
520. Chow, CC.T.., et al. ZBTB2 is Recruited to a Specific Subset of HIF-1 Target Loci to Facilitate Full Gene Expression Under Hypoxia. __J Mol Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.JMB.2023.168162).
521. Chow, CN., et al. Databases and prospects of dynamic gene regulation in eukaryotes: A mini review. __Comput Struct Biotechnol J__ (2023). [Link](https://doi.org/doi:10.1016/J.CSBJ.2023.03.032).
522. CN, C., et al. PlantPAN 4.0: updated database for identifying conserved non-coding sequences and exploring dynamic transcriptional regulation in plant promoters. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD945).
523. Dai, C., et al. Comprehensive analysis of the diagnostic and therapeutic value of the hypoxia-related gene PLAUR in the progression of atherosclerosis. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-35548-Z).
524. Diawara, M., et al. Regulatory mechanisms of SoxD transcription factors and their influences on male fertility. __Reprod Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.REPBIO.2023.100823).
525. Dong, L., et al. A functional crosstalk between the H3K9 methylation writers and their reader HP1 in safeguarding embryonic stem cell identity. __Stem Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.STEMCR.2023.08.004).
526. Escalante-Covarrubias, Q., et al. Time-of-day defines NAD+ efficacy to treat diet-induced metabolic disease by synchronizing the hepatic clock in mice. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-37286-2).
527. Etoh, K., et al. A web-based integrative transcriptome analysis, RNAseqChef, uncovers cell/tissue type-dependent action of sulforaphane. __J Biol Chem__ (2023). [Link](https://doi.org/doi:10.1016/J.JBC.2023.104810).
528. FC, Q., et al. SEanalysis 2.0: a comprehensive super-enhancer regulatory network analysis tool for human and mouse. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD408).
529. Feng, G., et al. CellPolaris: Decoding Cell Fate through Generalization Transfer Learning of Gene Regulatory Networks. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.25.559244).
530. Feng, K., et al. Gene regulatory network inference based on causal discovery integrating with graph neural network. __Quant Biol__ (2023). [Link](https://doi.org/doi:10.1002/QUB2.26).
531. Filipovic, D., et al. Interpretable predictive models of genome-wide aryl hydrocarbon receptor-DNA binding reveal tissue specific binding determinants. __Toxicol Sci__ (2023). [Link](https://doi.org/doi:10.1093/TOXSCI/KFAD094).
532. Francia, MG., et al. AKT1 induces Nanog promoter in a SUMOylation-dependent manner in different pluripotent contexts. __BMC Res Notes__ (2023). [Link](https://doi.org/doi:10.1186/S13104-023-06598-3).
533. Freeman, TF.., et al. Ribosome biogenesis disruption mediated chromatin structure changes revealed by SRAtac, a customizable end to end analysis pipeline for ATAC-seq. __BMC Genomics__ (2023). [Link](https://doi.org/doi:10.1186/S12864-023-09576-Y).
534. Frenkel, M., et al. Discovering chromatin dysregulation induced by protein-coding perturbations at scale. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.20.555752).
535. Fu, Y., et al. Fam72a functions as a cell-cycle-controlled gene during proliferation and antagonizes apoptosis through reprogramming PP2A substrates. __Dev Cell__ (2023). [Link](https://doi.org/doi:10.1016/J.DEVCEL.2023.02.006).
536. Gao, Z., et al. Phase Separation Modulates the Formation and Stabilities of DNA Guanine Quadruplex. __JACS Au__ (2023). [Link](https://doi.org/doi:10.1021/JACSAU.3C00106).
537. García‐Hernández, V., et al. β-Catenin activity induces an RNA biosynthesis program promoting therapy resistance in T-cell acute lymphoblastic leukemia. __EMBO Mol Med__ (2023). [Link](https://doi.org/doi:10.15252/EMMM.202216554).
538. Ghaibour, K., et al. An Efficient Protocol for CUT&RUN Analysis of FACS-Isolated Mouse Satellite Cells. __J Vis Exp__ (2023). [Link](https://doi.org/doi:10.3791/65215).
539. Goncalves, TM., et al. Unsupervised Machine Learning Identify Cis-Regulatory Network Involved in Neurodegeneration. __Alzheimers Dement__ (2023). [Link](https://doi.org/doi:10.1002/ALZ.082808).
540. Gong, B., et al. Epigenetic and transcriptional activation of the secretory kinase FAM20C as an oncogene in glioma. __J Genet Genom__ (2023). [Link](https://doi.org/doi:10.1016/J.JGG.2023.01.008).
541. Grey, W., et al. The CKS1/CKS2 Proteostasis Axis Is Crucial to Maintain Hematopoietic Stem Cell Function. __Hemasphere__ (2023). [Link](https://doi.org/doi:10.1097/HS9.0000000000000853).
542. Hagemann, S., et al. IGF2BP1 induces high-risk neuroblastoma and forms a druggable feedforward loop with MYCN promoting 17q oncogene expression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.18.533148).
543. Hagemann, S., et al. IGF2BP1 induces neuroblastoma via a druggable feedforward loop with MYCN promoting 17q oncogene expression. __Mol Cancer__ (2023). [Link](https://doi.org/doi:10.1186/S12943-023-01792-0).
544. Hamba, Y., et al. Topologically associating domain underlies tissue specific expression of long intergenic non-coding RNAs. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.106640).
545. Hanaki, S., et al. Impact of FKBP52 on cell proliferation and hormone-dependent cancers. __Cancer Sci__ (2023). [Link](https://doi.org/doi:10.1111/CAS.15811).
546. Harada, S., et al. The Lysophospholipase PNPLA7 Controls Hepatic Choline and Methionine Metabolism. __Biomolecules__ (2023). [Link](https://doi.org/doi:10.3390/BIOM13030471).
547. Herbert, A. Flipons and small RNAs accentuate the asymmetries of pervasive transcription by the reset and sequence-specific microcoding of promoter conformation. __J Biol Chem__ (2023). [Link](https://doi.org/doi:10.1016/J.JBC.2023.105140).
548. Hino, S., et al. Chromatin Immunoprecipitation Sequencing (ChIP-seq) for Detecting Histone Modifications and Modifiers. __Methods Mol Biol__ (2023). [Link](https://doi.org/doi:10.1007/978-1-0716-2724-2_4).
549. Hou, Y., et al. Transcriptome sequencing analysis reveals a potential role of lncRNA NONMMUT058932.2 and NONMMUT029203.2 in abnormal myelin development of male offspring following prenatal PM2.5 exposure. __Sci Total Environ__ (2023). [Link](https://doi.org/doi:10.1016/J.SCITOTENV.2023.165004).
550. Hsieh, Y‐T., et al. IFN-stimulated metabolite transporter ENT3 facilitates viral genome release. __EMBO Rep__ (2023). [Link](https://doi.org/doi:10.15252/EMBR.202255286).
551. Hu, Q., et al. GPX3 expression was down-regulated but positively correlated with poor outcome in human cancers. __Front Oncol__ (2023). [Link](https://doi.org/doi:10.3389/FONC.2023.990551).
552. Huang, D., et al. MAX transcriptionally enhances PD-L1 to inhibit CD8+ T cell-mediated killing of lung adenocarcinoma cells. __Cell Immunol__ (2023). [Link](https://doi.org/doi:10.1016/J.CELLIMM.2023.104706).
553. Huang, W., et al. NFAT and NF-κB dynamically co-regulate TCR and CAR signaling responses in human T cells. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112663).
554. Hughes, LA.., et al. Copy number variation in tRNA isodecoder genes impairs mammalian development and balanced translation. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-37843-9).
555. Huo, S., et al. ATF3/SPI1/SLC31A1 Signaling Promotes Cuproptosis Induced by Advanced Glycosylation End Products in Diabetic Myocardial Injury. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24021667).
556. Inagaki, T., et al. Prolonged Kaposi’s Sarcoma-associated Herpesvirus vIL-6 Exposure Enhances Inflammatory Responses by Epigenetic Reprogramming. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.06.25.546454).
557. Inubushi, T., et al. On-demand chlorine dioxide solution enhances odontoblast differentiation through desulfation of cell surface heparan sulfate proteoglycan and subsequent activation of canonical Wnt signaling. __Front Cell Dev Biol__ (2023). [Link](https://doi.org/doi:10.3389/FCELL.2023.1271455).
558. Ishikawa, M., et al. Inference of gene regulatory networks using time-series single-cell RNA-seq data with CRISPR perturbations. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.23.554254).
559. Isobe, S., et al. Reduced FOXF1 links unrepaired DNA damage to pulmonary arterial hypertension. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-43039-Y).
560. Itakura, H., et al. Tumor-suppressive role of the musculoaponeurotic fibrosarcoma gene in colorectal cancer. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/j.isci.2023.106478).
561. Ito, S., et al. Androgen receptor binding sites enabling genetic prediction of mortality due to prostate cancer in cancer-free subjects. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-39858-8).
562. Iyer, DR.., et al. Cyclic-AMP response element binding protein (CREB) and microRNA miR-29b regulate renalase gene expression under catecholamine excess conditions. __Life Sci__ (2023). [Link](https://doi.org/doi:10.1016/J.LFS.2023.121859).
563. Iyer, P., et al. MGA deletion leads to Richter’s transformation via modulation of mitochondrial OXPHOS. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.07.527502).
564. J, M., et al. Genetic variation is a key determinant of chromatin accessibility and drives differences in the regulatory landscape of C57BL/6J and 129S1/SvImJ mice. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD1225).
565. Jain, L., et al. The growth hormone receptor interacts with transcriptional regulator HMGN1 upon GH-induced nuclear translocation. __J Cell Commun Signal__ (2023). [Link](https://doi.org/doi:10.1007/S12079-023-00741-2).
566. Janacova, L., et al. Catechol-O-methyl transferase suppresses cell invasion and interplays with MET signaling in estrogen dependent breast cancer. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-28078-1).
567. Kaller, M., et al. c-MYC-Induced AP4 Attenuates DREAM-Mediated Repression by p53. __Cancers__ (2023). [Link](https://doi.org/doi:10.3390/CANCERS15041162).
568. Kamimoto, K., et al. Dissecting cell identity via network inference and in silico gene perturbation. __Nature__ (2023). [Link](https://doi.org/doi:10.1038/S41586-022-05688-9).
569. Kartiganer, Z., et al. Improved Cell-Type Identification and Comprehensive Mapping of Regulatory Features with Spatial Epigenomics 96-Channel Microfluidic Platform. __GEN Biotech__ (2023). [Link](https://doi.org/doi:10.1089/GENBIO.2023.0044).
570. Kawamura, G., et al. Optogenetic decoding of Akt2-regulated metabolic signaling pathways in skeletal muscle cells using transomics analysis. __Sci Signal__ (2023). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ABN0782).
571. Kernfeld, E., et al. Model-X knockoffs reveal data-dependent limits on regulatory network identification. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.23.541948).
572. Kirtana, R.., et al. KDM5A noncanonically binds antagonists MLL1/2 to mediate gene regulation and promotes epithelial to mesenchymal transition. __Biochim Biophys Acta__ (2023). [Link](https://doi.org/doi:10.1016/J.BBAGRM.2023.194986).
573. Kobayashi, H., et al. Post-fertilization transcription initiation in an ancestral LTR retrotransposon drives lineage-specific genomic imprinting of ZDBF2. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.10.30.564869).
574. Kojima, H., et al. MEF2C/p300-mediated epigenetic remodeling promotes the maturation of induced cardiomyocytes. __Stem Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.STEMCR.2023.05.001).
575. Koyama, Y., et al. Increased RUNX3 expression mediates tumor-promoting ability of human breast cancer-associated fibroblasts. __Cancer Med__ (2023). [Link](https://doi.org/doi:10.1002/CAM4.6421).
576. Kubo, A., et al. Protective effects of Mkl1/2 against lipodystrophy and muscle atrophy via PI3K/AKT-independent FoxO repression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.09.552644).
577. Kumegawa, K., et al. Chromatin profile-based identification of a novel ER-positive breast cancer subgroup with reduced ER-responsive element accessibility. __Br J Cancer__ (2023). [Link](https://doi.org/doi:10.1038/S41416-023-02178-1).
578. Kurosawa, K., et al. ncBAF enhances PXR-mediated transcriptional activation in the human and mouse liver. __Biochem Pharmacol__ (2023). [Link](https://doi.org/doi:10.1016/J.BCP.2023.115733).
579. Kurosawa, K., et al. ncBAF, a chromatin remodeler, enhances PXR-mediated transcriptional activation in the human and mouse liver. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.03.527063).
580. L, M., et al. Detecting and dissecting signaling crosstalk via the multilayer network integration of signaling and regulatory interactions. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD1035).
581. Lacher, SE.., et al. An antioxidant response element regulates the HIF1α axis in breast cancer cells. __Free Radic Biol Med__ (2023). [Link](https://doi.org/doi:10.1016/J.FREERADBIOMED.2023.05.003).
582. Ladias, P., et al. Cancer Associated PRDM9: Implications for Linking Genomic Instability and Meiotic Recombination. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS242216522).
583. Latyshev, P., et al. Unsupervised domain adaptation methods for cross-species transfer of regulatory code signals. __Front Big Data__ (2023). [Link](https://doi.org/doi:10.3389/FDATA.2023.1140663).
584. Le Ciclé, C., et al. The Neurod1/4-Ntrk3-Src pathway regulates gonadotrope cell adhesion and motility. __Cell Death Discov__ (2023). [Link](https://doi.org/doi:10.1038/S41420-023-01615-7).
585. Lee, J., et al. Dnmt1/Tet2-mediated changes in Cmip methylation regulate the development of nonalcoholic fatty liver disease by controlling the Gbp2-Pparγ-CD36 axis. __Exp Mol Med__ (2023). [Link](https://doi.org/doi:10.1038/S12276-022-00919-5).
586. Lee, SA., et al. FUS Facilitates Gene Body Export from Nuclear Condensates. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.25.542265).
587. Li, C., et al. Gp78 deficiency in hepatocytes alleviates hepatic ischemia-reperfusion injury via suppressing ACSL4-mediated ferroptosis. __Cell Death Dis__ (2023). [Link](https://doi.org/doi:10.1038/S41419-023-06294-X).
588. Li, F., et al. VGLL4 and MENIN function as TEAD1 corepressors to block pancreatic β cell proliferation. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111904).
589. Li, J., et al. Overexpression of Ultrabithorax Changes the Development of Silk Gland and the Expression of Fibroin Genes in Bombyx mori. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS24076670).
590. Li, J., et al. The Recurrent-Specific Regulation Network of Prognostic Stemness-Related Signatures in Low-Grade Glioma. __Dis Markers__ (2023). [Link](https://doi.org/doi:10.1155/2023/2243928).
591. Liang, H., et al. Targeting CBX3 with a Dual BET/PLK1 Inhibitor Enhances the Antitumor Efficacy of CDK4/6 Inhibitors in Prostate Cancer. __Adv Sci__ (2023). [Link](https://doi.org/doi:10.1002/ADVS.202302368).
592. Lim, JM., et al. CREB-Regulated Transcriptional Coactivator 2 Proteome Landscape is Modulated by SREBF1. __Mol Cell Proteom__ (2023). [Link](https://doi.org/doi:10.1016/J.MCPRO.2023.100637).
593. Lin, SC., et al. Targeting YAP1 ameliorates progesterone resistance in endometriosis. __Hum Reprod__ (2023). [Link](https://doi.org/doi:10.1093/HUMREP/DEAD071).
594. Ling, YH., et al. Cell cycle regulation of the psoriasis associated gene CCHCR1 by transcription factor E2F1. __PLoS One__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0294661).
595. Liu, R., et al. Nuclear respiratory factor 1 drives hepatocellular carcinoma progression by activating LPCAT1-ERK1/2-CREB axis. __Biol Direct__ (2023). [Link](https://doi.org/doi:10.1186/S13062-023-00428-Z).
596. Liu, Z., et al. NFYC-37 promotes tumor growth by activating the mevalonate pathway in bladder cancer. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112963).
597. Lu-Culligan, WJ.., et al. Acetyl-methyllysine marks chromatin at active transcription start sites. __Nature__ (2023). [Link](https://doi.org/doi:10.1038/S41586-023-06565-9).
598. Lu, J., et al. DPM2 serve as novel oncogene and prognostic marker transactivated by ESR1 in breast cancer. __Environ Toxicol__ (2023). [Link](https://doi.org/doi:10.1002/TOX.24059).
599. Lu, Y., et al. ER stress mediates Angiotensin II-augmented innate immunity memory and facilitates distinct susceptibilities of thoracic from abdominal aorta to aneurysm development. __Front Immunol__ (2023). [Link](https://doi.org/doi:10.3389/FIMMU.2023.1268916).
600. Ma, C., et al. Super Enhancer-Associated Circular RNA-CircKrt4 Regulates Hypoxic Pulmonary Artery Endothelial Cell Dysfunction in Mice. __Arterioscler Thromb Vasc Biol__ (2023). [Link](https://doi.org/doi:10.1161/ATVBAHA.122.318842).
601. Maehara, H., et al. DNA hypomethylation characterizes genes encoding tissue-dominant functional proteins in liver and skeletal muscle. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-46393-5).
602. Mangoni, D., et al. LINE-1 regulates cortical development by acting as long non-coding RNAs. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-40743-7).
603. Mannion, AJ.., et al. The junctional mechanosensor AmotL2 regulates YAP promotor accessibility. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.01.13.523596).
604. Mao, G., et al. Gene Regulatory Network Inference Using Convolutional Neural Networks from scRNA-seq Data. __J Comput Biol__ (2023). [Link](https://doi.org/doi:10.1089/CMB.2022.0355).
605. Marku, M., et al. From time-series transcriptomics to gene regulatory networks: A review on inference methods. __PLoS Comput Biol__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1011254).
606. Martins, KM.., et al. Prevalent and Disseminated Recombinant and Wild-Type Adeno-Associated Virus Integration in Macaques and Humans. __Hum Gene Ther__ (2023). [Link](https://doi.org/doi:10.1089/HUM.2023.134).
607. Mbebi, AJ.., et al. Gene regulatory network inference using mixed-norms regularized multivariate model with covariance selection. __PLoS Comput Biol__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1010832).
608. Medina-Jover, F., et al. A rise in double-strand breaks sensitizes tumours to oxidative metabolism inhibitors. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.12.19.572355).
609. Meng, Q., et al. Consensus peaks of chromatin accessibility in the human genome. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.30.542889).
610. Miyazawa, K., et al. Cross-ancestry genome-wide analysis of atrial fibrillation unveils disease biology and enables cardioembolic risk prediction. __Nat Genet__ (2023). [Link](https://doi.org/doi:10.1038/S41588-022-01284-9).
611. Moreira, P., et al. Nuclear Factor-Y is a Pervasive Regulator of Neuronal Gene Expression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.02.14.528575).
612. Morita, T., et al. Actin-related protein 5 suppresses the cooperative activation of cardiac gene transcription by myocardin and MEF2. __FEBS Open Bio__ (2023). [Link](https://doi.org/doi:10.1002/2211-5463.13549).
613. Mu, Q., et al. Identifying predictors of glioma evolution from longitudinal sequencing. __Sci Transl Med__ (2023). [Link](https://doi.org/doi:10.1126/SCITRANSLMED.ADH4181).
614. Murakami, T., et al. HOXA10 promotes Gdf5 expression in articular chondrocytes. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-50318-7).
615. Murao, N., et al. Homeostatic Regulation of Seizure Susceptibility and Cognitive Function by ER Quality Controller Derlin-1 through Maintenance of Adult Neurogenesis. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.27.538634).
616. Nagao, S., et al. Reduced DEFA5 Expression and STAT3 Activation Underlie the Submucosal Invasion of Early Gastric Cancers. __Digestion__ (2023). [Link](https://doi.org/doi:10.1159/000531790).
617. Nakamura, J., et al. TFEB-mediated lysosomal exocytosis alleviates high fat diet-induced lipotoxicity in the kidney. __JCI Insight__ (2023). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.162498).
618. Navarrete-López, P., et al. Loss of the importin Kpna2 causes infertility in male mice by disrupting the translocation of testis-specific transcription factors. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.107134).
619. Nethander, M., et al. An atlas of genetic determinants of forearm fracture. __Nat Genet__ (2023). [Link](https://doi.org/doi:10.1038/S41588-023-01527-3).
620. Neu, D., et al. Oncogenic reactivation of young L1s is a hallmark of colon cancer. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.17.541189).
621. Noh, M., et al. Particulate matter-induced metabolic recoding of epigenetics in macrophages drives pathogenesis of chronic obstructive pulmonary disease. __J Hazard Mater__ (2023). [Link](https://doi.org/doi:10.1016/J.JHAZMAT.2023.132932).
622. Odongoo, R., et al. Overlapping role of c-Jun N-terminal kinase (JNK) 1 and 2 in imidazole ketone erastin-induced ferroptosis. __Gene Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.GENREP.2023.101813).
623. Oey, NE.., et al. A Proteome-Wide Effect of PHF8 Knockdown on Cortical Neurons Shows Downregulation of Parkinson&rsquo;s Disease-Associated Protein Alpha-Synuclein and Its Interactors. __Biomedicines__ (2023). [Link](https://doi.org/doi:10.3390/BIOMEDICINES11020486).
624. Ohata, H., et al. PROX1 induction by autolysosomal activity stabilizes persister-like state of colon cancer via feedback repression of the NOX1-mTORC1 pathway. __Cell Rep__ (2023). [Link](https://doi.org/doi:10.1016/J.CELREP.2023.112519).
625. Ohishi, H., et al. Transcription-coupled changes in higher-order genomic structure and transcription hub viscosity prolong enhancer-promoter connectivity. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.11.27.568629).
626. Ojewunmi, OO.., et al. The genetic dissection of fetal haemoglobin persistence in sickle cell disease in Nigeria. __medRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.16.23289851).
627. Oliveros, W., et al. Systematic characterization of regulatory variants of blood pressure genes ll Systematic characterization of regulatory variants of blood pressure genes. __Cell Genom__ (2023). [Link](https://doi.org/doi:10.1016/j.xgen.2023.100330).
628. Onodera, T., et al. Protective roles of adiponectin and molecular signatures of HNF4α and PPARα as downstream targets of adiponectin in pancreatic β cells. __Mol Metab__ (2023). [Link](https://doi.org/doi:10.1016/J.MOLMET.2023.101821).
629. Osada, N., et al. c-FOS is an integral component of the IKZF1 transactivator complex and mediates lenalidomide resistance in multiple myeloma. __Clin Transl Med__ (2023). [Link](https://doi.org/doi:10.1002/CTM2.1364).
630. Ota, H., et al. Switching mechanism from AR to EGFR signaling via 3-O-sulfated heparan sulfate in castration-resistant prostate cancer. __Sci Rep__ (2023). [Link](https://doi.org/doi:10.1038/S41598-023-38746-X).
631. Pan, H., et al. A nutrient-deficient microenvironment facilitates ferroptosis resistance via the FAM60A-PPAR axis in pancreatic ductal adenocarcinoma. __Research (Lambertville)__ (2023). [Link](https://doi.org/doi:10.34133/RESEARCH.0300).
632. Peña-Martínez, EG.., et al. Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 and TBX5 Binding through an Integrative Computational Approach. __J Biol Chem__ (2023). [Link](https://doi.org/doi:10.1016/J.JBC.2023.105423).
633. Peña-Martínez, EG.., et al. Prioritizing Cardiovascular Disease-Associated Variants Altering NKX2-5 Binding through an Integrative Computational Approach. __medRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.01.23294951).
634. Peñuelas‐Haro, I., et al. A novel empeROR of pancreatic malignancy. __EMBO J__ (2023). [Link](https://doi.org/doi:10.15252/EMBJ.2023114282).
635. Pinton, A., et al. PHF6-altered T-ALL harbor epigenetic repressive switch at bivalent promoters and respond to 5-azacitidine and venetoclax. __Clin Cancer Res__ (2023). [Link](https://doi.org/doi:10.1158/1078-0432.CCR-23-2159).
636. Po, A., et al. An econometric lens resolves cell-state parallax. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.02.530529).
637. Rader, AE.., et al. Combined inactivation of RB and Hippo converts differentiating Drosophila photoreceptors into eye progenitor cells through derepression of homothorax. __Dev Cell__ (2023). [Link](https://doi.org/doi:10.1016/J.DEVCEL.2023.09.003).
638. Raicu, AM., et al. Retinoblastoma protein activity revealed by CRISPRi study of divergent Rbf1 and Rbf2 paralogs. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.19.541454).
639. Rakotopare, J., et al. A systematic approach identifies p53-DREAM pathway target genes associated with blood or brain abnormalities. __Dis Model Mech__ (2023). [Link](https://doi.org/doi:10.1242/DMM.050376).
640. Rakotopare, J., et al. p53 in the Molecular Circuitry of Bone Marrow Failure Syndromes. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS241914940).
641. Rojas-Quintero, J., et al. Spatial Transcriptomics Resolve an Emphysema-specific Lymphoid Follicle B Cell Signature in COPD. __Am J Respir Crit Care Med__ (2023). [Link](https://doi.org/doi:10.1164/RCCM.202303-0507LE).
642. Rommelfanger, MK.., et al. Gene regulatory network inference with popInfer reveals dynamic regulation of hematopoietic stem cell quiescence upon diet restriction and aging. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.18.537360).
643. Ross, MO., et al. PTPN2 copper-sensing rapidly relays copper level fluctuations into EGFR/CREB activation and associated CTR1 transcriptional repression. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.29.555401).
644. Ryan, TA.J.., et al. Dimethyl fumarate and 4-octyl itaconate are anticoagulants that suppress Tissue Factor in macrophages via inhibition of Type I Interferon. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-39174-1).
645. Rymuza, J., et al. Methods for constructing and evaluating consensus genomic interval sets. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.08.03.551899).
646. Sakai, K., et al. Testicular histone hyperacetylation in mice by valproic acid administration affects the next generation by changes in sperm DNA methylation. __PLoS One__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0282898).
647. Sakai, S., et al. d-Alanine Affects the Circadian Clock to Regulate Glucose Metabolism in Kidney. __Kidney360__ (2023). [Link](https://doi.org/doi:10.34067/KID.0000000000000345).
648. Sanchez-Hernandez, ES;., et al. Glucocorticoid Receptor Regulates and Interacts with LEDGF/p75 to Promote Docetaxel Resistance in Prostate Cancer Cells. __Cells__ (2023). [Link](https://doi.org/doi:10.3390/CELLS12162046).
649. Sarott, R., et al. Borrowing Transcriptional Kinases to Activate Apoptosis. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.10.23.563687).
650. Sekine, K., et al. Transposons contribute to the acquisition of cell type-specific cis-elements in the brain. __Commun Biol__ (2023). [Link](https://doi.org/doi:10.1038/S42003-023-04989-7).
651. Shan, W., et al. Identification of canonical pyroptosis-related genes, associated regulation axis, and related traditional Chinese medicine in spinal cord injury. __Front Aging Neurosci__ (2023). [Link](https://doi.org/doi:10.3389/FNAGI.2023.1152297).
652. Shen, J., et al. Current insights into the oncogenic roles of lncRNA LINC00355. __Cancer Innov__ (2023). [Link](https://doi.org/doi:10.1002/CAI2.91).
653. Shimoda, M., et al. KSHV uses viral IL6 to expand infected immunosuppressive macrophages. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.05.531224).
654. Shimoda, M., et al. Virally encoded interleukin-6 facilitates KSHV replication in monocytes and induction of dysfunctional macrophages. __PLoS Pathog__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PPAT.1011703).
655. Shinzawa, K., et al. GREB1 isoform 4 is specifically transcribed by MITF and required for melanoma proliferation. __Oncogene__ (2023). [Link](https://doi.org/doi:10.1038/S41388-023-02803-6).
656. Shojaee, A., et al. Robust discovery of gene regulatory networks from single-cell gene expression data by Causal Inference Using Composition of Transactions. __Brief Bioinform__ (2023). [Link](https://doi.org/doi:10.1093/BIB/BBAD370).
657. Silva-Llanes, I., et al. The Transcription Factor NRF2 Has Epigenetic Regulatory Functions Modulating HDACs, DNMTs, and miRNA Biogenesis. __Antioxidants__ (2023). [Link](https://doi.org/doi:10.3390/ANTIOX12030641).
658. Sokolov, DK.., et al. Dexamethasone Inhibits Heparan Sulfate Biosynthetic System and Decreases Heparan Sulfate Content in Orthotopic Glioblastoma Tumors in Mice. __Int J Mol Sci__ (2023). [Link](https://doi.org/doi:10.3390/IJMS241210243).
659. Sugimoto, H., et al. iTraNet: A Web-Based Platform for integrated Trans-Omics Network Visualization and Analysis. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.11.30.569499).
660. Sugimoto, J., et al. Controlling Trophoblast Cell Fusion in the Human Placenta—Transcriptional Regulation of Suppressyn, an Endogenous Inhibitor of Syncytin-1. __Biomolecules__ (2023). [Link](https://doi.org/doi:10.3390/BIOM13111627).
661. Sun, W., et al. Catalytic domain-dependent and -independent transcriptional activities of the tumour suppressor histone H3K27 demethylase UTX/KDM6A in specific cancer types. __Epigenetics__ (2023). [Link](https://doi.org/doi:10.1080/15592294.2023.2222245).
662. Suzuki, A., et al. Ablation of Max expression induces meiotic onset in sexually undifferentiated germ cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.06.10.544477).
663. Tabata, S., et al. NFκB nuclear dynamics orchestrate inflammatory aging. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.18.536673).
664. Tabe-Bordbar, S., et al. Integrative modeling of lncRNA-chromatin interaction maps reveals diverse mechanisms of nuclear retention. __BMC Genomics__ (2023). [Link](https://doi.org/doi:10.1186/S12864-023-09498-9).
665. Tagore, M., et al. GABA regulates electrical activity and tumor initiation in melanoma. __Cancer Discov__ (2023). [Link](https://doi.org/doi:10.1158/2159-8290.CD-23-0389).
666. Tahara, S., et al. Transcription factor-binding k-mer analysis clarifies the cell type dependency of binding specificities and cis-regulatory SNPs in humans. __BMC Genomics__ (2023). [Link](https://doi.org/doi:10.1186/S12864-023-09692-9).
667. Taing, L., et al. Cistrome Data Browser: integrated search, analysis and visualization of chromatin data. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD1069).
668. Tamura, I., et al. Genome-wide analysis of histone modifications that underlie the dynamic changes in gene expression during decidualization in human endometrial stromal cells. __Mol Hum Reprod__ (2023). [Link](https://doi.org/doi:10.1093/MOLEHR/GAAD019).
669. Tanaka, Y., et al. Ror1 promotes PPARα-mediated fatty acid metabolism in astrocytes. __Genes Cells__ (2023). [Link](https://doi.org/doi:10.1111/GTC.13013).
670. Tang, Y., et al. "Meiosis arrester" C-natriuretic peptide directly stimulates oocyte mtDNA accumulation and is implicated in aging-associated oocyte mtDNA loss. __FASEB J__ (2023). [Link](https://doi.org/doi:10.1096/FJ.202300886R).
671. Toyoda, K., et al. HTLV-1 bZIP factor-induced reprogramming of lactate metabolism and epigenetic status promote leukemic cell expansion. __Blood Cancer Discov__ (2023). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-22-0139).
672. Trovato, M., et al. Histone H3.3 lysine 9 and 27 control repressive chromatin states at cryptic cis-regulatory elements and bivalent promoters in mouse embryonic stem cells. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.08.539859).
673. Tzani, A., et al. FAM222A, Part of the BET-Regulated Basal Endothelial Transcriptome, Is a Novel Determinant of Endothelial Biology and Angiogenesis. __Arterioscler Thromb Vasc Biol__ (2023). [Link](https://doi.org/doi:10.1161/ATVBAHA.123.319909).
674. Vahab, N., et al. Uncovering Co-regulatory Modules and Gene Regulatory Networks in the Heart through Machine Learning-based Analysis of Large-scale Epigenomic Data. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.28.538783).
675. Vičić Bočkor, V., et al. Transcription Factors HNF1A, HNF4A, and FOXA2 Regulate Hepatic Cell Protein N-Glycosylation. __Engineering (Beijing)__ (2023). [Link](https://doi.org/doi:10.1016/J.ENG.2023.09.019).
676. Waisman, A., et al. The transcription factor OCT6 promotes the dissolution of the naïve pluripotent state by repressing Nanog and activating a formative state gene regulatory network. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.07.13.548918).
677. Wang, J., et al. Inferring gene regulatory network from single-cell transcriptomes with graph autoencoder model. __PLoS Genet__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PGEN.1010942).
678. Wang, M., et al. Deletions of singular U1 snRNA gene significantly interfere with transcription and 3'-end mRNA formation. __PLoS Genet__ (2023). [Link](https://doi.org/doi:10.1371/JOURNAL.PGEN.1011021).
679. Wang, X., et al. Decoding key cell sub-populations and molecular alterations in glioblastoma at recurrence by single-cell analysis. __Acta Neuropathol Commun__ (2023). [Link](https://doi.org/doi:10.1186/S40478-023-01613-X).
680. Wang, Y., et al. Large-scale loss-of-function perturbations reveal a comprehensive epigenetic regulatory network in breast cancer. __Cancer Biol Med__ (2023). [Link](https://doi.org/doi:10.20892/J.ISSN.2095-3941.2023.0276).
681. Watabe, S., et al. Regulation of 4-HNE via SMARCA4 Is Associated with Worse Clinical Outcomes in Hepatocellular Carcinoma. __Biomedicines__ (2023). [Link](https://doi.org/doi:10.3390/BIOMEDICINES11082278).
682. Watanabe, M., et al. The kinetics of signaling through the common FcRγ chain determine cytokine profiles in dendritic cells. __Sci Signal__ (2023). [Link](https://doi.org/doi:10.1126/SCISIGNAL.ABN9909).
683. Weinstock, JS.., et al. Gene regulatory network inference from CRISPR perturbations in primary CD4+ T cells elucidates the genomic basis of immune disease. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.09.17.557749).
684. Wen, L., et al. Tubular aryl hydratocarbon receptor upregulates EZH2 to promote cellular senescence in cisplatin-induced acute kidney injury. __Cell Death Dis__ (2023). [Link](https://doi.org/doi:10.1038/S41419-022-05492-3).
685. Wernig-Zorc, S., et al. The long non-coding RNA MALAT1 modulates NR4A1 expression through a downstream regulatory element in specific cancer-cell-types. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.03.09.531856).
686. Włodarczyk, T., et al. Epiregulon: Inference of single-cell transcription factor activity to dissect mechanisms of lineage plasticity and drug response. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.11.27.568955).
687. Xiang, J., et al. ATF4 May Be Essential for Adaption of the Ocular Lens to Its Avascular Environment. __Cells__ (2023). [Link](https://doi.org/doi:10.3390/CELLS12222636).
688. Xu, H., et al. Inferring CTCF binding patterns and anchored loops across human tissues and cell types. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2022.06.15.496356).
689. Xu, J., et al. STGRNS: an interpretable transformer-based method for inferring gene regulatory networks from single-cell transcriptomic data. __Bioinformatics__ (2023). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAD165).
690. Xue, B., et al. Opportunities and challenges in sharing and reusing genomic interval data. __Front Genet__ (2023). [Link](https://doi.org/doi:10.3389/FGENE.2023.1155809).
691. Y, D., et al. SCAN: Spatiotemporal Cloud Atlas for Neural cells. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD895).
692. Yadav, P., et al. Hypoxia-induced loss of SRSF2-dependent DNA methylation promotes CTCF-mediated alternative splicing of VEGFA in breast cancer. __iScience__ (2023). [Link](https://doi.org/doi:10.1016/J.ISCI.2023.106804).
693. Yamakado, N., et al. Chemical inhibition of LSD1 leads to epithelial to mesenchymal transition in vitro of an oral squamous cell carcinoma OM-1 cell line via release from LSD1-dependent suppression of ZEB1. __Biochem Biophys Res Commun__ (2023). [Link](https://doi.org/doi:10.1016/J.BBRC.2023.01.062).
694. Yamamoto, R., et al. Computer model of IL-6 dependent rheumatoid arthritis in F759 mice. __Int Immunol__ (2023). [Link](https://doi.org/doi:10.1093/INTIMM/DXAD016).
695. Yamazaki, M., et al. YAP/BRD4-controlled ROR1 promotes tumor-initiating cells and hyperproliferation in pancreatic cancer. __EMBO J__ (2023). [Link](https://doi.org/doi:10.15252/EMBJ.2022112614).
696. Yan, Q., et al. LINC00941: a novel player involved in the progression of human cancers. __Hum Cell__ (2023). [Link](https://doi.org/doi:10.1007/S13577-023-01002-5).
697. Yang, C., et al. TRIM15 forms a regulatory loop with the AKT/FOXO1 axis and LASP1 to modulate the sensitivity of HCC cells to TKIs. __Cell Death Dis__ (2023). [Link](https://doi.org/doi:10.1038/S41419-023-05577-7).
698. Yari, S., et al. JAK inhibition ameliorates bone destruction by simultaneously targeting mature osteoclasts and their precursors. __Inflamm Regen__ (2023). [Link](https://doi.org/doi:10.1186/S41232-023-00268-4).
699. Yonezawa, S., et al. Meta-analysis of heat-stressed transcriptomes using the public gene expression database from human and mouse samples. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.04.30.537112).
700. Yumimoto, K., et al. Molecular evolution of Keap1 was essential for adaptation of vertebrates to terrestrial life. __Sci Adv__ (2023). [Link](https://doi.org/doi:10.1126/SCIADV.ADG2379).
701. Z, Z., et al. MethMarkerDB: a comprehensive cancer DNA methylation biomarker database. __Nucleic Acids Res__ (2023). [Link](https://doi.org/doi:10.1093/NAR/GKAD923).
702. Zhang, N., et al. CRISPR screening reveals gleason score and castration resistance related oncodriver ring finger protein 19 A (RNF19A) in prostate cancer. __Drug Resist Update__ (2023). [Link](https://doi.org/doi:10.1016/J.DRUP.2022.100912).
703. Zhang, R., et al. G-Quadruplex Structures Are Key Modulators of Somatic Structural Variants in Cancers. __Cancer Res__ (2023). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-22-3089).
704. Zhang, W., et al. FOXP3 recognizes microsatellites and bridges DNA through multimerization. __Nature__ (2023). [Link](https://doi.org/doi:10.1038/S41586-023-06793-Z).
705. Zhang, Z., et al. Therapeutic potency of compound RMY-205 for pulmonary fibrosis induced by SARS-CoV-2 nucleocapsid protein. __Cell Chem Biol__ (2023). [Link](https://doi.org/doi:10.1016/J.CHEMBIOL.2023.02.004).
706. Zhao, W., et al. Progesterone activates histone lactylation-Hif1α-glycolysis feedback loop to promote decidualization. __Endocrinology__ (2023). [Link](https://doi.org/doi:10.1210/ENDOCR/BQAD169).
707. Zimmermann, B., et al. Topological structures and syntenic conservation in sea anemone genomes. __Nat Commun__ (2023). [Link](https://doi.org/doi:10.1038/S41467-023-44080-7).
708. Zou, Z., et al. Elucidating disease-associated mechanisms triggered by pollutants via the epigenetic landscape using large-scale ChIP-Seq data. __bioRxiv__ (2023). [Link](https://doi.org/doi:10.1101/2023.05.18.541391).
709. Abdelfattah, N., et al. Sex-biased Yap1 oncogene function. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.21.521494).
710. Acevedo, AF., et al. BCL6 regulates the endothelial pro-immunogenic phenotype relevant to organ transplant rejection. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514941).
711. Acevedo, AF., et al. The DNA binding protein BCL6 regulates NFκB-controlled endothelial inflammatory gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.514942).
712. Ambrosini, G., et al. The Breast Cancer Epigenomics Track Hub. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.01.490187).
713. Anderson, CJ.., et al. Strand-resolved mutagenicity of DNA damage and repair. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.10.495644).
714. Anerillas, C., et al. A BDNF-TrkB autocrine loop enhances senescent cell viability. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-33709-8).
715. Atindaana, E., et al. Bimodal Expression Patterns, and Not Viral Burst Sizes, Predict the Effects of Vpr on HIV-1 Proviral Populations in Jurkat Cells. __mBio__ (2022). [Link](https://doi.org/doi:10.1128/MBIO.03748-21).
716. Awamleh, Z., et al. DNA methylation signature associated with Bohring-Opitz syndrome: a new tool for functional classification of variants in ASXL genes. __Eur J Hum Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41431-022-01083-0).
717. Baessler, A., et al. Tet2 coordinates with Foxo1 and Runx1 to balance T follicular helper cell and T helper 1 cell differentiation. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABM4982).
718. Barlier, Cé., et al. RNetDys: identification of disease-related impaired regulatory interactions due to SNPs. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.08.511312).
719. Bideyan, L., et al. Hepatic GATA4 regulates cholesterol and triglyceride homeostasis in collaboration with LXRs. __Genes Dev__ (2022). [Link](https://doi.org/doi:10.1101/GAD.350145.122).
720. Bideyan, L., et al. Integrative analysis reveals multiple modes of LXR transcriptional regulation in liver. __Proc Natl Acad Sci U S A__ (2022). [Link](https://doi.org/doi:10.1073/PNAS.2122683119).
721. Booth, LN.., et al. Males induce premature demise of the opposite sex by multifaceted strategies. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00276-y).
722. Bujnicki, J., et al. Editorial: the 20th annual Nucleic Acids Research Web Server Issue 2022. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC525).
723. Butera, A., et al. p53-driven lipidome influences non-cell-autonomous lysophospholipids in pancreatic cancer. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00319-9).
724. Cao, M., et al. The synergistic interaction landscape of chromatin regulators reveals their epigenetic regulation mechanisms across five cancer cell lines. __Comput Struct Biotechnol J__ (2022). [Link](https://doi.org/doi:10.1016/J.CSBJ.2022.09.008).
725. Chai, G., et al. Sevoflurane inhibits histone acetylation and contributes to cognitive dysfunction by enhancing the expression of ANP32A in aging mice. __Behav Brain Res__ (2022). [Link](https://doi.org/doi:10.1016/J.BBR.2022.113949).
726. Chandra, O., et al. Inferring functions of coding and non-coding genes using epigenomic patterns and deciphering the effect of combinatorics of transcription factors binding at promoters. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.17.488570).
727. Chauvistré, H., et al. Persister state-directed transitioning and vulnerability in melanoma. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30641-9).
728. Chen, F., et al. Placental DNA methylation changes in gestational diabetes mellitus. __Epigenetics__ (2022). [Link](https://doi.org/doi:10.1080/15592294.2022.2110193).
729. Chen, G., et al. Graph attention network for link prediction of gene regulations from single cell RNA-sequencing data. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC559).
730. Chen, H., et al. BET Inhibitors Target the SCLC-N subtype of Small Cell Lung Cancer by Blocking NEUROD1 Transactivation. __Mol Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-22-0594).
731. Chen, J., et al. MYT1L is required for suppressing earlier neuronal development programs in the adult mouse brain. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.17.512591).
732. Chen, KY., et al. Adipocyte-Specific Ablation of PU.1 Promotes Energy Expenditure and Ameliorates Metabolic Syndrome in Aging Mice. __Front Aging__ (2022). [Link](https://doi.org/doi:10.3389/FRAGI.2021.803482).
733. Chen, M., et al. Integrative analyses for the identification of idiopathic pulmonary fibrosis-associated genes and shared loci with other diseases. __Thorax__ (2022). [Link](https://doi.org/doi:10.1136/THORAX-2021-217703).
734. Chen, T., et al. GSTM3 deficiency impedes DNA mismatch repair to promote gastric tumorigenesis via CAND1/NRF2-KEAP1 signaling. __Cancer Lett__ (2022). [Link](https://doi.org/doi:10.1016/J.CANLET.2022.215692).
735. Chirichella, M., et al. RFX transcription factors control a miR-150/PDAP1 axis that restrains the proliferation of human T cells. __PLoS Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PBIO.3001538).
736. Corleone, G., et al. Enhancer plasticity sustains oncogenic transformation and progression of B-Cell Acute Lymphoblastic leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.18.492497).
737. Del Giudice, M., et al. FOXA1 regulates alternative splicing in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111404).
738. Dong, C., et al. Curcumin Attenuates Hydrocephalus via Activation of E2F Transcription Factor 4. __Cerebrovasc Dis__ (2022). [Link](https://doi.org/doi:10.1159/000523994).
739. Dufour, CR.., et al. The mTOR chromatin-bound interactome in prostate cancer. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110534).
740. Eguchi, R., et al. TRANSDIRE: data-driven direct reprogramming by a pioneer factor-guided trans-omics approach. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC209).
741. Escalante-Covarrubias, Q., et al. Time-of-day defines the efficacy of NAD+ to treat diet-induced metabolic disease by adjusting oscillations of the hepatic circadian clock. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503453).
742. Eustice, M., et al. Nutrient sensing pathways regulating adult reproductive diapause in C. elegans. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0274076).
743. Fang, XQ., et al. PGC1α Cooperates with FOXA1 to Regulate Epithelial Mesenchymal Transition through the TCF4-TWIST1. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS23158247).
744. Feng, C., et al. Arborinine from Glycosmis parva leaf extract inhibits clear-cell renal cell carcinoma by inhibiting KDM1A/UBE2O signaling. __Food Nutr Res__ (2022). [Link](https://doi.org/doi:10.29219/FNR.V66.8714).
745. Filipovic, D., et al. Predictive Models of Genome-wide Aryl Hydrocarbon Receptor DNA Binding Reveal Tissue Specific Binding Determinants. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.13.491754).
746. Freimer, JW.., et al. Systematic discovery and perturbation of regulatory genes in human T cells reveals the architecture of immune networks. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01106-Y).
747. Fu, LY., et al. ChIP-Hub provides an integrative platform for exploring plant regulome. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-30770-1).
748. Gao, Y., et al. TRmir: A Comprehensive Resource for Human Transcriptional Regulatory Information of MiRNAs. __Front Genet__ (2022). [Link](https://doi.org/doi:10.3389/FGENE.2022.808950).
749. Gourisankar, S., et al. Rewiring Cancer Drivers to Activate Apoptosis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.04.517548).
750. Guan, J., et al. SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference. __BMC Genomics__ (2022). [Link](https://doi.org/doi:10.1186/S12864-022-09020-7).
751. Gupta, A., et al. Dynamic Control Balancing Cell Proliferation and Inflammation is Crucial for an Effective Immune Response to Malaria. __Front Mol Biosci__ (2022). [Link](https://doi.org/doi:10.3389/FMOLB.2021.800721).
752. Habano, W., et al. Aryl hydrocarbon receptor as a DNA methylation reader in the stress response pathway. __CRC Crit Rev Toxicol__ (2022). [Link](https://doi.org/doi:10.1016/J.TOX.2022.153154).
753. Harada, T., et al. Novel antimyeloma therapeutic option with inhibition of the HDAC1-IRF4 axis and PIM kinase. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022007155).
754. Hasegawa, A., et al. Heterozygous variants in GATA2 contribute to DCML deficiency in mice by disrupting tandem protein binding. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-03316-W).
755. Hassan, HM.., et al. Regulation of Chromatin Accessibility by the Farnesoid X Receptor Is Essential for Circadian and Bile Acid Homeostasis In Vivo. __Cancers__ (2022). [Link](https://doi.org/doi:10.3390/CANCERS14246191).
756. He, S., et al. The Role of the SOX9/lncRNA ANXA2P2/miR-361-3p/SOX9 Regulatory Loop in Cervical Cancer Cell Growth and Resistance to Cisplatin. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.784525).
757. He, X., et al. PBRM1 presents a potential prognostic marker and therapeutic target in duodenal papillary carcinoma. __Clin Transl Med__ (2022). [Link](https://doi.org/doi:10.1002/CTM2.1062).
758. Hersbach, BA., et al. Probing cell identity hierarchies by fate titration and collision during direct reprogramming. __Mol Syst Biol__ (2022). [Link](https://doi.org/doi:10.15252/MSB.202211129).
759. Hinton, A., et al. OPA1 Downregulation in Skeletal Muscle Induces MERC formation in an ATF4-Dependent Manner. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.12.507669).
760. Hirata, H., et al. A novel role of helix-loop-helix transcriptional factor Bhlhe40 in osteoclast activation. __J Cell Physiol__ (2022). [Link](https://doi.org/doi:10.1002/JCP.30844).
761. Hoffmann, M., et al. TF-Prioritizer: a java pipeline to prioritize condition-specific transcription factors. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.19.512881).
762. Hsieh, YT., et al. Equilibrative Nucleoside Transporter 3 is an IFN-stimulated Gene that Facilitates Viral Genome Release. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.27.489746).
763. Hu, X., et al. Aryl hydrocarbon receptor utilises cellular zinc signals to maintain the gut epithelial barrier. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.03.515052).
764. Hu, Y., et al. RUNX1 inhibits the antiviral immune response against influenza A virus through attenuating type I interferon signaling. __Virol J__ (2022). [Link](https://doi.org/doi:10.1186/S12985-022-01764-8).
765. Huang, T., et al. Identification of upstream transcription factor binding sites in orthologous genes using mixed Student's t-test statistics. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009773).
766. Ideno, H., et al. Human PSCs determine the competency of cerebral organoid differentiation via FGF signaling and epigenetic mechanisms. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105140).
767. Ito, N., et al. The Ccl17 gene encoding TARC is synergistically transactivated by PU.1 and IRF4 driven by the mammalian common promoter in dendritic cells. __Allergy__ (2022). [Link](https://doi.org/doi:10.1111/ALL.15184).
768. Itokazu, M., et al. Adipose-derived exosomes block muscular stem cell proliferation in aged mouse by delivering miRNA Let-7d-3p that targets transcription factor HMGA2. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102098).
769. Iwata, M., et al. Regulome-based characterization of drug activity across the human diseasome. __NPJ Syst Biol Appl__ (2022). [Link](https://doi.org/doi:10.1038/S41540-022-00255-4).
770. J, H., et al. ChIPBase v3.0: the encyclopedia of transcriptional regulations of non-coding RNAs and protein-coding genes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC1067).
771. J, W., et al. CohesinDB: a comprehensive database for decoding cohesin-related epigenomes, 3D genomes and transcriptomes in human cells. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC795).
772. Jiang, J., et al. IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.105359).
773. Jing, F., et al. Prediction of the transcription factor binding sites with meta-learning. __Methods__ (2022). [Link](https://doi.org/doi:10.1016/J.YMETH.2022.04.010).
774. Kaizuka, T., et al. Developmental dynamics of the postsynaptic proteome to understand synaptic maturation and dysmaturation. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.05.490828).
775. Kanemoto, Y., et al. A long non-coding RNA as a direct vitamin D target transcribed from the anti-sense strand of the human HSD17B2 locus. __Biosci Rep__ (2022). [Link](https://doi.org/doi:10.1042/BSR20220321).
776. Kim, DK., et al. Aβ-induced mitochondrial dysfunction in neural progenitors controls KDM5A to influence neuronal differentiation. __Exp Mol Med__ (2022). [Link](https://doi.org/doi:10.1038/S12276-022-00841-W).
777. Kim, I., et al. AURKB, in concert with REST, acts as an oxygen-sensitive epigenetic regulator of the hypoxic induction of MDM2. __BMB Rep__ (2022). [Link](https://doi.org/doi:10.5483/BMBREP.2022.55.6.017).
778. Kitami, K., et al. Peritoneal restoration by repurposing vitamin D inhibits ovarian cancer dissemination via blockade of the TGF-β1/thrombospondin-1 axis. __Matrix Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.MATBIO.2022.03.003).
779. Kobelyatskaya, AA.., et al. ALDH3A2, ODF2, QSOX2, and MicroRNA-503-5p Expression to Forecast Recurrence in TMPRSS2-ERG-Positive Prostate Cancer. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911695).
780. Kojima, S., et al. Epigenome editing reveals core DNA methylation for imprinting control in the Dlk1-Dio3 imprinted domain. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC344).
781. Kokaji, T., et al. Construction of an inter-organ transomic network for whole-body metabolism. __Curr Opin Endocr Metab Res__ (2022). [Link](https://doi.org/doi:10.1016/J.COEMR.2022.100361).
782. Kokaji, T., et al. In vivo transomic analyses of glucose-responsive metabolism in skeletal muscle reveal core differences between the healthy and obese states. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.03.27.486003).
783. Kondo, J., et al. Transcription factor SP1 regulates haptoglobin fucosylation via induction of GDP-fucose transporter 1 in the hepatoma cell line HepG2. __Biochem Biophys Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.BBREP.2022.101372).
784. Kong, S., et al. Nucleosome-Omics: A Perspective on the Epigenetic Code and 3D Genome Landscape. __Genes__ (2022). [Link](https://doi.org/doi:10.3390/GENES13071114).
785. Kozlova, AA.., et al. Assessment of DDAH1 and DDAH2 Contributions to Psychiatric Disorders via In Silico Methods. __Int J Mol Sci__ (2022). [Link](https://doi.org/doi:10.3390/IJMS231911902).
786. Kreiß, M., et al. Human 5-lipoxygenase regulates transcription by association to euchromatin. __Biochem Pharmacol__ (2022). [Link](https://doi.org/doi:10.1016/J.BCP.2022.115187).
787. Labaf, M., et al. Increased AR expression in castration-resistant prostate cancer rapidly induces AR signaling reprogramming with the collaboration of EZH2. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2022.1021845).
788. Larsen, EG.., et al. Transmembrane protein TMEM184B is necessary for interleukin-31-induced itch. __Pain__ (2022). [Link](https://doi.org/doi:10.1097/J.PAIN.0000000000002452).
789. Lavrekha, VV.., et al. CisCross: A gene list enrichment analysis to predict upstream regulators in Arabidopsis thaliana. __Front Plant Sci__ (2022). [Link](https://doi.org/doi:10.3389/FPLS.2022.942710).
790. Lee, SH., et al. Activation function 1 of progesterone receptor is required for progesterone antagonism of oestrogen action in the uterus. __BMC Biol__ (2022). [Link](https://doi.org/doi:10.1186/S12915-022-01410-3).
791. Letson, CT.., et al. Targeting BET Proteins downregulates miR-33a to promote synergy with PIM inhibitors in CMML. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.01.514753).
792. Li, F., et al. TEAD1 regulates cell proliferation through a pocket-independent transcription repression mechanism. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC1063).
793. Li, R., et al. Elucidation of an mTORC2-PKC-NRF2 pathway that sustains the ATF4 stress response and identification of Sirt5 as a key ATF4 effector. __Cell Death Discov__ (2022). [Link](https://doi.org/doi:10.1038/S41420-022-01156-5).
794. Li, W., et al. YTHDC1 is downregulated by the YY1/HDAC2 complex and controls the sensitivity of ccRCC to sunitinib by targeting the ANXA1-MAPK pathway. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02460-9).
795. Li, Y., et al. ARID3A promotes the chemosensitivity of colon cancer by inhibiting AKR1C3. __Cell Biol Int__ (2022). [Link](https://doi.org/doi:10.1002/CBIN.11789).
796. Li, Y., et al. Immunoglobulin superfamily 9 (IGSF9) is trans-activated by p53, inhibits breast cancer metastasis via FAK. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02459-8).
797. Liao, Y., et al. Downregulation of LNMAS orchestrates partial EMT and immune escape from macrophage phagocytosis to promote lymph node metastasis of cervical cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02202-3).
798. Lisacek, F., et al. Worldwide Glycoscience Informatics Infrastructure: The GlySpace Alliance. __JACS Au__ (2022). [Link](https://doi.org/doi:10.1021/JACSAU.2C00477).
799. Liu, T., et al. Computational identification of preneoplastic cells displaying high stemness and risk of cancer progression. __Cancer Res__ (2022). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-22-0668).
800. Liu, W., et al. A novel FBW7/NFAT1 axis regulates cancer immunity in sunitinib-resistant renal cancer by inducing PD-L1 expression. __J Exp Clin Cancer Res__ (2022). [Link](https://doi.org/doi:10.1186/S13046-022-02253-0).
801. Liu, Z., et al. JAK2/STAT3 inhibition attenuates intestinal ischemia-reperfusion injury via promoting autophagy: in vitro and in vivo study. __Mol Biol Rep__ (2022). [Link](https://doi.org/doi:10.1007/S11033-021-07099-X).
802. Lu, Y., et al. Thioredoxin-interacting protein-activated intracellular potassium deprivation mediates the anti-tumour effect of a novel histone acetylation inhibitor HL23, a fangchinoline derivative, in human hepatocellular carcinoma. __J Adv Res__ (2022). [Link](https://doi.org/doi:10.1016/J.JARE.2022.10.017).
803. Lyu, Q., et al. A small proportion of X-linked genes contribute to X chromosome upregulation in early embryos via BRD4-mediated transcriptional activation. __Curr Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.CUB.2022.08.059).
804. M, W., et al. Hexokinase 2 is a transcriptional target and a positive modulator of AHR signalling. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC360).
805. M, Z., et al. MethBank 4.0: an updated database of DNA methylation across a variety of species. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC969).
806. Ma, X., et al. Single-cell RNA sequencing reveals B cell-T cell interactions in vascular adventitia of hyperhomocysteinemia-accelerated atherosclerosis. __Protein Cell__ (2022). [Link](https://doi.org/doi:10.1007/S13238-021-00904-0).
807. MacKay, H., et al. Sex-specific epigenetic development in the mouse hypothalamic arcuate nucleus pinpoints human genomic regions associated with body mass index. __Sci Adv__ (2022). [Link](https://doi.org/doi:10.1126/SCIADV.ABO3991).
808. Maity, AK.., et al. Inference of age-associated transcription factor regulatory activity changes in single cells. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00233-9).
809. Maixner, N., et al. Visceral Adipose Tissue E2F1-miRNA206/210 Pathway Associates with Type 2 Diabetes in Humans with Extreme Obesity. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11193046).
810. Mamivand, A., et al. Data mining of bulk and single-cell RNA sequencing introduces OBI1-AS1 as an astrocyte marker with possible role in glioma recurrence and progression. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01260-4).
811. Marques, IJ.., et al. Wt1 transcription factor impairs cardiomyocyte specification and drives a phenotypic switch from myocardium to epicardium. __Development__ (2022). [Link](https://doi.org/doi:10.1242/DEV.200375).
812. Martínez Corrales, G., et al. Transcriptional memory of dFOXO activation in youth curtails later-life mortality through chromatin remodeling and Xbp1. __Nat Aging__ (2022). [Link](https://doi.org/doi:10.1038/s43587-022-00312-x).
813. Masamoto, Y., et al. EVI1 exerts distinct roles in AML via ERG and cyclin D1 promoting a chemoresistance and immune-suppressive environment. __Blood Adv__ (2022). [Link](https://doi.org/doi:10.1182/BLOODADVANCES.2022008018).
814. Matsumoto, A., et al. C11orf21, a novel RUNX1 target gene, is down-regulated by RUNX1-ETO. __BBA Adv__ (2022). [Link](https://doi.org/doi:10.1016/J.BBADVA.2022.100047).
815. Mbebi, AJ.., et al. Gene regulatory network inference using mixed-norms regularized multivariate model with covariance selection. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.21.521538).
816. McKowen, JK., et al. The Drosophila BEAF Insulator Protein Interacts with the Polybromo Subunit of the PBAP Chromatin Remodeling Complex. __Genes Genomes Genet__ (2022). [Link](https://doi.org/doi:10.1093/G3JOURNAL/JKAC223).
817. Mei, H., et al. Autophagy Facilitates Cancer-Intrinsic Evasion of CAR T Cell-Mediated Cytotoxicity in B-Cell Malignancies. __Blood__ (2022). [Link](https://doi.org/doi:10.1182/BLOOD-2022-168590).
818. Meller, A., et al. Mining Proteomics Datasets to Uncover Functional Pseudogenes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2124-0_16).
819. Metcalf, MG.., et al. Cell non-autonomous control of autophagy and metabolism by glial cells. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.12.15.520639).
820. Miao, J., et al. Sp1 Inhibits PGC-1α via HDAC2-Catalyzed Histone Deacetylation in Chronic Constriction Injury-Induced Neuropathic Pain. __ACS Chem Neurosci__ (2022). [Link](https://doi.org/doi:10.1021/ACSCHEMNEURO.2C00440).
821. Migdał, M., et al. xcore: an R package for inference of gene expression regulators. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.23.481130).
822. Mori, J., et al. Identification of cell cycle-associated and -unassociated regulators for expression of a hepatocellular carcinoma oncogene cyclin-dependent kinase inhibitor 3. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.07.088).
823. Mori, T., et al. Depletion of NIMA-related kinase Nek2 induces aberrant self-renewal and apoptosis in stem/progenitor cells of aged muscular tissues. __Mech Ageing Dev__ (2022). [Link](https://doi.org/doi:10.1016/J.MAD.2022.111619).
824. Morin, A., et al. Characterizing the targets of transcription regulators by aggregating ChIP-seq and perturbation expression data sets. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.30.505909).
825. Morita, T., et al. Actin-related protein 5 functions as a novel modulator of MyoD and MyoG in skeletal muscle and in rhabdomyosarcoma. __Elife__ (2022). [Link](https://doi.org/doi:10.7554/ELIFE.77746).
826. Mouri, K., et al. Prioritization of autoimmune disease-associated genetic variants that perturb regulatory element activity in T cells. __Nat Genet__ (2022). [Link](https://doi.org/doi:10.1038/S41588-022-01056-5).
827. Muromoto, R., et al. Regulation of NFKBIZ gene promoter activity by STAT3, C/EBPβ, and STAT1. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.140).
828. Nair, PR.., et al. MLL1 regulates cytokine-driven cell migration and metastasis. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.10.18.512715).
829. Nakamura, H., et al. Hypoxia-inducible factor-1α and poly [ADP ribose] polymerase 1 cooperatively regulate Notch3 expression under hypoxia via a noncanonical mechanism. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102137).
830. Neal, A., et al. Finding and Verifying Enhancers for Endothelial-Expressed Genes. __Methods Mol Biol__ (2022). [Link](https://doi.org/doi:10.1007/978-1-0716-2059-5_28).
831. Ng, KJ., et al. Sox2 in the dermal papilla regulates hair follicle pigmentation. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111100).
832. Nie, M., et al. Targeting acetylcholine signaling modulates persistent drug tolerance in EGFR-mutant lung cancer and impedes tumor relapse. __J Clin Invest__ (2022). [Link](https://doi.org/doi:10.1172/JCI160152).
833. Nishi, K., et al. Novel estrogen-responsive genes (ERGs) for the evaluation of estrogenic activity. __PLoS One__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0273164).
834. Ochsner, SA.., et al. Transcriptional regulatory networks of circulating immune cells in type 1 diabetes: A community knowledgebase. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104581).
835. Ohnishi, T., et al. Periodontitis promotes the expression of gingival transmembrane serine protease 2 (TMPRSS2), a priming protease for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). __J Oral Biosci__ (2022). [Link](https://doi.org/doi:10.1016/J.JOB.2022.04.004).
836. Ono, K., et al. Congenital sideroblastic anemia model due to ALAS2 mutation is susceptible to ferroptosis. __Sci Rep__ (2022). [Link](https://doi.org/doi:10.1038/S41598-022-12940-9).
837. Panatta, E., et al. Metabolic regulation by p53 prevents R-loop-associated genomic instability. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111568).
838. Panatta, E., et al. p53 regulates expression of nuclear envelope components in cancer cells. __Biol Direct__ (2022). [Link](https://doi.org/doi:10.1186/S13062-022-00349-3).
839. Peng, L., et al. Bombesin receptor-activated protein exacerbates cisplatin-induced AKI by regulating the degradation of SIRT2. __Nephrol Dial Transplant__ (2022). [Link](https://doi.org/doi:10.1093/NDT/GFAC164).
840. Pontis, J., et al. Primate-specific transposable elements shape transcriptional networks during human development. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34800-W).
841. Q, Z., et al. ChromLoops: a comprehensive database for specific protein-mediated chromatin loops in diverse organisms. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC893).
842. Reagor, CC.., et al. Depicting pseudotime-lagged causality across single-cell trajectories for accurate gene-regulatory inference. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.25.489377).
843. Ren, J., et al. Downregulation of CBX7 induced by EZH2 upregulates FGFR3 expression to reduce sensitivity to cisplatin in bladder cancer. __Br J Cancer__ (2022). [Link](https://doi.org/doi:10.1038/S41416-022-02058-0).
844. Rhizobium, GE. TFSyntax: a database of transcription factors binding syntax in mammalian genomes. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC849).
845. Richard, D., et al. Intronic regulation of SARS-CoV-2 receptor (ACE2) expression mediated by immune signaling and oxidative stress pathways. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104614).
846. Richard, D., et al. Lineage-Specific Differences and Regulatory Networks Governing Human Chondrocyte Development. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.05.25.493388).
847. Rykaczewska, U., et al. Plaque Evaluation by Ultrasound and Transcriptomics Reveals BCLAF1 as a Regulator of Smooth Muscle Cell Lipid Transdifferentiation in Atherosclerosis. __Arterioscler Thromb Vasc Biol__ (2022). [Link](https://doi.org/doi:10.1161/ATVBAHA.121.317018).
848. Saadey, AA.., et al. Rebalancing TGFβ1/BMP signals in exhausted T cells unlocks responsiveness to immune checkpoint blockade therapy. __Nat Immunol__ (2022). [Link](https://doi.org/doi:10.1038/S41590-022-01384-Y).
849. Sakai, H., et al. Uhrf1 governs the proliferation and differentiation of muscle satellite cells. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.103928).
850. Samelson, AJ.., et al. BRD2 inhibition blocks SARS-CoV-2 infection by reducing transcription of the host cell receptor ACE2. __Nat Cell Biol__ (2022). [Link](https://doi.org/doi:10.1038/S41556-021-00821-8).
851. Sasaki, N., et al. Gp130-Mediated STAT3 Activation Contributes to the Aggressiveness of Pancreatic Cancer through H19 Long Non-Coding RNA Expression. __Cancers__ (2022). [Link](https://doi.org/doi:10.3390/CANCERS14092055).
852. Sasaki, N., et al. PRC2-dependent regulation of ganglioside expression during dedifferentiation contributes to the proliferation and migration of vascular smooth muscle cells. __Front Cell Dev Biol__ (2022). [Link](https://doi.org/doi:10.3389/FCELL.2022.1003349).
853. Sato, F., et al. COL8A1 facilitates the growth of triple-negative breast cancer via FAK/Src activation. __Breast Cancer Res Treat__ (2022). [Link](https://doi.org/doi:10.1007/S10549-022-06635-Y).
854. Sawada, T., et al. Antagonistic action of a synthetic androgen ligand mediated by chromatin remodeling in a human prostate cancer cell line. __Biochem Biophys Res Commun__ (2022). [Link](https://doi.org/doi:10.1016/J.BBRC.2022.04.109).
855. Schihada, H., et al. Isoforms of GPR35 have distinct extracellular N-termini that allosterically modify receptor-transducer coupling and mediate intracellular pathway bias. __J Biol Chem__ (2022). [Link](https://doi.org/doi:10.1016/J.JBC.2022.102328).
856. Schuhwerk, H., et al. The EMT transcription factor ZEB1 governs a fitness-promoting but vulnerable DNA replication stress response. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.111819).
857. Serrano-Zayas, C., et al. Regulation of cellular LDL uptake by PROX1 and CHD7. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.09.20.507601).
858. Shen, SH., et al. Bromodomain-containing protein 4 activates cardiotrophin-like cytokine factor 1, an unfavorable prognostic biomarker, and promotes glioblastoma in vitro. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-1164).
859. Shin, J., et al. SARS-CoV-2 infection impairs the insulin/IGF signaling pathway in the lung, liver, adipose tissue, and pancreatic cells via IRF1. __Metabolism__ (2022). [Link](https://doi.org/doi:10.1016/J.METABOL.2022.155236).
860. Silvennoinen, K., et al. SCN1A overexpression, associated with a genomic region marked by a risk variant for a common epilepsy, raises seizure susceptibility. __Acta Neuropathol__ (2022). [Link](https://doi.org/doi:10.1007/S00401-022-02429-0).
861. Singh, A., et al. Conserved neuropeptidergic regulation of intestinal integrity in invertebrate models of aging. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.02.24.481867).
862. Siswanto, FM., et al. Chlorogenic acid activates Nrf2/SKN-1 and prolongs the lifespan of Caenorhabditis elegans via the Akt-FOXO3/DAF16a-DDB1 pathway and activation of DAF16f. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.1093/GERONA/GLAC062).
863. Smirnov, D., et al. SIRT6 is a key regulator of mitochondrial function in the brain. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.08.515572).
864. Sokolowska, KE., et al. Identified in blood diet-related methylation changes stratify liver biopsies of NAFLD patients according to fibrosis grade. __Clin Epigenetics__ (2022). [Link](https://doi.org/doi:10.1186/S13148-022-01377-6).
865. Sokolowski, DJ.., et al. Differential Expression Enrichment Tool (DEET): An interactive atlas of human differential gene expression. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.29.505468).
866. Su, Y., et al. Epigenetic histone acetylation modulating prenatal Poly I:C induced neuroinflammation in the prefrontal cortex of rats: a study in a maternal immune activation model. __Front Cell Neurosci__ (2022). [Link](https://doi.org/doi:10.3389/FNCEL.2022.1037105).
867. Sugawara, S., et al. RNaseH2A downregulation drives inflammatory gene expression via genomic DNA fragmentation in senescent and cancer cells. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-04369-7).
868. Sun, J., et al. The lncRNA CASC2 Modulates Hepatocellular Carcinoma Cell Sensitivity and Resistance to TRAIL Through Apoptotic and Non-Apoptotic Signaling. __Front Oncol__ (2022). [Link](https://doi.org/doi:10.3389/FONC.2021.726622).
869. Tahara, S., et al. MOCCS profile analysis clarifies the cell type dependency of transcription factor-binding sequences and cis-regulatory SNPs in humans. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.08.487641).
870. Takayama, S., et al. GATA6 regulates expression of annexin A10 (ANXA10) associated with epithelial-mesenchymal transition of oral squamous cell carcinoma. __Arch Oral Biol__ (2022). [Link](https://doi.org/doi:10.1016/J.ARCHORALBIO.2022.105569).
871. Takeuchi, C., et al. Mod(mdg4) variants repress telomeric retrotransposon HeT-A by blocking subtelomeric enhancers. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.16.476534).
872. Talamanca, L., et al. Sex-dimorphic and age-dependent organization of 24 hour gene expression rhythms in human. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.06.13.494714).
873. Tanigawa, Y., et al. WhichTF is functionally important in your open chromatin data?. __PLoS Comput Biol__ (2022). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1010378).
874. Terakawa, A., et al. Trans-omics analysis of insulin action reveals a cell growth subnetwork which co-regulates anabolic processes. __iScience__ (2022). [Link](https://doi.org/doi:10.1016/J.ISCI.2022.104231).
875. Toyoda, K., et al. Reprogramming of lactate metabolism is linked to the oncogenesis of the virus-induced leukemia. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.12.503711).
876. Tsujino, T., et al. CRISPR screens reveal genetic determinants of PARP inhibitor sensitivity and resistance in prostate cancer. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.04.13.488115).
877. Vorontsova, JE.., et al. A new insight into the aryl hydrocarbon receptor/cytochrome 450 signaling pathway in MG63, HOS, SAOS2, and U2OS cell lines. __Biochimie__ (2022). [Link](https://doi.org/doi:10.1016/J.BIOCHI.2022.10.018).
878. Wang, Y., et al. NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks. __Commun Biol__ (2022). [Link](https://doi.org/doi:10.1038/S42003-022-04226-7).
879. Wang, Z., et al. Towards a better understanding of TF-DNA binding prediction from genomic features. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105993).
880. Woogeng, IN., et al. Inducing human retinal pigment epithelium-like cells from somatic tissue. __Stem Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.12.008).
881. Wu, B., et al. ORAI1 Limits SARS-CoV-2 Infection by Regulating Tonic Type I IFN Signaling. __J Immunol__ (2022). [Link](https://doi.org/doi:10.4049/JIMMUNOL.2100742).
882. Wu, L., et al. Natural Coevolution of Tumor and Immunoenvironment in Glioblastoma. __Cancer Discov__ (2022). [Link](https://doi.org/doi:10.1158/2159-8290.CD-22-0196).
883. Wu, S., et al. Bromodomain containing 4 transcriptionally activated Deltex E3 ubiquitin ligase 2 contributes to glioma progression and predicts an unfavorable prognosis. __Ann Transl Med__ (2022). [Link](https://doi.org/doi:10.21037/ATM-22-555).
884. Y, H., et al. Mitochondrial stress induces AREG expression and epigenomic remodeling through c-JUN and YAP-mediated enhancer activation. __Nucleic Acids Res__ (2022). [Link](https://doi.org/doi:10.1093/NAR/GKAC735).
885. Yamada, M., et al. KLF5-regulated extracellular matrix remodeling secures biliary epithelial tissue integrity against cholestatic liver injury. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.01.25.477619).
886. Yamamoto-Imoto, H., et al. Age-associated decline of MondoA drives cellular senescence through impaired autophagy and mitochondrial homeostasis. __Cell Rep__ (2022). [Link](https://doi.org/doi:10.1016/J.CELREP.2022.110444).
887. Yamashita, Y., et al. Osteoblast lineage cells-derived Sema3A regulates bone homeostasis independently of androgens. __Endocrinology__ (2022). [Link](https://doi.org/doi:10.1210/ENDOCR/BQAC126).
888. Yamazaki, M., et al. ROR1 plays a critical role in pancreatic tumor-initiating cells with a partial EMT signature. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.07.13.499868).
889. Yan, B., et al. A comprehensive single cell data analysis of in lymphoblastoid cells reveals the role of Super-enhancers in maintaining EBV latency. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.08.10.503552).
890. Yang, K., et al. Prognostic and Immunotherapeutic Roles of KRAS in Pan-Cancer. __Cells__ (2022). [Link](https://doi.org/doi:10.3390/CELLS11091427).
891. Yang, L., et al. Sevoflurane Offers Neuroprotection in a Cerebral Ischemia/Reperfusion Injury Rat Model Through the E2F1/EZH2/TIMP2 Regulatory Axis. __Mol Neurobiol__ (2022). [Link](https://doi.org/doi:10.1007/S12035-021-02602-8).
892. Yashar, WM.., et al. GoPeaks: histone modification peak calling for CUT&Tag. __Genome Biol__ (2022). [Link](https://doi.org/doi:10.1186/S13059-022-02707-W).
893. Yheskel, M., et al. Proximity labeling reveals a new in vivo network of interactors for the histone demethylase KDM5. __bioRxiv__ (2022). [Link](https://doi.org/doi:10.1101/2022.11.20.517232).
894. Yonezawa, Y., et al. Identification of a Functional Susceptibility Variant for Adolescent Idiopathic Scoliosis that Upregulates Early Growth Response 1 (EGR1)-Mediated UNCX Expression. __J Bone Miner Res__ (2022). [Link](https://doi.org/doi:10.1002/JBMR.4738).
895. Zelenka, T., et al. The 3D enhancer network of the developing T cell genome is shaped by SATB1. __Nat Commun__ (2022). [Link](https://doi.org/doi:10.1038/S41467-022-34345-Y).
896. Zhang, H., et al. CBX3 Regulated By YBX1 Promotes Smoking-induced Pancreatic Cancer Progression via Inhibiting SMURF2 Expression. __Int J Biol Sci__ (2022). [Link](https://doi.org/doi:10.7150/IJBS.68995).
897. Zhang, L., et al. STAT3 is a biologically relevant therapeutic target in H3K27M-mutant diffuse midline glioma. __Neuro Oncol__ (2022). [Link](https://doi.org/doi:10.1093/NEUONC/NOAC093).
898. Zhang, T., et al. ADAR1 masks the cancer immunotherapeutic promise of ZBP1-driven necroptosis. __Nature__ (2022). [Link](https://doi.org/doi:10.1038/S41586-022-04753-7).
899. Zhang, Y., et al. Fusion prior gene network for high reliable single-cell gene regulatory network inference. __Comput Biol Med__ (2022). [Link](https://doi.org/doi:10.1016/J.COMPBIOMED.2022.105279).
900. Zhao, J., et al. CanMethdb: a database for genome-wide DNA methylation annotation in cancers. __Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAC783).
901. Zhao, M., et al. A hybrid deep learning framework for gene regulatory network inference from single-cell transcriptomic data. __Brief Bioinform__ (2022). [Link](https://doi.org/doi:10.1093/BIB/BBAB568).
902. Zheng, X., et al. Downregulation of HINFP induces senescence-associated secretory phenotype to promote metastasis in a non-cell-autonomous manner in bladder cancer. __Oncogene__ (2022). [Link](https://doi.org/doi:10.1038/S41388-022-02371-1).
903. Zhong, X., et al. CBX3 is associated with metastasis and glutathione/ glycosphingolipid metabolism in colon adenocarcinoma. __J Gerontol A Biol Sci Med Sci__ (2022). [Link](https://doi.org/doi:10.21037/JGO-22-97/COIF)).
904. Zhu, T., et al. A pan-tissue DNA methylation atlas enables in silico decomposition of human tissue methylomes at cell-type resolution. __Nat Methods__ (2022). [Link](https://doi.org/doi:10.1038/S41592-022-01412-7).
905. Zou, Z., et al. Epigenetic landscape of drug responses revealed through large-scale ChIP-seq data analyses. __BMC Bioinformatics__ (2022). [Link](https://doi.org/doi:10.1186/S12859-022-04571-8).
906. A, M., et al. AnnoMiner is a new web-tool to integrate epigenetics, transcription factor occupancy and transcriptomics data to predict transcriptional regulators. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-94805-1).
907. A, M., et al. Integrative Bioinformatics and Experimental Investigation Introduces OBI1-AS1 as a Key LncRNA in the Progression of Low-grade Glioma to Glioblastoma. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-215387/V1).
908. A, N., et al. Hyperleptinemia in obese state renders luminal breast cancers refractory to tamoxifen by coordinating a crosstalk between Med1, miR205 and ErbB. __NPJ Breast Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41523-021-00314-9).
909. A, W., et al. E-cadherin is regulated by GATA-2 and marks the early commitment of mouse hematopoietic progenitors to the basophil and mast cell fates. __Sci Immunol__ (2021). [Link](https://doi.org/doi:10.1126/SCIIMMUNOL.ABA0178).
910. Agudelo, LZ.., et al. Metabolic resilience is encoded in genome plasticity. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.25.449953).
911. Aizawa, S., et al. Early reactivation of clustered genes on the inactive X chromosome during somatic cell reprogramming. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/J.STEMCR.2021.11.008).
912. Alfeghaly, C., et al.  Implication of repeat insertion domains in the trans -activity of the long non-coding RNA ANRIL . __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/nar/gkab245).
913. AP, W., et al. Bayesian information sharing enhances detection of regulatory associations in rare cell types. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB269).
914. Astudillo, P. Analysis in silico of the functional interaction between WNT5A and YAP/TEAD signaling in cancer. __PeerJ__ (2021). [Link](https://doi.org/doi:10.7717/peerj.10869).
915. Łasut-Szyszka, B., et al. Transcriptome Analysis of Cells Exposed to Actinomycin D and Nutlin-3a Reveals New Candidate p53-Target Genes and Indicates That CHIR-98014 Is an Important Inhibitor of p53 Activity. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222011072).
916. B. Tóth, Bá., et al. Regulatory modules of human thermogenic adipocytes: functional genomics of large cohort and Meta-analysis derived marker-genes. __BMC Genomics__ (2021). [Link](https://doi.org/doi:10.1186/S12864-021-08126-8).
917. Bang, J., et al. Identification of Signaling Pathways for Early Embryonic Lethality and Developmental Retardation in Sephs1 -/- Mice. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222111647).
918. Biani, M.C.., et al. Small extracellular vesicles from iPSC-MSC lose their regenerative potential upon UV-C irradiation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.27.461979).
919. Bono, H. Meta-analysis of oxidative transcriptomes in insects. __Antioxidants__ (2021). [Link](https://doi.org/doi:10.3390/antiox10030345).
920. C, F., et al. PFKFB4 is overexpressed in clear-cell renal cell carcinoma promoting pentose phosphate pathway that mediates Sunitinib resistance. __J Exp Clin Cancer Res__ (2021). [Link](https://doi.org/doi:10.1186/S13046-021-02103-5).
921. CA, A., et al. ACSNI: An unsupervised machine-learning tool for prediction of tissue-specific pathway components using gene expression profiles. __Patterns__ (2021). [Link](https://doi.org/doi:10.1016/J.PATTER.2021.100270).
922. Centenera, MM.., et al. ELOVL5 is a critical and targetable fatty acid elongase in prostate cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-2511).
923. Cerase, A., et al. Chd8 regulates X chromosome inactivation in mouse through fine-tuning control of Xist expression. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-01945-1).
924. Chen, H., et al. BET Bromodomain Inhibitors Target the NEUROD1-subtype SCLC by Blocking NEUROD1 Transactivation. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.25.465771).
925. Chen, M., et al. Integrative Analyses Reveal Novel Disease-associated Loci and Genes for Idiopathic Pulmonary Fibrosis. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.11.21257064).
926. Choi, S., et al. Identification of a Putative Enhancer RNA for EGFR in Hyper-Accessible Regions in Esophageal Squamous Cell Carcinoma Cells by Analysis of Chromatin Accessibility Landscapes. __Front Oncol__ (2021). [Link](https://doi.org/doi:10.3389/FONC.2021.724687).
927. Cofer, EM.., et al. Modeling transcriptional regulation of model species with deep learning. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/gr.266171.120).
928. de Roover, A., et al. Hypoxia induces DOT1L in articular cartilage to protect against osteoarthritis. __JCI Insight__ (2021). [Link](https://doi.org/doi:10.1172/JCI.INSIGHT.150451).
929. De Smedt, J., et al. PU.1 drives specification of pluripotent stem cell-derived endothelial cells to LSEC-like cells. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-020-03356-2).
930. Dhaka, B., et al. Differential chromatin accessibility landscape of gain-of-function mutant p53 tumours. __BMC Cancer__ (2021). [Link](https://doi.org/doi:10.1186/s12885-021-08362-x).
931. Doi, H., et al. Early-life midazolam exposure persistently changes chromatin accessibility to impair adult hippocampal neurogenesis and cognition. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2107596118/-/DCSUPPLEMENTAL).
932. Droin, C., et al. Space-time logic of liver gene expression at sub-lobular scale. __Nat Metab__ (2021). [Link](https://doi.org/doi:10.1038/s42255-020-00323-1).
933. Durand, Sé., et al. RSL24D1 sustains steady-state ribosome biogenesis and pluripotency translational programs in embryonic stem cells. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.27.443845).
934. E, M., et al. NUAK2 and RCan2 participate in the p53 mutant pro-tumorigenic network. __Biol Direct__ (2021). [Link](https://doi.org/doi:10.1186/S13062-021-00296-5).
935. Feng, H., et al. Identification of core miRNAs and regulatory pathways in breast cancer by integrated bioinformatics analysis. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/d0mo00171f).
936. Feng, L., et al. Comprehensive Analysis of E3 Ubiquitin Ligases Reveals Ring Finger Protein 223 as a Novel Oncogene Activated by KLF4 in Pancreatic Cancer. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.738709).
937. Feng, P., et al. A machine learning-based framework for modeling transcription elongation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/pnas.2007450118).
938. Fernandez Lahore, G., et al. Polymorphic estrogen receptor binding site causes Cd2-dependent sex bias in the susceptibility to autoimmune diseases. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25828-5).
939. Franco, ED., et al. Primate-specific ZNF808 is essential for pancreatic development in humans. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.23.21262262).
940. Fujino, K., et al. A human endogenous bornavirus-like nucleoprotein encodes a mitochondrial protein associated with cell viability. __J Virol__ (2021). [Link](https://doi.org/doi:10.1128/JVI.02030-20).
941. Fujino, S., et al. HNF1A regulates colorectal cancer progression and drug resistance as a downstream of POU5F1. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89126-2).
942. G, D., et al. Comprehensive analysis of epigenetic signatures of human transcription control. __Mol Omics__ (2021). [Link](https://doi.org/doi:10.1039/D0MO00130A).
943. Gallo, A., et al. NF-Y subunits overexpression in gastric adenocarcinomas (STAD). __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-03027-Y).
944. Gharavi, E., et al. Embeddings of genomic region sets capture rich biological associations in lower dimensions. __Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1093/BIOINFORMATICS/BTAB439).
945. H, D., et al. Early-life midazolam exposure persistently changes chromatin accessibility to impair adult hippocampal neurogenesis and cognition. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2107596118).
946. H, M., et al. Evi1 upregulates Fbp1 and supports progression of acute myeloid leukemia through pentose phosphate pathway activation. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/CAS.15098).
947. H, O., et al. Lysine Demethylase 5A is Required for MYC Driven Transcription in Multiple Myeloma. __Blood Cancer Discov__ (2021). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0108).
948. Hamano, M., et al. Prediction of single-cell mechanisms for disease progression in hypertrophic remodelling by a trans-omics approach. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86821-y).
949. Hammal, F., et al. ReMap 2022: a database of Human, Mouse, Drosophila and Arabidopsis regulatory regions from an integrative analysis of DNA-binding sequencing experiments. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB996).
950. Hanaki, S., et al. PP1 regulatory subunit NIPP1 regulates transcription of E2F1 target genes following DNA damage. __Cancer Sci__ (2021). [Link](https://doi.org/doi:10.1111/cas.14924).
951. Harada, Y., et al. ETS‐dependent enhancers for endothelial‐specific expression of serum/glucocorticoid‐regulated kinase 1 during mouse embryo development. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/gtc.12874).
952. He, L., et al. An optimized two-step chromatin immunoprecipitation protocol to quantify the associations of two separate proteins and their common target DNA. __STAR Protoc__ (2021). [Link](https://doi.org/doi:10.1016/j.xpro.2021.100504).
953. Hernández-Nava, E., et al. Transcriptional and Epigenetic Bioinformatic Analysis of Claudin-9 Regulation in Gastric Cancer. __J Oncol__ (2021). [Link](https://doi.org/doi:10.1155/2021/5936905).
954. Hiebert, P. The Nrf2 transcription factor: A multifaceted regulator of the extracellular matrix. __Matrix Biol Plus__ (2021). [Link](https://doi.org/doi:10.1016/j.mbplus.2021.100057).
955. Hisamatsu, Y., et al. Matrix Metalloproteinases in Human Decidualized Endometrial Stromal Cells. __Curr Issues Mol Biol__ (2021). [Link](https://doi.org/doi:10.3390/CIMB43030146).
956. Hisano, O., et al. Short single-stranded DNAs with putative non-canonical structures comprise a new class of plasma cell-free DNA. __BMC Biol__ (2021). [Link](https://doi.org/doi:10.1186/S12915-021-01160-8).
957. Hosoi, H., et al. Super-enhancers for RUNX3 are required for cell proliferation in EBV-infected B cell lines. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145421).
958. Hosoya, T., et al. Chondroprotective effects of CDK4/6 inhibition via enhanced ubiquitin-dependent degradation of JUN in synovial fibroblasts. __Baillieres Clin Rheumatol__ (2021). [Link](https://doi.org/doi:10.1093/RHEUMATOLOGY/KEAB874).
959. Hossain, SMM., et al. Discovering key transcriptomic regulators in pancreatic ductal adenocarcinoma using Dirichlet process Gaussian mixture model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87234-7).
960. Huang, CCF., et al. Functional mapping of androgen receptor enhancer activity. __Genome Biol__ (2021). [Link](https://doi.org/doi:10.1186/s13059-021-02339-6).
961. Hudecova, I., et al. Characteristics, origin, and potential for cancer diagnostics of ultrashort plasma cell-free DNA. __Genome Res__ (2021). [Link](https://doi.org/doi:10.1101/GR.275691.121).
962. Inazumi, H., et al. NRSF-GNAO1 Pathway Contributes to the Regulation of Cardiac Ca 2+ Homeostasis. __Circ Res__ (2021). [Link](https://doi.org/doi:10.1161/CIRCRESAHA.121.318898).
963. J, D., et al. Systematic identification of genomic elements that regulate FCGR2A expression and harbor variants linked with autoimmune disease. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB372).
964. J, S., et al. Cancer drivers and clonal dynamics in acute lymphoblastic leukaemia subtypes. __Research Square__ (2021). [Link](https://doi.org/doi:10.21203/RS.3.RS-366981/V1).
965. J, W., et al. JUNB suppresses distant metastasis by influencing the initial metastatic stage. __Clin Exp Metastasis__ (2021). [Link](https://doi.org/doi:10.1007/S10585-021-10108-9).
966. Jang, M., et al. Matrix stiffness epigenetically regulates the oncogenic activation of the Yes-associated protein in gastric cancer. __Nat Biomed Eng__ (2021). [Link](https://doi.org/doi:10.1038/s41551-020-00657-x).
967. Jarvis, LB.., et al. Therapeutically expanded human regulatory T-cells are super-suppressive due to HIF1A induced expression of CD73. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02721-X).
968. Jia, Y., et al. Super enhancer-mediated upregulation of HJURP promotes growth and survival of t(4;14)-positive multiple myeloma. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-0921).
969. Jiang, JC., et al. Integrated transcription factor profiling with transcriptome analysis identifies L1PA2 transposons as global regulatory modulators in a breast cancer model. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-86395-9).
970. Jiang, JC., et al. Widespread Exaptation of L1 Transposons for Transcription Factor Binding in Breast Cancer. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115625).
971. Jiang, X., et al. Heat shock induces the nuclear accumulation of YAP1 via SRC. __Exp Cell Res__ (2021). [Link](https://doi.org/doi:10.1016/j.yexcr.2020.112439).
972. Jin, X., et al. Smoking-associated upregulation of CBX3 suppresses ARHGAP24 expression to activate Rac1 signaling and promote tumor progression in lung adenocarcinoma. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/S41388-021-02114-8).
973. Jung, S., et al. A computer-guided design tool to increase the efficiency of cellular conversions. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/s41467-021-21801-4).
974. Jungwirth, E., et al. Meta-analysis and Consolidation of Farnesoid X Receptor Chromatin Immunoprecipitation Sequencing Data Across Different Species and Conditions. __Hepatol Commun__ (2021). [Link](https://doi.org/doi:10.1002/HEP4.1749).
975. K, K., et al. ATF4-mediated transcriptional regulation protects against β-cell loss during endoplasmic reticulum stress in a mouse model. __Mol Metab__ (2021). [Link](https://doi.org/doi:10.1016/J.MOLMET.2021.101338).
976. K, N., et al. Profiling of Androgen-Dependent Enhancer RNAs Expression in Human Prostate Tumors: Search for Malignancy Transition Markers. __Res Rep Urol__ (2021). [Link](https://doi.org/doi:10.2147/RRU.S328661).
977. K, S., et al. HMGB1 coordinates SASP-related chromatin folding and RNA homeostasis on the path to senescence. __Mol Syst Biol__ (2021). [Link](https://doi.org/doi:10.15252/MSB.20209760).
978. Kaiser, M., et al. Regulation of otocyst patterning by <i>Tbx2</i> and <i>Tbx3</i> is required for inner ear morphogenesis in the mouse. __Development__ (2021). [Link](https://doi.org/doi:10.1242/dev.195651).
979. Kalman, ZE., et al. A preliminary study on the cistrome of human postsynaptic density from an evolutionary and network-based perspective. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.01.25.428072).
980. Kanai, M., et al. Insights from complex trait fine-mapping across diverse populations. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.09.03.21262975).
981. Kanamori, Y., et al. Iron-rich Kupffer cells exhibit phenotypic changes during the development of liver fibrosis in NASH. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/j.isci.2020.102032).
982. Katada, S., et al. Neural stem/precursor cells dynamically change their epigenetic landscape to differentially respond to BMP signaling for fate switching during brain development. __Genes Dev__ (2021). [Link](https://doi.org/doi:10.1101/GAD.348797.121).
983. Kidwell, A., et al. Translation rescue by targeting Ppp1r15a upstream open reading frame in vivo. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.12.11.472232).
984. Kikutake, C., et al. Pan-cancer analysis of non-coding recurrent mutations and their possible involvement in cancer pathogenesis. __NAR Cancer__ (2021). [Link](https://doi.org/doi:10.1093/narcan/zcab008).
985. Kim, J., et al. The RNA helicase DDX6 controls early mouse embryogenesis by repressing aberrant inhibition of BMP signaling through miRNA-mediated gene silencing. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.11.29.470397).
986. Kohrogi, K., et al. LSD1 de fi nes erythroleukemia metabolism by controlling the lineage-speci fi c transcription factors GATA1 and C / EBP a. __Blood Adv__ (2021). [Link](https://doi.org/doi:10.1182/bloodadvances.2020003521).
987. Koike, Y., et al. Age-related demethylation of the TDP-43 autoregulatory region in the human motor cortex. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02621-0).
988. Kosinsky, RL., et al. RNF20 and RNF40 regulate vitamin D receptor-dependent signaling in inflammatory bowel disease. __Cell Death Differ__ (2021). [Link](https://doi.org/doi:10.1038/s41418-021-00808-w).
989. Kwon, ATJ., et al. Development of p53 Knockout U87MG Cell Line for Unbiased Drug Delivery Testing System Using CRISPR-Cas9 and Transcriptomic Analysis. __J Biotechnol__ (2021). [Link](https://doi.org/doi:10.1016/j.jbiotec.2021.03.017).
990. L, W., et al. The Ubiquitin Sensor and Adaptor Protein p62 Mediates Signal Transduction of a Viral Oncogenic Pathway. __mBio__ (2021). [Link](https://doi.org/doi:10.1128/MBIO.01097-21).
991. Labade, AS.., et al. Nup93 and CTCF modulate spatiotemporal dynamics and function of the HOXA gene locus during differentiation. __J Cell Sci__ (2021). [Link](https://doi.org/doi:10.1242/JCS.259307).
992. Lee, HC., et al. YAP1 overexpression contributes to the development of enzalutamide resistance by induction of cancer stemness and lipid metabolism in prostate cancer. __Oncogene__ (2021). [Link](https://doi.org/doi:10.1038/s41388-021-01718-4).
993. Lee, SA., et al. METTL8 mRNA Methyltransferase Enhances Cancer Cell Migration via Direct Binding to ARID1A. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/ijms22115432).
994. Li, JN., et al. Expression of SnoRNA U50A Is Associated with Better Prognosis and Prolonged Mitosis in Breast Cancer. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13246304).
995. Liu, Y., et al. Tumors exploit FTO-mediated regulation of glycolytic metabolism to evade immune surveillance. __Cell Metab__ (2021). [Link](https://doi.org/doi:10.1016/j.cmet.2021.04.001).
996. Luginbühl, J., et al. Decoding Neuronal Diversification by Multiplexed Single-cell RNA-Seq. __Stem Cell Rep__ (2021). [Link](https://doi.org/doi:10.1016/j.stemcr.2021.02.006).
997. M, F., et al. SUMO conjugation susceptibility of Akt/protein kinase B affects the expression of the pluripotency transcription factor Nanog in embryonic stem cells. __PLoS One__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PONE.0254447).
998. M, S., et al. PHLDA3 is an important downstream mediator of p53 in squamous cell carcinogenesis. __J Invest Dermatol__ (2021). [Link](https://doi.org/doi:10.1016/J.JID.2021.09.007).
999. M, Y., et al. Chronological genome and single-cell transcriptome integration characterizes the evolutionary process of adult T cell leukemia-lymphoma. __Nat Commun__ (2021). [Link](https://doi.org/doi:10.1038/S41467-021-25101-9).
1000. Ma, S., et al. Powerful gene-based testing by integrating long-range chromatin interactions and knockoff genotypes. __medRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.14.21260405).
1001. Ma, W., et al. BARTweb: a web server for transcriptional regulator association analysis. __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqab022).
1002. Markouli, C.., et al. Sustained intrinsic WNT and BMP4 activation impairs hESC differentiation to definitive endoderm and drives the cells towards extra-embryonic mesoderm. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-87547-7).
1003. Marques, IJ.., et al. Downregulation of WT1 transcription factor gene expression is required to promote myocardial fate. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.07.06.451274).
1004. Matsuura, N., et al. NOTCH3 limits the epithelial–mesenchymal transition and predicts a favorable clinical outcome in esophageal cancer. __Cancer Med__ (2021). [Link](https://doi.org/doi:10.1002/cam4.3933).
1005. Milanesi, E., et al. Whole Blood Expression Pattern of Inflammation and Redox Genes in Mild Alzheimer's Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S334337).
1006. Minnoye, L., et al. Chromatin accessibility profiling methods. __Nat Rev Methods Primers__ (2021). [Link](https://doi.org/doi:10.1038/s43586-020-00008-9).
1007. Misawa, A., et al. Long Noncoding RNA HOXA11-AS and Transcription Factor HOXB13 Modulate the Expression of Bone Metastasis-Related Genes in Prostate Cancer. __Genes__ (2021). [Link](https://doi.org/doi:10.3390/genes12020182).
1008. Miyake, M., et al. Integrated stress response regulates GDF15 secretion from adipocytes, preferentially suppresses appetite for a high-fat diet and improves obesity. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103448).
1009. Morishita, M., et al. Characterization of mouse embryonic fibroblasts derived from Rassf6 knockout mice shows the implication of Rassf6 in the regulation of NF-κB signaling. __Genes Cells__ (2021). [Link](https://doi.org/doi:10.1111/GTC.12901).
1010. Morrow, AK., et al. Epitome: predicting epigenetic events in novel cell types with multi-cell deep ensemble learning. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB676).
1011. Mourao, L., et al. Hyperactive WNT/CTNNB1 signaling induces a competing cell proliferation and epidermal differentiation response in the mouse mammary epithelium. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.06.22.449461).
1012. N, O., et al. mTOR inhibitors sensitize multiple myeloma cells to venetoclax via IKZF3-and Blimp-1-mediated BCL-2 up-regulation. __Haematologica__ (2021). [Link](https://doi.org/doi:10.3324/HAEMATOL.2021.278506).
1013. Nagai, Y., et al. Glucotoxicity-induced suppression of Cox6a2 expression provokes β-cell dysfunction via augmented ROS production. __Biochem Biophys Res Commun__ (2021). [Link](https://doi.org/doi:10.1016/j.bbrc.2021.03.148).
1014. Namba, S., et al. Transcript-targeted analysis reveals isoform alterations and double-hop fusions in breast cancer. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/S42003-021-02833-4).
1015. Nurtdinov, R., et al. Alu-mediated weak CEBPA binding and slow B cell transdifferentiation in human. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.10.28.466072).
1016. Ogino, T., et al. Post-transcriptional repression of circadian component CLOCK regulates cancer-stemness in murine breast cancer cells. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/eLife.66155).
1017. Oh, TI., et al. PGC1α Loss Promotes Lung Cancer Metastasis through Epithelial-Mesenchymal Transition. __Cancers__ (2021). [Link](https://doi.org/doi:10.3390/CANCERS13081772).
1018. Omiya, H., et al. BMP signaling suppresses Gemc1 expression and ependymal differentiation of mouse telencephalic progenitors. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79610-6).
1019. P, ME., et al. DNA Methylome Alterations are Associated with Airway Macrophage Differentiation and Phenotype During Lung Fibrosis. __Am J Respir Crit Care Med__ (2021). [Link](https://doi.org/doi:10.1164/RCCM.202101-0004OC).
1020. P, Wü., et al. Transcription factors: Bridge between cell signaling and gene regulation. __Proteomics__ (2021). [Link](https://doi.org/doi:10.1002/PMIC.202000034).
1021. Papagiannopoulos, CI., et al. The histone methyltransferase inhibitor A-366 enhances hemoglobin expression in erythroleukemia cells upon co-exposure with chemical inducers in culture. __J Biol Res__ (2021). [Link](https://doi.org/doi:10.1186/s40709-020-00132-3).
1022. Pecori, F., et al. A defined glycosylation regulatory network modulates total glycome dynamics during pluripotency state transition. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-020-79666-4).
1023. Petrone, MíV., et al. The pluripotency transcription factor OCT4 represses heme oxygenase-1 gene expression. __FEBS Lett__ (2021). [Link](https://doi.org/doi:10.1002/1873-3468.14138).
1024. Piette, BL.., et al. Comprehensive interactome profiling of the human Hsp70 network highlights functional differentiation of J domains. __Mol Cell__ (2021). [Link](https://doi.org/doi:10.1016/j.molcel.2021.04.012).
1025. PM, T., et al. Human autoinflammatory disease reveals ELF4 as a transcriptional regulator of inflammation. __Nat Immunol__ (2021). [Link](https://doi.org/doi:10.1038/S41590-021-00984-4).
1026. Q, L., et al. SIGNET: single-cell RNA-seq-based gene regulatory network prediction using multiple-layer perceptron bagging. __Brief Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/BIB/BBAB547).
1027. Quinn, HM., et al. YAP and β-catenin cooperate to drive oncogenesis in basal breast cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.can-20-2801).
1028. RC, Z., et al. Conserved immunomodulatory transcriptional networks underlie antipsychotic-induced weight gain. __Transl Psychiatry__ (2021). [Link](https://doi.org/doi:10.1038/S41398-021-01528-Y).
1029. Reimegård, J., et al. A combined approach for single-cell mRNA and intracellular protein expression analysis. __Commun Biol__ (2021). [Link](https://doi.org/doi:10.1038/s42003-021-02142-w).
1030. Ruiz, JéL., et al.  The regulatory genome of the malaria vector Anopheles gambiae : integrating chromatin accessibility and gene expression . __NAR Genom Bioinform__ (2021). [Link](https://doi.org/doi:10.1093/nargab/lqaa113).
1031. S, H., et al. Targeting ATF4-dependent pro-survival autophagy to synergize glutaminolysis inhibition. __Theranostics__ (2021). [Link](https://doi.org/doi:10.7150/THNO.60028).
1032. S, L., et al. Long noncoding RNA HOTAIR interacts with Y-Box Protein-1 (YBX1) to regulate cell proliferation. __Life Sci Alliance__ (2021). [Link](https://doi.org/doi:10.26508/LSA.202101139).
1033. S, M., et al. The alternative serotonin transporter promoter P2 impacts gene function in females with irritable bowel syndrome. __J Cell Mol Med__ (2021). [Link](https://doi.org/doi:10.1111/JCMM.16736).
1034. Sakuratani, T., et al. Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2021). [Link](https://doi.org/doi:10.1007/S00428-020-02899-1).
1035. Sarkar, A., et al. Genome-wide DNA methylation and multi-omics study of human chondrocyte ontogeny and an epigenetic clock analysis of adult chondrocytes. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.08.02.454544).
1036. Sekita, Y., et al. AKT signaling is associated with epigenetic reprogramming via the upregulation of TET and its cofactor, alpha-ketoglutarate during iPSC generation. __Stem Cell Res Ther__ (2021). [Link](https://doi.org/doi:10.1186/S13287-021-02578-1).
1037. Sharma, A. Inferring molecular mechanisms of dexamethasone therapy in severe COVID-19 from existing transcriptomic data. __Gene__ (2021). [Link](https://doi.org/doi:10.1016/j.gene.2021.145665).
1038. Shimizu, T., et al. Osteocytes as main responders to low-intensity pulsed ultrasound treatment during fracture healing. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/s41598-021-89672-9).
1039. Shin, J., et al. Possible Involvement of Adipose Tissue in Patients With Older Age, Obesity, and Diabetes With Coronavirus SARS-CoV-2 Infection (COVID-19) via GRP78 (BIP/HSPA5): Significance of Hyperinsulinemia Management in COVID-19. __Diabetes__ (2021). [Link](https://doi.org/doi:10.2337/DB20-1094).
1040. Shinohara, M., et al. Upregulated expression of a subset of genes in <i>APP</i> ; <i>ob</i> / <i>ob</i> mice: Evidence of an interaction between diabetes‐linked obesity and Alzheimer's disease. __FASEB Bioadv__ (2021). [Link](https://doi.org/doi:10.1096/fba.2020-00151).
1041. Shirai, Y., et al. Elucidation of disease etiology by trans-layer omics analysis. __Inflamm Regen__ (2021). [Link](https://doi.org/doi:10.1186/s41232-021-00155-w).
1042. Singh, I., et al. Heterogeneity in the Epigenetic Landscape of Murine Testis-Specific Histone Variants TH2A and TH2B Sharing the Same Bi-Directional Promoter. __Front Cell Dev Biol__ (2021). [Link](https://doi.org/doi:10.3389/FCELL.2021.755751).
1043. Stirparo, GG.., et al. OCT4 induces embryonic pluripotency via STAT3 signaling and metabolic mechanisms. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2008890118).
1044. Stuparević, I., et al. Regulation of the conserved 3′-5′ exoribonuclease EXOSC10/Rrp6 during cell division, development and cancer. __Biol Rev__ (2021). [Link](https://doi.org/doi:10.1111/brv.12693).
1045. Su, EY., et al. Reconstruction of dynamic regulatory networks reveals signaling-induced topology 3 changes associated with germ layer specification 4 5 6. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.06.443021).
1046. Sun, Y., et al. Oleic Acid and Eicosapentaenoic Acid Reverse Palmitic Acid-induced Insulin Resistance in Human HepG2 Cells via the Reactive Oxygen Species / JUN Pathway. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/j.gpb.2019.06.005).
1047. SY, W., et al. MYC suppresses STING-dependent innate immunity by transcriptionally upregulating DNMT1 in triple-negative breast cancer. __J Immunother Cancer__ (2021). [Link](https://doi.org/doi:10.1136/JITC-2021-002528).
1048. T, S., et al. Androgen-dependent and DNA binding-independent association of androgen receptor with chromatic regions coding androgen-induced non-coding RNAs. __Biosci Biotechnol Biochem__ (2021). [Link](https://doi.org/doi:10.1093/BBB/ZBAB135).
1049. T, Y., et al. Saikosaponin-D Alleviates Renal Inflammation and Cell Apoptosis in a Mouse Model of Sepsis via TCF7/FOSL1/MMP9 Inhibition. __Mol Cell Biol__ (2021). [Link](https://doi.org/doi:10.1128/MCB.00332-21).
1050. Takai, J., et al. Gata2 heterozygous mutant mice exhibit reduced inflammatory responses and impaired bacterial clearance. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.102836).
1051. Takeda, T., et al. A stem cell marker KLF5 regulates CCAT1 via three-dimensional genome structure in colorectal cancer cells. __Br J Cancer__ (2021). [Link](https://doi.org/doi:10.1038/S41416-021-01579-4).
1052. Takeuchi, Y., et al. The membrane-linked adaptor FRS2β fashions a cytokine-rich inflammatory microenvironment that promotes breast cancer carcinogenesis. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2103658118).
1053. Tan, D., et al. Quantitative control of noise in mammalian gene expression by dynamic histone regulation. __Elife__ (2021). [Link](https://doi.org/doi:10.7554/ELIFE.65654).
1054. Teschendorff, AE., et al. Statistical mechanics meets single-cell biology. __Nat Rev Genet__ (2021). [Link](https://doi.org/doi:10.1038/s41576-021-00341-z).
1055. Tian, Y., et al. Novel role of prostate cancer risk variant rs7247241 on PPP1R14A isoform transition through allelic TF binding and CpG methylation. __Hum Mol Genet__ (2021). [Link](https://doi.org/doi:10.1093/HMG/DDAB347).
1056. Umarov, R., et al. DeepCellState: An autoencoder-based framework for predicting cell type specific transcriptional states induced by drug treatment. __PLoS Comput Biol__ (2021). [Link](https://doi.org/doi:10.1371/JOURNAL.PCBI.1009465).
1057. Unno, K., et al. Activated ALK Cooperates with N-Myc via Wnt/β-catenin Signaling to Induce Neuroendocrine Prostate Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-20-3351).
1058. Wan, R., et al. Direct Reconstruction of Gene Regulatory Networks underlying Cellular state Transitions without Pseudo-time Inference. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.12.443928).
1059. Wang, J., et al. HPV E7 affects the function of cervical cancer cells via the TAL1/lnc‑EBIC/KLHDC7B axis. __Oncol Rep__ (2021). [Link](https://doi.org/doi:10.3892/or.2021.8002).
1060. Wang, L., et al. ASXL1 promotes adrenocortical carcinoma and is associated with chemoresistance to EDP regimen. __Aging__ (2021). [Link](https://doi.org/doi:10.18632/AGING.203534).
1061. Wei, KC., et al. An innovative targeted therapy for fluoroscopy-induced chronic radiation dermatitis. __J Mol Med__ (2021). [Link](https://doi.org/doi:10.1007/S00109-021-02146-3).
1062. Wen, L., et al. Selective EZH2 inhibitor zld1039 alleviates inflammation in cisplatin-induced acute kidney injury partially by enhancing RKIP and suppressing NF-κB p65 pathway. __Zhongguo Yao Li Xue Bao__ (2021). [Link](https://doi.org/doi:10.1038/S41401-021-00837-8).
1063. Wierzbicka, JM.., et al. The Effects of Vitamin D on the Expression of IL-33 and Its Receptor ST2 in Skin Cells; Potential Implication for Psoriasis. __Int J Mol Sci__ (2021). [Link](https://doi.org/doi:10.3390/IJMS222312907).
1064. Wu, B., et al. ORAI1 establishes resistance to SARS-CoV-2 infection by regulating tonic type I interferon signaling. __bioRxiv__ (2021). [Link](https://doi.org/doi:10.1101/2021.05.04.442548).
1065. X, C., et al. Poly-L-arginine promotes asthma angiogenesis through induction of FGFBP1 in airway epithelial cells via activation of the mTORC1-STAT3 pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04055-2).
1066. X, L., et al. Defining Proximity Proteomics of Histone Modifications by Antibody-mediated Protein A-APEX2 Labeling. __Genomics Proteomics Bioinformatics__ (2021). [Link](https://doi.org/doi:10.1016/J.GPB.2021.09.003).
1067. X, Z., et al. Combinatorial CRISPR/Cas9 Screening Reveals Epistatic Networks of Interacting Tumor Suppressor Genes and Therapeutic Targets in Human Breast Cancer. __Cancer Res__ (2021). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-21-2555).
1068. Xiang, T., et al. The novel ZEB1-upregulated protein PRTG induced by Helicobacter pylori infection promotes gastric carcinogenesis through the cGMP/PKG signaling pathway. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/s41419-021-03440-1).
1069. Y, H., et al. Norrie disease protein is essential for cochlear hair cell maturation. __Proc Natl Acad Sci U S A__ (2021). [Link](https://doi.org/doi:10.1073/PNAS.2106369118).
1070. Y, O., et al. Multi-Omic Meta-Analysis of Transcriptomes and the Bibliome Uncovers Novel Hypoxia-Inducible Genes. __Biomedicines__ (2021). [Link](https://doi.org/doi:10.3390/BIOMEDICINES9050582).
1071. Y, Z., et al. TcoFBase: a comprehensive database for decoding the regulatory transcription co-factors in human and mouse. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB950).
1072. Yagi, M., et al. Mitochondrial translation deficiency impairs NAD <sup>+</sup> ‐mediated lysosomal acidification. __EMBO J__ (2021). [Link](https://doi.org/doi:10.15252/embj.2020105268).
1073. Yao, X., et al. TP53/miR-129/MDM2/4/TP53 feedback loop modulates cell proliferation and apoptosis in retinoblastoma. __Cell Cycle__ (2021). [Link](https://doi.org/doi:10.1080/15384101.2021.1892321).
1074. Yazar, V., et al. Integrative genome-wide analysis of dopaminergic neuron-specific PARIS expression in Drosophila dissects recognition of multiple PPAR-γ associated gene regulation. __Sci Rep__ (2021). [Link](https://doi.org/doi:10.1038/S41598-021-00858-7).
1075. YH, C., et al. The Induction of Alpha-1 Antitrypsin by Vitamin D in Human T Cells Is TGF-β Dependent: A Proposed Anti-inflammatory Role in Airway Disease. __Front Nutr__ (2021). [Link](https://doi.org/doi:10.3389/FNUT.2021.667203).
1076. Yi, L., et al. The FOXM1/RNF26/p57 axis regulates the cell cycle to promote the aggressiveness of bladder cancer. __Cell Death Dis__ (2021). [Link](https://doi.org/doi:10.1038/S41419-021-04260-Z).
1077. Yi, X., et al. Interrogating cell type-specific cooperation of transcriptional regulators in 3D chromatin. __iScience__ (2021). [Link](https://doi.org/doi:10.1016/J.ISCI.2021.103468).
1078. Yoneda, M., et al. Nucleosome assembly protein 1 (NAP-1) is a regulator of histone H1 acetylation. __J Biochem__ (2021). [Link](https://doi.org/doi:10.1093/JB/MVAB098).
1079. Yoshizaki, K., et al. Paternal age affects offspring via an epigenetic mechanism involving REST/NRSF. __EMBO Rep__ (2021). [Link](https://doi.org/doi:10.15252/embr.202051524).
1080. Zampieri, C., et al. p53 mutations define the chromatin landscape to confer drug tolerance in pancreatic cancer. __Mol Oncol__ (2021). [Link](https://doi.org/doi:10.1002/1878-0261.13161).
1081. Zhang, S., et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. __Sci Adv__ (2021). [Link](https://doi.org/doi:10.1126/SCIADV.ABG8205).
1082. Zhao, X., et al. CircleBase: an integrated resource and analysis platform for human eccDNAs. __Nucleic Acids Res__ (2021). [Link](https://doi.org/doi:10.1093/NAR/GKAB1104).
1083. Zhao, Y., et al. Identification of Unique Transcriptomic Signatures and Hub Genes Through RNA Sequencing and Integrated WGCNA and PPI Network Analysis in Nonerosive Reflux Disease. __J Inflamm Res__ (2021). [Link](https://doi.org/doi:10.2147/JIR.S340452).
1084. Ahmed, M., et al. Integrating binding and expression data to predict transcription factors combined function. __BMC Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12864-020-06977-1).
1085. Azim, K., et al. Resolving the transcriptional transitions associated with oligodendrocyte generation from adult neural stem cells by 2 single cell sequencing Running Title: Oligodendroglial transcriptional networks 4 5 6. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.12.18.423285).
1086. B, Z., et al. Mi-2β-targeted inhibition induces immunotherapy response in melanoma. __Research Square__ (2020). [Link](https://doi.org/doi:10.21203/RS.3.RS-118153/V1).
1087. Beknazarov, N., et al. Deep learning approach for predicting functional Z-DNA regions using omics data. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-76203-1).
1088. Bono, H., et al. Meta-analysis of hypoxic transcriptomes from public databases. __Biomedicines__ (2020). [Link](https://doi.org/doi:10.3390/biomedicines8010010).
1089. Booth, LN., et al. Males deploy multifaceted strategies and hijack longevity pathways to induce premature demise of the opposite sex. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.30.181008).
1090. Bunina, D., et al. Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons. __Cell Syst__ (2020). [Link](https://doi.org/doi:10.1016/j.cels.2020.05.003).
1091. Caruana, BT., et al. The NF-κB signalling pathway regulates GLUT6 expression in endometrial cancer. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109688).
1092. Cataldo, A., et al. MiR-302b as a Combinatorial Therapeutic Approach to Improve Cisplatin Chemotherapy Efficacy in Human Triple-Negative Breast Cancer. __Cancers__ (2020). [Link](https://doi.org/doi:10.3390/cancers12082261).
1093. Cavalli, M., et al. A Multi-Omics Approach to Liver Diseases: Integration of Single Nuclei Transcriptomics with Proteomics and HiCap Bulk Data in Human Liver. __OMICS__ (2020). [Link](https://doi.org/doi:10.1089/omi.2019.0215).
1094. Charmpi, K., et al. Convergent network effects along the axis of gene expression during prostate cancer progression. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02188-9).
1095. Chauvistré, H., et al. The Janus-faced role of KDM5B heterogeneity in melanoma: differentiation as a situational driver of both growth arrest and drug-resistance. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.01.999847).
1096. Chen, CCL., et al. Histone H3.3G34-Mutant Interneuron Progenitors Co-opt PDGFRA for Gliomagenesis. __Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.cell.2020.11.012).
1097. Chen, H., et al. PARK2 promotes mitochondrial pathway of apoptosis and antimicrotubule drugs chemosensitivity via degradation of phospho-BCL-2. __Theranostics__ (2020). [Link](https://doi.org/doi:10.7150/thno.47044).
1098. Chen, J., et al. LncSEA: a platform for long non-coding RNA related sets and enrichment analysis. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa806).
1099. Choi, JH., et al. Histone demethylase KDM4D cooperates with NFIB and MLL1 complex to regulate adipogenic differentiation of C3H10T1/2 mesenchymal stem cells. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-60049-8).
1100. Choudhury, A., et al. Meningioma epigenetic grouping reveals biologic drivers and therapeutic vulnerabilities. __medRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.11.23.20237495).
1101. Czipa, E., et al. ChIPSummitDB: a ChIP-seq-based database of human transcription factor binding sites and the topological arrangements of the proteins bound to them. __Database__ (2020). [Link](https://doi.org/doi:10.1093/database/baz141).
1102. Devailly, G., et al. Comprehensive analysis of epigenetic signatures of human transcrip-tion control †. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.23.309625).
1103. Dorotea, D., et al. Recent Insights Into SREBP as a Direct Mediator of Kidney Fibrosis via Lipid-Independent Pathways. __Front Pharmacol__ (2020). [Link](https://doi.org/doi:10.3389/fphar.2020.00265).
1104. Farahmand, S., et al. ModEx: A text mining system for extracting mode of regulation of transcription factor-gene regulatory interaction. __J Biomed Inform__ (2020). [Link](https://doi.org/doi:10.1016/j.jbi.2019.103353).
1105. Feng, C., et al. KnockTF: A comprehensive human gene expression profile database with knockdown/knockout of transcription factors. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz881).
1106. Fornes, O., et al. JASPAR 2020: Update of the open-Access database of transcription factor binding profiles. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1001).
1107. Gao, L., et al. Ozone therapy promotes the differentiation of basal keratinocytes via increasing Tp63-mediated transcription of KRT10 to improve psoriasis. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15160).
1108. Gerisch, B., et al. HLH-30/TFEB Is a Master Regulator of Reproductive Quiescence. __Dev Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.devcel.2020.03.014).
1109. Grujic, O., et al. Identification and characterization of constrained non-exonic bases lacking predictive epigenomic and transcription factor binding annotations. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-19962-9).
1110. Guan, S., et al. TGF-β1 induces CREB1-mediated miR-1290 upregulation to antagonize lung fibrosis via Napsin A. __Int J Mol Med__ (2020). [Link](https://doi.org/doi:10.3892/ijmm.2020.4565).
1111. Hayashi, A., et al. A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma. __Nat Cancer__ (2020). [Link](https://doi.org/doi:10.1038/s43018-019-0010-1).
1112. Hayashi, A., et al. Evolutionary Dynamics of Non-Coding Regions in Pancreatic Ductal Adenocarcinoma. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.11.294389).
1113. Hirano, K., et al. Indonesian ginger (Bangle) extract promotes neurogenesis of human neural stem cells through WNT pathway activation. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134772).
1114. Hänsel-Hertsch, R., et al. Landscape of G-quadruplex DNA structural regions in breast cancer. __Nat Genet__ (2020). [Link](https://doi.org/doi:10.1038/s41588-020-0672-8).
1115. Horn, M., et al. Hexosamine Pathway Activation Improves Protein Homeostasis through the Integrated Stress Response. __iScience__ (2020). [Link](https://doi.org/doi:10.1016/j.isci.2020.100887).
1116. Hsiao, WY., et al. The Lipid Handling Capacity of Subcutaneous Fat Is Programmed by mTORC2 during Development. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108223).
1117. Kaminuma, E., et al. DDBJ Data Analysis Challenge: a machine learning competition to predict Arabidopsis chromatin feature annotations from DNA sequences. __Genes Genet Syst__ (2020). [Link](https://doi.org/doi:10.1266/GGS.19-00034).
1118. Katahira, J., et al. Human THO maintains the stability of repetitive DNA. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12760).
1119. Kikuyama, T., et al. 25(OH)D3 stimulates the expression of vitamin D target genes in renal tubular cells when Cyp27b1 is abrogated. __J Steroid Biochem Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jsbmb.2020.105593).
1120. Kim, YJ., et al. UBE2C Overexpression Aggravates Patient Outcome by Promoting Estrogen-Dependent/Independent Cell Proliferation in Early Hormone Receptor-Positive and HER2-Negative Breast Cancer. __Front Oncol__ (2020). [Link](https://doi.org/doi:10.3389/fonc.2019.01574).
1121. Knaupp, AS.., et al. TINC- A Method to Dissect Regulatory Complexes at Single-Locus Resolution- Reveals an Extensive Protein Complex at the Nanog Promoter. __Stem Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.STEMCR.2020.11.005).
1122. Koga, T., et al. Expression of leukotriene B4 receptor 1 defines functionally distinct DCs that control allergic skin inflammation. __Cell Mol Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41423-020-00559-7).
1123. Kohmoto, T., et al. Claudin-6 is a single prognostic marker and functions as a tumor-promoting gene in a subgroup of intestinal type gastric cancer. __Gastric Cancer__ (2020). [Link](https://doi.org/doi:10.1007/s10120-019-01014-x).
1124. Kokaji, T., et al. Transomics analysis reveals allosteric and gene regulation axes for altered hepatic glucose-responsive metabolism in obesity. __Sci Signal__ (2020). [Link](https://doi.org/doi:10.1126/SCISIGNAL.AAZ1236).
1125. Kolmykov, S., et al. GTRD: an integrated view of transcription regulation. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa1057).
1126. Kremsky, I., et al. Protection from DNA re-methylation by transcription factors in primordial germ cells and pre-implantation embryos can explain trans-generational epigenetic inheritance. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02036-w).
1127. Krześniak, Mł., et al. Synergistic activation of p53 by actinomycin D and nutlin-3a is associated with the upregulation of crucial regulators and effectors of innate immunity. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109552).
1128. Kubota, N., et al. An integrated analysis of public genomic data unveils a possible functional mechanism of psoriasis risk via a long-range ERRFI1 enhancer. __BMC Med Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12920-020-0662-9).
1129. Kutscher, L., et al. Functional loss of a non-canonical BCOR-PRC1.1 complex accelerates SHH-driven medulloblastoma formation. __Genes Dev__ (2020). [Link](https://doi.org/doi:10.1101/2020.02.06.938035).
1130. Kwiecien, K., et al. The methylation status of the chemerin promoter region located from − 252 to + 258 bp regulates constitutive but not acute-phase cytokine-inducible chemerin expression levels. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-70625-7).
1131. Kwon, ATJ., et al. Efficient Development of Platform Cell Lines Using CRISPR-Cas9 and Transcriptomics Analysis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.16.299248).
1132. L, H., et al. A Regulation Loop between YAP and NR4A1 Balances Cell Proliferation and Apoptosis. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.CELREP.2020.108284).
1133. Larsen, EG.., et al. TMEM184B controls pruriceptor specification and function. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.01.25.919902).
1134. Lei, X., et al. Crystal Structures of Ternary Complexes of MEF2 and NKX2–5 Bound to DNA Reveal a Disease Related Protein–Protein Interaction Interface. __J Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jmb.2020.07.004).
1135. Leporcq, Cé., et al. TFmotifView: a webserver for the visualization of transcription factor motifs in genomic regions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa252).
1136. Li, X., et al. ERRα activates SHMT2 transcription to enhance the resistance of breast cancer to lapatinib via modulating the mitochondrial metabolic adaption. __Biosci Rep__ (2020). [Link](https://doi.org/doi:10.1042/BSR20192465).
1137. Li, Y., et al. TRlnc: a comprehensive database for human transcriptional regulatory information of lncRNAs. __Brief Bioinform__ (2020). [Link](https://doi.org/doi:10.1093/bib/bbaa011).
1138. Li, YH., et al. FGF9 is a downstream target of SRY and sufficient to determine male sex fate in ex vivo XX gonad culture. __Biol Reprod__ (2020). [Link](https://doi.org/doi:10.1093/biolre/ioaa154).
1139. Li, Z., et al. Runx2 (Runt-Related Transcription Factor 2)-Mediated Microcalcification Is a Novel Pathological Characteristic and Potential Mediator of Abdominal Aortic Aneurysm. __Arterioscler Thromb Vasc Biol__ (2020). [Link](https://doi.org/doi:10.1161/ATVBAHA.119.314113).
1140. Lin, QXX., et al. TFregulomeR reveals transcription factors' context-specific features and functions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1088).
1141. LM, K., et al. Cytosolic pH regulates proliferation and tumour growth by promoting expression of cyclin D1. __Nat Metab__ (2020). [Link](https://doi.org/doi:10.1038/S42255-020-00297-0).
1142. Lorenzo, JP., et al. APOBEC2 binds Chromatin and Represses Transcription during Myoblast Differentiation. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.07.29.223594).
1143. Lou, S., et al. TopicNet: a framework for measuring transcriptional regulatory network change. __Bioinformatics__ (2020). [Link](https://doi.org/doi:10.1093/bioinformatics/btaa403).
1144. Lyu, J., et al. Synthetic lethality of RB1 and aurora A is driven by stathmin-mediated disruption of microtubule dynamics. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18872-0).
1145. Maharjan, M., et al. Overlapping but Distinct Sequences Play Roles in the Insulator and Promoter Activities of the Drosophila BEAF-Dependent scs' Insulator. __Genetics__ (2020). [Link](https://doi.org/doi:10.1534/genetics.120.303344).
1146. Mao, N., et al. Oncogenic ERG represses PI3K signaling through downregulation of IRS2. __Cancer Res__ (2020). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-19-1394).
1147. McErlean, P., et al. Profiling of H3K27Ac Reveals the Influence of Asthma on the Epigenome of the Airway Epithelium. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.585746).
1148. Meng, Y., et al. Transcriptomic landscape profiling of metformin-treated healthy mice: Implication for potential hypertension risk when prophylactically used. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15472).
1149. Mercatelli, D., et al. Gene regulatory network inference resources: A practical overview. __Biochim Biophys Acta Gene Regul Mech__ (2020). [Link](https://doi.org/doi:10.1016/j.bbagrm.2019.194430).
1150. Merrett, JE., et al. Identification of DNA response elements regulating expression of CCAAT/enhancer-binding protein (C/EBP) β and δ and MAP kinase-interacting kinases during early adipogenesis. __Adipocyte__ (2020). [Link](https://doi.org/doi:10.1080/21623945.2020.1796361).
1151. Miyamoto, Y., et al. Genetic loss of importin α4 causes abnormal sperm morphology and impacts on male fertility in mouse. __FASEB J__ (2020). [Link](https://doi.org/doi:10.1096/fj.202000768RR).
1152. Nam, A.R., et al. Alternative methylation of intron motifs is associated with cancer-related gene expression in both canine mammary tumor and human breast cancer. __Clin Epigenetics__ (2020). [Link](https://doi.org/doi:10.1186/s13148-020-00888-4).
1153. Namekawa, T., et al. ALDH1A1 in patient-derived bladder cancer spheroids activates retinoic acid signaling leading to TUBB3 overexpression and tumor progression. __Int J Cancer__ (2020). [Link](https://doi.org/doi:10.1002/ijc.32505).
1154. Namekawa, T., et al. HIF1α inhibitor 2-methoxyestradiol decreases NRN1 expression and represses in vivo and in vitro growth of patient-derived testicular germ cell tumor spheroids. __Cancer Lett__ (2020). [Link](https://doi.org/doi:10.1016/j.canlet.2020.05.040).
1155. Ochiai, K., et al. Chromatin Protein PC4 Orchestrates B Cell Differentiation by Collaborating with IKAROS and IRF4. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108517).
1156. Ochsner, SA.., et al. Consensus transcriptional regulatory networks of coronavirus-infected human cells. __Sci Data__ (2020). [Link](https://doi.org/doi:10.1038/s41597-020-00628-6).
1157. Ohishi, H., et al. Characterization of genetic-origin-dependent monoallelic expression in mouse embryonic stem cells. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12736).
1158. Okada, R., et al. RNaseH2A Downregulation Drives Chromosomal DNA Fragmentation and Accumulation of RNA-DNA Hybrids in Senescent Cells. __SSRN__ (2020). [Link](https://doi.org/doi:10.2139/ssrn.3588831).
1159. Okazaki, Y., et al. Increased glucose metabolism in Arid5b-/- skeletal muscle is associated with the down-regulation of TBC1 domain family member 1 (TBC1D1). __Biol Res__ (2020). [Link](https://doi.org/doi:10.1186/s40659-020-00313-3).
1160. P, Sé., et al. Chromatin interactions in differentiating keratinocytes reveal novel atopic dermatitis- and psoriasis-associated genes. __J Allergy Clin Immunol__ (2020). [Link](https://doi.org/doi:10.1016/J.JACI.2020.09.035).
1161. Pecori, F., et al. Mucin-type O-glycosylation controls pluripotency in mouse embryonic stem cells via Wnt receptor endocytosis. __J Cell Sci__ (2020). [Link](https://doi.org/doi:10.1242/jcs.245845).
1162. Pradhananga, S., et al. Promoter anchored interaction landscape of THP-1 macrophages captures early immune response processes. __Cell Immunol__ (2020). [Link](https://doi.org/doi:10.1016/j.cellimm.2020.104148).
1163. Pratapa, A., et al. Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data. __Nat Methods__ (2020). [Link](https://doi.org/doi:10.1038/s41592-019-0690-6).
1164. Q, P., et al. VARAdb: a comprehensive variation annotation database for human. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/NAR/GKAA922).
1165. Ray, JP., et al. Prioritizing disease and trait causal variants at the TNFAIP3 locus using functional and genomic features. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-15022-4).
1166. Rikitake, M., et al. Analysis of GPI-anchored proteins involved in germline stem cell proliferation in the Caenorhabditis elegans germline stem cell niche. __J Biochem__ (2020). [Link](https://doi.org/doi:10.1093/jb/mvaa075).
1167. Roels, J., et al. Aging of preleukemic thymocytes drives CpG island hypermethylation in T-cell acute lymphoblastic leukemia. __Blood Cancer Discov__ (2020). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0059).
1168. Roels, J., et al. Distinct and temporary-restricted epigenetic mechanisms regulate human αβ and γδ T cell development. __Nat Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41590-020-0747-9).
1169. Ronzio, M., et al. Integrating Peak Colocalization and Motif Enrichment Analysis for the Discovery of Genome-Wide Regulatory Modules and Transcription Factor Recruitment Rules. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00072).
1170. Sakuratani, T., et al. Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2020). [Link](https://doi.org/doi:10.1007/s00428-020-02899-1).
1171. Sharipov, RN., et al. Assessment of transcriptional importance of cell line-specific features based on GTRD and FANTOM5 data. __PLoS One__ (2020). [Link](https://doi.org/doi:10.1371/journal.pone.0243332).
1172. Shikov, AE., et al. Phenome-wide functional dissection of pleiotropic effects highlights key molecular pathways for human complex traits. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-58040-4).
1173. SN, R., et al. STAT3 in the dorsal raphe gates behavioural reactivity and regulates gene networks associated with psychopathology. __Mol Psychiatry__ (2020). [Link](https://doi.org/doi:10.1038/S41380-020-00904-2).
1174. Sugasawa, T., et al. Influence of intermittent cold stimulations on CREB and its targeting genes in muscle: Investigations into molecular mechanisms of local cryotherapy. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134588).
1175. Sukumaran, A., et al. Insight on Transcriptional Regulation of the Energy Sensing AMPK and Biosynthetic mTOR Pathway Genes. __Front Cell Dev Biol__ (2020). [Link](https://doi.org/doi:10.3389/fcell.2020.00671).
1176. Swoboda, A., et al. STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pathway. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01584-6).
1177. Tamura, I., et al. Transcription factor C/EBPβ induces genome-wide H3K27ac and upregulates gene expression during decidualization of human endometrial stromal cells. __Molec Cell Endocr__ (2020). [Link](https://doi.org/doi:10.1016/j.mce.2020.111085).
1178. Tanaka, H., et al. The NSD2/WHSC1/MMSET methyltransferase prevents cellular senescence-associated epigenomic remodeling. __Aging Cell__ (2020). [Link](https://doi.org/doi:10.1111/acel.13173).
1179. Teschendorff, AE., et al. Improved detection of tumor suppressor events in single-cell RNA-Seq data. __NPJ Genom Med__ (2020). [Link](https://doi.org/doi:10.1038/s41525-020-00151-y).
1180. Teschendorff, AE.., et al. EPISCORE: Cell type deconvolution of bulk tissue DNA methylomes from single-cell RNA-Seq data. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02126-9).
1181. Thonsri, U., et al. High glucose-ROS conditions enhance the progression in cholangiocarcinoma via upregulation of MAN2A2 and CHD8. __Cancer Sci__ (2020). [Link](https://doi.org/doi:10.1111/cas.14719).
1182. Toro, Aé., et al. Novel Interplay between p53 and HO-1 in Embryonic Stem Cells. __Cells__ (2020). [Link](https://doi.org/doi:10.3390/cells10010035).
1183. Totani, H., et al. Autocrine HGF/c-Met signaling pathway confers aggressiveness in lymph node adult T-cell leukemia/lymphoma. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01393-x).
1184. Ueda, MT., et al. Comprehensive genomic analysis reveals dynamic evolution of endogenous retroviruses that code for retroviral-like protein domains. __Mob DNA__ (2020). [Link](https://doi.org/doi:10.1186/s13100-020-00224-w).
1185. Wang, A., et al. Single-cell multiomic profiling of human lungs reveals cell-type-specific and age-dynamic control of SARS-CoV2 host genes. __Elife__ (2020). [Link](https://doi.org/doi:10.7554/ELIFE.62522).
1186. Wang, R., et al. Transient IGF-1R inhibition combined with osimertinib eradicates AXL-low expressing EGFR mutated lung cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18442-4).
1187. Wu, X., et al. Regulation of cellular sterol homeostasis by the oxygen responsive noncoding RNA lincNORS. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18411-x).
1188. X, L., et al. The role of MYB proto-oncogene like 2 in tamoxifen resistance in breast cancer. __J Mol Histol__ (2020). [Link](https://doi.org/doi:10.1007/S10735-020-09920-6).
1189. Y, L., et al. Single-Cell Transcriptome Analysis Reveals Dynamic Cell Populations and Differential Gene Expression Patterns in Control and Aneurysmal Human Aortic Tissue. __Circulation__ (2020). [Link](https://doi.org/doi:10.1161/CIRCULATIONAHA.120.046528).
1190. Y, S., et al. UTX maintains functional integrity of murine hematopoietic system by globally regulating aging-associated genes. __Blood__ (2020). [Link](https://doi.org/doi:10.1182/BLOOD.2019001044).
1191. Y, Y., et al. SAA1 is upregulated in gastric cancer-associated fibroblasts possibly by its enhancer activation. __Carcinogenesis__ (2020). [Link](https://doi.org/doi:10.1093/CARCIN/BGAA131).
1192. Yumimoto, K., et al. Recent insight into the role of FBXW7 as a tumor suppressor. __Semin Cancer Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.semcancer.2020.02.017).
1193. Zhao, W., et al. Epigenetic Regulation of m6A Modifications in Human Cancer. __Mol Ther Nucleic Acids__ (2020). [Link](https://doi.org/doi:10.1016/j.omtn.2019.11.022).
1194. Zhou, S., et al. Noncoding mutations target cis-regulatory elements of the FOXA1 plexus in prostate cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-14318-9).
1195. Zhou, Y., et al. epiCOLOC: Integrating Large-Scale and Context-Dependent Epigenomics Features for Comprehensive Colocalization Analysis. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00053).
1196. Abugessaisa, I., et al. refTSS: A Reference Data Set for Human and Mouse Transcription Start Sites. __J Mol Biol__ (2019). [Link](https://doi.org/doi:10.1016/j.jmb.2019.04.045).
1197. Ahn, H., et al. Propanet: Time-varying condition-specific transcriptional network construction by network propagation. __Front Plant Sci__ (2019). [Link](https://doi.org/doi:10.3389/fpls.2019.00698).
1198. Ando, M., et al. Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-09937-w).
1199. Aono, S., et al. β-catenin/TCF4 complex-mediated induction of the NRF3 (NFE2L3) gene in cancer cells. __Int J Mol Sci__ (2019). [Link](https://doi.org/doi:10.3390/ijms20133344).
1200. Arata, Y., et al. Defective induction of the proteasome associated with T-cell receptor signaling underlies T-cell senescence. __Genes Cells__ (2019). [Link](https://doi.org/doi:10.1111/gtc.12728).
1201. Bakhmet, EI., et al. hnRNP-K Targets Open Chromatin in Mouse Embryonic Stem Cells in Concert with Multiple Regulators. __Stem Cells__ (2019). [Link](https://doi.org/doi:10.1002/stem.3025).
1202. Banerjee, S., et al. Identifying transcriptional regulatory modules among different chromatin states in mouse neural stem cells. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
1203. Bocci, M., et al. Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__ (2019). [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
1204. Bravo González-Blas, C., et al. cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. __Nat Methods__ (2019). [Link](https://doi.org/doi:10.1038/s41592-019-0367-1).
1205. Chang, HC., et al. Investigating the role of super-enhancer RNAs underlying embryonic stem cell differentiation. __BMC Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12864-019-6293-x).
1206. Chereji, RăV., et al. Quantitative MNase-seq accurately maps nucleosome occupancy levels. __Genome Biol__ (2019). [Link](https://doi.org/doi:10.1186/s13059-019-1815-z).
1207. Chow, KT.., et al. IRF5 regulates unique subset of genes in dendritic cells during West Nile virus infection. __J Leukoc Biol__ (2019). [Link](https://doi.org/doi:10.1002/JLB.MA0318-136RRR).
1208. Delicado, D., et al. Multilocus phylogeny, species delimitation and biogeography of Iberian valvatiform springsnails (Caenogastropoda: Hydrobiidae), with the description of a new genus. __Zool J Linn Soc__ (2019). [Link](https://doi.org/doi:10.1093/zoolinnean/zly093).
1209. Duforestel, M., et al. Glyphosate primes mammary cells for tumorigenesis by reprogramming the epigenome in a TET3-dependent manner. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2019.00885).
1210. Farahmand, S., et al. Causal Inference Engine: A platform for directional gene set enrichment analysis and inference of active transcriptional regulators. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz1046).
1211. Ferris, E., et al. Parallel Accelerated Evolution in Distant Hibernators Reveals Candidate Cis Elements and Genetic Circuits Regulating Mammalian Obesity. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.10.102).
1212. Fleming, AM.., et al. Location dependence of the transcriptional response of a potential G-quadruplex in gene promoters under oxidative stress. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz207).
1213. Fontela, MG., et al. The conserved non-coding sequence 2 (CNS2) enhances CD69 transcription through cooperation between the transcription factors oct1 and RUNX1. __Genes__ (2019). [Link](https://doi.org/doi:10.3390/genes10090651).
1214. Horie, K., et al. Down-regulation of GATA1-dependent erythrocyte-related genes in the spleens of mice exposed to a space travel. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-44067-9).
1215. Hotta, M., et al. Farnesoid X receptor induces cell death and sensitizes to TRAIL-induced inhibition of growth in colorectal cancer cells through the up-regulation of death receptor 5. __Biochem Biophys Res Commun__ (2019). [Link](https://doi.org/doi:10.1016/j.bbrc.2019.09.033).
1216. Hurtz, C., et al. Rationale for targeting BCL6 in MLL-rearranged acute lymphoblastic leukemia. __Genes Dev__ (2019). [Link](https://doi.org/doi:10.1101/gad.327593.119).
1217. Ishijima, Y., et al. The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Molec Cell Endocr__ (2019). [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
1218. Kang, H., et al. HSF1 regulates mevalonate and cholesterol biosynthesis pathways. __Cancers__ (2019). [Link](https://doi.org/doi:10.3390/cancers11091363).
1219. Kehl, T., et al. The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int J Cancer__ (2019). [Link](https://doi.org/doi:10.1002/ijc.31834).
1220. Åkerborg, Ö., et al. High-Resolution Regulatory Maps Connect Vascular Risk Variants to Disease-Related Pathways. __Circulation__ (2019). [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
1221. Kikuchi, M., et al. Enhancer variants associated with Alzheimer's disease affect gene expression via chromatin looping. __BMC Med Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12920-019-0574-8).
1222. Klein, JC., et al. Functional testing of thousands of osteoarthritis-associated variants for regulatory activity. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-10439-y).
1223. Kojima, M., et al. The histone methyltransferase WHSC1 is regulated by EZH2 and is important for ovarian clear cell carcinoma cell proliferation. __BMC Cancer__ (2019). [Link](https://doi.org/doi:10.1186/s12885-019-5638-9).
1224. Kolmykov, SK., et al. Population size estimation for quality control of ChIP-Seq datasets. __PLoS One__ (2019). [Link](https://doi.org/doi:10.1371/journal.pone.0221760).
1225. Kondo, T., et al. Two-step regulation of trachealess ensures tight coupling of cell fate with morphogenesis in the drosophila trachea. __Elife__ (2019). [Link](https://doi.org/doi:10.7554/eLife.45145).
1226. Krug, B., et al. Pervasive H3K27 Acetylation Leads to ERV Expression and a Therapeutic Vulnerability in H3K27M Gliomas. __Cancer Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.ccell.2019.04.004).
1227. Kulyté, Aé., et al. MicroRNA-27a/b-3p and PPARG regulate SCAMP3 through a feed-forward loop during adipogenesis. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-50210-3).
1228. Kunii, Y., et al. Differential protein expression of DARPP-32 versus Calcineurin in the prefrontal cortex and nucleus accumbens in schizophrenia and bipolar disorder. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-51456-7).
1229. Kuno, A., et al. Conserved DNA-binding motif loci reflect functional transcription factor binding sites. __Research Square__ (2019). [Link](https://doi.org/doi:10.21203/rs.2.15866/v1).
1230. Lin, S., et al. Transcriptome and DNA methylome signatures associated with retinal Müller glia development, injury response, and aging. __Invest Ophthalmol Vis Sci__ (2019). [Link](https://doi.org/doi:10.1167/iovs.19-27361).
1231. Maedera, S., et al. GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13298).
1232. Matsuda, T., et al. Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__ (2019). [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
1233. Menzel, M., et al. Enhort: A platform for deep analysis of genomic positions. __PeerJ Comput Sci__ (2019). [Link](https://doi.org/doi:10.7717/peerj-cs.198).
1234. Murayama, Y., et al. Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13328).
1235. Neiman, G., et al. Integrin alpha-5 subunit is critical for the early stages of human pluripotent stem cell cardiac differentiation. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-54352-2).
1236. Ochsner, SA., et al. The Signaling Pathways Project, an integrated ‘omics knowledgebase for mammalian cellular signaling pathways. __Sci Data__ (2019). [Link](https://doi.org/doi:10.1038/s41597-019-0193-4).
1237. Oulès, Béé., et al. Mutant Lef1 controls Gata6 in sebaceous gland development and cancer. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018100526).
1238. Peggion, C., et al. The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol Neurobiol__ (2019). [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
1239. Qian, FC., et al. SEanalysis: A web tool for super-enhancer associated regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz302).
1240. Ranjit, M., et al. Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
1241. Rauschmeier, Ré., et al. Bhlhe40 and Bhlhe41 transcription factors regulate alveolar macrophage self‐renewal and identity. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018101233).
1242. Robinson, CM., et al. AHypoxia-inducible HIF1-GAL3ST1-sulfatide axis enhances ccRCC immune evasion via increased tumor cell-platelet binding. __Mol Cancer Res__ (2019). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-19-0461).
1243. Saito, M., et al. A Polymorphic Variant in p19Arf Confers Resistance to Chemically Induced Skin Tumors by Activating the p53 Pathway. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
1244. Salameti, V., et al. NOTCH1 signaling in oral squamous cell carcinoma via a TEL2/ SERPINE1 axis. __Oncotarget__ (2019). [Link](https://doi.org/doi:10.18632/oncotarget.27306).
1245. Simak, M., et al. Boolean function network analysis of time course liver transcriptome data to reveal novel circadian transcriptional regulators in mammals. __J Chin Med Assoc__ (2019). [Link](https://doi.org/doi:10.1097/JCMA.0000000000000180).
1246. Singh, R., et al. Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
1247. Solari, C., et al. The pluripotency transcription factor Nanog represses glutathione reductase gene expression in mouse embryonic stem cells. __BMC Res Notes__ (2019). [Link](https://doi.org/doi:10.1186/s13104-019-4411-0).
1248. Studd, JB., et al. Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__ (2019). [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
1249. Suda, K., et al. Reduction of Rpd3 suppresses defects in locomotive ability and neuronal morphology induced by the knockdown of Drosophila SLC25A46 via an epigenetic pathway. __Exp Cell Res__ (2019). [Link](https://doi.org/doi:10.1016/j.yexcr.2019.111673).
1250. Tano, A., et al. The juvenility-associated long noncoding RNA Gm14230 maintains cellular juvenescence. __J Cell Sci__ (2019). [Link](https://doi.org/doi:10.1242/jcs.227801).
1251. Tong, Z., et al. TransmiR v2.0: An updated transcription factor-microRNA regulation database. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1023).
1252. Ulirsch, JC., et al. Interrogation of human hematopoiesis at single-cell and single-variant resolution. __Nat Genet__ (2019). [Link](https://doi.org/doi:10.1038/s41588-019-0362-6).
1253. Velychko, S., et al. Excluding Oct4 from Yamanaka Cocktail Unleashes the Developmental Potential of iPSCs. __Cell Stem Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.stem.2019.10.002).
1254. Vijayakrishnan, J., et al. Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-13069-6).
1255. Vogel, FCE., et al. Targeting the H3K4 Demethylase KDM5B Reprograms the Metabolome and Phenotype of Melanoma Cells. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2019.06.124).
1256. Wu, CJ., et al. MiR-23~27~24–mediated control of humoral immunity reveals a TOX-driven regulatory circuit in follicular helper T cell differentiation. __Sci Adv__ (2019). [Link](https://doi.org/doi:10.1126/sciadv.aaw1715).
1257. Wu, Z., et al. Cis-control of Six1 expression in neural crest cells during craniofacial development. __Dev Dyn__ (2019). [Link](https://doi.org/doi:10.1002/dvdy.109).
1258. Wu, Z., et al. Cis-control of Six1 expression in neural crest cells during craniofacial development. __Dev Dyn__ (2019). [Link](https://doi.org/doi:10.1002/DVDY.109).
1259. Yamada, T., et al. Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __WIREs RNA__ (2019). [Link](https://doi.org/doi:10.1002/wrna.1508).
1260. Yashiro, T., et al. PU.1 plays a pivotal role in dendritic cell migration from the periphery to secondary lymphoid organs via regulating CCR7 expression. __FASEB J__ (2019). [Link](https://doi.org/doi:10.1096/fj.201900379RR).
1261. Yevshin, I., et al. GTRD: A database on gene transcription regulation - 2019 update. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1128).
1262. Yoshizaki, K., et al. Paternal Age Affects Offspring'S Behavior Possibly Via Epigenetic Mechanism Recruiting a Transcriptional Repressor Rest. __Eur Neuropsychopharmacol__ (2019). [Link](https://doi.org/doi:10.1016/j.euroneuro.2017.08.089).
1263. Zhang, L., et al. DeepHINT: Understanding HIV-1 integration via deep learning with attention. __Bioinformatics__ (2019). [Link](https://doi.org/doi:10.1093/bioinformatics/bty842).
1264. Zheng, R., et al. Cistrome Data Browser: Expanded datasets and new tools for gene regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1094).
1265. Anan, K., et al. LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/NAR/GKY234).
1266. Baldi, S., et al. Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol Cell__ (2018). [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
1267. Chatterjee, S., et al. Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res Treat__ (2018). [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
1268. Chen, Y., et al. PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci__ (2018). [Link](https://doi.org/doi:10.1111/cas.13796).
1269. Chishima, T., et al. Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes__ (2018). [Link](https://doi.org/doi:10.3390/genes9010023).
1270. Chèneby, J., et al. ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx1092).
1271. Cui, C., et al. Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics Proteomics Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
1272. Dréos, Ré., et al. MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx995).
1273. Egashira, Y., et al. Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
1274. Ferris, E., et al. Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
1275. Fiziev, P., et al. ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
1276. Groff, AF.., et al. Enhancers in the Peril lincRNA locus regulate distant but not local genes. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1589-8).
1277. Imrichova, H., et al. ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/388934).
1278. Kawaji, H. dirHub: a trackHub configurator with directory structure projection. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/314807).
1279. Kawata, K., et al. Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__ (2018). [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
1280. Kehl, T., et al. REGGAE: A novel approach for the identification of key transcriptional regulators. __Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
1281. Lasar, D., et al. Peroxisome Proliferator Activated Receptor Gamma Controls Mature Brown Adipocyte Inducibility through Glycerol Kinase. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2017.12.067).
1282. Mao, F., et al. EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx918).
1283. Misawa, A., et al. lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif Tissue Int__ (2018). [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
1284. Miura, I., et al. DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J Psychiatr Res__ (2018). [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
1285. Mochizuki, K., et al. SETDB1 is essential for mouse primordial germ cell fate determination by ensuring bmp signaling. __Development__ (2018). [Link](https://doi.org/doi:10.1242/dev.164160).
1286. Mourad, Rë., et al. Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
1287. Nishizawa, Y., et al. Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__ (2018). [Link](https://doi.org/doi:10.18632/oncotarget.23554).
1288. Oki, S., et al. ChIP-Atlas: a data-mining suite powered by full integration of public ChIP-seq data. __EMBO Rep__ (2018). [Link](https://doi.org/doi:10.15252/EMBR.201846255).
1289. Onodera, Y., et al. Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__ (2018). [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
1290. Polouliakh, N., et al. Sequence homology in eukaryotes (SHOE): Interactive visual tool for promoter analysis 06 Biological Sciences 0604 Genetics. __BMC Genomics__ (2018). [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
1291. Raffield, LM., et al. Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet__ (2018). [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
1292. Romero-Cordoba, SL., et al. Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
1293. Wang, H., et al. Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J Biol Chem__ (2018). [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
1294. Chen, Y., et al. Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol__ (2017). [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
1295. Govaere, O., et al. The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__ (2017). [Link](https://doi.org/doi:10.1038/onc.2017.260).
1296. Guoa, MH., et al. Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc Natl Acad Sci U S A__ (2017). [Link](https://doi.org/doi:10.1073/pnas.1619052114).
1297. Ishigaki, K., et al. Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat Genet__ (2017). [Link](https://doi.org/doi:10.1038/ng.3885).
1298. Kalender Atak, Z., et al. Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med__ (2017). [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
1299. Kehl, T., et al. RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkx350).
1300. Matsuda, K., et al. ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__ (2017). [Link](https://doi.org/doi:10.1242/dev.143479).
1301. Naderi, A. C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__ (2017). [Link](https://doi.org/doi:10.18632/oncotarget.17826).
1302. Onodera, Y., et al. miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__ (2017). [Link](https://doi.org/doi:10.1111/acel.12680).
1303. Sanosaka, T., et al. DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
1304. Tanegashima, K., et al. Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes Cells__ (2017). [Link](https://doi.org/doi:10.1111/gtc.12456).
1305. Turrell, FK., et al. Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev__ (2017). [Link](https://doi.org/doi:10.1101/gad.298463.117).
1306. Umeyama, T., et al. DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
1307. Yevshin, I., et al. GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkw951).
1308. Yoshihara, M., et al. Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
1309. Sugeno, N., et al. α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci Rep__ (2016). [Link](https://doi.org/doi:10.1038/srep36328).

